create table metabolite_gene_change_results
(
    `index`        int  null,
    host_type      text null,
    Alteration     text null,
    substrate_name text null,
    throughput     text null,
    description    text null,
    Evidence       int  null
)collate = utf8mb4_bin;

INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3059, 'human', 'null', 'D-Glucose', 'null', 'Major end products of metabolism of glucose are acetic and butyric acids.', 21357455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3060, 'human', 'null', 'Salicin', 'null', 'Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.', 21357455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3061, 'human', 'null', 'D-Xylose', 'null', 'Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.', 21357455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3062, 'human', 'null', 'L-Arabinose', 'null', 'Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.', 21357455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3063, 'human', 'null', 'L-Rhamnose', 'null', 'Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.', 21357455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3064, 'human', 'null', 'D-Mannose', 'null', 'Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.', 21357455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3065, 'human', 'null', 'D-Glucose', 'null', 'Major end products of metabolism of glucose are acetic and butyric acids.', 21357455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3066, 'human', 'null', 'Salicin', 'null', 'Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.', 21357455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3067, 'human', 'null', 'D-Xylose', 'null', 'Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.', 21357455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3068, 'human', 'null', 'L-Arabinose', 'null', 'Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.', 21357455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3069, 'human', 'null', 'L-Rhamnose', 'null', 'Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.', 21357455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3070, 'human', 'null', 'D-Mannose', 'null', 'Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.', 21357455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3071, 'human', 'null', 'Orientin', 'null', 'This strain, which was able to cleave the C-glycoside of orientin to produce luteolin and generate some other metabolites.', 24316522);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3072, 'human', 'null', 'Bile acid', 'null', 'Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.', 23836456);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3073, 'human', 'null', 'Cholic acid', 'null', 'Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.', 23836456);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3074, 'human', 'null', 'Chenodeoxycholic acid', 'null', 'Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.', 23836456);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3075, 'human', 'null', 'Bile acid', 'null', 'Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.', 23836456);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3076, 'human', 'null', 'Cholic acid', 'null', 'Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.', 23836456);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3077, 'human', 'null', 'Chenodeoxycholic acid', 'null', 'Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.', 23836456);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3080, 'human', 'increase', 'Cholesterol', 'low-throughput', 'Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.', 31142855);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3081, 'human', 'increase', 'Cholesterol', 'low-throughput', 'Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.', 31142855);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3082, 'human', 'increase', 'Cholesterol', 'low-throughput', 'Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.', 31142855);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3083, 'human', 'null', 'Cholesterol', 'null', 'Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.', 31142855);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3088, 'human', 'null', 'Cynaroside', 'null', 'Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3089, 'human', 'null', 'Taxifolin', 'null', 'Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3090, 'human', 'null', 'Phloretin', 'null', 'Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3091, 'human', 'null', 'Luteolin', 'null', 'Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3092, 'human', 'null', '5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one', 'null', 'Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3093, 'human', 'null', 'Eriodictyol', 'null', 'Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3094, 'human', 'null', 'Kaempferol', 'null', 'Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3095, 'human', 'null', 'Rutin', 'null', 'Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3096, 'human', 'null', 'Quercetin', 'null', 'Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3097, 'human', 'null', 'Isoquercitrin', 'null', 'Schneider et al. (1999) isolated strains of Eubacterium ramulus from human feces that are capable of degrading quercetin-3-glucoside to 3,4-dihydroxyphenylacetic acid (3,4-DIOHA), butyrate, and acetate.Phloroglucinol was formed as an intermediate.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3098, 'human', 'null', 'Isoquercitrin', 'null', 'Schneider et al. (1999) isolated strains of Eubacterium ramulus from human feces that are capable of degrading quercetin-3-glucoside to 3,4-dihydroxyphenylacetic acid (3,4-DIOHA), butyrate, and acetate.Phloroglucinol was formed as an intermediate.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3099, 'human', 'null', 'Isoquercitrin', 'null', 'Schneider et al. (1999) isolated strains of Eubacterium ramulus from human feces that are capable of degrading quercetin-3-glucoside to 3,4-dihydroxyphenylacetic acid (3,4-DIOHA), butyrate, and acetate.Phloroglucinol was formed as an intermediate.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3100, 'human', 'null', 'Isoquercitrin', 'null', 'In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3101, 'human', 'null', 'Taxifolin', 'null', 'In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3102, 'human', 'null', 'Luteolin', 'null', 'In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3103, 'human', 'null', 'Eriodictyol', 'null', 'In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3104, 'human', 'null', 'Kaempferol', 'null', 'In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3105, 'human', 'null', '5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one', 'null', 'This assumption is supported by the degradation of eriodictyol and naringenin (IX) to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid (XII), respectively.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3106, 'human', 'null', 'Kaempferol-3-sorphoroside-7-glucoside', 'null', 'This assumption is supported by the degradation of eriodictyol and naringenin (IX) to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid (XII), respectively.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3107, 'human', 'null', 'Quercetin', 'null', 'Luteolin-5-glucoside, diosmetin-7rutinoside, naringenin-7-neohesperidoside, (+)-catechin, and (�?-epicatechin were not degraded. Cell extracts of E. ramulus contained α- and β-D-glucosidase activities, but were devoid of α-L-rhamnosidase activity.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3108, 'human', 'null', 'null', 'null', 'Schneider et al. (1999) isolated strains of Eubacterium ramulus from human feces that are capable of degrading quercetin-3-glucoside to 3,4-dihydroxyphenylacetic acid (3,4-DIOHA), butyrate, and acetate.Phloroglucinol was formed as an intermediate.', 10648107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3109, 'human', 'decrease', 'null', 'low-throughput', 'Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).', 26241311);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3110, 'human', 'decrease', 'null', 'low-throughput', 'Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).', 26241311);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3111, 'human', 'decrease', 'null', 'low-throughput', 'Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).', 26241311);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3112, 'human', 'increase', 'null', 'low-throughput', 'we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.', 28322790);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3113, 'human', 'increase', 'null', 'low-throughput', 'we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.', 28322790);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3114, 'human', 'decrease', 'null', 'low-throughput', 'we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.', 28322790);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3115, 'human', 'increase', 'null', 'low-throughput', 'we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.', 28322790);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3116, 'human', 'increase', 'null', 'low-throughput', 'we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.', 28322790);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3117, 'human', 'decrease', 'null', 'low-throughput', 'we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.', 28322790);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3118, 'human', 'increase', 'null', 'low-throughput', 'we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.', 28322790);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3119, 'human', 'increase', 'null', 'low-throughput', 'we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.', 28322790);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3120, 'human', 'decrease', 'null', 'low-throughput', 'we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.', 28322790);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3121, 'human', 'null', 'Daidzein', 'null', 'One strain (strain TM-40) produced dihydrodaidzein both from daidzein and daidzin.', 17113326);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3122, 'human', 'null', 'Daidzin', 'null', 'One strain (strain TM-40) produced dihydrodaidzein both from daidzein and daidzin.', 17113326);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3123, 'human', 'null', 'Daidzein', 'null', 'A newly identified equol-producing bacterium, Slackia sp.strain NATTS, with a high equol-producing activity was isolated from human feces in Japanese adults.', 24761887);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3124, 'human', 'null', 'D-Glucose', 'null', 'endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.', 27931827);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3125, 'human', 'null', 'Pyruvate', 'null', 'endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.', 27931827);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3126, 'human', 'null', 'D-Glucose', 'null', 'endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.', 27931827);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3127, 'human', 'null', 'D-Glucose', 'null', 'endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.', 27931827);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3128, 'human', 'null', 'D-Glucose', 'null', 'endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.', 27931827);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3129, 'human', 'null', 'Pyruvate', 'null', 'endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.', 27931827);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3130, 'human', 'null', 'D-Glucose', 'null', 'endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.', 27931827);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3131, 'human', 'null', 'D-Glucose', 'null', 'endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.', 27931827);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3132, 'human', 'null', 'Choline', 'null', 'Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).', 28785417);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3133, 'human', 'null', 'Carnitine', 'null', 'Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).', 28785417);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3134, 'human', 'null', 'Trimethylamine oxide', 'null', 'Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).', 28785417);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3135, 'human', 'null', 'Choline', 'null', 'Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).', 28785417);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3136, 'human', 'null', 'Carnitine', 'null', 'Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).', 28785417);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3137, 'human', 'null', 'Trimethylamine oxide', 'null', 'Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).', 28785417);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3138, 'human', 'null', 'Hesperetin dihydrochalcone', 'null', 'C.orbiscindens was not able to convert hesperetindihydrochalcone 4�?â-D-glucoside within a period of 43 h, whereas the aglycon was cleaved by this species. C. orbiscindens formed 0.18 mM 3-(3-hydroxy-4-methoxyphenyl)propionic acid from 0.26 mM hesperetin dihydrochalcone within 23 h of incubation.', 15740074);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3139, 'human', 'null', 'beta-D-Glucopyranoside, 3,5-dihydroxy-4-(3-(3-hydroxy-4-methoxyphenyl)-1-oxopropyl)phenyl', 'null', 'However, hesperetin dihydrochalcone 4�?â-D-glucoside was converted by E. ramulus to hesperetin dihydrochalcone and further to 3-(3-hydroxy-4-methoxyphenyl)propionic acid, but not by C. orbiscindens.', 15740074);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3140, 'human', 'null', 'Hesperetin dihydrochalcone', 'null', 'However, hesperetin dihydrochalcone 4�?â-D-glucoside was converted by E. ramulus to hesperetin dihydrochalcone and further to 3-(3-hydroxy-4-methoxyphenyl)propionic acid, but not by C. orbiscindens.', 15740074);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3147, 'human', 'null', 'Daidzein', 'null', 'Strain YY7918 converted substrates daidzein and dihydrodaidzein into S-equol.', 18838805);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3148, 'human', 'null', 'Dihydrodaidzein', 'null', 'Strain YY7918 converted substrates daidzein and dihydrodaidzein into S-equol.', 18838805);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3156, 'human', 'increase', 'null', 'low-throughput', 'As Erk is a ‘downstream�?signaling component of TLR pathways, we investigated whether succinate alone or in combination with TLR ligands was able to induce proinflammatory cytokine production by human DCs. Succinate alone did not have any great effect on a variety of cytokines and chemokines produced by activated human DCs. However, succinate augmented the expression of TNF and IL-1b by iMoDCs when used in combination with the TLR3 ligand poly(I:C) or the TLR7 agonist imiquimod.', 18820681);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3157, 'human', 'increase', 'null', 'low-throughput', 'As Erk is a ‘downstream�?signaling component of TLR pathways, we investigated whether succinate alone or in combination with TLR ligands was able to induce proinflammatory cytokine production by human DCs. Succinate alone did not have any great effect on a variety of cytokines and chemokines produced by activated human DCs. However, succinate augmented the expression of TNF and IL-1b by iMoDCs when used in combination with the TLR3 ligand poly(I:C) or the TLR7 agonist imiquimod.', 18820681);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3158, 'human', 'increase', 'null', 'low-throughput', 'Succinate alone was able to induce phosphorylation of Erk1/2 in human iMoDCs,in agreement with the reported activity of succinate on GPR91 transfectants. Polyinosinic-polycytidylic acid (poly(I:C)) alone showed a very modest activation of this signaling pathway, but when both poly(I:C) and succinate were present, Erk1/2 phosphorylation was synergistically induced.', 18820681);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3159, 'human', 'increase', 'null', 'low-throughput', 'Succinate alone was able to induce phosphorylation of Erk1/2 in human iMoDCs,in agreement with the reported activity of succinate on GPR91 transfectants. Polyinosinic-polycytidylic acid (poly(I:C)) alone showed a very modest activation of this signaling pathway, but when both poly(I:C) and succinate were present, Erk1/2 phosphorylation was synergistically induced.', 18820681);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3160, 'human', 'increase', 'null', 'low-throughput', 'In addition, activation of iMoDCs with poly(I:C) or imiquimod in the presence of succinate induced substantial enhancement of the expression of CCL2 mRNA and CXCL8 mRNA (data not shown). Therefore, we concluded that succinate potentiates the production and secretion of proinflam matory mediators by iMoDCs when the cells are stimulated through some TLRs.', 18820681);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3161, 'human', 'increase', 'null', 'low-throughput', 'In addition, activation of iMoDCs with poly(I:C) or imiquimod in the presence of succinate induced substantial enhancement of the expression of CCL2 mRNA and CXCL8 mRNA (data not shown). Therefore, we concluded that succinate potentiates the production and secretion of proinflam matory mediators by iMoDCs when the cells are stimulated through some TLRs.', 18820681);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3162, 'human', 'increase', 'null', 'low-throughput', 'butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.', 23300800);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3163, 'human', 'increase', 'null', 'low-throughput', 'butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.', 23300800);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3164, 'human', 'increase', 'null', 'low-throughput', 'butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.', 23300800);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3165, 'human', 'increase', 'null', 'low-throughput', 'The activation of AP-1 was strongly correlated with the presence of butyrate in the CM.', 23300800);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3166, 'human', 'increase', 'null', 'low-throughput', 'butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.', 23300800);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3167, 'human', 'increase', 'null', 'low-throughput', 'butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.', 23300800);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3168, 'human', 'increase', 'null', 'low-throughput', 'butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.', 23300800);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3169, 'human', 'increase', 'null', 'low-throughput', 'The activation of AP-1 was strongly correlated with the presence of butyrate in the CM.', 23300800);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3170, 'human', 'increase', 'null', 'low-throughput', 'We also measured MUC2 mRNA levels in the butyrate-treated cells, and found that 1 mM butyrate treatment for 36 h increased the cellular MUC2 mRNA levels, although the fold increase was less than that for MUC2 proteins (Fig. 3B). These results show that buty rate treatment inhibits the proliferation of LS174T cells but concomitantly increases MUC2 production per cell.', 17374366);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3171, 'human', 'decrease', 'null', 'low-throughput', 'Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.', 17374366);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3172, 'human', 'decrease', 'null', 'low-throughput', 'Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.', 17374366);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3173, 'human', 'decrease', 'null', 'low-throughput', 'Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.', 17374366);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3174, 'human', 'decrease', 'null', 'low-throughput', 'Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.', 17374366);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3175, 'human', 'decrease', 'null', 'low-throughput', 'Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.', 17374366);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3176, 'human', 'decrease', 'null', 'low-throughput', 'Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.', 17374366);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3177, 'human', 'decrease', 'null', 'low-throughput', 'Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.', 17374366);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3178, 'human', 'decrease', 'null', 'low-throughput', 'Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.', 17374366);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3179, 'human', 'decrease', 'null', 'low-throughput', 'Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.', 17374366);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3186, 'human', 'null', 'Daidzein', 'null', 'One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.', 15578160);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3187, 'human', 'null', 'Daidzein', 'null', 'One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.', 15578160);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3188, 'human', 'null', 'Daidzein', 'null', 'One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.', 15578160);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3189, 'human', 'null', 'Daidzein', 'null', 'One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.', 15578160);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3190, 'human', 'null', 'Daidzein', 'null', 'One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.', 15578160);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3191, 'human', 'null', 'Daidzein', 'null', 'One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.', 15578160);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3194, 'human', 'null', 'null', 'null', 'Strains of Roseburia sp. and F. prausnitzii possessed butyryl coenzyme A (CoA):acetate-CoA transferase and acetate kinase activities, but butyrate kinase activity was not detectable either in growing or in stationary-phase cultures.', 12324374);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3195, 'human', 'null', 'null', 'null', 'Strains of Roseburia sp. and F. prausnitzii possessed butyryl coenzyme A (CoA):acetate-CoA transferase and acetate kinase activities, but butyrate kinase activity was not detectable either in growing or in stationary-phase cultures.', 12324374);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3196, 'human', 'null', 'null', 'null', 'Strains of Roseburia sp. and F. prausnitzii possessed butyryl coenzyme A (CoA):acetate-CoA transferase and acetate kinase activities, but butyrate kinase activity was not detectable either in growing or in stationary-phase cultures.', 12324374);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3197, 'human', 'null', 'null', 'null', 'On the other hand, Coprococcus sp. strain L2-50 produces acetate and possesses both butyrate kinase and butyryl-CoA transferase.', 12324374);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3198, 'human', 'null', 'null', 'null', 'A. muciniphila has previously been shown to produce acetate and propionate during its growth on mucin (24).', 28687644);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3199, 'human', 'null', 'null', 'null', 'A. muciniphila has previously been shown to produce acetate and propionate during its growth on mucin (24).', 28687644);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3200, 'human', 'null', 'Cianidanol', 'null', 'The conversion of both epicatechin and catechin resulted in the formation of metabolite M1 subsequently characterized as 1�?3,4‐dihydroxyphenyl)�?�?2,4,6‐trihydroxyphenyl)propan�?‐ol (3,4‐diHPP�?‐ol, structure in Fig. 3). In accordance with the initial conversion of the catechins, 3,4‐diHPP�?‐ol was formed more rapidly from catechin than from epicatechin.', 21457417);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3201, 'human', 'null', '(-)-Epicatechin', 'null', 'The conversion of both epicatechin and catechin resulted in the formation of metabolite M1 subsequently characterized as 1�?3,4‐dihydroxyphenyl)�?�?2,4,6‐trihydroxyphenyl)propan�?‐ol (3,4‐diHPP�?‐ol, structure in Fig. 3). In accordance with the initial conversion of the catechins, 3,4‐diHPP�?‐ol was formed more rapidly from catechin than from epicatechin.', 21457417);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3202, 'human', 'null', '1-(3,4-Dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl)-2-propanol', 'null', 'Two metabolites, M2 and M3, were first detected after 4 and 6 h, respectively, of incubation with F. plautii aK2. The concentration of M2 was maximal after 6 h and slightly decreased afterwards. The concentration of M3 increased rapidly until 8 h followed by a minor increase during further incubation. Characterization of the observed metabolites by MS analyses and taking literature data into account led to the assignment of M2 as 5�?3,4‐dihydroxyphenyl)‐γ‐valerolactone (3,4‐diHPVL) and M3 as 4‐hydroxy�?�?3,4‐dihydroxyphenyl)valeric acid (4‐H�?,4‐diHPVA) (structures in Fig. 3).', 21457417);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3203, 'human', 'null', '1-(3,4-Dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl)-2-propanol', 'null', 'Two metabolites, M2 and M3, were first detected after 4 and 6 h, respectively, of incubation with F. plautii aK2. The concentration of M2 was maximal after 6 h and slightly decreased afterwards. The concentration of M3 increased rapidly until 8 h followed by a minor increase during further incubation. Characterization of the observed metabolites by MS analyses and taking literature data into account led to the assignment of M2 as 5�?3,4‐dihydroxyphenyl)‐γ‐valerolactone (3,4‐diHPVL) and M3 as 4‐hydroxy�?�?3,4‐dihydroxyphenyl)valeric acid (4‐H�?,4‐diHPVA) (structures in Fig. 3).', 21457417);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3204, 'human', 'null', 'null', 'null', 'Additionally,the abundances of genera Blautia (r = 0.307, p = 0.004), Bifidobacterium (r = 0.36, p = 0.001), Collinsella(r = 0.254, p = 0.018), and Lactobacillus (r = 0.428, p < 0.001) (Figure 2C) were positively correlated withthe urinary TMAO level.', 30469463);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3205, 'human', 'null', 'null', 'null', 'Additionally,the abundances of genera Blautia (r = 0.307, p = 0.004), Bifidobacterium (r = 0.36, p = 0.001), Collinsella(r = 0.254, p = 0.018), and Lactobacillus (r = 0.428, p < 0.001) (Figure 2C) were positively correlated withthe urinary TMAO level.', 30469463);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3206, 'human', 'null', 'null', 'null', 'Additionally,the abundances of genera Blautia (r = 0.307, p = 0.004), Bifidobacterium (r = 0.36, p = 0.001), Collinsella(r = 0.254, p = 0.018), and Lactobacillus (r = 0.428, p < 0.001) (Figure 2C) were positively correlated withthe urinary TMAO level.', 30469463);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3207, 'human', 'null', 'null', 'null', 'Additionally,the abundances of genera Blautia (r = 0.307, p = 0.004), Bifidobacterium (r = 0.36, p = 0.001), Collinsella(r = 0.254, p = 0.018), and Lactobacillus (r = 0.428, p < 0.001) (Figure 2C) were positively correlated withthe urinary TMAO level.', 30469463);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3208, 'human', 'null', 'null', 'null', 'The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.', 12508881);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3209, 'human', 'null', 'null', 'null', 'The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.', 12508881);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3210, 'human', 'null', 'null', 'null', 'The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.', 12508881);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3211, 'human', 'null', 'null', 'null', 'The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.', 12508881);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3212, 'human', 'null', 'null', 'null', 'The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.', 12508881);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3213, 'human', 'null', 'null', 'null', 'The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.', 12508881);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3214, 'human', 'null', 'null', 'null', 'The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.', 12508881);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3215, 'human', 'null', 'null', 'null', 'The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.', 12508881);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3216, 'human', 'null', 'null', 'null', 'A higher relative abundance of succinate-producing Prevotellaceae (P) and Veillonellaceae (V), and a lower relative abundance of succinate-consuming Odoribacteraceae (O) and Clostridaceae (C) in obese individuals, with the (P + V/O + C) ratio being a main determinant of plasma succinate.', 29434314);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3217, 'human', 'null', 'null', 'null', 'A higher relative abundance of succinate-producing Prevotellaceae (P) and Veillonellaceae (V), and a lower relative abundance of succinate-consuming Odoribacteraceae (O) and Clostridaceae (C) in obese individuals, with the (P + V/O + C) ratio being a main determinant of plasma succinate.', 29434314);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3218, 'human', 'null', 'null', 'null', 'We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.', 31030641);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3219, 'human', 'null', 'null', 'null', 'We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.', 31030641);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3220, 'human', 'null', 'null', 'null', 'We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.', 31030641);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3221, 'human', 'null', 'null', 'null', 'We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.', 31030641);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3222, 'human', 'null', 'null', 'null', 'We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.', 31030641);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3223, 'human', 'null', 'null', 'null', 'The strongest associations seen were positive associations with RF39 assigned to the Tenericutes phylum and negative associations with OTUs from the Lachnospiraceae family.', 31030641);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3224, 'human', 'null', 'null', 'null', 'The strongest associations seen were positive associations with RF39 assigned to the Tenericutes phylum and negative associations with OTUs from the Lachnospiraceae family.', 31030641);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3225, 'human', 'null', 'null', 'null', 'Negative correlations instead were observed with several Eubacterium species including E. dolichum that has been previously reported to be associated with increased visceral fat (Figure 1).', 31030641);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3226, 'human', 'null', 'null', 'null', 'Negative correlations instead were observed with several Eubacterium species including E. dolichum that has been previously reported to be associated with increased visceral fat (Figure 1).', 31030641);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3227, 'human', 'null', 'Chlorogenic acid', 'null', 'The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.', 24550206);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3228, 'human', 'null', 'Chlorogenic acid', 'null', 'The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.', 24550206);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3229, 'human', 'null', 'Chlorogenic acid', 'null', 'The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.', 24550206);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3230, 'human', 'null', 'Chlorogenic acid', 'null', 'The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.', 24550206);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3231, 'human', 'null', 'Chlorogenic acid', 'null', 'The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.', 24550206);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3232, 'human', 'null', 'Chlorogenic acid', 'null', 'The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.', 24550206);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3233, 'human', 'null', 'Chlorogenic acid', 'null', 'The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.', 24550206);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3234, 'human', 'null', 'Chlorogenic acid', 'null', 'The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.', 24550206);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3235, 'human', 'null', 'Chlorogenic acid', 'null', 'The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.', 24550206);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3236, 'human', 'null', 'Rutin', 'null', 'α-L-rhamnosidase from Bifidobacterium breve was produced by using Escherichia coli BL21 for biotransformation of rutin to isoquercitrin.', 25724715);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3242, 'human', 'null', 'Taxifolin', 'null', 'The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.', 14532034);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3243, 'human', 'null', 'Quercetin', 'null', 'The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.', 14532034);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3244, 'human', 'null', 'Eriodictyol', 'null', 'The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.', 14532034);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3245, 'human', 'null', 'Luteolin', 'null', 'The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.', 14532034);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3246, 'human', 'null', 'Apigenin', 'null', 'The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.', 14532034);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3247, 'human', 'null', '5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one', 'null', 'The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.', 14532034);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3248, 'human', 'null', 'Phloretin', 'null', 'The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.', 14532034);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3249, 'human', 'null', 'Daidzein', 'null', 'Daidzein was converted to equol by strain DZE.', 18670101);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3250, 'human', 'null', 'Succinate', 'null', 'PY broth tube tests indicated that strains YIT 12067T and YIT 12068 grew well only in PY medium supplemented with succinate. HPLC analysis showed that succinate was quantitatively converted to propionate. These results indicate that succinate decarboxylation stimulated the growth of strains YIT 12067T and YIT 12068.', 22081579);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3251, 'human', 'null', 'Succinate', 'null', 'PY broth tube tests indicated that strains YIT 12067T and YIT 12068 grew well only in PY medium supplemented with succinate. HPLC analysis showed that succinate was quantitatively converted to propionate. These results indicate that succinate decarboxylation stimulated the growth of strains YIT 12067T and YIT 12068.', 22081579);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3252, 'human', 'null', 'Daidzein', 'null', 'In this study, we isolated a human intestinal bacterium capable of transforming daidzein to equol and genistein to 5-hydroxy-equol.', 19139227);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3253, 'human', 'null', 'Genistein', 'null', 'In this study, we isolated a human intestinal bacterium capable of transforming daidzein to equol and genistein to 5-hydroxy-equol.', 19139227);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3254, 'human', 'null', 'Genistein 7-O-glucoside', 'null', 'Genistein-7-O-glucoside was partially transformed by way of genistein to the product 2-(4-hydroxyphenyl)-propionic acid.', 11959436);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3255, 'human', 'null', 'Genistein', 'null', 'Genistein was completely degraded by E. ramulus via 6''-hydroxy-O-desmethylangolensin to 2-(4-hydroxyphenyl)-propionic acid.', 11959436);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3256, 'human', 'null', 'Daidzein', 'null', 'Daidzein was in part degraded to O-desmethylangolensin, the corresponding metabolite to 6''-hydroxy-O-desmethylangolensin.', 11959436);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3257, 'human', 'null', 'Isoxanthohumol', 'null', 'The intestinal bacterium E. limosum was able to perform this conversion, and after strain selection, the bacterium rapidly converted all IX into 8-PN.', 16076107);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3258, 'human', 'null', 'null', 'null', 'In reactions using CF faecal inocula, butyrate levels were strongly correlated with the relative abundance of Clostridium sensu stricto cluster 1.', 30359203);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3259, 'human', 'null', 'null', 'null', 'Propionate concentrations in CF reactions were significantly correlated with the relative abundance of the Veillonella genus.', 30359203);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3260, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3261, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3262, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3263, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3264, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3265, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3266, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3267, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3268, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3269, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3270, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3271, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3272, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3273, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3274, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3275, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3276, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3277, 'human', 'null', 'null', 'null', 'Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3278, 'human', 'null', 'null', 'null', 'Genus Turicibacter and species Bifdobacterium longum,Lactobacillus mucosae, Lactobacillus zeae were positively correlated with 3,4 dihydroxyhydrocinnmaic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3279, 'human', 'null', 'null', 'null', 'Genus Turicibacter and species Bifdobacterium longum,Lactobacillus mucosae, Lactobacillus zeae were positively correlated with 3,4 dihydroxyhydrocinnmaic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3280, 'human', 'null', 'null', 'null', 'Genus Turicibacter and species Bifdobacterium longum,Lactobacillus mucosae, Lactobacillus zeae were positively correlated with 3,4 dihydroxyhydrocinnmaic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3281, 'human', 'null', 'null', 'null', 'Genus Turicibacter and species Bifdobacterium longum,Lactobacillus mucosae, Lactobacillus zeae were positively correlated with 3,4 dihydroxyhydrocinnmaic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3282, 'human', 'null', 'null', 'null', 'Similarly, a positive correlation of Eubacterium with 3-hydroxybenzoic acid and 4-hydroxybenzoic acid concurs with the previous studies.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3283, 'human', 'null', 'null', 'null', 'Similarly, a positive correlation of Eubacterium with 3-hydroxybenzoic acid and 4-hydroxybenzoic acid concurs with the previous studies.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3284, 'human', 'null', 'null', 'null', 'Collinsella aerofaciens showed a positive correlation with sedoheptulose, citric acid, tricarballylic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3285, 'human', 'null', 'null', 'null', 'Collinsella aerofaciens showed a positive correlation with sedoheptulose, citric acid, tricarballylic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3286, 'human', 'null', 'null', 'null', 'Collinsella aerofaciens showed a positive correlation with sedoheptulose, citric acid, tricarballylic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3287, 'human', 'null', 'null', 'null', 'E. coli was positively correlated with pipecolic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3288, 'human', 'null', 'null', 'null', 'Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3289, 'human', 'null', 'null', 'null', 'Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3290, 'human', 'null', 'null', 'null', 'Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3291, 'human', 'null', 'null', 'null', 'Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3292, 'human', 'null', 'null', 'null', 'Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3293, 'human', 'null', 'null', 'null', 'Lactobacillus ruminis showed a negative correlation with glycine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3294, 'human', 'null', 'null', 'null', 'Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3295, 'human', 'null', 'null', 'null', 'Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3296, 'human', 'null', 'null', 'null', 'Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3297, 'human', 'null', 'null', 'null', 'Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3298, 'human', 'null', 'null', 'null', 'Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3299, 'human', 'null', 'null', 'null', 'Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3300, 'human', 'null', 'null', 'null', 'Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3301, 'human', 'null', 'null', 'null', 'Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3302, 'human', 'null', 'null', 'null', 'Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3303, 'human', 'null', 'null', 'null', 'Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3304, 'human', 'null', 'null', 'null', 'Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3305, 'human', 'null', 'null', 'null', 'Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3306, 'human', 'null', 'null', 'null', 'Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3307, 'human', 'null', 'null', 'null', 'Genera Faecalibacterium, Succinivibrio, Macrococcus are negatively correlated with creatinine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3308, 'human', 'null', 'null', 'null', 'Genera Faecalibacterium, Succinivibrio, Macrococcus are negatively correlated with creatinine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3309, 'human', 'null', 'null', 'null', 'Genera Faecalibacterium, Succinivibrio, Macrococcus are negatively correlated with creatinine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3310, 'human', 'null', 'null', 'null', 'Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3311, 'human', 'null', 'null', 'null', 'Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3312, 'human', 'null', 'null', 'null', 'Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3313, 'human', 'null', 'null', 'null', 'Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3314, 'human', 'null', 'null', 'null', 'Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3315, 'human', 'null', 'null', 'null', 'Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3316, 'human', 'null', 'null', 'null', 'Genera Escherichia, Paraprevotella, Akkermansia have a positive correlation with glycocholic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3317, 'human', 'null', 'null', 'null', 'Genera Escherichia, Paraprevotella, Akkermansia have a positive correlation with glycocholic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3318, 'human', 'null', 'null', 'null', 'Genera Escherichia, Paraprevotella, Akkermansia have a positive correlation with glycocholic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3319, 'human', 'null', 'null', 'null', 'Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3320, 'human', 'null', 'null', 'null', 'Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3321, 'human', 'null', 'null', 'null', 'Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3322, 'human', 'null', 'null', 'null', 'Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3323, 'human', 'null', 'null', 'null', 'Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3324, 'human', 'null', 'null', 'null', 'Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3325, 'human', 'null', 'null', 'null', 'Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3326, 'human', 'null', 'null', 'null', 'Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3327, 'human', 'null', 'null', 'null', 'Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3328, 'human', 'null', 'null', 'null', 'Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3329, 'human', 'null', 'null', 'null', 'Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3330, 'human', 'null', 'null', 'null', 'Genus Oscillospira is positively correlated with citric acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3331, 'human', 'null', 'null', 'null', 'Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3332, 'human', 'null', 'null', 'null', 'Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3333, 'human', 'null', 'null', 'null', 'Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3334, 'human', 'null', 'null', 'null', 'Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3335, 'human', 'null', 'null', 'null', 'Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3336, 'human', 'null', 'null', 'null', 'Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3337, 'human', 'null', 'null', 'null', 'Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3338, 'human', 'null', 'null', 'null', 'Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3339, 'human', 'null', 'null', 'null', 'Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.', 31235863);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3340, 'human', 'null', 'null', 'null', 'We cultured fecal specimens from healthy young adults in media containing defatted flaxseeds and detected END from the culture supernatant. Following selection through successive subcultures of the fecal microbiota with defatted flaxseeds as the only carbon source, we obtained a bacterial consortium, designated as END-49, which contained the smallest number of bacterial types still capable of metabolizing defatted flaxseeds to produce END.', 20398397);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3341, 'human', 'increase', 'null', 'low-throughput', 'We provide direct evidence that IS activates EGFR and downstream MAPK signaling, which stimulates AhR−ARNT nuclear translocation and results in enhanced OAT1 expression and function in the kidney.', 31341083);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3342, 'human', 'increase', 'null', 'low-throughput', 'We provide direct evidence that IS activates EGFR and downstream MAPK signaling, which stimulates AhR−ARNT nuclear translocation and results in enhanced OAT1 expression and function in the kidney.', 31341083);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3343, 'human', 'increase', 'Choline', 'low-throughput', 'Interestingly, we found that plasma TMAO was positively correlated with serum sCD14 and sCD163 levels which are biomarkers of monocyte activation and macrophage inflammation and have been reported to be associated with carotid artery atherosclerosis and CVD risk in HIV infected individuals.', 29912352);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3344, 'human', 'increase', 'Choline', 'low-throughput', 'Interestingly, we found that plasma TMAO was positively correlated with serum sCD14 and sCD163 levels which are biomarkers of monocyte activation and macrophage inflammation and have been reported to be associated with carotid artery atherosclerosis and CVD risk in HIV infected individuals.', 29912352);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3345, 'human', 'increase', 'null', 'low-throughput', 'For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.', 28713353);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3346, 'human', 'increase', 'null', 'low-throughput', 'For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.', 28713353);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3347, 'human', 'increase', 'null', 'low-throughput', 'For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.', 28713353);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3348, 'human', 'increase', 'null', 'low-throughput', 'For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.', 28713353);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3358, 'human', 'null', 'Choline', 'null', 'Further, in vitro conversion of choline to TMA was checked in presence/absence of trigonelline and in presence of trigonelline there was significant inhibition of growth of C. freundii and hence significantly reduced levels of TMA was estimated.', 29980011);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3359, 'human', 'null', 'Trimethylamine', 'null', 'Butin presence oftrigonelline, significantly reduced levels of TMAO was estimated which was, of course, due to flavin monooxygenase (FMO3) enzyme inhibition by trigonelline.', 29980011);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3360, 'human', 'null', 'Quercetin', 'null', 'The degradation of the flavonol quercetin and the flavone luteolin by Eubacterium ramulus, a strict anaerobe of the human intestinal tract, was studied. Resting cells converted these flavonoids to 3,4-dihydroxyphenylacetic acid and 3-(3,4-dihydroxyphenyl)propionic acid, respectively.', 11722907);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3361, 'human', 'null', 'Luteolin', 'null', 'The degradation of the flavonol quercetin and the flavone luteolin by Eubacterium ramulus, a strict anaerobe of the human intestinal tract, was studied. Resting cells converted these flavonoids to 3,4-dihydroxyphenylacetic acid and 3-(3,4-dihydroxyphenyl)propionic acid, respectively.', 11722907);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3362, 'human', 'null', 'Hesperidin', 'null', 'The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.', 25875120);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3363, 'human', 'null', 'Hesperidin', 'null', 'The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.', 25875120);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3364, 'human', 'null', 'Hesperidin', 'null', 'The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.', 25875120);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3365, 'human', 'null', 'Hesperidin', 'null', 'The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.', 25875120);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3366, 'human', 'null', 'Hesperidin', 'null', 'The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.', 25875120);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3367, 'human', 'increase', 'null', 'low-throughput', 'In the present study, we focused on the impact of SCFAs and the role of PPAR-γin intestinal ANGPTL4 expression. Here, we show that ANGPTL4 expression can be regulated by SCFAs both in vitro and in vivo, with butyrate being the most efficient activator.', 23518684);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3368, 'human', 'null', 'null', 'null', 'Faecalibacterium prausnitzii is one of the most abundant bacteria in the human gut ecosystem and it is an important supplier of butyrate to the colonic epithelium.', 22357539);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3369, 'human', 'null', 'Cianidanol', 'null', 'When (+)-catechin was anaerobically incubated with Eg. sp. CAT-1, and the incubation mixture was analyzed by LC/MS, two metabolites, designated CAM1 and CAM2, were detected.', 23207778);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3370, 'human', 'null', 'Cianidanol', 'null', 'When (+)-catechin was anaerobically incubated with Eg. sp. CAT-1, and the incubation mixture was analyzed by LC/MS, two metabolites, designated CAM1 and CAM2, were detected.', 23207778);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3371, 'human', 'decrease', 'null', 'low-throughput', 'On the other hand, we confirmed in both ex vivo epithelial cultures and whole mucosal samples that the inflammatory milieu, in particular the presence of TNFɑ, does impact butyrate uptake and oxidation. Indeed, TNFɑ may be responsible for the downregulation of butyrate transporters in the human intestinal epithelium observed in patient biopsies, and that may result in decreased butyrate consumption.', 31211831);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3372, 'human', 'null', 'Exopolysaccharides', 'null', 'In conclusion, this study revealed that relevant changes in the populations of some major anaerobic intestinal bacterial groups as well as noticeable shifts in SCFA and lactic acid production can result from the presence of bifidobacterial EPS. Microbial population dynamics correlated well and supported the SCFA and lactic acid variations detected. EPS could reach the colon by two different ways: they could be either synthesized by probiotics in food products or by intestinal bacteria in the intestinal environment, thus contributing to the modulation of the intestinal microbiota.', 19735956);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3373, 'human', 'null', 'null', 'null', 'These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.', 18820905);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3374, 'human', 'null', 'null', 'null', 'These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.', 18820905);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3375, 'human', 'null', 'null', 'null', 'These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.', 18820905);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3376, 'human', 'null', 'null', 'null', 'These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.', 18820905);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3377, 'human', 'null', 'null', 'null', 'These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.', 18820905);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3378, 'human', 'null', 'null', 'null', 'These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.', 18820905);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3379, 'human', 'null', 'null', 'null', 'These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.', 18820905);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3380, 'human', 'null', 'null', 'null', 'These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.', 18820905);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3381, 'human', 'null', 'null', 'null', 'These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.', 18820905);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3382, 'human', 'null', 'null', 'null', 'These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.', 18820905);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3383, 'human', 'null', 'null', 'null', 'These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.', 18820905);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3384, 'human', 'null', 'null', 'null', 'These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.', 18820905);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3385, 'human', 'null', 'null', 'null', 'We also show that E. lenta further metabolizes the dopamine produced by L-dopa decarboxylation using a distinctly microbial reaction, catechol dehydroxylation.', 31196984);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3386, 'human', 'null', 'null', 'null', 'Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.', 28894218);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3387, 'human', 'null', 'null', 'null', 'Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.', 28894218);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3388, 'human', 'null', 'null', 'null', 'Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.', 28894218);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3389, 'human', 'null', 'null', 'null', 'Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.', 28894218);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3390, 'human', 'null', 'null', 'null', 'Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.', 28894218);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3391, 'human', 'null', 'null', 'null', 'Streptococcus sp. MRG-ICA-B isolated from human intestine were reacted with icariin under anaerobic conditions. The MRG-ICA-B and E strains hydrolyzed only the glucose moiety of icariin, and icariside II was the only metabolite.', 27589718);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3392, 'human', 'null', 'null', 'null', 'Enterococcus sp. MRG-ICA-E isolated from human intestine were reacted with icariin under anaerobic conditions. The MRG-ICA-B and E strains hydrolyzed only the glucose moiety of icariin, and icariside II was the only metabolite.', 27589718);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3393, 'human', 'null', 'null', 'null', 'Blautia sp. MRG-PMF-1 isolated from human intestine were reacted with icariin under anaerobic conditions. It not only produced icariside II  but strain metabolized icariin further to desmethylicaritin via icariside II and icaritin.', 27589718);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3394, 'human', 'null', 'null', 'null', 'Blautia sp. MRG-PMF-1 isolated from human intestine were reacted with icariin under anaerobic conditions. It not only produced icariside II  but strain metabolized icariin further to desmethylicaritin via icariside II and icaritin.', 27589718);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3395, 'human', 'null', 'Puerarin', 'null', 'Puerarin was completely converted to daidzein within 100 min by Lactococcus sp. MRG-IFC-1 and Enterococcus sp. MRG-IFC-2, which is the fastest conversion among the reported human intestinal bacteria.', 25199800);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3396, 'human', 'null', 'Puerarin', 'null', 'Puerarin was completely converted to daidzein within 100 min by Lactococcus sp. MRG-IFC-1 and Enterococcus sp. MRG-IFC-2, which is the fastest conversion among the reported human intestinal bacteria.', 25199800);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3397, 'human', 'null', 'Isoquercitrin', 'null', 'From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.', 9914304);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3398, 'human', 'null', 'Isoquercitrin', 'null', 'From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.', 9914304);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3399, 'human', 'null', 'Isoquercitrin', 'null', 'From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.', 9914304);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3400, 'human', 'null', 'Isoquercitrin', 'null', 'From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.', 9914304);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3401, 'human', 'null', 'Isoquercitrin', 'null', 'From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.', 9914304);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3402, 'human', 'null', 'Isoquercitrin', 'null', 'From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.', 9914304);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3403, 'human', 'null', 'Isoquercitrin', 'null', 'From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.', 9914304);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3404, 'human', 'null', 'null', 'null', 'Finally, a total of seven metabolites were detected and tentatively identified, such as the demethylation metabolite (M1), deoxidation metabolites (M3, M7) and hydrolysis at the C-28 oligosaccharide metabolites (M5,M6), which were first discovered in this experiment. The results indicate that hydrolysis, demethylation, dehydroxylation, and acetylation were the major metabolic pathways of PDin vitro. Additionally, four bacterial strains from human feces including Enterococcus sp.41, Bacillus sp.46, Escherichia sp.49A and Escherichia sp.64 were detected and further identified with 16S rRNA gene sequencing due to their relatively strong metabolic capacity', 27222071);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3405, 'human', 'null', 'null', 'null', 'Finally, a total of seven metabolites were detected and tentatively identified, such as the demethylation metabolite (M1), deoxidation metabolites (M3, M7) and hydrolysis at the C-28 oligosaccharide metabolites (M5,M6), which were first discovered in this experiment. The results indicate that hydrolysis, demethylation, dehydroxylation, and acetylation were the major metabolic pathways of PDin vitro. Additionally, four bacterial strains from human feces including Enterococcus sp.41, Bacillus sp.46, Escherichia sp.49A and Escherichia sp.64 were detected and further identified with 16S rRNA gene sequencing due to their relatively strong metabolic capacity', 27222071);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3406, 'human', 'null', 'null', 'null', 'Finally, a total of seven metabolites were detected and tentatively identified, such as the demethylation metabolite (M1), deoxidation metabolites (M3, M7) and hydrolysis at the C-28 oligosaccharide metabolites (M5,M6), which were first discovered in this experiment. The results indicate that hydrolysis, demethylation, dehydroxylation, and acetylation were the major metabolic pathways of PDin vitro. Additionally, four bacterial strains from human feces including Enterococcus sp.41, Bacillus sp.46, Escherichia sp.49A and Escherichia sp.64 were detected and further identified with 16S rRNA gene sequencing due to their relatively strong metabolic capacity', 27222071);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3407, 'human', 'null', 'null', 'null', 'Finally, a total of seven metabolites were detected and tentatively identified, such as the demethylation metabolite (M1), deoxidation metabolites (M3, M7) and hydrolysis at the C-28 oligosaccharide metabolites (M5,M6), which were first discovered in this experiment. The results indicate that hydrolysis, demethylation, dehydroxylation, and acetylation were the major metabolic pathways of PDin vitro. Additionally, four bacterial strains from human feces including Enterococcus sp.41, Bacillus sp.46, Escherichia sp.49A and Escherichia sp.64 were detected and further identified with 16S rRNA gene sequencing due to their relatively strong metabolic capacity', 27222071);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3408, 'human', 'null', 'Daidzein', 'null', 'We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.', 18450717);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3409, 'human', 'null', 'Daidzein', 'null', 'We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.', 18450717);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3410, 'human', 'null', 'Daidzein', 'null', 'We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.', 18450717);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3411, 'human', 'null', 'Daidzein', 'null', 'We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.', 18450717);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3412, 'human', 'null', 'Daidzein', 'null', 'We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.', 18450717);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3413, 'human', 'null', 'Daidzein', 'null', 'We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.', 18450717);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3414, 'human', 'null', 'Daidzein', 'null', 'We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.', 18450717);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3415, 'human', 'null', 'Dihydrodaidzein', 'null', 'We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.', 18450717);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3416, 'human', 'null', 'Dihydrodaidzein', 'null', 'We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.', 18450717);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3425, 'human', 'null', 'D-Glucose', 'null', 'Acetate is readily absorbed from the rumen and subsequently used in fatty acid metabolism.', 8481001);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3426, 'human', 'null', 'D-Glucose', 'null', 'Succinate and formate, on the other hand, are considered intermediates of rumen fermentation which are further metabolized by other organisms. Succinate is thought to be an important intermediate for propionate formation , with the latter being readily absorbed from the rumen for gluconeogenesis.', 8481001);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3427, 'human', 'null', 'D-Glucose', 'null', 'Succinate and formate, on the other hand, are considered intermediates of rumen fermentation which are further metabolized by other organisms. Formate has been shown to be metabolized to methane by rumen methanogens.', 8481001);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3428, 'human', 'null', 'D-Glucose', 'null', 'Lactate is also an intermediate of the overall rumen fermentation and, when present in excess, has been correlated with acidosis.', 8481001);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3429, 'human', 'null', 'D-Glucose', 'null', 'Lactic acid bacteria, because the bifidobacterial pathway yields 2.5 ATP molecules from 1 mol of fermented glucose, as well as 1.5 mol of acetate and 1 mol of lactate [94]. In contrast, the homofermenta tive group of lactic acid bacteria (LAB) produces 2 mol of ATP and 2 mol of lactic acid from 1 mol of glucose, whereas heterofermentative LAB produce 1 mol each of lactic acid, ethanol and ATP per 1 mol of fermented glu cose.', 21484167);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3430, 'human', 'null', 'D-Glucose', 'null', 'Lactic acid bacteria, because the bifidobacterial pathway yields 2.5 ATP molecules from 1 mol of fermented glucose, as well as 1.5 mol of acetate and 1 mol of lactate [94]. In contrast, the homofermenta tive group of lactic acid bacteria (LAB) produces 2 mol of ATP and 2 mol of lactic acid from 1 mol of glucose, whereas heterofermentative LAB produce 1 mol each of lactic acid, ethanol and ATP per 1 mol of fermented glu cose.', 21484167);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3431, 'human', 'null', 'D-Glucose', 'null', 'Lactic acid bacteria, because the bifidobacterial pathway yields 2.5 ATP molecules from 1 mol of fermented glucose, as well as 1.5 mol of acetate and 1 mol of lactate [94]. In contrast, the homofermenta tive group of lactic acid bacteria (LAB) produces 2 mol of ATP and 2 mol of lactic acid from 1 mol of glucose, whereas heterofermentative LAB produce 1 mol each of lactic acid, ethanol and ATP per 1 mol of fermented glu cose.', 21484167);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3432, 'human', 'null', 'null', 'null', 'Due to the fact that bifidobacteria produce lactic acid as one of their main fermentation end products, they are often included in the lactic acid bacteria (LAB).', 21484167);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3433, 'human', 'null', '5,7-Dimethoxyflavone', 'null', 'Activity-guided screening for PMF-metabolizing intestinal bacteria under anaerobic conditions resulted in the isolation of a strict anaerobic bacterium, which was identified as Blautia sp. MRG-PMF1. The isolated MRGPMF1 was able to metabolize various PMFs to the corresponding demethylated flavones.The microbial conversion of bioactive 5,7-dimethoxyflavone (5,7-DMF) and 5,7,4�?trimethoxyflavone (5,7,4�?TMF) was studied in detail. 5,7-DMF and 5,7,4�?TMF were completely metabolized to 5,7-dihydroxyflavone (chrysin) and 5,7,4�?trihydroxyflavone (apigenin), respectively.', 25437273);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3434, 'human', 'null', '4'',5,7-Trimethoxyflavone', 'null', 'Activity-guided screening for PMF-metabolizing intestinal bacteria under anaerobic conditions resulted in the isolation of a strict anaerobic bacterium, which was identified as Blautia sp. MRG-PMF1. The isolated MRGPMF1 was able to metabolize various PMFs to the corresponding demethylated flavones.The microbial conversion of bioactive 5,7-dimethoxyflavone (5,7-DMF) and 5,7,4�?trimethoxyflavone (5,7,4�?TMF) was studied in detail. 5,7-DMF and 5,7,4�?TMF were completely metabolized to 5,7-dihydroxyflavone (chrysin) and 5,7,4�?trihydroxyflavone (apigenin), respectively.', 25437273);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3435, 'human', 'null', 'Ellagic Acid', 'null', 'Four gut bacteria strains (CEBAS 4A1, 4A2, 4A3, 4A4)isolated from a healthy donor and able to produce urolithins M6, C and isourolithin A, are described in the present study.Strains were found at high concentrations (�?07 cfu g�?feces).Comparison of the 16S rRNA gene sequences of the strains showed that the four isolates belong to the same species and that they are phylogenetically members of the family Eggerthellaceae.', 28824607);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3436, 'human', 'null', 'Ellagic Acid', 'null', 'Four gut bacteria strains (CEBAS 4A1, 4A2, 4A3, 4A4)isolated from a healthy donor and able to produce urolithins M6, C and isourolithin A, are described in the present study.Strains were found at high concentrations (�?07 cfu g�?feces).Comparison of the 16S rRNA gene sequences of the strains showed that the four isolates belong to the same species and that they are phylogenetically members of the family Eggerthellaceae.', 28824607);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3437, 'human', 'null', 'Ellagic Acid', 'null', 'Four gut bacteria strains (CEBAS 4A1, 4A2, 4A3, 4A4)isolated from a healthy donor and able to produce urolithins M6, C and isourolithin A, are described in the present study.Strains were found at high concentrations (�?07 cfu g�?feces).Comparison of the 16S rRNA gene sequences of the strains showed that the four isolates belong to the same species and that they are phylogenetically members of the family Eggerthellaceae.', 28824607);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3438, 'human', 'null', 'Ellagic Acid', 'null', 'Four gut bacteria strains (CEBAS 4A1, 4A2, 4A3, 4A4)isolated from a healthy donor and able to produce urolithins M6, C and isourolithin A, are described in the present study.Strains were found at high concentrations (�?07 cfu g�?feces).Comparison of the 16S rRNA gene sequences of the strains showed that the four isolates belong to the same species and that they are phylogenetically members of the family Eggerthellaceae.', 28824607);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3439, 'human', 'null', 'null', 'null', 'Recently, we isolated a dihydrodaidzein (DHD)-producing Clostridium-like bacterium, strain TM-40, from human feces.', 25045313);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3440, 'human', 'null', 'DL-Alanine', 'null', 'For example, if R1 is a methyl group (-CH3), then the donor amino acid is an alanine, and the product of the oxidative deamination would be pyruvate, a central compounds of the intermediary metabolism.', 22325238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3441, 'human', 'null', 'Glycine', 'null', 'For example when R2 is a hydrogen (thus the acceptor amino acid is a glycine) this product corresponds to acetyl-phosphate.', 22325238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3442, 'human', 'increase', 'null', 'low-throughput', 'Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3443, 'human', 'increase', 'null', 'low-throughput', 'Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3444, 'human', 'increase', 'null', 'low-throughput', 'Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3445, 'human', 'increase', 'null', 'low-throughput', 'Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3446, 'human', 'increase', 'null', 'low-throughput', 'And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3447, 'human', 'increase', 'null', 'low-throughput', 'And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3448, 'human', 'increase', 'null', 'low-throughput', 'And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3449, 'human', 'increase', 'null', 'low-throughput', 'And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3450, 'human', 'increase', 'null', 'low-throughput', 'And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3451, 'human', 'increase', 'null', 'low-throughput', 'These data show that B dentium–secreted products, including acetate, can modulate 5-HT production by host enterochromaffin cells.', 32795610);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3452, 'human', 'increase', 'null', 'low-throughput', 'Gene expression of LGR5 was increased more than 2-fold in human duodenal enteroids treated with 10 mM ACET', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3453, 'human', 'decrease', 'null', 'low-throughput', 'Conversely, enteroids treated with 10 mM BUT and PROP had a greater than 10-fold decrease in LGR5 compared to controls (CON; P < 0.05; Fig 4b). Only 1mM treatment of BUT was able to significantly decrease LGR5 expression (P < 0.05; Fig 4a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3454, 'human', 'decrease', 'null', 'low-throughput', 'Conversely, enteroids treated with 10 mM BUT and PROP had a greater than 10-fold decrease in LGR5 compared to controls (CON; P < 0.05; Fig 4b). Only 1mM treatment of BUT was able to significantly decrease LGR5 expression (P < 0.05; Fig 4a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3455, 'human', 'increase', 'null', 'low-throughput', 'Treatment with 10 mM SCFA significantly increased ALPI expression treatments but was highest (11-fold) in enteroids treated with PROP. There was no difference in ALPI expression in enteroids treated with 1 mM SCFAs (P > 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3456, 'human', 'increase', 'null', 'low-throughput', 'Treatment with 10 mM SCFA significantly increased ALPI expression treatments but was highest (11-fold) in enteroids treated with PROP. There was no difference in ALPI expression in enteroids treated with 1 mM SCFAs (P > 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3457, 'human', 'increase', 'null', 'low-throughput', 'Treatment with 10 mM SCFA significantly increased ALPI expression treatments but was highest (11-fold) in enteroids treated with PROP. There was no difference in ALPI expression in enteroids treated with 1 mM SCFAs (P > 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3458, 'human', 'increase', 'null', 'low-throughput', 'Gene expression of MUC2 was increased 2-fold due to ACET treatment and 5-fold due to BUT treatment (P < 0.05; Fig 4b) but was not increased due to PROP treatment. There was no difference in MUC2 expression in enteroids treated with 1 mM SCFAs (P > 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3459, 'human', 'increase', 'null', 'low-throughput', 'Gene expression of MUC2 was increased 2-fold due to ACET treatment and 5-fold due to BUT treatment (P < 0.05; Fig 4b) but was not increased due to PROP treatment. There was no difference in MUC2 expression in enteroids treated with 1 mM SCFAs (P > 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3460, 'human', 'increase', 'null', 'low-throughput', 'Lysozyme (LYZ) increased due to BUT alone (P < 0.05)however there was no difference in ALPI expression in enteroids treated with 1 mM SCFAs(P > 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3461, 'human', 'increase', 'null', 'low-throughput', 'Enteroendocrine cell marker CHGA was significantly increased (170-fold) in BUT treated enteroids. There was no difference in CHGA expression in enteroids treated with 1
mM SCFAs (P > 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3462, 'human', 'increase', 'null', 'low-throughput', 'Tuft cell marker DCLK1 was significantly elevated due to ACET(7-fold) and BUT (14-fold) treatment compared to CON (P < 0.05; Fig 4b). 1 mM ACET,BUT and PROP significantly increased DCLK1 expression (5-fold, 3.6-fold, and 5.8-fold,respectively; P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3463, 'human', 'increase', 'null', 'low-throughput', 'Tuft cell marker DCLK1 was significantly elevated due to ACET(7-fold) and BUT (14-fold) treatment compared to CON (P < 0.05; Fig 4b). 1 mM ACET,BUT and PROP significantly increased DCLK1 expression (5-fold, 3.6-fold, and 5.8-fold,respectively; P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3464, 'human', 'increase', 'null', 'low-throughput', 'Tuft cell marker DCLK1 was significantly elevated due to ACET(7-fold) and BUT (14-fold) treatment compared to CON (P < 0.05; Fig 4b). 1 mM ACET,BUT and PROP significantly increased DCLK1 expression (5-fold, 3.6-fold, and 5.8-fold,respectively; P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3465, 'human', 'increase', 'null', 'low-throughput', 'Anti-microbial peptide gene expression of REG3β and REG3γ (Fig 4b)were not increased due to ACET treatment, however, were significantly increased due to BUT(2.5-fold and 3-fold, respectively) compared to CON. There was no difference in REG3β and REG3γ expression in enteroids treated with 1 mM SCFAs (P > 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3466, 'human', 'increase', 'null', 'low-throughput', 'Anti-microbial peptide gene expression of REG3β and REG3γ (Fig 4b)were not increased due to ACET treatment, however, were significantly increased due to BUT(2.5-fold and 3-fold, respectively) compared to CON. There was no difference in REG3β and REG3γ expression in enteroids treated with 1 mM SCFAs (P > 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3467, 'human', 'increase', 'null', 'low-throughput', 'β-defensin 1 (DEFB1)was increased 75-fold and 66-fold respectively in BUT and PROP treated enteroids (P < 0.05; Fig4b).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3468, 'human', 'increase', 'null', 'low-throughput', 'β-defensin 1 (DEFB1)was increased 75-fold and 66-fold respectively in BUT and PROP treated enteroids (P < 0.05; Fig4b).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3469, 'human', 'increase', 'null', 'low-throughput', 'Tight junction marker expression of OCLN was only significantly elevated in ACET treated enteroids (4-fold; P < 0.05,Fig 4b).There was no difference in OCLN expression in enteroids treated with 1 mM SCFAs(P > 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3470, 'human', 'increase', 'null', 'low-throughput', 'CLDN3 and TJP1 were increased due to both ACET and BUT treatments(P < 0.05, Fig 4b). There was no difference in CLDN3 expression in enteroids treated with 1 mM SCFAs (P > 0.05), however 1 mM BUT decreased TJP1 expression more than 2-fold (P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3471, 'human', 'increase', 'null', 'low-throughput', 'CLDN3 and TJP1 were increased due to both ACET and BUT treatments(P < 0.05, Fig 4b). There was no difference in CLDN3 expression in enteroids treated with 1 mM SCFAs (P > 0.05), however 1 mM BUT decreased TJP1 expression more than 2-fold (P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3472, 'human', 'increase', 'null', 'low-throughput', 'CLDN3 and TJP1 were increased due to both ACET and BUT treatments(P < 0.05, Fig 4b). There was no difference in CLDN3 expression in enteroids treated with 1 mM SCFAs (P > 0.05), however 1 mM BUT decreased TJP1 expression more than 2-fold (P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3473, 'human', 'increase', 'null', 'low-throughput', 'CLDN3 and TJP1 were increased due to both ACET and BUT treatments(P < 0.05, Fig 4b). There was no difference in CLDN3 expression in enteroids treated with 1 mM SCFAs (P > 0.05), however 1 mM BUT decreased TJP1 expression more than 2-fold (P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3474, 'human', 'increase', 'null', 'low-throughput', 'Tight junction related marker claudin-3 (CLDN3) expression was significantly increased by 10 mM BUT and 10 mM PROP (1.5 and 1.4-fold, respectively; P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3475, 'human', 'increase', 'null', 'low-throughput', 'Occludin (OCLN) expression was significantly increased by 10 mM BUT treatment alone (P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3476, 'human', 'increase', 'null', 'low-throughput', 'Ghrelin was significantly increased by all SCFAs in cell supernatants, with BUT and PROP showing highest concentrations (P < 0.05). Ghrelin was also significantly increased by 10 mM BUT alone in basolateral media (P < 0.05; Table 1).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3477, 'human', 'increase', 'null', 'low-throughput', 'Ghrelin was significantly increased by all SCFAs in cell supernatants, with BUT and PROP showing highest concentrations (P < 0.05). Ghrelin was also significantly increased by 10 mM BUT alone in basolateral media (P < 0.05; Table 1).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3478, 'human', 'increase', 'null', 'low-throughput', 'Ghrelin was significantly increased by all SCFAs in cell supernatants, with BUT and PROP showing highest concentrations (P < 0.05). Ghrelin was also significantly increased by 10 mM BUT alone in basolateral media (P < 0.05; Table 1).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3479, 'human', 'increase', 'null', 'low-throughput', 'Interleukin 4 was significantly increased in 10 mM BUT and PROP treated 2D enteroids(P < 0.05) and was only increased significantly in basolateral media by 10 mM BUT (P < 0.05;Table 1).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3480, 'human', 'increase', 'null', 'low-throughput', 'Interleukin 4 was significantly increased in 10 mM BUT and PROP treated 2D enteroids(P < 0.05) and was only increased significantly in basolateral media by 10 mM BUT (P < 0.05;Table 1).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3481, 'human', 'increase', 'null', 'low-throughput', 'Interleukin 8 was significantly increased in cell supernatants of 10 mM BUT treated enteroids as well as basolateral media (P < 0.05). Interleukin 18 was not significantly different in cell supernatant, however, there was a tendency for 10 mM BUT to increase expression (P = 0.08;Table 1). Interleukin 18, however was increased by 10 mM BUT treatment in basolateral media(P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3482, 'human', 'null', 'D-Glucose', 'null', 'B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).', 32873608);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3483, 'human', 'null', 'D-Glucose', 'null', 'B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).', 32873608);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3484, 'human', 'null', 'D-Glucose', 'null', 'B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).', 32873608);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3485, 'human', 'null', 'D-Glucose', 'null', 'B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).', 32873608);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3486, 'human', 'null', 'D-Glucose', 'null', 'B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).', 32873608);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3487, 'human', 'null', 'D-Glucose', 'null', 'B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).', 32873608);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3488, 'human', 'null', 'D-Glucose', 'null', 'B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).', 32873608);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3489, 'human', 'null', 'D-Glucose', 'null', 'B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).', 32873608);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3490, 'human', 'null', 'D-Glucose', 'null', 'B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).', 32873608);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3491, 'human', 'null', 'D-Glucose', 'null', 'B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).', 32873608);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3492, 'human', 'null', 'D-Glucose', 'null', 'B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).', 32873608);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3493, 'human', 'null', 'Levodopa', 'null', 'Here, we show that C. sporogenes deaminates levodopa, the main treatment in Parkinson’s disease, and identify the aromatic aminotransferase responsible for the initiation of the deamination pathway. The deaminate metabolite from levodopa, 3-(3,4-dihydroxyphenyl)propionic acid, elicits an inhibitory effect on ileal motility in an ex vivo model.', 33076930);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3494, 'human', 'null', '5-Hydroxytryptophan', 'null', '5-HTP was converted into two new unknown peaks (Additional File1: Fig. S2A, S2B), albeit to a much lesser extent compared to levodopa. Only the first peak could be detected and assigned by LC-MS as 5-hydroxyindole-3-lactic acid (5-HILA) by its predicted exact mass (Additional File 1:Fig. S2C).', 33076930);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3495, 'human', 'null', 'Tryptophan', 'null', 'Tryptophan and tyrosine were converted to their intermediates ILA (indole-3-lactic acid) and 4-HPLA (3-(4-hydroxyphenyl)lactic acid), respectively, as previously shown [11].', 33076930);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3496, 'human', 'null', 'Tyrosine', 'null', 'Tryptophan and tyrosine were converted to their intermediates ILA (indole-3-lactic acid) and 4-HPLA (3-(4-hydroxyphenyl)lactic acid), respectively, as previously shown [11].', 33076930);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3497, 'human', 'null', 'Levodopa', 'null', 'Analogous to tryptophan and tyrosine, levodopa was no longer deaminated to DHPPA but to its intermediate product 3-(3,4-dihydroxy phenyl)lactic acid (DHPLA) (Fig. 1d, e). ', 33076930);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3498, 'human', 'increase', 'null', 'low-throughput', 'The human ortholog HM74a is also a nicotinic acid receptor and likely has a similar role in anti-lipolysis.', 15929991);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3499, 'human', 'increase', 'null', 'low-throughput', 'Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.', 15929991);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3500, 'human', 'increase', 'null', 'low-throughput', 'Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.', 15929991);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3501, 'human', 'increase', 'null', 'low-throughput', 'Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.', 15929991);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3502, 'human', 'increase', 'null', 'low-throughput', 'Both (D)- and (L)-OHB, but not acetoacetate or free acetone, elicited a Ca2 response in cells expressing HM74a but not HM74.', 15929991);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3503, 'human', 'increase', 'null', 'low-throughput', 'Racemic (DL)-OHB also showed some degree of activity on all receptors; however, it was more potent on HM74a and its ortholog PUMA-G  than on HM74.', 15929991);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3504, 'human', 'decrease', 'Diosmetin 7-O-beta-D-glucopyranoside', 'low-throughput', 'A Gram-negative anaerobic bacterium, strain 4, capable of metabolizing diosmetin-7-O-glucoside was newly isolated.Its 16S rRNA sequence displayed 99% similarity with that of Escherichia. Then strain 4 was identified as a species of thegenus Escherichia and was named Escherichia sp. 4. Additionally,  an ultraperformance liquid chromatography/quadrupole-time-of-flight mass spectrometry  technique combined with Metabolynx software method was established to screenthe metabolites of diosmetin-7-O-glucoside.', 25382172);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3505, 'human', 'null', 'Diosmetin 7-O-beta-D-glucopyranoside', 'null', 'A Gram-negative anaerobic bacterium, strain 4, capable of metabolizing diosmetin-7-O-glucoside was newly isolated.Its 16S rRNA sequence displayed 99% similarity with that of Escherichia. Then strain 4 was identified as a species of thegenus Escherichia and was named Escherichia sp. 4. Additionally,  an ultraperformance liquid chromatography/quadrupole-time-of-flight mass spectrometry  technique combined with Metabolynx software method was established to screenthe metabolites of diosmetin-7-O-glucoside.', 25382172);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3506, 'human', 'increase', 'null', 'low-throughput', 'We analyzed 50 colon adenomas from AA patients for both microsatellite instability (MSI) and the methylation status of SLC5A8 gene. This gene''s product is involved in the transport of butyrate that has anti-proliferative properties through its effects on histone acetylation and gene expression. A proteomic analysis to check the expressed histones in adenoma and normal tissues was also performed.No association was found between SLC5A8 methylation and the MSI status. Also, there was no association between SLC5A8 methylation and the sex and age of the patients. However, there were more right sided adenomas with SLC5A8 methylation than the left sided ones. The proteomic analysis revealed distinct histone expression profiles between normal and adenoma tissues.SLC5A8 is highly methylated in AA colon adenomas which points to its potential use as a marker for early detection. The MSI rate is similar to that found in colon cancer tumors in AAs. These findings suggest that both processes stem from the same epigenetic and genetic events occurring at an early stage in colon carcinogenesis in AAs.', 21687703);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3507, 'human', 'increase', 'null', 'low-throughput', 'We analyzed 50 colon adenomas from AA patients for both microsatellite instability (MSI) and the methylation status of SLC5A8 gene. This gene''s product is involved in the transport of butyrate that has anti-proliferative properties through its effects on histone acetylation and gene expression. A proteomic analysis to check the expressed histones in adenoma and normal tissues was also performed.No association was found between SLC5A8 methylation and the MSI status. Also, there was no association between SLC5A8 methylation and the sex and age of the patients. However, there were more right sided adenomas with SLC5A8 methylation than the left sided ones. The proteomic analysis revealed distinct histone expression profiles between normal and adenoma tissues.SLC5A8 is highly methylated in AA colon adenomas which points to its potential use as a marker for early detection. The MSI rate is similar to that found in colon cancer tumors in AAs. These findings suggest that both processes stem from the same epigenetic and genetic events occurring at an early stage in colon carcinogenesis in AAs.', 21687703);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3508, 'human', 'increase', 'null', 'low-throughput', 'We analyzed 50 colon adenomas from AA patients for both microsatellite instability (MSI) and the methylation status of SLC5A8 gene. This gene''s product is involved in the transport of butyrate that has anti-proliferative properties through its effects on histone acetylation and gene expression. A proteomic analysis to check the expressed histones in adenoma and normal tissues was also performed.No association was found between SLC5A8 methylation and the MSI status. Also, there was no association between SLC5A8 methylation and the sex and age of the patients. However, there were more right sided adenomas with SLC5A8 methylation than the left sided ones. The proteomic analysis revealed distinct histone expression profiles between normal and adenoma tissues.SLC5A8 is highly methylated in AA colon adenomas which points to its potential use as a marker for early detection. The MSI rate is similar to that found in colon cancer tumors in AAs. These findings suggest that both processes stem from the same epigenetic and genetic events occurring at an early stage in colon carcinogenesis in AAs.', 21687703);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3509, 'human', 'null', 'null', 'null', 'In vitro cultivation of PD demonstrated its capacity to transform bile acids and production of succinate. Succinate supplementation in the diet decreased hyperglycemia in ob/ob mice via the activation of intestinal gluconeogenesis (IGN). Gavage with a mixture of LCA and UDCA reduced hyperlipidemia by activating the FXR pathway and repairing gut barrier integrity. Co-treatment with succinate and LCA/UDCA mirrored the benefits of LPD. The binding target of succinate was identified as fructose-1,6-bisphosphatase, the rate-limiting enzyme in IGN. The succinate and secondary bile acids produced by P. distasonis played key roles in the modulation of host metabolism.', 30605678);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3510, 'human', 'null', 'null', 'null', 'In vitro cultivation of PD demonstrated its capacity to transform bile acids and production of succinate. Succinate supplementation in the diet decreased hyperglycemia in ob/ob mice via the activation of intestinal gluconeogenesis (IGN). Gavage with a mixture of LCA and UDCA reduced hyperlipidemia by activating the FXR pathway and repairing gut barrier integrity. Co-treatment with succinate and LCA/UDCA mirrored the benefits of LPD. The binding target of succinate was identified as fructose-1,6-bisphosphatase, the rate-limiting enzyme in IGN. The succinate and secondary bile acids produced by P. distasonis played key roles in the modulation of host metabolism.', 30605678);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3516, 'human', 'null', 'Clindamycin', 'null', 'In the CDI chemostat model, clindamycin decreased valerate and deoxycholic acid concentrations and increased C difficile total viable counts (TVC) and valerate precursors, taurocholic acid, and succinate concentrations. After we stopped adding clindamycin, levels of bile acids and succinate recovered, whereas levels of valerate and valerate precursors did not. In the CDI chemostat model FMT increased valerate concentrations and decreased C difficile TVC (94% reduction), spore counts (86% reduction), and valerate precursor concentrations—concentrations of bile acids were unchanged.', 30025704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3517, 'human', 'null', 'Clindamycin', 'null', 'In the CDI chemostat model, clindamycin decreased valerate and deoxycholic acid concentrations and increased C difficile total viable counts (TVC) and valerate precursors, taurocholic acid, and succinate concentrations. After we stopped adding clindamycin, levels of bile acids and succinate recovered, whereas levels of valerate and valerate precursors did not. In the CDI chemostat model FMT increased valerate concentrations and decreased C difficile TVC (94% reduction), spore counts (86% reduction), and valerate precursor concentrations—concentrations of bile acids were unchanged.', 30025704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3518, 'human', 'null', 'Clindamycin', 'null', 'In the CDI chemostat model, clindamycin decreased valerate and deoxycholic acid concentrations and increased C difficile total viable counts (TVC) and valerate precursors, taurocholic acid, and succinate concentrations. After we stopped adding clindamycin, levels of bile acids and succinate recovered, whereas levels of valerate and valerate precursors did not. In the CDI chemostat model FMT increased valerate concentrations and decreased C difficile TVC (94% reduction), spore counts (86% reduction), and valerate precursor concentrations—concentrations of bile acids were unchanged.', 30025704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3519, 'human', 'null', 'Clindamycin', 'null', 'In the CDI chemostat model, clindamycin decreased valerate and deoxycholic acid concentrations and increased C difficile total viable counts (TVC) and valerate precursors, taurocholic acid, and succinate concentrations. After we stopped adding clindamycin, levels of bile acids and succinate recovered, whereas levels of valerate and valerate precursors did not. In the CDI chemostat model FMT increased valerate concentrations and decreased C difficile TVC (94% reduction), spore counts (86% reduction), and valerate precursor concentrations—concentrations of bile acids were unchanged.', 30025704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3520, 'human', 'decrease', 'null', 'low-throughput', 'In our study, we demonstrate that n-butyrate, a bacterial metabolite found in high abundance in the colon, regulates macrophage function through the inhibition of HDACs. n-Butyrate may behave as a microbial signal to inform the host immune system that the relative levels of n-butyrate–producing bacteria are within the desired range. In this model, HDACs act as sensors for n-butyrate, and their inhibition by high amounts of this metabolite causes intestinal macrophages to reduce production of proinflammatory mediators such as NO, IL-6, and IL-12.', 24390544);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3521, 'human', 'decrease', 'null', 'low-throughput', 'In our study, we demonstrate that n-butyrate, a bacterial metabolite found in high abundance in the colon, regulates macrophage function through the inhibition of HDACs. n-Butyrate may behave as a microbial signal to inform the host immune system that the relative levels of n-butyrate–producing bacteria are within the desired range. In this model, HDACs act as sensors for n-butyrate, and their inhibition by high amounts of this metabolite causes intestinal macrophages to reduce production of proinflammatory mediators such as NO, IL-6, and IL-12.', 24390544);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3522, 'human', 'decrease', 'null', 'low-throughput', 'In our study, we demonstrate that n-butyrate, a bacterial metabolite found in high abundance in the colon, regulates macrophage function through the inhibition of HDACs. n-Butyrate may behave as a microbial signal to inform the host immune system that the relative levels of n-butyrate–producing bacteria are within the desired range. In this model, HDACs act as sensors for n-butyrate, and their inhibition by high amounts of this metabolite causes intestinal macrophages to reduce production of proinflammatory mediators such as NO, IL-6, and IL-12.', 24390544);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3523, 'human', 'null', 'Resveratrol', 'null', 'In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.', 23283496);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3524, 'human', 'null', 'Resveratrol', 'null', 'In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.', 23283496);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3525, 'human', 'null', 'Resveratrol', 'null', 'In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.', 23283496);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3526, 'human', 'null', 'Resveratrol', 'null', 'In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.', 23283496);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3527, 'human', 'null', 'Resveratrol', 'null', 'In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.', 23283496);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3528, 'human', 'null', 'Resveratrol', 'null', 'In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.', 23283496);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3529, 'human', 'increase', '1-((3,5-Dichloro)-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone', 'low-throughput', 'DIF-3 induces proteolysis of cyclin D1 and c-Myc in HCT-116 cells via activation of GSK-3b.', 25913757);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3530, 'human', 'decrease', '1-((3,5-Dichloro)-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone', 'low-throughput', 'The amount of TCF7L2 increased gradually over time in control cells, but it was markedly reduced by DIF-3 in a time and dose-dependent manner.', 25913757);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3531, 'human', 'decrease', '1-((3,5-Dichloro)-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone', 'low-throughput', 'DIF-3 decreased the expression of cyclin D1 and cMyc to a level comparable to the reduction induced by DIF-1.', 25913757);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3532, 'human', 'decrease', '1-((3,5-Dichloro)-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone', 'low-throughput', 'DIF-3 decreased the expression of cyclin D1 and cMyc to a level comparable to the reduction induced by DIF-1.', 25913757);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3533, 'human', 'decrease', 'null', 'low-throughput', 'Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC3) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.', 30683619);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3534, 'human', 'decrease', 'null', 'low-throughput', 'Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC4) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.', 30683619);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3535, 'human', 'decrease', 'null', 'low-throughput', 'Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC5) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.', 30683619);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3536, 'human', 'null', 'Xyloglucan Heptasaccharide', 'null', 'Commensal bifidobacteria colonize the human gastrointestinal tract and catabolize glycans that are impervious to host digestion. Accordingly, Bifidobacterium longum typically secretes acetate and lactate as fermentative end products. This study tested the hypothesis that B. longum utilizes cranberry-derived xyloglucans in a strain-dependent manner. Interestingly, the B. longum strain that efficiently utilizes cranberry xyloglucans secretes 2.0 to 2.5 mol of acetate-lactate. The 1.5 acetate:lactate ratio theoretical yield obtained in hexose fermentations shifts during xyloglucan metabolism. Accordingly, this metabolic shift is characterized by increased acetate and formate production at the expense of lactate. α-l-Arabinofuranosidase, an arabinan endo-1,5-α-l-arabinosidase, and a β-xylosidase with a carbohydrate substrate-binding protein and carbohydrate ABC transporter membrane proteins are upregulated (>2-fold change), which suggests carbon flux through this catabolic pathway. Finally, syntrophic interactions occurred with strains that utilize carbohydrate products derived from initial degradation from heterologous bacteria.', 28667113);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3537, 'human', 'null', 'Xyloglucan Heptasaccharide', 'null', 'Commensal bifidobacteria colonize the human gastrointestinal tract and catabolize glycans that are impervious to host digestion. Accordingly, Bifidobacterium longum typically secretes acetate and lactate as fermentative end products. This study tested the hypothesis that B. longum utilizes cranberry-derived xyloglucans in a strain-dependent manner. Interestingly, the B. longum strain that efficiently utilizes cranberry xyloglucans secretes 2.0 to 2.5 mol of acetate-lactate. The 1.5 acetate:lactate ratio theoretical yield obtained in hexose fermentations shifts during xyloglucan metabolism. Accordingly, this metabolic shift is characterized by increased acetate and formate production at the expense of lactate. α-l-Arabinofuranosidase, an arabinan endo-1,5-α-l-arabinosidase, and a β-xylosidase with a carbohydrate substrate-binding protein and carbohydrate ABC transporter membrane proteins are upregulated (>2-fold change), which suggests carbon flux through this catabolic pathway. Finally, syntrophic interactions occurred with strains that utilize carbohydrate products derived from initial degradation from heterologous bacteria.', 28667113);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3538, 'human', 'null', 'null', 'null', 'We have presented direct evidence that the gut bacterial genotoxin colibactin alkylates DNA in vivo. The ability of pks+ E. coli to generate DNA adducts in mammalian cells and in mice strengthens support for the involvement of colibactin in cancer development or progression since misrepaired mono- and cross-linked adducts may generate mutations in oncogenes or tumor suppressor genes, contributing to tumorigenesis . Our findings will enable efforts to decipher the molecular details of this process. Importantly, this work has also uncovered a candidate metabolite biomarker of colibactin exposure and cancer risk. The ability to directly assess whether exposure to colibactin has occurred in animal models and human patients will help address the critical question of whether pks+ E. coli contribute to colorectal carcinogenesis in patient cohorts .', 30765538);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3539, 'human', 'null', 'null', 'null', 'We have presented direct evidence that the gut bacterial genotoxin colibactin alkylates DNA in vivo. The ability of pks+ E. coli to generate DNA adducts in mammalian cells and in mice strengthens support for the involvement of colibactin in cancer development or progression since misrepaired mono- and cross-linked adducts may generate mutations in oncogenes or tumor suppressor genes, contributing to tumorigenesis . Our findings will enable efforts to decipher the molecular details of this process. Importantly, this work has also uncovered a candidate metabolite biomarker of colibactin exposure and cancer risk. The ability to directly assess whether exposure to colibactin has occurred in animal models and human patients will help address the critical question of whether pks+ E. coli contribute to colorectal carcinogenesis in patient cohorts .', 30765538);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3542, 'human', 'null', 'Soybean Isoflavones', 'null', 'An equol-producing bacterium, strain DZET , which was isolated from human faeces, was characterized by morphological, biochemical and molecular methods. The isolate was Grampositive, obligately anaerobic, non-spore forming, asaccharolytic and rod-shaped. 16S rRNA sequence analysis showed 92.8, 91.0, 91.1 and 90.6 % similarities with Slackia faecicanis, Slackia exigua, Slackia heliotrinireducens and Slackia isoflavoniconvertens, respectively. Basedon these data, we propose a novel species of the genus Slackia, Slackia equolifaciens sp. nov.The major cellular fatty acids are C14 : 0, C18 : 1v9c and C18 : 1v9c DMA (dimethyl acetal). The DNA G+C content of the strain is 60.8 mol%. The type strain of S.equolifaciens sp. nov. is DZE.', 19734283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3543, 'human', 'null', 'Daidzein', 'null', 'Equol and 5-hydroxy-equol were found in intestinal contents, feces, and urine of SIA rats but not in the corresponding samples of the control rats. 5-Hydroxy-equol was present at much lower concentrations than equol and the main metabolite produced from genistein was the intermediate dihydrogenistein. The plasma of SIA rats contained equol but no 5-hydroxy-equol. Equol formation had no effect on plasma concentrations of the insulin-like growth factor I. The concentrations of daidzein and genistein were considerably lower in all samples of the SIA rats than in those of the control rats. Male SIA rats had higher intestinal and fecal concentrations of the isoflavones and their metabolites than female SIA rats. The observed activity in the rat model indicates that S. isoflavoniconvertens is capable of contributing in vivo to the bioactivation of daidzein and genistein by formation of equol and 5-hydroxy-equol.', 22113864);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3544, 'human', 'null', 'Genistein', 'null', 'Equol and 5-hydroxy-equol were found in intestinal contents, feces, and urine of SIA rats but not in the corresponding samples of the control rats. 5-Hydroxy-equol was present at much lower concentrations than equol and the main metabolite produced from genistein was the intermediate dihydrogenistein. The plasma of SIA rats contained equol but no 5-hydroxy-equol. Equol formation had no effect on plasma concentrations of the insulin-like growth factor I. The concentrations of daidzein and genistein were considerably lower in all samples of the SIA rats than in those of the control rats. Male SIA rats had higher intestinal and fecal concentrations of the isoflavones and their metabolites than female SIA rats. The observed activity in the rat model indicates that S. isoflavoniconvertens is capable of contributing in vivo to the bioactivation of daidzein and genistein by formation of equol and 5-hydroxy-equol.', 22113864);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3545, 'human', 'null', 'null', 'null', 'The alignment results showed that homologous proteins for FOS transporters and glycosidases were distributed in 237 of the 453 strains of gut bacteria. La506 msmK encoding the ATP-binding protein and Aec45 fosGH1 encoding glycoside hydrolase were most widely distributed, in 155 and 55 strains, respectively. Seven of eight strains with both transporters and glycosidases were proven to be capable of metabolizing FOS, while five strains without either transporters or glycosidases were not. Fifteen species isolated from human feces and 11 species from the alignment search were identified to be FOS-metabolizing, of which Cronobacter sakazakii, Marvinbryantia formatexigens, Ruminococcus gnavus, and Weissella paramesenteroides are reported here for the first time.', 25653214);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3546, 'human', 'null', 'null', 'null', 'The alignment results showed that homologous proteins for FOS transporters and glycosidases were distributed in 237 of the 453 strains of gut bacteria. La506 msmK encoding the ATP-binding protein and Aec45 fosGH1 encoding glycoside hydrolase were most widely distributed, in 155 and 55 strains, respectively. Seven of eight strains with both transporters and glycosidases were proven to be capable of metabolizing FOS, while five strains without either transporters or glycosidases were not. Fifteen species isolated from human feces and 11 species from the alignment search were identified to be FOS-metabolizing, of which Cronobacter sakazakii, Marvinbryantia formatexigens, Ruminococcus gnavus, and Weissella paramesenteroides are reported here for the first time.', 25653214);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3547, 'human', 'null', 'null', 'null', 'The alignment results showed that homologous proteins for FOS transporters and glycosidases were distributed in 237 of the 453 strains of gut bacteria. La506 msmK encoding the ATP-binding protein and Aec45 fosGH1 encoding glycoside hydrolase were most widely distributed, in 155 and 55 strains, respectively. Seven of eight strains with both transporters and glycosidases were proven to be capable of metabolizing FOS, while five strains without either transporters or glycosidases were not. Fifteen species isolated from human feces and 11 species from the alignment search were identified to be FOS-metabolizing, of which Cronobacter sakazakii, Marvinbryantia formatexigens, Ruminococcus gnavus, and Weissella paramesenteroides are reported here for the first time.', 25653214);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3548, 'human', 'null', 'null', 'null', 'The alignment results showed that homologous proteins for FOS transporters and glycosidases were distributed in 237 of the 453 strains of gut bacteria. La506 msmK encoding the ATP-binding protein and Aec45 fosGH1 encoding glycoside hydrolase were most widely distributed, in 155 and 55 strains, respectively. Seven of eight strains with both transporters and glycosidases were proven to be capable of metabolizing FOS, while five strains without either transporters or glycosidases were not. Fifteen species isolated from human feces and 11 species from the alignment search were identified to be FOS-metabolizing, of which Cronobacter sakazakii, Marvinbryantia formatexigens, Ruminococcus gnavus, and Weissella paramesenteroides are reported here for the first time.', 25653214);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3549, 'human', 'null', 'Daidzin', 'null', 'Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.', 11195098);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3550, 'human', 'null', 'Genistin', 'null', 'Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.', 11195098);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3551, 'human', 'null', 'Daidzin', 'null', 'Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.', 11195098);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3552, 'human', 'null', 'Genistin', 'null', 'Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.', 11195098);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3553, 'human', 'null', 'Daidzein', 'null', 'Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.', 11195098);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3554, 'human', 'null', 'Genistein', 'null', 'Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.', 11195098);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3555, 'human', 'increase', 'null', 'high-throughput', 'Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3556, 'human', 'increase', 'null', 'high-throughput', 'Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3557, 'human', 'increase', 'null', 'high-throughput', 'Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3558, 'human', 'increase', 'null', 'high-throughput', 'Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3559, 'human', 'increase', 'null', 'high-throughput', 'Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3560, 'human', 'increase', 'null', 'high-throughput', 'Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3561, 'human', 'increase', 'null', 'high-throughput', 'Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3562, 'human', 'increase', 'null', 'high-throughput', 'Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3563, 'human', 'increase', 'null', 'high-throughput', 'Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3564, 'human', 'increase', 'null', 'high-throughput', 'Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3565, 'human', 'increase', 'null', 'high-throughput', 'Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3573, 'human', 'null', 'D-Glucose', 'null', 'Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.', 19667375);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3574, 'human', 'null', 'D-Glucose', 'null', 'Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.', 19667375);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3575, 'human', 'null', 'D-Glucose', 'null', 'Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.', 19667375);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3576, 'human', 'null', 'D-Glucose', 'null', 'Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.', 19667375);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3577, 'human', 'null', 'D-Glucose', 'null', 'Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.', 19667375);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3578, 'human', 'null', 'D-Glucose', 'null', 'Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.', 19667375);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3579, 'human', 'null', 'D-Glucose', 'null', 'Two anaerobic, non-spore-forming, pleomorphic, Gram-negative rods, designated YIT 11840T and YIT 11841T, were isolated from human faeces. The organisms were catalase-negative, produced succinic and acetic acids as end products of glucose metabolism and had DNA G+C contents of approximately 48�?9 mol%. Although the phenotypic characteristics of these two strains were very similar, analysis of their 16S rRNA sequence showed that they are only distantly related (93.8 %), indicating that they represent two different species. A comparative sequence analysis revealed that these two species are members of the family ‘Prevotellaceae�?but are phylogenetically distant (<88 % sequence similarity) from the known genera belonging to this family, including Prevotella, Hallela and Xylanibacter.', 19567577);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3580, 'human', 'null', 'D-Glucose', 'null', 'Two anaerobic, non-spore-forming, pleomorphic, Gram-negative rods, designated YIT 11840T and YIT 11841T, were isolated from human faeces. The organisms were catalase-negative, produced succinic and acetic acids as end products of glucose metabolism and had DNA G+C contents of approximately 48�?9 mol%. Although the phenotypic characteristics of these two strains were very similar, analysis of their 16S rRNA sequence showed that they are only distantly related (93.8 %), indicating that they represent two different species. A comparative sequence analysis revealed that these two species are members of the family ‘Prevotellaceae�?but are phylogenetically distant (<88 % sequence similarity) from the known genera belonging to this family, including Prevotella, Hallela and Xylanibacter.', 19567577);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3581, 'human', 'null', 'D-Glucose', 'null', 'Two anaerobic, non-spore-forming, pleomorphic, Gram-negative rods, designated YIT 11840T and YIT 11841T, were isolated from human faeces. The organisms were catalase-negative, produced succinic and acetic acids as end products of glucose metabolism and had DNA G+C contents of approximately 48�?9 mol%. Although the phenotypic characteristics of these two strains were very similar, analysis of their 16S rRNA sequence showed that they are only distantly related (93.8 %), indicating that they represent two different species. A comparative sequence analysis revealed that these two species are members of the family ‘Prevotellaceae�?but are phylogenetically distant (<88 % sequence similarity) from the known genera belonging to this family, including Prevotella, Hallela and Xylanibacter.', 19567577);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3582, 'human', 'null', 'D-Glucose', 'null', 'Two anaerobic, non-spore-forming, pleomorphic, Gram-negative rods, designated YIT 11840T and YIT 11841T, were isolated from human faeces. The organisms were catalase-negative, produced succinic and acetic acids as end products of glucose metabolism and had DNA G+C contents of approximately 48�?9 mol%. Although the phenotypic characteristics of these two strains were very similar, analysis of their 16S rRNA sequence showed that they are only distantly related (93.8 %), indicating that they represent two different species. A comparative sequence analysis revealed that these two species are members of the family ‘Prevotellaceae�?but are phylogenetically distant (<88 % sequence similarity) from the known genera belonging to this family, including Prevotella, Hallela and Xylanibacter.', 19567577);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3583, 'human', 'null', 'null', 'null', 'F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.', 25900655);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3584, 'human', 'null', 'null', 'null', 'F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.', 25900655);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3585, 'human', 'null', 'null', 'null', 'F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.', 25900655);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3589, 'human', 'decrease', 'Retinol', 'low-throughput', 'Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.', 30566883);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3590, 'human', 'increase', 'Retinol', 'low-throughput', 'Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.', 30566883);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3591, 'human', 'increase', 'Retinol', 'low-throughput', 'Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.', 30566883);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3596, 'human', 'null', 'Trimethylamine', 'null', 'We now show gut microbes, through generation of trimethylamine N-oxide (TMAO), directly contribute to platelet hyperreactivity and enhanced thrombosis potential. Plasma TMAO levels in subjects (n > 4,000) independently predicted incident (3 years) thrombosis (heart attack, stroke) risk. Direct exposure of platelets to TMAO enhanced sub maximal stimulus-dependent platelet activation from multiple agonists through augmented Ca2+ release from intracellular stores. Animal model studies employing dietary choline or TMAO, germ-free mice, and microbial transplantation collectively confirm a role for gut microbiota and TMAO in modulating platelet hyperresponsiveness and thrombosis potential and identify microbial taxa associated with plasma TMAO and thrombosis potential.', 26972052);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3597, 'human', 'null', 'Trimethylamine', 'null', 'We now show gut microbes, through generation of trimethylamine N-oxide (TMAO), directly contribute to platelet hyperreactivity and enhanced thrombosis potential. Plasma TMAO levels in subjects (n > 4,000) independently predicted incident (3 years) thrombosis (heart attack, stroke) risk. Direct exposure of platelets to TMAO enhanced sub maximal stimulus-dependent platelet activation from multiple agonists through augmented Ca2+ release from intracellular stores. Animal model studies employing dietary choline or TMAO, germ-free mice, and microbial transplantation collectively confirm a role for gut microbiota and TMAO in modulating platelet hyperresponsiveness and thrombosis potential and identify microbial taxa associated with plasma TMAO and thrombosis potential.', 26972052);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3609, 'human', 'increase', 'Linoleic acid', 'low-throughput', 'In this study, we investigated the effects of LA and LA-derived fatty acids on the activation of peroxisome proliferator-activated receptors (PPARs) which regulate whole-body energy metabolism. None of the fatty acids activated PPARd, whereas almost all activated PPARa in luciferase assays. Two fatty acids potently activated PPARg, a master regulator of adipocyte differentiation, with 10-oxo-12(Z)-octadecenoic acid (KetoA) having the most potency. In 3T3- L1 cells, KetoA induced adipocyte differentiation via the activation of PPARg, and increased adiponectin production and insulin-stimulated glucose uptake. These findings suggest that fatty acids, including KetoA, generated in gut by lactic acid bacteria may be involved in the regulation of host energy metabolism.', 25749343);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3610, 'human', 'increase', 'D-Fructose', 'low-throughput', 'A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice.', 31255176);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3614, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.', 31506421);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3615, 'human', 'decrease', 'Tryptophan', 'low-throughput', 'Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.', 31506421);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3616, 'human', 'decrease', 'Tryptophan', 'low-throughput', 'Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.', 31506421);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3617, 'human', 'decrease', 'Tryptophan', 'low-throughput', 'Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.', 31506421);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3618, 'human', 'decrease', 'Tryptophan', 'low-throughput', 'Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.', 31506421);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3619, 'human', 'null', 'null', 'null', 'Gut flora and plasma microbiome-related metabolites were evaluated by amplicon sequencing of the bacterial 16S ribosomal RNA gene and capillary electrophoresis time-of-flight mass spectrometry, respectively. HF patients were evaluated in both the Decomp and Comp phases during hospitalization. The phylum Actinobacteria was enriched in HF patients compared with control subjects. At the genus level, Bifiodobacterium was abundant while Megamonas was depleted in HF patients. Meanwhile, plasma concentration of trimethylamine N-oxide (TMAO), a gut microbiome-derived metabolite, was increased in HF patients. A correlation analysis revealed positive correlations between the abundance of the genus Escherichia/Shigella and levels of TMAO and indoxyl sulfate (IS, a microbe-dependent uremic toxin) in Comp HF . Escherichia/Shigella was more abundant in Decomp than in Comp HF (P=0.030).Conclusions: Our results suggest that gut microbiome composition and microbiome-related metabolites are altered in HF patient.', 30487369);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3620, 'human', 'null', 'null', 'null', 'Gut flora and plasma microbiome-related metabolites were evaluated by amplicon sequencing of the bacterial 16S ribosomal RNA gene and capillary electrophoresis time-of-flight mass spectrometry, respectively. HF patients were evaluated in both the Decomp and Comp phases during hospitalization. The phylum Actinobacteria was enriched in HF patients compared with control subjects. At the genus level, Bifiodobacterium was abundant while Megamonas was depleted in HF patients. Meanwhile, plasma concentration of trimethylamine N-oxide (TMAO), a gut microbiome-derived metabolite, was increased in HF patients. A correlation analysis revealed positive correlations between the abundance of the genus Escherichia/Shigella and levels of TMAO and indoxyl sulfate (IS, a microbe-dependent uremic toxin) in Comp HF . Escherichia/Shigella was more abundant in Decomp than in Comp HF (P=0.030).Conclusions: Our results suggest that gut microbiome composition and microbiome-related metabolites are altered in HF patient.', 30487369);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3621, 'human', 'null', 'null', 'null', 'Gut flora and plasma microbiome-related metabolites were evaluated by amplicon sequencing of the bacterial 16S ribosomal RNA gene and capillary electrophoresis time-of-flight mass spectrometry, respectively. HF patients were evaluated in both the Decomp and Comp phases during hospitalization. The phylum Actinobacteria was enriched in HF patients compared with control subjects. At the genus level, Bifiodobacterium was abundant while Megamonas was depleted in HF patients. Meanwhile, plasma concentration of trimethylamine N-oxide (TMAO), a gut microbiome-derived metabolite, was increased in HF patients. A correlation analysis revealed positive correlations between the abundance of the genus Escherichia/Shigella and levels of TMAO and indoxyl sulfate (IS, a microbe-dependent uremic toxin) in Comp HF . Escherichia/Shigella was more abundant in Decomp than in Comp HF (P=0.030).Conclusions: Our results suggest that gut microbiome composition and microbiome-related metabolites are altered in HF patient.', 30487369);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3622, 'human', 'null', 'null', 'null', 'Gut flora and plasma microbiome-related metabolites were evaluated by amplicon sequencing of the bacterial 16S ribosomal RNA gene and capillary electrophoresis time-of-flight mass spectrometry, respectively. HF patients were evaluated in both the Decomp and Comp phases during hospitalization. The phylum Actinobacteria was enriched in HF patients compared with control subjects. At the genus level, Bifiodobacterium was abundant while Megamonas was depleted in HF patients. Meanwhile, plasma concentration of trimethylamine N-oxide (TMAO), a gut microbiome-derived metabolite, was increased in HF patients. A correlation analysis revealed positive correlations between the abundance of the genus Escherichia/Shigella and levels of TMAO and indoxyl sulfate (IS, a microbe-dependent uremic toxin) in Comp HF . Escherichia/Shigella was more abundant in Decomp than in Comp HF (P=0.030).Conclusions: Our results suggest that gut microbiome composition and microbiome-related metabolites are altered in HF patient.', 30487369);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3626, 'human', 'null', 'Quercitrin', 'null', 'Ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC/Q-TOF MS) technique combined with MetabolynxTM software was used for analysis of the metabolites of quercitrin by the isolated human intestinal bacteria from the human feces. Four metabolites of quercitrin were detected and tentatively identified based on the characteristics of their protonated ions. The metabolites were metabolized by four main metabolic pathwaysincluding hydroxylation, demethylation, deglycosylation and ring-cleavage. Quercitrin was metabolized to the hydroxyquercitrin and desmethylquercitrin by the majority of the isolated intestinal bacteria such as Bacteroidessp. 54, and was degraded to the deglycosylated product quercetin by rhamnosidase and further ring cleavage metabolite 3,4-dihydroxybenzoic acid by the minority of the isolated bacteria such as Bacteroides sp. 45. The metabolic pathways and most of the metabolites of quercitrin were reported for the first time.', 23754166);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3627, 'human', 'null', 'Quercitrin', 'null', 'Ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC/Q-TOF MS) technique combined with MetabolynxTM software was used for analysis of the metabolites of quercitrin by the isolated human intestinal bacteria from the human feces. Four metabolites of quercitrin were detected and tentatively identified based on the characteristics of their protonated ions. The metabolites were metabolized by four main metabolic pathwaysincluding hydroxylation, demethylation, deglycosylation and ring-cleavage. Quercitrin was metabolized to the hydroxyquercitrin and desmethylquercitrin by the majority of the isolated intestinal bacteria such as Bacteroidessp. 54, and was degraded to the deglycosylated product quercetin by rhamnosidase and further ring cleavage metabolite 3,4-dihydroxybenzoic acid by the minority of the isolated bacteria such as Bacteroides sp. 45. The metabolic pathways and most of the metabolites of quercitrin were reported for the first time.', 23754166);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3628, 'human', 'null', 'Quercitrin', 'null', 'Ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC/Q-TOF MS) technique combined with MetabolynxTM software was used for analysis of the metabolites of quercitrin by the isolated human intestinal bacteria from the human feces. Four metabolites of quercitrin were detected and tentatively identified based on the characteristics of their protonated ions. The metabolites were metabolized by four main metabolic pathwaysincluding hydroxylation, demethylation, deglycosylation and ring-cleavage. Quercitrin was metabolized to the hydroxyquercitrin and desmethylquercitrin by the majority of the isolated intestinal bacteria such as Bacteroidessp. 54, and was degraded to the deglycosylated product quercetin by rhamnosidase and further ring cleavage metabolite 3,4-dihydroxybenzoic acid by the minority of the isolated bacteria such as Bacteroides sp. 45. The metabolic pathways and most of the metabolites of quercitrin were reported for the first time.', 23754166);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3632, 'human', 'decrease', 'null', 'low-throughput', 'Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host''s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p39 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.', 27505423);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3633, 'human', 'increase', 'null', 'low-throughput', 'Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host''s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p40 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.', 27505423);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3634, 'human', 'increase', 'null', 'low-throughput', 'Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host''s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p41 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.', 27505423);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3635, 'human', 'increase', 'null', 'low-throughput', 'Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host''s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p42 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.', 27505423);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3636, 'human', 'increase', 'null', 'low-throughput', 'Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host''s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p43 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.', 27505423);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3637, 'human', 'null', 'D-Sorbitol', 'null', 'Three sequential fermentative batches were carried out with cell recycle in four simultaneously operating bioreactors maintained at pH 6.5, 30°C, and 100 rpm. P. acidipropionici ATCC 4875 was able to produce propionic and succinic acid from sorbitol. The concentration of propionic acid decreased slightly from 39.5 ± 5.2 g L�? to 34.4 ± 1.9 g L�?,and that of succinic acid increased significantly from 6.1 ± 2.1 g L�? to 14.8 ± 0.9 g L�? through the sequential batches. In addition, a small amount of acetic acid was produced that decreased from 3.3 ± 0.4 g L�? to 2.0 ± 0.3 g L�? through the batches. The major yield for propionic acid was 0.613 g g�? in the first batch and succinic acid it was 0.212 g g�? in the third batch. The minor yield of acetic acid was 0.029 g g�? ,in the second and third batches.', 25852990);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3638, 'human', 'null', 'D-Sorbitol', 'null', 'Three sequential fermentative batches were carried out with cell recycle in four simultaneously operating bioreactors maintained at pH 6.5, 30°C, and 100 rpm. P. acidipropionici ATCC 4875 was able to produce propionic and succinic acid from sorbitol. The concentration of propionic acid decreased slightly from 39.5 ± 5.2 g L�? to 34.4 ± 1.9 g L�?,and that of succinic acid increased significantly from 6.1 ± 2.1 g L�? to 14.8 ± 0.9 g L�? through the sequential batches. In addition, a small amount of acetic acid was produced that decreased from 3.3 ± 0.4 g L�? to 2.0 ± 0.3 g L�? through the batches. The major yield for propionic acid was 0.613 g g�? in the first batch and succinic acid it was 0.212 g g�? in the third batch. The minor yield of acetic acid was 0.029 g g�? ,in the second and third batches.', 25852990);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3639, 'human', 'increase', 'null', 'low-throughput', 'In the present study, we found that TMAO significantly upregulated the expression of proinflammatory markers, such as IL-1β, IL-6, and TNF-α in TH positive neurons, as well as NMDARs, which might further result in sympathetic hyperactivation within the LSG and promote the incidence of VA.', 30954457);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3640, 'human', 'increase', 'null', 'low-throughput', 'In the present study, we found that TMAO significantly upregulated the expression of proinflammatory markers, such as IL-1β, IL-6, and TNF-α in TH positive neurons, as well as NMDARs, which might further result in sympathetic hyperactivation within the LSG and promote the incidence of VA.', 30954457);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3641, 'human', 'increase', 'null', 'low-throughput', 'In the present study, we found that TMAO significantly upregulated the expression of proinflammatory markers, such as IL-1β, IL-6, and TNF-α in TH positive neurons, as well as NMDARs, which might further result in sympathetic hyperactivation within the LSG and promote the incidence of VA.', 30954457);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3642, 'human', 'increase', '4-N-(3-Chloro-4-fluorophenyl)-6-N-[1-(iodomethyl)piperidin-4-yl]pyrimido[5,4-d]pyrimidine-4,6-diamine', 'low-throughput', 'The present study identified the metabolites from DN through analysis of organ-specific biotransformation and validated the cardioprotective effects of the screened metabolites using an isoprenaline-induced myocardial ischemia rat model. The findings of the present study provide evidence that, first, diosgenin is generated from DN by intestinal microflora; second, diosgenin can protect the myocardium against ischemic insult in a dose-dependent manner, almost comparable to the effect of DN extract; and, third, DN''s protective effect can be attributed to the increase of enzymatic and nonenzymatic antioxidants (SOD, CAT, GPx, and T-AOC) in vivo and to a decrease in lipid peroxidation. These phenomena help explain the clinical efficacy of DN as an anti-MI drug.', 29085554);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3643, 'human', 'increase', '4-N-(3-Chloro-4-fluorophenyl)-6-N-[1-(iodomethyl)piperidin-4-yl]pyrimido[5,4-d]pyrimidine-4,6-diamine', 'low-throughput', 'The present study identified the metabolites from DN through analysis of organ-specific biotransformation and validated the cardioprotective effects of the screened metabolites using an isoprenaline-induced myocardial ischemia rat model. The findings of the present study provide evidence that, first, diosgenin is generated from DN by intestinal microflora; second, diosgenin can protect the myocardium against ischemic insult in a dose-dependent manner, almost comparable to the effect of DN extract; and, third, DN''s protective effect can be attributed to the increase of enzymatic and nonenzymatic antioxidants (SOD, CAT, GPx, and T-AOC) in vivo and to a decrease in lipid peroxidation. These phenomena help explain the clinical efficacy of DN as an anti-MI drug.', 29085554);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3644, 'human', 'increase', '4-N-(3-Chloro-4-fluorophenyl)-6-N-[1-(iodomethyl)piperidin-4-yl]pyrimido[5,4-d]pyrimidine-4,6-diamine', 'low-throughput', 'The present study identified the metabolites from DN through analysis of organ-specific biotransformation and validated the cardioprotective effects of the screened metabolites using an isoprenaline-induced myocardial ischemia rat model. The findings of the present study provide evidence that, first, diosgenin is generated from DN by intestinal microflora; second, diosgenin can protect the myocardium against ischemic insult in a dose-dependent manner, almost comparable to the effect of DN extract; and, third, DN''s protective effect can be attributed to the increase of enzymatic and nonenzymatic antioxidants (SOD, CAT, GPx, and T-AOC) in vivo and to a decrease in lipid peroxidation. These phenomena help explain the clinical efficacy of DN as an anti-MI drug.', 29085554);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3645, 'human', 'decrease', 'Arbutin', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 21889493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3646, 'human', 'increase', 'Arbutin', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 21889493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3647, 'human', 'increase', 'Arbutin', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 21889493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3648, 'human', 'decrease', 'Arbutin', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 21889493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3649, 'human', 'increase', 'Arbutin', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 21889493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3650, 'human', 'increase', 'Arbutin', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 21889493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3651, 'human', 'decrease', 'Arbutin', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 21889493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3652, 'human', 'increase', 'Arbutin', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 21889493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3653, 'human', 'increase', 'Arbutin', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 21889493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3654, 'human', 'decrease', 'Arbutin', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 21889493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3655, 'human', 'increase', 'Arbutin', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 21889493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3656, 'human', 'increase', 'Arbutin', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 21889493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3667, 'human', 'null', 'Dihydrodaidzein', 'null', 'In this study, chemically synthesized dihydrodaidzein was used as the substrate administered to a newly isolated rodshaped, gram-negative, anaerobic bacterium from human feces which can convert dihydrodaidzein to S-equol.', 29112296);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3668, 'human', 'null', 'daidzein', 'null', 'Clostridium sp. strain HGH6, that is capable of reducing daidzein to dihydrodaidzein, but further metabolism to tetrahydrodaidzein or equol did not occur.', 15640190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3669, 'human', 'decrease', 'null', 'low-throughput', 'Our result also demonstrated that butyrate intervention enhanced fecal SCFAs concentrations, and inhibited endotoxin levels in feces and serum. Correlation analysis indicated positive relation between endotoxin level and Desulfovibrionaceae abundance. Furthermore,  butyrate intervention inhibited expressions of IL-1β, IL-6 and MCP1/CCL2 in liver, as well as TLR4 in adipose tissue.', 15640190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3670, 'human', 'decrease', 'null', 'low-throughput', 'Our result also demonstrated that butyrate intervention enhanced fecal SCFAs concentrations, and inhibited endotoxin levels in feces and serum. Correlation analysis indicated positive relation between endotoxin level and Desulfovibrionaceae abundance. Furthermore,  butyrate intervention inhibited expressions of IL-1β, IL-6 and MCP1/CCL2 in liver, as well as TLR4 in adipose tissue.', 31295342);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3671, 'human', 'decrease', 'null', 'low-throughput', 'Our result also demonstrated that butyrate intervention enhanced fecal SCFAs concentrations, and inhibited endotoxin levels in feces and serum. Correlation analysis indicated positive relation between endotoxin level and Desulfovibrionaceae abundance. Furthermore,  butyrate intervention inhibited expressions of IL-1β, IL-6 and MCP1/CCL2 in liver, as well as TLR4 in adipose tissue.', 31295342);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3672, 'human', 'decrease', 'null', 'low-throughput', 'Our result also demonstrated that butyrate intervention enhanced fecal SCFAs concentrations, and inhibited endotoxin levels in feces and serum. Correlation analysis indicated positive relation between endotoxin level and Desulfovibrionaceae abundance. Furthermore,  butyrate intervention inhibited expressions of IL-1β, IL-6 and MCP1/CCL2 in liver, as well as TLR4 in adipose tissue.', 31295342);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3673, 'human', 'null', 'Biochanin A', 'null', 'Eubacterium limosum (ATCC 8486), a strict anaerobe from the human intestinal tract that is capable of O-demethylation of several compounds, was tested for the ability to metabolize three methoxylated isoflavonoids, biochanin A, formononetin, and glycitein. High performance liquid chromatography elution profiles of metabolites produced from biochanin A, formononetin, and glycitein showed peaks that had identical retention times to authentic genistein, daidzein, and 6,7,4P-trihydroxyisoflavone, respectively.', 31295342);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3674, 'human', 'null', 'Formononetin', 'null', 'Eubacterium limosum (ATCC 8486), a strict anaerobe from the human intestinal tract that is capable of O-demethylation of several compounds, was tested for the ability to metabolize three methoxylated isoflavonoids, biochanin A, formononetin, and glycitein. High performance liquid chromatography elution profiles of metabolites produced from biochanin A, formononetin, and glycitein showed peaks that had identical retention times to authentic genistein, daidzein, and 6,7,4P-trihydroxyisoflavone, respectively.', 11040423);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3675, 'human', 'null', 'Glycitein', 'null', 'Eubacterium limosum (ATCC 8486), a strict anaerobe from the human intestinal tract that is capable of O-demethylation of several compounds, was tested for the ability to metabolize three methoxylated isoflavonoids, biochanin A, formononetin, and glycitein. High performance liquid chromatography elution profiles of metabolites produced from biochanin A, formononetin, and glycitein showed peaks that had identical retention times to authentic genistein, daidzein, and 6,7,4P-trihydroxyisoflavone, respectively.', 11040423);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3680, 'human', 'null', '(-)-Epigallocatechin gallate', 'null', 'The results showed that L. plantarum IFPL935 exhibited higher resistance to nongalloylated monomeric flavan-3-ols, A-type dimeric procyanidins, and cranberry extract than to (�?-epicatechin-3-O-gallate and B-type dimeric procyanidins. Despite these findings,the strain was capable of rapidly degrading (�?-epicatechin-3-O-gallate, but not A- or B-type dimeric procyanidins. However, it was able to produce large changes in the phenolic profile of the cranberry extract mainly due to the catabolism of hydroxycinnamic and hydroxybenzoic acids. Of most relevance was the fact that L. plantarum IFPL935 cleaved the heterocyclic ring of monomeric flavan-3-ols, giving rise to 1-(3�?4�?dihydroxyphenyl)-3-(2�?4�?6�?trihydroxyphenyl)propan-2-ol, activity exhibited by only a few human intestinal bacteria.', 22646528);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3681, 'human', 'null', '(-)-Epigallocatechin gallate', 'null', 'The results showed that L. plantarum IFPL935 exhibited higher resistance to nongalloylated monomeric flavan-3-ols, A-type dimeric procyanidins, and cranberry extract than to (�?-epicatechin-3-O-gallate and B-type dimeric procyanidins. Despite these findings,the strain was capable of rapidly degrading (�?-epicatechin-3-O-gallate, but not A- or B-type dimeric procyanidins. However, it was able to produce large changes in the phenolic profile of the cranberry extract mainly due to the catabolism of hydroxycinnamic and hydroxybenzoic acids. Of most relevance was the fact that L. plantarum IFPL935 cleaved the heterocyclic ring of monomeric flavan-3-ols, giving rise to 1-(3�?4�?dihydroxyphenyl)-3-(2�?4�?6�?trihydroxyphenyl)propan-2-ol, activity exhibited by only a few human intestinal bacteria.', 22646528);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3682, 'human', 'null', 'null', 'null', 'One important secondary metabolite of bifidobacteria is folate, which contributes to the one-carbon metabolism of the host and therefore enables maintenance of DNA methylation marks as explained below (41).', 23594611);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3683, 'human', 'increase', 'null', 'low-throughput', 'Furthermore, TMAO increased TGF-β receptor I expression, which promoted the phosphorylation of Smad2 and up-regulated the expression of α-SMA and collagen I. The ubiquitination of TGF-βRI was decreased in neonatal mouse fibroblasts after TMAO treatment. TMAO also inhibited the expression of smurf2. Inhibition of TGF-β1 receptor with the small molecule inhibitor SB431542 decreased TGF-β receptor I expression, reduced the phosphorylation of Smad2, down-regulated TMAO-induced α-SMA and collagen I expression in cardiac fibroblasts. Conclusions: Cardiac function and cardiac fibrosis were significantly exacerbated in mice fed diets supplemented with either choline or TMAO, probably through accelerating the transformation of fibroblasts into myofibroblasts, indicating activation of TGF-βRI/Smad2 pathway', 31299216);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3684, 'human', 'increase', 'null', 'low-throughput', 'Furthermore, TMAO increased TGF-β receptor I expression, which promoted the phosphorylation of Smad2 and up-regulated the expression of α-SMA and collagen I. The ubiquitination of TGF-βRI was decreased in neonatal mouse fibroblasts after TMAO treatment. TMAO also inhibited the expression of smurf2. Inhibition of TGF-β1 receptor with the small molecule inhibitor SB431542 decreased TGF-β receptor I expression, reduced the phosphorylation of Smad2, down-regulated TMAO-induced α-SMA and collagen I expression in cardiac fibroblasts. Conclusions: Cardiac function and cardiac fibrosis were significantly exacerbated in mice fed diets supplemented with either choline or TMAO, probably through accelerating the transformation of fibroblasts into myofibroblasts, indicating activation of TGF-βRI/Smad2 pathway', 31299216);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3685, 'human', 'decrease', 'null', 'low-throughput', 'Furthermore, TMAO increased TGF-β receptor I expression, which promoted the phosphorylation of Smad2 and up-regulated the expression of α-SMA and collagen I. The ubiquitination of TGF-βRI was decreased in neonatal mouse fibroblasts after TMAO treatment. TMAO also inhibited the expression of smurf2. Inhibition of TGF-β1 receptor with the small molecule inhibitor SB431542 decreased TGF-β receptor I expression, reduced the phosphorylation of Smad2, down-regulated TMAO-induced α-SMA and collagen I expression in cardiac fibroblasts. Conclusions: Cardiac function and cardiac fibrosis were significantly exacerbated in mice fed diets supplemented with either choline or TMAO, probably through accelerating the transformation of fibroblasts into myofibroblasts, indicating activation of TGF-βRI/Smad2 pathway', 31299216);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3686, 'human', 'null', 'Glycitein', 'null', 'The dichotomy between the biotransformation of polyphenols into their metabolites by gut microbiota and the modulation of gut microbiota composition by polyphenols contributes to positive health outcomes. Although there are many studies on the in vivo bioavailability of polyphenols, the mutual relationship between polyphenols and gut microbiota is not fully understood. This review focuses on the biotransformation of polyphenols by gut microbiota, modulation of gut microbiota by polyphenols, and the effects of these two-way mutual interactions on polyphenol bioavailability, and ultimately, human health.', 26861391);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3687, 'human', 'null', 'Glycitein', 'null', 'The dichotomy between the biotransformation of polyphenols into their metabolites by gut microbiota and the modulation of gut microbiota composition by polyphenols contributes to positive health outcomes. Although there are many studies on the in vivo bioavailability of polyphenols, the mutual relationship between polyphenols and gut microbiota is not fully understood. This review focuses on the biotransformation of polyphenols by gut microbiota, modulation of gut microbiota by polyphenols, and the effects of these two-way mutual interactions on polyphenol bioavailability, and ultimately, human health.', 26861391);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3688, 'human', 'null', 'Chlorogenic acid', 'null', 'Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.', 11412317);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3689, 'human', 'null', 'Chlorogenic acid', 'null', 'Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.', 11412317);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3690, 'human', 'null', 'Chlorogenic acid', 'null', 'Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.', 11412317);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3691, 'human', 'null', 'Chlorogenic acid', 'null', 'Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.', 11412317);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3692, 'human', 'null', 'Chlorogenic acid', 'null', 'Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.', 11412317);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3693, 'human', 'null', 'Chlorogenic acid', 'null', 'Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.', 11412317);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3694, 'human', 'null', 'D-Glucose', 'null', 'To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.', 26858714);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3695, 'human', 'null', 'D-Glucose', 'null', 'To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.', 26858714);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3696, 'human', 'null', 'D-Fructose', 'null', 'To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.', 26858714);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3697, 'human', 'null', 'D-Glucose', 'null', 'To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.', 26858714);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3698, 'human', 'null', 'D-Fructose', 'null', 'To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.', 26858714);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3699, 'human', 'null', 'Phosphoenolpyruvate', 'null', 'Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.', 22018234);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3700, 'human', 'null', 'Fumarate', 'null', 'Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.', 22018234);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3701, 'human', 'null', 'Succinate', 'null', 'Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.', 22018234);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3702, 'human', 'null', 'Acetate', 'null', 'Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.', 22018234);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3703, 'human', 'null', 'D-Glucose', 'null', 'Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.', 22018234);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3704, 'human', 'null', 'D-Glucose', 'null', 'Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.', 22018234);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3705, 'human', 'null', 'Daidzein', 'null', 'Equol is a metabolite produced from daidzein by enteric microflora, and it has attracted a great deal of attention because of its protective or ameliorative ability against several sex hormone-dependent diseases (e.g., menopausal disorder and lower bone density), which is more potent than that of other isoflavonoids. We purified a novel NADP(H)-dependent daidzein reductase (L-DZNR) from Lactococcus strain 20-92 (Lactococcus 20-92; S. Uchiyama, T. Ueno, and T. Suzuki, international patent WO2005/000042) that is involved in the metabolism of soy isoflavones and equol production and converts daidzein to dihydrodaidzein.', 20639368);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3706, 'human', 'null', 'Daidzein', 'null', 'Equol is a metabolite produced from daidzein by enteric microflora, and it has attracted a great deal of attention because of its protective or ameliorative ability against several sex hormone-dependent diseases (e.g., menopausal disorder and lower bone density), which is more potent than that of other isoflavonoids. We purified a novel NADP(H)-dependent daidzein reductase (L-DZNR) from Lactococcus strain 20-92 (Lactococcus 20-92; S. Uchiyama, T. Ueno, and T. Suzuki, international patent WO2005/000042) that is involved in the metabolism of soy isoflavones and equol production and converts daidzein to dihydrodaidzein.', 20639368);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3707, 'human', 'null', 'Mangiferin', 'null', 'The C-glucosyl bond of C-glucosides generally tolerates acid and enzymatic hydrolysis. Many C-glucosides are cleaved by human intestinal bacteria. We isolated the specific bacterium involved in the metabolism of mangiferin (2-b-D-glucopyranosyl-1,3,6,7-tetrahydroxyxanthone), C-glucosyl xanthone, from a mixture of human fecal bacteria. The anaerobic Bacteroides species named MANG, transformed mangiferin to the aglycone, norathyriol, suggesting cleavage of a C-glucosyl bond.', 16141538);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3708, 'human', 'null', 'Mangiferin', 'null', 'The C-glucosyl bond of C-glucosides generally tolerates acid and enzymatic hydrolysis. Many C-glucosides are cleaved by human intestinal bacteria. We isolated the specific bacterium involved in the metabolism of mangiferin (2-b-D-glucopyranosyl-1,3,6,7-tetrahydroxyxanthone), C-glucosyl xanthone, from a mixture of human fecal bacteria. The anaerobic Bacteroides species named MANG, transformed mangiferin to the aglycone, norathyriol, suggesting cleavage of a C-glucosyl bond.', 16141538);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3709, 'human', 'increase', 'null', 'low-throughput', '5-HT storing enterochromaffin (EC) cells are believed to respond to nutrient and gut microbial components, and 5-HT receptor-expressing afferent vagal neurons have been described to be the major sensors of nutrients in the GI-tract. However, the molecular mechanism through which EC cells sense nutrients and gut microbiota is still unclear.Nutrient metabolites apparently do not stimulate EC cells of the small intestine directly but through a paracrine mechanism involving GLP-1 secreted from neighboring enteroendocrine cells. In contrast, colonic EC cells are able to sense a multitude of different metabolites generated by the gut microbiota as well as gut hormones, including GLP-1.', 29576437);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3710, 'human', 'increase', 'null', 'low-throughput', '5-HT storing enterochromaffin (EC) cells are believed to respond to nutrient and gut microbial components, and 5-HT receptor-expressing afferent vagal neurons have been described to be the major sensors of nutrients in the GI-tract. However, the molecular mechanism through which EC cells sense nutrients and gut microbiota is still unclear.Nutrient metabolites apparently do not stimulate EC cells of the small intestine directly but through a paracrine mechanism involving GLP-1 secreted from neighboring enteroendocrine cells. In contrast, colonic EC cells are able to sense a multitude of different metabolites generated by the gut microbiota as well as gut hormones, including GLP-1.', 29576437);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3711, 'human', 'increase', 'null', 'low-throughput', '5-HT storing enterochromaffin (EC) cells are believed to respond to nutrient and gut microbial components, and 5-HT receptor-expressing afferent vagal neurons have been described to be the major sensors of nutrients in the GI-tract. However, the molecular mechanism through which EC cells sense nutrients and gut microbiota is still unclear.Nutrient metabolites apparently do not stimulate EC cells of the small intestine directly but through a paracrine mechanism involving GLP-1 secreted from neighboring enteroendocrine cells. In contrast, colonic EC cells are able to sense a multitude of different metabolites generated by the gut microbiota as well as gut hormones, including GLP-1.', 29576437);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3712, 'human', 'decrease', 'null', 'low-throughput', 'The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.', 16534848);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3713, 'human', 'decrease', 'null', 'low-throughput', 'The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.', 16534848);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3714, 'human', 'decrease', 'null', 'low-throughput', 'The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.', 16534848);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3715, 'human', 'decrease', 'null', 'low-throughput', 'The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.', 16534848);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3716, 'human', 'null', 'Rutin', 'null', 'In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.', 22583754);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3717, 'human', 'null', 'Rutin', 'null', 'In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.', 22583754);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3718, 'human', 'null', 'Rutin', 'null', 'In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.', 22583754);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3719, 'human', 'null', 'Rutin', 'null', 'In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.', 22583754);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3720, 'human', 'null', 'Rutin', 'null', 'In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.', 22583754);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3721, 'human', 'null', 'Rutin', 'null', 'In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.', 22583754);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3722, 'human', 'null', 'Rutin', 'null', 'In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.', 22583754);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3723, 'human', 'null', 'Phytoestrogens', 'null', 'Nine healthy male adult subjects ingested 0.3 g/kg/day flaxseeds during 1 week. Gut bacteria as well as blood and fecal metabolites were analyzed. Ingestion of flaxseeds triggered a significant increase in the blood concentration of enterolignans, accompanied by fecal excretion of propionate and glycerol. Overall diversity and composition of dominant fecal bacteria remained individual specific throughout the study.', 25988339);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3724, 'human', 'null', 'Phytoestrogens', 'null', 'Nine healthy male adult subjects ingested 0.3 g/kg/day flaxseeds during 2 week. Gut bacteria as well as blood and fecal metabolites were analyzed. Ingestion of flaxseeds triggered a significant increase in the blood concentration of enterolignans, accompanied by fecal excretion of propionate and glycerol. Overall diversity and composition of dominant fecal bacteria remained individual specific throughout the study.', 25988339);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3725, 'human', 'null', 'Geniposide', 'null', 'Genipin, a deglycosylated form of geniposide, was cytotoxic at the concentrations that geniposide was not. As metabolic activation systems for geniposide, human intestinal bacterial cultures, fecal preparation (fecalase) and intestinal microbial enzyme mix were employed in the present study. When geniposide was incubated with human intestinal bacteria, either Bifidobacterium longum HY8001 or Bacteroides fragilis, for 24 h, the cultured media caused cytotoxicity in HepG3 cells.', 22553067);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3726, 'human', 'null', 'Geniposide', 'null', 'Genipin, a deglycosylated form of geniposide, was cytotoxic at the concentrations that geniposide was not. As metabolic activation systems for geniposide, human intestinal bacterial cultures, fecal preparation (fecalase) and intestinal microbial enzyme mix were employed in the present study. When geniposide was incubated with human intestinal bacteria, either Bifidobacterium longum HY8001 or Bacteroides fragilis, for 24 h, the cultured media caused cytotoxicity in HepG3 cells.', 22553067);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3740, 'human', 'increase', 'Retinol', 'low-throughput', 'We reported here that microbiota metabolite short-chain fatty acid acetate promoted intestinal IgA responses, which was mediated by "metabolite-sensing" GPR43. GPR43-/- mice demonstrated lower levels of intestinal IgA and IgA+ gut bacteria compared with those in wild type (WT) mice. Feeding WT but not GPR43-/- mice acetate but not butyrate promoted intestinal IgA response independent of T cells.', 27966553);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3741, 'human', 'null', 'Bile acid', 'null', 'The most important bile acid transformation is 7α-dehydroxylation, producing deoxycholic acid (DCA) and lithocholic acid (LCA). The enzymatic pathway carrying out 7α-dehydroxylation is restricted to a narrow phylogenetic group of commensal bacteria, the best-characterized of which is Clostridium scindens.we used a recently established, standardized gnotobiotic mouse model that is stably associated with a simplified murine 12-species "oligo-mouse microbiota" (Oligo-MM12).', 28066726);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3742, 'human', 'null', 'Bile acid', 'null', 'The most important bile acid transformation is 7α-dehydroxylation, producing deoxycholic acid (DCA) and lithocholic acid (LCA). The enzymatic pathway carrying out 7α-dehydroxylation is restricted to a narrow phylogenetic group of commensal bacteria, the best-characterized of which is Clostridium scindens.we used a recently established, standardized gnotobiotic mouse model that is stably associated with a simplified murine 12-species "oligo-mouse microbiota" (Oligo-MM12).', 28066726);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3743, 'human', 'decrease', 'null', 'low-throughput', 'Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a ''diet-gut microbiota-physiology axis'' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind ''metabolite-sensing'' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.', 28658542);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3744, 'human', 'decrease', 'null', 'low-throughput', 'Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a ''diet-gut microbiota-physiology axis'' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind ''metabolite-sensing'' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.', 28658542);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3745, 'human', 'decrease', 'null', 'low-throughput', 'Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a ''diet-gut microbiota-physiology axis'' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind ''metabolite-sensing'' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.', 28658542);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3746, 'human', 'decrease', 'null', 'low-throughput', 'Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a ''diet-gut microbiota-physiology axis'' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind ''metabolite-sensing'' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.', 28658542);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3747, 'human', 'increase', 'null', 'low-throughput', 'Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a ''diet-gut microbiota-physiology axis'' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind ''metabolite-sensing'' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.', 28658542);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3748, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a ''diet-gut microbiota-physiology axis'' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind ''metabolite-sensing'' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.', 28658542);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3749, 'human', 'increase', 'Linoleic Acid', 'low-throughput', 'In this study, we investigated the effects of novel fatty acid metabolite derivatives of linoleic acid generated by the gut lactic acid bacteria Lactobacillus plantarum on the Nrf2-ARE pathway. 10-Oxo-trans-11-octadecenoic acid (KetoC) protected HepG2 cells from cytotoxicity induced by hydrogen peroxide. KetoC also significantly increased cellular Nrf2 protein levels, ARE-dependent transcription, and the gene expression of antioxidative enzymes such as heme oxygenase-1 (HO-1), glutamate-cysteine ligase modifier subunit (GCLM), and Nad(p)h: quinone oxidoreductase 1 (NQO1) in HepG2 cells.', 26879219);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3750, 'human', 'increase', 'Linoleic Acid', 'low-throughput', 'In this study, we investigated the effects of novel fatty acid metabolite derivatives of linoleic acid generated by the gut lactic acid bacteria Lactobacillus plantarum on the Nrf2-ARE pathway. 10-Oxo-trans-11-octadecenoic acid (KetoC) protected HepG2 cells from cytotoxicity induced by hydrogen peroxide. KetoC also significantly increased cellular Nrf2 protein levels, ARE-dependent transcription, and the gene expression of antioxidative enzymes such as heme oxygenase-1 (HO-1), glutamate-cysteine ligase modifier subunit (GCLM), and Nad(p)h: quinone oxidoreductase 1 (NQO1) in HepG2 cells.', 26879219);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3751, 'human', 'null', 'Rutin', 'null', 'The average α-Lrhamnosidase activities on the p-nitrophenyl-α-L-rhamnopyranoside, rutin, and poncirin subtrates were 0.10 ± 0.07, 0.25 ± 0.08, and 0.15 ± 0.09 pmol/min/mg, respectively. To investigate the enzymatic properties, α-L-rhamnosidase-producing bacteria were isolated from the specimens, and the α-L-rhamnosidase gene was cloned from a selected organism, Bifidobacterium dentium, and expressed in E. coli. The cloned α-L-rhamnosidase gene contained a 2,673 bp sequcence encoding 890 amino acid residues. The cloned gene was expressed using the pET 26b(+) vector in E. coli BL21, and the expressed enzyme was purified using Ni(2+)-NTA and Q-HP column chromatography. The specific activity of the purified α-L-rhamnosidase was 23.3 μmol/min/mg.', 25179902);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3752, 'human', 'null', 'Poncirin', 'null', 'The average α-Lrhamnosidase activities on the p-nitrophenyl-α-L-rhamnopyranoside, rutin, and poncirin subtrates were 0.10 ± 0.07, 0.25 ± 0.08, and 0.15 ± 0.09 pmol/min/mg, respectively. To investigate the enzymatic properties, α-L-rhamnosidase-producing bacteria were isolated from the specimens, and the α-L-rhamnosidase gene was cloned from a selected organism, Bifidobacterium dentium, and expressed in E. coli. The cloned α-L-rhamnosidase gene contained a 2,673 bp sequcence encoding 890 amino acid residues. The cloned gene was expressed using the pET 26b(+) vector in E. coli BL21, and the expressed enzyme was purified using Ni(2+)-NTA and Q-HP column chromatography. The specific activity of the purified α-L-rhamnosidase was 23.3 μmol/min/mg.', 25179902);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3753, 'human', 'decrease', 'null', 'low-throughput', 'A feature of human ulcerative colitis and other colitic diseases is a change in ''healthy'' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.', 19865172);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3754, 'human', 'decrease', 'null', 'low-throughput', 'A feature of human ulcerative colitis and other colitic diseases is a change in ''healthy'' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.', 19865172);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3755, 'human', 'decrease', 'null', 'low-throughput', 'A feature of human ulcerative colitis and other colitic diseases is a change in ''healthy'' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.', 19865172);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3756, 'human', 'decrease', 'null', 'low-throughput', 'A feature of human ulcerative colitis and other colitic diseases is a change in ''healthy'' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.', 19865172);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3757, 'human', 'decrease', 'Isoflavone', 'low-throughput', 'The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.', 30767231);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3758, 'human', 'decrease', 'Isoflavone', 'low-throughput', 'The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.', 30767231);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3759, 'human', 'decrease', 'Isoflavone', 'low-throughput', 'The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.', 30767231);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3760, 'human', 'decrease', 'Isoflavone', 'low-throughput', 'The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.', 30767231);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3761, 'human', 'decrease', 'Isoflavone', 'low-throughput', 'The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.', 30767231);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3762, 'human', 'decrease', 'Isoflavone', 'low-throughput', 'The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.', 30767231);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3763, 'human', 'null', 'Galbeta1,3GlcNAc', 'null', 'In the current study, we isolated a novel strain of Bifidobacterium kashiwanohense, named APCKJ1, from the faeces of a four-week old breastfed infant, based on the ability of the strain to utilise the HMO component fucosyllactose. We then determined the full genome sequence of this strain, and employed the generated data to analyze fucosyllactose metabolism in B. kashiwanohense APCKJ1. Transcriptomic and growth analyses, combined with metabolite analysis, in vitro hydrolysis assays and heterologous expression, allowed us to elucidate the pathway for fucosyllactose metabolism in B. kashiwanohense APCKJ1. Homologs of the key genes for this metabolic pathway were identified in particular in infant-derived members of the Bifdobacterium genus, revealing the apparent niche-specific nature of this pathway, and allowing a broad perspective on bifidobacterial fucosyllactose and L-fucose metabolism.', 31659215);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3764, 'human', 'null', 'N-Acetyl-D-lactosamine', 'null', 'In the current study, we isolated a novel strain of Bifidobacterium kashiwanohense, named APCKJ1, from the faeces of a four-week old breastfed infant, based on the ability of the strain to utilise the HMO component fucosyllactose. We then determined the full genome sequence of this strain, and employed the generated data to analyze fucosyllactose metabolism in B. kashiwanohense APCKJ1. Transcriptomic and growth analyses, combined with metabolite analysis, in vitro hydrolysis assays and heterologous expression, allowed us to elucidate the pathway for fucosyllactose metabolism in B. kashiwanohense APCKJ1. Homologs of the key genes for this metabolic pathway were identified in particular in infant-derived members of the Bifdobacterium genus, revealing the apparent niche-specific nature of this pathway, and allowing a broad perspective on bifidobacterial fucosyllactose and L-fucose metabolism.', 31659215);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3765, 'human', 'increase', 'null', 'low-throughput', 'A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host''s health through transcriptional regulation.', 28266623);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3766, 'human', 'increase', 'null', 'low-throughput', 'A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host''s health through transcriptional regulation.', 28266623);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3767, 'human', 'increase', 'null', 'low-throughput', 'A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host''s health through transcriptional regulation.', 28266623);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3768, 'human', 'increase', 'null', 'low-throughput', 'A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host''s health through transcriptional regulation.', 28266623);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3769, 'human', 'null', 'Daidzein', 'null', 'Several kinds of carbohydrates such as sorbose, adonitol, and melezitose were found to enhance equol production from daidzein in an in vitro human fecal culture. Sorbose, one of the most effective carbohydrates, was used as a carbohydrate source for isolating the NATTS strain, which was a Gram-positive, non-spore-forming rod bacterium with high ability to convert daidzein to equol isolated from the 7th maintenance culture. The strain was found to belong to the genus Slackia family Coriobacteriaceae by 16S rRNA sequence-based analysis, and the prevalence of the Slackia sp. in Japanese adults was examined by reverse transcription-quantitative PCR (RT-qPCR), which was found to be 40% at a mean population level of 10(6) cells per gram of feces.', 20237913);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3770, 'human', 'null', 'Daidzin', 'null', 'Several kinds of carbohydrates such as sorbose, adonitol, and melezitose were found to enhance equol production from daidzein in an in vitro human fecal culture. Sorbose, one of the most effective carbohydrates, was used as a carbohydrate source for isolating the NATTS strain, which was a Gram-positive, non-spore-forming rod bacterium with high ability to convert daidzein to equol isolated from the 7th maintenance culture. The strain was found to belong to the genus Slackia family Coriobacteriaceae by 16S rRNA sequence-based analysis, and the prevalence of the Slackia sp. in Japanese adults was examined by reverse transcription-quantitative PCR (RT-qPCR), which was found to be 40% at a mean population level of 10(6) cells per gram of feces.', 20237913);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3771, 'human', 'null', 'Daidzein', 'null', 'Several kinds of carbohydrates such as sorbose, adonitol, and melezitose were found to enhance equol production from daidzein in an in vitro human fecal culture. Sorbose, one of the most effective carbohydrates, was used as a carbohydrate source for isolating the NATTS strain, which was a Gram-positive, non-spore-forming rod bacterium with high ability to convert daidzein to equol isolated from the 7th maintenance culture. The strain was found to belong to the genus Slackia family Coriobacteriaceae by 16S rRNA sequence-based analysis, and the prevalence of the Slackia sp. in Japanese adults was examined by reverse transcription-quantitative PCR (RT-qPCR), which was found to be 40% at a mean population level of 10(6) cells per gram of feces.', 20237913);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3772, 'human', 'null', 'Daidzein', 'null', 'Several kinds of carbohydrates such as sorbose, adonitol, and melezitose were found to enhance equol production from daidzein in an in vitro human fecal culture. Sorbose, one of the most effective carbohydrates, was used as a carbohydrate source for isolating the NATTS strain, which was a Gram-positive, non-spore-forming rod bacterium with high ability to convert daidzein to equol isolated from the 7th maintenance culture. The strain was found to belong to the genus Slackia family Coriobacteriaceae by 16S rRNA sequence-based analysis, and the prevalence of the Slackia sp. in Japanese adults was examined by reverse transcription-quantitative PCR (RT-qPCR), which was found to be 40% at a mean population level of 10(6) cells per gram of feces.', 20237913);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3773, 'human', 'null', 'Puerarin', 'null', 'A rod-shaped Gram-positive bacterium, strain CG19-1, capable of deglycosylating puerarin to daidzein was isolated from the corresponding suspension. However, the strictly anaerobic isolate was unable to utilize puerarin as sole carbon and energy source nor any of the tested carbohydrates. Comparative 16S rRNA sequence analysis indicated that strain CG19-1 is a new species of the Lachnospiraceae. Strain CG19-1 also converted other aromatic C-glucosides in addition to puerarin. The xanthone C-glucoside mangiferin was deglycosylated to norathyriol. The flavone C-glucosides homoorientin and vitexin were degraded to 3-(3,4-dihydroxyphenyl)propionic acid via luteolin and 3-(4-hydroxyphenyl)propionic acid respectively. In addition, strain CG19-1 converted flavonoid O-glucosides, but at rates that were lower than those of the C-glucosides tested. The isolate deglycosylated the isoflavone O-glucosides daidzin and genistin to daidzein and genistein respectively. Several O-glucosides of the flavones luteolin and apigenin undergoing deglycosylation were subsequently cleaved to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid respectively. Moreover, strain CG19-1 cleaved both O-desmethylangolensin and 6''-hydroxy-O-desmethylangolensin to yield 2-(4-dihydroxyphenyl)propionic acid. The corresponding cleavage product, resorcinol, was only observed for O-desmethylangolensin.', 20946528);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3774, 'human', 'null', 'Vitexin', 'null', 'A rod-shaped Gram-positive bacterium, strain CG19-1, capable of deglycosylating puerarin to daidzein was isolated from the corresponding suspension. However, the strictly anaerobic isolate was unable to utilize puerarin as sole carbon and energy source nor any of the tested carbohydrates. Comparative 16S rRNA sequence analysis indicated that strain CG19-1 is a new species of the Lachnospiraceae. Strain CG19-1 also converted other aromatic C-glucosides in addition to puerarin. The xanthone C-glucoside mangiferin was deglycosylated to norathyriol. The flavone C-glucosides homoorientin and vitexin were degraded to 3-(3,4-dihydroxyphenyl)propionic acid via luteolin and 3-(4-hydroxyphenyl)propionic acid respectively. In addition, strain CG19-1 converted flavonoid O-glucosides, but at rates that were lower than those of the C-glucosides tested. The isolate deglycosylated the isoflavone O-glucosides daidzin and genistin to daidzein and genistein respectively. Several O-glucosides of the flavones luteolin and apigenin undergoing deglycosylation were subsequently cleaved to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid respectively. Moreover, strain CG19-1 cleaved both O-desmethylangolensin and 6''-hydroxy-O-desmethylangolensin to yield 2-(4-dihydroxyphenyl)propionic acid. The corresponding cleavage product, resorcinol, was only observed for O-desmethylangolensin.', 20946528);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3775, 'human', 'null', 'O-Desmethylangolensin', 'null', 'A rod-shaped Gram-positive bacterium, strain CG19-1, capable of deglycosylating puerarin to daidzein was isolated from the corresponding suspension. However, the strictly anaerobic isolate was unable to utilize puerarin as sole carbon and energy source nor any of the tested carbohydrates. Comparative 16S rRNA sequence analysis indicated that strain CG19-1 is a new species of the Lachnospiraceae. Strain CG19-1 also converted other aromatic C-glucosides in addition to puerarin. The xanthone C-glucoside mangiferin was deglycosylated to norathyriol. The flavone C-glucosides homoorientin and vitexin were degraded to 3-(3,4-dihydroxyphenyl)propionic acid via luteolin and 3-(4-hydroxyphenyl)propionic acid respectively. In addition, strain CG19-1 converted flavonoid O-glucosides, but at rates that were lower than those of the C-glucosides tested. The isolate deglycosylated the isoflavone O-glucosides daidzin and genistin to daidzein and genistein respectively. Several O-glucosides of the flavones luteolin and apigenin undergoing deglycosylation were subsequently cleaved to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid respectively. Moreover, strain CG19-1 cleaved both O-desmethylangolensin and 6''-hydroxy-O-desmethylangolensin to yield 2-(4-dihydroxyphenyl)propionic acid. The corresponding cleavage product, resorcinol, was only observed for O-desmethylangolensin.', 20946528);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3776, 'human', 'null', '6''-Hydroxy-O-desmethylangolensin', 'null', 'A rod-shaped Gram-positive bacterium, strain CG19-1, capable of deglycosylating puerarin to daidzein was isolated from the corresponding suspension. However, the strictly anaerobic isolate was unable to utilize puerarin as sole carbon and energy source nor any of the tested carbohydrates. Comparative 16S rRNA sequence analysis indicated that strain CG19-1 is a new species of the Lachnospiraceae. Strain CG19-1 also converted other aromatic C-glucosides in addition to puerarin. The xanthone C-glucoside mangiferin was deglycosylated to norathyriol. The flavone C-glucosides homoorientin and vitexin were degraded to 3-(3,4-dihydroxyphenyl)propionic acid via luteolin and 3-(4-hydroxyphenyl)propionic acid respectively. In addition, strain CG19-1 converted flavonoid O-glucosides, but at rates that were lower than those of the C-glucosides tested. The isolate deglycosylated the isoflavone O-glucosides daidzin and genistin to daidzein and genistein respectively. Several O-glucosides of the flavones luteolin and apigenin undergoing deglycosylation were subsequently cleaved to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid respectively. Moreover, strain CG19-1 cleaved both O-desmethylangolensin and 6''-hydroxy-O-desmethylangolensin to yield 2-(4-dihydroxyphenyl)propionic acid. The corresponding cleavage product, resorcinol, was only observed for O-desmethylangolensin.', 20946528);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3777, 'human', 'null', '(3R)-flavan-3-ols', 'null', 'A human intestinal bacterium, Eubacterium (E.) sp. strain SDG-2, was tested for its ability to metabolize various (3R)- and (3S)-flavan-3-ols and their 3-O-gallates. This bacterium cleaved the C-ring of (3R)- and (3S)-flavan-3-ols to give 1,3-diphenylpropan-2-ol derivatives, but not their 3-O-gallates. Furthermore, E. sp. strain SDG-2 had the ability of p-dehydroxylation in the B-ring of (3R)-flavan-3-ols, such as (-)-catechin, (-)-epicatechin, (-)-gallocatechin and (-)-epigallocatechin, but not of (3S)-flavan-3-ols, such as (+)-catechin and (+)-epicatechin.', 11767089);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3778, 'human', 'null', '(3S)-flavan-3-ols', 'null', 'A human intestinal bacterium, Eubacterium (E.) sp. strain SDG-2, was tested for its ability to metabolize various (3R)- and (3S)-flavan-3-ols and their 3-O-gallates. This bacterium cleaved the C-ring of (3R)- and (3S)-flavan-3-ols to give 1,3-diphenylpropan-2-ol derivatives, but not their 3-O-gallates. Furthermore, E. sp. strain SDG-2 had the ability of p-dehydroxylation in the B-ring of (3R)-flavan-3-ols, such as (-)-catechin, (-)-epicatechin, (-)-gallocatechin and (-)-epigallocatechin, but not of (3S)-flavan-3-ols, such as (+)-catechin and (+)-epicatechin.', 11767089);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3803, 'human', 'increase', 'Ginsenoside rb1', 'low-throughput', 'Compound K significantly up‑regulated expression of p21 (CDKN1A), and downregulated the expressions of CDK6, cyclin D and cyclin E, which are G1 phase arrest regulators.', 29805567);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3804, 'human', 'decrease', 'Ginsenoside rb1', 'low-throughput', 'Compound K significantly up‑regulated expression of p21 (CDKN1A), and downregulated the expressions of CDK6, cyclin D and cyclin E, which are G1 phase arrest regulators.', 29805567);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3805, 'human', 'decrease', 'Ginsenoside rb1', 'low-throughput', 'Compound K significantly up‑regulated expression of p21 (CDKN1A), and downregulated the expressions of CDK6, cyclin D and cyclin E, which are G1 phase arrest regulators.', 29805567);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3806, 'human', 'decrease', 'Ginsenoside rb1', 'low-throughput', 'Compound K significantly up‑regulated expression of p21 (CDKN1A), and downregulated the expressions of CDK6, cyclin D and cyclin E, which are G1 phase arrest regulators.', 29805567);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3807, 'human', 'increase', 'Ginsenoside rb1', 'low-throughput', 'Compound K up-regulated the expressions of p53 and Bax,  which are pro-apoptotic genes. In contrast, compound K down-regulated the expression of Bcl2, which is an anti-apoptotic gene.', 29805567);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3808, 'human', 'increase', 'Ginsenoside rb1', 'low-throughput', 'Compound K up-regulated the expressions of p53 and Bax,  which are pro-apoptotic genes. In contrast, compound K down-regulated the expression of Bcl2, which is an anti-apoptotic gene.', 29805567);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3809, 'human', 'decrease', 'Ginsenoside rb1', 'low-throughput', 'Compound K up-regulated the expressions of p53 and Bax,  which are pro-apoptotic genes. In contrast, compound K down-regulated the expression of Bcl2, which is an anti-apoptotic gene.', 29805567);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3810, 'human', 'decrease', 'Ginsenoside rb1', 'low-throughput', 'Compound K significantly reduced the IL-8 production in HT-29 cells.', 29805567);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3817, 'human', 'null', 'Levodopa', 'null', 'Gut-associated bacteria have the abality to decarboxylate levodopa to dopamine via tyrosine decarboxylases.', 30659181);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3818, 'human', 'null', 'PhIP-N2-β-D-glucuronide', 'null', 'B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.', 31096909);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3819, 'human', 'null', '2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine', 'null', 'B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.', 31096909);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3820, 'human', 'null', '2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine', 'null', 'B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.', 31096909);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3821, 'human', 'null', '2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine', 'null', 'B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.', 31096909);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3822, 'human', 'null', '2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine', 'null', 'B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.', 31096909);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3823, 'human', 'null', 'Rutin', 'null', 'Enterococcus avium EFEL009 is capable of converting rutin to quercetin.', 26505733);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3832, 'human', 'decrease', 'null', 'low-throughput', 'In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.', 21178864);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3833, 'human', 'decrease', 'null', 'low-throughput', 'In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.', 21178864);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3834, 'human', 'decrease', 'null', 'low-throughput', 'In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.', 21178864);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3835, 'human', 'increase', 'null', 'low-throughput', 'In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.', 21178864);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3836, 'human', 'decrease', 'null', 'low-throughput', 'In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.', 21178864);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3838, 'human', 'null', 'null', 'null', 'Lactobacillus casei strain Shirota intake increased defecation frequency, normalized stool consistency, and improved constipation related symptoms in Chinese adults with constipation. Through the use of metabolomic approaches, PIPA, which is a non-volatile metabolite in the gut, was identified as a potential target mediator in LcS-mediated constipation alleviation. Furthermore, PIPA could improve constipation symptoms, as shown by increases in the small intestine propulsive rate and the number of fecal pellets in a constipation mouse model, raising its possibility in the constipation therapy. Future studies should verify the anti-constipation function of PIPA and elucidate its potential mechanisms.', 30846982);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3841, 'human', 'decrease', 'Geniposide', 'low-throughput', 'Western blot analysis revealed that the activated GS inhibited Bcl-2 expression with a subsequent increase in Bax expression. Likewise, GS activation by IM stimulated caspase-3 and the production of reactive oxygen species (ROS). In addition, activated GS-induced apoptosis was confirmed by apoptosis and ROS assays; N-acetyl-l-cysteine (NAC) suppressed ROS production and apoptotic cell death. Activated GS induced sustained JNK phosphorylation. Moreover, activated GS-induced cell death was reversed by SP600125. Taken together, these findings suggest that human IM is able to metabolize GS into GP, and the related biological activities induce apoptosis through ROS/JNK signaling.', 22245672);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3842, 'human', 'increase', 'Geniposide', 'low-throughput', 'Western blot analysis revealed that the activated GS inhibited Bcl-2 expression with a subsequent increase in Bax expression. Likewise, GS activation by IM stimulated caspase-3 and the production of reactive oxygen species (ROS). In addition, activated GS-induced apoptosis was confirmed by apoptosis and ROS assays; N-acetyl-l-cysteine (NAC) suppressed ROS production and apoptotic cell death. Activated GS induced sustained JNK phosphorylation. Moreover, activated GS-induced cell death was reversed by SP600125. Taken together, these findings suggest that human IM is able to metabolize GS into GP, and the related biological activities induce apoptosis through ROS/JNK signaling.', 22245672);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3843, 'human', 'increase', 'Geniposide', 'low-throughput', 'Western blot analysis revealed that the activated GS inhibited Bcl-2 expression with a subsequent increase in Bax expression. Likewise, GS activation by IM stimulated caspase-3 and the production of reactive oxygen species (ROS). In addition, activated GS-induced apoptosis was confirmed by apoptosis and ROS assays; N-acetyl-l-cysteine (NAC) suppressed ROS production and apoptotic cell death. Activated GS induced sustained JNK phosphorylation. Moreover, activated GS-induced cell death was reversed by SP600125. Taken together, these findings suggest that human IM is able to metabolize GS into GP, and the related biological activities induce apoptosis through ROS/JNK signaling.', 22245672);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3844, 'human', 'increase', 'Geniposide', 'low-throughput', 'Western blot analysis revealed that the activated GS inhibited Bcl-2 expression with a subsequent increase in Bax expression. Likewise, GS activation by IM stimulated caspase-3 and the production of reactive oxygen species (ROS). In addition, activated GS-induced apoptosis was confirmed by apoptosis and ROS assays; N-acetyl-l-cysteine (NAC) suppressed ROS production and apoptotic cell death. Activated GS induced sustained JNK phosphorylation. Moreover, activated GS-induced cell death was reversed by SP600125. Taken together, these findings suggest that human IM is able to metabolize GS into GP, and the related biological activities induce apoptosis through ROS/JNK signaling.', 22245672);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3845, 'human', 'decrease', 'null', 'low-throughput', 'We examined the effects of UV-killed F. prausnitzii, bacterial fractions,supernatants, or sterile medium on IL-8 secretion in Caco-2 cells,with or without IL-1 stimulation (SI Materials and Methods and Fig. 3A). Either UV-killed F. prausnitzii, membrane extracts, or cytoplasmic extracts, but not DNA or sterile medium, induce low levels of IL-8 secretion. UV-killed F. prausnitzii, bacterial fractions,and sterile medium did not inhibit IL-8 secretion induced by IL-1,compared with negative control. In contrast, supernatant from F.prausnitzii cultures significantly reduces IL-8 secretion induced by IL-1.', 18936492);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3846, 'human', 'decrease', 'null', 'low-throughput', 'We examined the effects of UV-killed F. prausnitzii, bacterial fractions,supernatants, or sterile medium on IL-8 secretion in Caco-2 cells,with or without IL-1 stimulation (SI Materials and Methods and Fig. 3A). Either UV-killed F. prausnitzii, membrane extracts, or cytoplasmic extracts, but not DNA or sterile medium, induce low levels of IL-8 secretion. UV-killed F. prausnitzii, bacterial fractions,and sterile medium did not inhibit IL-8 secretion induced by IL-1,compared with negative control. In contrast, supernatant from F.prausnitzii cultures significantly reduces IL-8 secretion induced by IL-1.', 18936492);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3850, 'human', 'increase', '3,8,9-Trihydroxy-6H-benzo[c]chromen-6-one', 'low-throughput', 'The expression of p53 was significantly increased in the HCT- 116 cells (wild type p53) by Uro-A and UM-A treatments.', 32179165);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3851, 'human', 'increase', '3,8,9-Trihydroxy-6H-benzo[c]chromen-6-one', 'low-throughput', 'A significant induction of the of p21Cip1/Waf1 in the cells treated with Uro-A (2.2 ± 0.2) and UM-A (2.2 ± 0.5) was observed.', 32179165);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3852, 'human', 'increase', 'null', 'low-throughput', 'Both gain- and loss-of-function studies employing genetic and pharmacological tools reveal PAGln mediates cellular events through G-protein coupled receptors, including α2A, α2B, and β2-adrenergic receptors.', 32142679);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3853, 'human', 'increase', 'null', 'low-throughput', 'Both gain- and loss-of-function studies employing genetic and pharmacological tools reveal PAGln mediates cellular events through G-protein coupled receptors, including α2A, α2B, and β2-adrenergic receptors.', 32142679);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3854, 'human', 'increase', 'null', 'low-throughput', 'Both gain- and loss-of-function studies employing genetic and pharmacological tools reveal PAGln mediates cellular events through G-protein coupled receptors, including α2A, α2B, and β2-adrenergic receptors.', 32142679);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3855, 'human', 'decrease', 'Ellagic acid', 'low-throughput', 'As a result, a 24 h treatment with UA (20 µm) significantly decreased Rac1 and PAK1 transcript levels and activity, depolymerized actin and wound healing.', 31976617);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3856, 'human', 'decrease', 'Ellagic acid', 'low-throughput', 'As a result, a 24 h treatment with UA (20 µm) significantly decreased Rac1 and PAK1 transcript levels and activity, depolymerized actin and wound healing.', 31976617);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3857, 'human', 'null', '(-)-Epicatechin', 'null', 'All the isoflavone-metabolizing bacteria catalyzed C-ring cleavage of –EC to produce (2S)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1S), and the formation was accelerated in the presence of hydrogen.', 26312950);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3858, 'human', 'null', '(-)-Epicatechin', 'null', 'All the isoflavone-metabolizing bacteria catalyzed C-ring cleavage of –EC to produce (2S)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1S), and the formation was accelerated in the presence of hydrogen.', 26312950);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3859, 'human', 'null', '(-)-Epicatechin', 'null', 'All the isoflavone-metabolizing bacteria catalyzed C-ring cleavage of –EC to produce (2S)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1S), and the formation was accelerated in the presence of hydrogen.', 26312950);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3860, 'human', 'null', '(-)-Epicatechin', 'null', 'All the isoflavone-metabolizing bacteria catalyzed C-ring cleavage of –EC to produce (2S)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1S), and the formation was accelerated in the presence of hydrogen.', 26312950);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3861, 'human', 'null', 'Cianidanol', 'null', 'The bacterial strains also showed ring-cleaving ability on +C to yield (2R)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1R), and this reaction was enhanced by the presence of hydrogen, with the exception of S. equolifaciens JCM 16059.', 26312950);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3862, 'human', 'null', 'Cianidanol', 'null', 'The bacterial strains also showed ring-cleaving ability on +C to yield (2R)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1R), and this reaction was enhanced by the presence of hydrogen, with the exception of S. equolifaciens JCM 16059.', 26312950);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3863, 'human', 'null', 'Cianidanol', 'null', 'The bacterial strains also showed ring-cleaving ability on +C to yield (2R)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1R), and this reaction was enhanced by the presence of hydrogen, with the exception of S. equolifaciens JCM 16059.', 26312950);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3864, 'human', 'null', 'Daidzein', 'null', 'Equol producing-related species Adlercreutzia equolifaciens and Bifdobacterium bifdum, showed higher relative abundance in EP than in NP.', 31168326);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3865, 'human', 'null', 'Daidzein', 'null', 'Equol producing-related species Adlercreutzia equolifaciens and Bifdobacterium bifdum, showed higher relative abundance in EP than in NP.', 31168326);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3866, 'human', 'null', 'Ginsenoside Rb1', 'null', 'The enzyme potently biotransformed ginsenoside Rb1, loganin, arctiin and arbutin to ginsenoside Rd, loganetin, arctigenin and hydroquinone, respectively, but was not active in the case of hesperidin, and kakkalide.', 22466563);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3867, 'human', 'null', 'Loganin', 'null', 'The enzyme potently biotransformed ginsenoside Rb1, loganin, arctiin and arbutin to ginsenoside Rd, loganetin, arctigenin and hydroquinone, respectively, but was not active in the case of hesperidin, and kakkalide.', 22466563);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3868, 'human', 'null', 'Arctiin', 'null', 'The enzyme potently biotransformed ginsenoside Rb1, loganin, arctiin and arbutin to ginsenoside Rd, loganetin, arctigenin and hydroquinone, respectively, but was not active in the case of hesperidin, and kakkalide.', 22466563);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3869, 'human', 'null', 'Arbutin', 'null', 'The enzyme potently biotransformed ginsenoside Rb1, loganin, arctiin and arbutin to ginsenoside Rd, loganetin, arctigenin and hydroquinone, respectively, but was not active in the case of hesperidin, and kakkalide.', 22466563);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3870, 'human', 'increase', 'null', 'low-throughput', 'Compared with the control, the stimulation of HCT116 cells with C. butyricum supernatant and sodium butyrate induced 4.5-fold and 4.2-fold upregulation of the GPR43 mRNA levels, 2.6-fold and 7.0-fold increases in the GPR109A mRNA levels.', 31734354);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3871, 'human', 'increase', 'null', 'low-throughput', 'Compared with the control, the stimulation of HCT116 cells with C. butyricum supernatant and sodium butyrate induced 4.5-fold and 4.2-fold upregulation of the GPR43 mRNA levels, 2.6-fold and 7.0-fold increases in the GPR109A mRNA levels.', 31734354);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3872, 'human', 'increase', 'null', 'low-throughput', 'The mRNA level of P21WAF1 was increased in HCT116 cells treated with C. butyricum.', 31734354);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3873, 'human', 'increase', 'null', 'low-throughput', 'Indole promoted mRNA expression of TJ-associated molecules, such as Cldn7, Ocln, and Tjp1 in the human adenocarcinoma cell line, Caco-2.', 24278294);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3874, 'human', 'increase', 'null', 'low-throughput', 'Indole promoted mRNA expression of TJ-associated molecules, such as Cldn7, Ocln, and Tjp1 in the human adenocarcinoma cell line, Caco-2.', 24278294);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3875, 'human', 'increase', 'null', 'low-throughput', 'Indole promoted mRNA expression of TJ-associated molecules, such as Cldn7, Ocln, and Tjp1 in the human adenocarcinoma cell line, Caco-2.', 24278294);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3876, 'human', 'increase', 'null', 'low-throughput', 'The function of butyrate as a GPR109A agonist was investigated using a heterologous expression system in which the coupling of pertussis toxin-sensitive Gi proteins to the receptor was monitored with the activity of inwardly-rectifying potassium (GIRK) channels as the read-out.', 19276343);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3877, 'human', 'null', 'null', 'null', 'The concentration of ursodesoxycholic acid in stool samples was highly positively correlated with the level of Ruminococcus.', 28587349);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3878, 'human', 'null', 'null', 'null', 'The concentrations of phenylalanine and glutamic acid were highly positively correlated with the levels of Phascolarctobacterium and Acidiminobacter.', 28587349);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3879, 'human', 'null', 'null', 'null', 'The concentrations of phenylalanine and glutamic acid were highly positively correlated with the levels of Phascolarctobacterium and Acidiminobacter.', 28587349);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3880, 'human', 'null', 'null', 'null', 'The concentrations of serine and threonine were moderately positively correlated with the levels of Phascolarctobacterium and Acidiminobacter.', 28587349);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3881, 'human', 'null', 'null', 'null', 'The concentrations of serine and threonine were moderately positively correlated with the levels of Phascolarctobacterium and Acidiminobacter.', 28587349);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3882, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Exposure to 1 mM indole led to an increase in the expression of two Toll-like receptor (TLR) genes—TLR-3 and TLR-9.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3883, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Exposure to 1 mM indole led to an increase in the expression of two Toll-like receptor (TLR) genes—TLR-3 and TLR-9.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3884, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3885, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3886, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3887, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Similarly, genes involved in the phosphoinositide-3-kinase (PI3K) pathway, which modulates TLR signaling and leads to antiinflammatory cytokine IL-10 induction, were also induced by indole.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3888, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3889, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3890, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3891, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Downstream target genes of AKT1, including the cAMP response element-binding protein and IL-10, were also induced upon addition of indole.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3892, 'human', 'increase', 'Tryptophan', 'low-throughput', 'In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3893, 'human', 'increase', 'Tryptophan', 'low-throughput', 'In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3894, 'human', 'increase', 'Tryptophan', 'low-throughput', 'In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3895, 'human', 'increase', 'Tryptophan', 'low-throughput', 'In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3896, 'human', 'increase', 'Tryptophan', 'low-throughput', 'In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3897, 'human', 'increase', 'Tryptophan', 'low-throughput', 'In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3898, 'human', 'increase', 'Tryptophan', 'low-throughput', 'In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3899, 'human', 'increase', 'Tryptophan', 'low-throughput', 'In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3900, 'human', 'increase', 'Tryptophan', 'low-throughput', 'In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3901, 'human', 'increase', 'Tryptophan', 'low-throughput', 'In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3902, 'human', 'increase', 'Tryptophan', 'low-throughput', 'In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3903, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3904, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3905, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3906, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3907, 'human', 'increase', 'Tryptophan', 'low-throughput', 'Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.', 19966295);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3908, 'human', 'null', 'Daidzin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3909, 'human', 'null', 'Daidzin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3910, 'human', 'null', 'Daidzin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3911, 'human', 'null', 'Daidzin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3912, 'human', 'null', 'Daidzin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3913, 'human', 'null', 'Genistin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3914, 'human', 'null', 'Genistin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3915, 'human', 'null', 'Genistin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3916, 'human', 'null', 'Genistin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3917, 'human', 'null', 'Genistin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3918, 'human', 'null', 'Glycitin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3919, 'human', 'null', 'Glycitin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3920, 'human', 'null', 'Glycitin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3921, 'human', 'null', 'Glycitin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3922, 'human', 'null', 'Glycitin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3923, 'human', 'null', 'Astragalin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3924, 'human', 'null', 'Astragalin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3925, 'human', 'null', 'Astragalin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3926, 'human', 'null', 'Astragalin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3927, 'human', 'null', 'Astragalin', 'null', 'All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.', 17439230);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3928, 'human', 'null', 'gamma-Butyrobetaine hydrochloride', 'null', 'Only 1 Clostridiales bacterium, Emergencia timonensis, that converted γBB to TMA.', 30530985);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3929, 'human', 'null', 'Phenylalanine', 'null', 'After incubation of wild-type C. sporogenes with phenylalanine, tyrosine and tryptophan, we observed robust production of PPA, 4-OH-PPA and IPA.', 29168502);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3930, 'human', 'null', 'Tyrosine', 'null', 'After incubation of wild-type C. sporogenes with phenylalanine, tyrosine and tryptophan, we observed robust production of PPA, 4-OH-PPA and IPA.', 29168502);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3931, 'human', 'null', 'Tryptophan', 'null', 'After incubation of wild-type C. sporogenes with phenylalanine, tyrosine and tryptophan, we observed robust production of PPA, 4-OH-PPA and IPA.', 29168502);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3932, 'human', 'null', 'Tryptophan', 'null', 'Notably, we identified four new IPA-producing gut bacteria: Peptostreptococcus anaerobius CC14N and three strains of Clostridium cadaveris.', 29168502);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3933, 'human', 'null', 'Tryptophan', 'null', 'Notably, we identified four new IPA-producing gut bacteria: Peptostreptococcus anaerobius CC14N and three strains of Clostridium cadaveris.', 29168502);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3934, 'human', 'null', 'Tryptophan', 'null', 'Notably, we identified four new IPA-producing gut bacteria: Peptostreptococcus anaerobius CC14N and three strains of Clostridium cadaveris.', 29168502);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3935, 'human', 'null', 'Tryptophan', 'null', 'Notably, we identified four new IPA-producing gut bacteria: Peptostreptococcus anaerobius CC14N and three strains of Clostridium cadaveris.', 29168502);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3936, 'human', 'decrease', 'Urocanic acid', 'low-throughput', 'We also noted that IRS1 levels were reduced by 8 hr treatment of imidazole propionate in the human embryonic kidney cell line HEK293.', 30401435);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3937, 'human', 'increase', 'Urocanic acid', 'low-throughput', 'Imidazole propionate activates mTORC1.', 30401435);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3938, 'human', 'decrease', 'Urocanic acid', 'low-throughput', 'We also noted that IRS1 levels were reduced by 8 hr treatment of imidazole propionate in the human embryonic kidney cell line HEK293.', 30401435);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3939, 'human', 'increase', 'Urocanic acid', 'low-throughput', 'Imidazole propionate activates mTORC1.', 30401435);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3940, 'human', 'increase', 'null', 'low-throughput', 'Propionate was the most potent agonist for both GPR41 and GPR43.', 12711604);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3941, 'human', 'increase', 'null', 'low-throughput', 'Propionate was the most potent agonist for both GPR41 and GPR43.', 12711604);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3942, 'human', 'increase', 'null', 'low-throughput', 'Acetate was more selective for GPR43, whereas butyrate and isobutyrate were more active on GPR41', 12711604);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3943, 'human', 'increase', 'null', 'low-throughput', 'Acetate was more selective for GPR43, whereas butyrate and isobutyrate were more active on GPR41', 12711604);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3949, 'human', 'null', 'Ginsenoside Rg3', 'null', 'Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.', 11824558);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3950, 'human', 'null', 'Ginsenoside Rg3', 'null', 'Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.', 11824558);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3951, 'human', 'null', 'Ginsenoside Rg3', 'null', 'Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.', 11824558);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3952, 'human', 'null', 'Ginsenoside Rh2', 'null', 'Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.', 11824558);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3953, 'human', 'null', 'Ginsenoside Rh2', 'null', 'Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.', 11824558);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3954, 'human', 'null', '(20R)-Ginsenoside Rg3', 'null', 'Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.', 11824558);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3955, 'human', 'null', '(20R)-Ginsenoside Rg3', 'null', 'Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.', 11824558);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3956, 'human', 'null', '20(R)-Ginsenoside Rh2', 'null', 'Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.', 11824558);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3957, 'human', 'null', 'Arctigenin', 'null', 'In the present study, we isolated one single bacterium, which we named Blautia sp. AUH-JLD56, capable of solely biotransforming arctiin or arctigenin to (-)-3''-DMAG.', 24236649);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3958, 'human', 'null', 'null', 'null', 'Ruminococcus, showed positive correlations with plasma glycerophospholipids.', 31748797);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3959, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that cytochrome P450 1A1 (Cyp1A1) is among the top 3 UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3960, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3961, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3962, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3963, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3964, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3965, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3966, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3967, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3968, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3969, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3970, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3971, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3972, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3973, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3974, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3975, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3976, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3977, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3978, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA upregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3979, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3980, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3981, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3982, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3983, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3984, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3985, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3986, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3987, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3988, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3989, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3990, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3991, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3992, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3993, 'human', 'decrease', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3994, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3995, 'human', 'increase', 'null', 'high-throughput', 'The RNA-Seq analysis showed that UroA downregulated genes.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3996, 'human', 'increase', 'null', 'high-throughput', 'The fold changes in mRNA levels of claudin 4 (Cldn4), occludin (Ocln), and Zona occludens 1 (ZO1) in HT29 cells were determined by RT PCR method.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3997, 'human', 'increase', 'null', 'high-throughput', 'The fold changes in mRNA levels of claudin 4 (Cldn4), occludin (Ocln), and Zona occludens 1 (ZO1) in HT29 cells were determined by RT PCR method.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3998, 'human', 'increase', 'null', 'high-throughput', 'The fold changes in mRNA levels of claudin 4 (Cldn4), occludin (Ocln), and Zona occludens 1 (ZO1) in HT29 cells were determined by RT PCR method.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (3999, 'human', 'increase', 'null', 'high-throughput', 'UroA/UAS03 induced NQO1 expression was also confirmed in HT29 cells (Supplementary Fig. 9d).', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4000, 'human', 'null', 'L-Fucose', 'null', 'R. inulinivorans A2-194 is also able to grow on the host-derived sugar fucose, following a lag period, producing propionate and propanol as additional fermentation products.', 16740940);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4001, 'human', 'null', 'L-Fucose', 'null', 'R. inulinivorans A2-194 is also able to grow on the host-derived sugar fucose, following a lag period, producing propionate and propanol as additional fermentation products.', 16740940);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4002, 'human', 'null', 'L-Fucose', 'null', 'R. inulinivorans A2-194 is also able to grow on the host-derived sugar fucose, following a lag period, producing propionate and propanol as additional fermentation products.', 16740940);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4003, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4004, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4005, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4006, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4007, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4008, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4009, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4010, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4011, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4012, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4013, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4014, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4015, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4016, 'human', 'null', 'null', 'null', 'Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.', 29572891);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4023, 'human', 'null', 'D-Glucose', 'null', 'The major end products of glucose metabolism are lactic, acetic and formic acids.', 20023061);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4024, 'human', 'null', 'D-Glucose', 'null', 'The major end products of glucose metabolism are lactic, acetic and formic acids.', 20023061);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4025, 'human', 'null', 'D-Glucose', 'null', 'The major end products of glucose metabolism are lactic, acetic and formic acids.', 20023061);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4026, 'human', 'null', 'D-Glucose', 'null', 'The major end products of glucose metabolism are lactic, acetic and formic acids.', 20023061);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4027, 'human', 'null', 'D-Glucose', 'null', 'The major end products of glucose metabolism were succinate and acetate.', 19749029);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4028, 'human', 'null', 'D-Glucose', 'null', 'The major end products of glucose metabolism were succinate and acetate.', 19749029);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4029, 'human', 'null', 'Avicularin', 'null', 'Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.', 24463398);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4030, 'human', 'null', 'Quercetin', 'null', 'Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.', 24463398);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4031, 'human', 'null', 'Quercetin', 'null', 'Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.', 24463398);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4032, 'human', 'null', 'Quercetin', 'null', 'Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.', 24463398);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4033, 'human', 'null', 'Avicularin', 'null', 'Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.', 24463398);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4034, 'human', 'increase', 'null', 'low-throughput', 'Sodium acetate (10�?0 mM) significantly increased TPH1mRNA expression.', 25550456);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4035, 'human', 'null', 'D-Glucose', 'null', 'C. scindens fermented glucose mainly to ethanol, acetate, formate, and H2.', 30737348);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4036, 'human', 'null', 'D-Glucose', 'null', 'C. scindens fermented glucose mainly to ethanol, acetate, formate, and H2.', 30737348);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4037, 'human', 'null', 'D-Glucose', 'null', 'C. scindens fermented glucose mainly to ethanol, acetate, formate, and H2.', 30737348);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4038, 'human', 'null', 'D-Glucose', 'null', 'C. scindens fermented glucose mainly to ethanol, acetate, formate, and H2.', 30737348);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4039, 'human', 'null', 'null', 'null', 'A trace amount of propionate was detected as an end product of metabolism.', 19542131);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4040, 'human', 'null', 'D-Glucose', 'null', 'The two strains produced significant levels of butyric and isobutyric acids as end products from glucose.', 19542124);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4041, 'human', 'null', 'D-Glucose', 'null', 'The two strains produced significant levels of butyric and isobutyric acids as end products from glucose.', 19542124);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4042, 'human', 'null', 'D-Glucose', 'null', 'The two strains produced significant levels of butyric and isobutyric acids as end products from glucose.', 19542124);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4043, 'human', 'null', 'D-Glucose', 'null', 'The two strains produced significant levels of butyric and isobutyric acids as end products from glucose.', 19542124);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4044, 'human', 'increase', 'null', 'low-throughput', 'Butyrate stimulated a rapid twofold increase of IL- 22R1 mRNA expression that, after a total 8 h incubation period, translated to the protein level.', 27801948);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4045, 'human', 'null', '(-)-Epicatechin', 'null', 'Strain MT4s-5 readily converted −EC and −C into (2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6- trihydroxyphenyl) propan-2-ol (1S) and +C into (2R)-1-(3,4- dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol (1R).', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4046, 'human', 'null', '(-)-Catechin', 'null', 'Strain MT4s-5 readily converted −EC and −C into (2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6- trihydroxyphenyl) propan-2-ol (1S) and +C into (2R)-1-(3,4- dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol (1R).', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4047, 'human', 'null', 'Cianidanol', 'null', 'Strain MT4s-5 readily converted −EC and −C into (2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6- trihydroxyphenyl) propan-2-ol (1S) and +C into (2R)-1-(3,4- dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol (1R).', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4048, 'human', 'null', '(-)-Epicatechin', 'null', 'Strain JCM 9979 easily converted −EC and +C into 1S and 1R, respectively.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4049, 'human', 'null', 'Cianidanol', 'null', 'Strain JCM 9979 easily converted −EC and +C into 1S and 1R, respectively.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4050, 'human', 'null', '(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii MT42 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4051, 'human', 'null', '(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii MT42 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4052, 'human', 'null', '(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii MT42 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4053, 'human', 'null', '(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii MT42 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4054, 'human', 'null', '(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'Similarly, strain MT42 could convert 2S and 2R to their corresponding 4-hydroxy-5-(3-hydroxyphenyl) valeric acids (5R from 2S, 5S from 2R) and 5-(3-hydroxyphenyl)-γ-valerolactones (6R from 2S, 6S from 2R).', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4055, 'human', 'null', '(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'Similarly, strain MT42 could convert 2S and 2R to their corresponding 4-hydroxy-5-(3-hydroxyphenyl) valeric acids (5R from 2S, 5S from 2R) and 5-(3-hydroxyphenyl)-γ-valerolactones (6R from 2S, 6S from 2R).', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4056, 'human', 'null', '(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'Similarly, strain MT42 could convert 2S and 2R to their corresponding 4-hydroxy-5-(3-hydroxyphenyl) valeric acids (5R from 2S, 5S from 2R) and 5-(3-hydroxyphenyl)-γ-valerolactones (6R from 2S, 6S from 2R).', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4057, 'human', 'null', '(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'Similarly, strain MT42 could convert 2S and 2R to their corresponding 4-hydroxy-5-(3-hydroxyphenyl) valeric acids (5R from 2S, 5S from 2R) and 5-(3-hydroxyphenyl)-γ-valerolactones (6R from 2S, 6S from 2R).', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4058, 'human', 'null', '(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4059, 'human', 'null', '(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4060, 'human', 'null', '(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4061, 'human', 'null', '(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4062, 'human', 'null', '(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4063, 'human', 'null', '(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4064, 'human', 'null', '(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4065, 'human', 'null', '(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4066, 'human', 'null', '(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4067, 'human', 'null', '(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4068, 'human', 'null', '(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4069, 'human', 'null', '(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4070, 'human', 'null', '(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4071, 'human', 'null', '(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4072, 'human', 'null', '(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4073, 'human', 'null', '(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol', 'null', 'F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.', 25947926);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4075, 'human', 'null', 'Daidzein', 'null', 'The bacterium cleaved the C-ring of daidzein to produce O-demethylangolensin ( O-Dma).', 12070764);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4076, 'human', 'null', 'Daidzein', 'null', 'The human intestinal bacterium Slackia isoflavoniconvertens converts the main soybean isoflavones daidzein and genistein to equol and 5-hydroxy-equol, respectively.', 23542626);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4077, 'human', 'null', 'Genistein', 'null', 'The human intestinal bacterium Slackia isoflavoniconvertens converts the main soybean isoflavones daidzein and genistein to equol and 5-hydroxy-equol, respectively.', 23542626);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4083, 'human', 'null', 'D-Glucose', 'null', 'The organism produced acetate, lactate, and ethanol as the major products of glucose fermentation.', 16337765);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4084, 'human', 'null', 'D-Glucose', 'null', 'The organism produced acetate, lactate, and ethanol as the major products of glucose fermentation.', 16337765);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4085, 'human', 'null', 'D-Glucose', 'null', 'The organism produced acetate, lactate, and ethanol as the major products of glucose fermentation.', 16337765);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4091, 'human', 'null', 'Myricitrin', 'null', 'Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.', 24882500);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4092, 'human', 'null', 'Myricitrin', 'null', 'Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.', 24882500);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4093, 'human', 'null', 'Myricitrin', 'null', 'Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.', 24882500);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4094, 'human', 'null', 'Myricitrin', 'null', 'Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.', 24882500);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4095, 'human', 'null', 'Myricitrin', 'null', 'Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.', 24882500);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4096, 'human', 'null', 'Myricitrin', 'null', 'Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.', 24882500);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4099, 'human', 'null', 'D-Glucose', 'null', 'Strain M25(T) was negative for both catalase and oxidase activity, utilized carbohydrates as fermentable substrates, produced lactate and acetate as the major end products of glucose fermentation in PYG medium.', 22544782);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4100, 'human', 'null', 'D-Glucose', 'null', 'Strain M25(T) was negative for both catalase and oxidase activity, utilized carbohydrates as fermentable substrates, produced lactate and acetate as the major end products of glucose fermentation in PYG medium.', 22544782);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4101, 'human', 'null', 'Succinate', 'null', 'However, strain YIT 11850(T) was able to decarboxylate succinate to propionate.', 19060046);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4102, 'human', 'null', 'D-Glucose', 'null', 'Cells of the other isolate, strain YIT 11860(T), were non-motile, rod-shaped, positive for aesculin hydrolysis, negative for indole production, produced succinic and acetic acids as end products of glucose metabolism and possessed a DNA G+C content of 45.5 mol%.', 19060046);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4103, 'human', 'null', 'D-Glucose', 'null', 'Cells of the other isolate, strain YIT 11860(T), were non-motile, rod-shaped, positive for aesculin hydrolysis, negative for indole production, produced succinic and acetic acids as end products of glucose metabolism and possessed a DNA G+C content of 45.5 mol%.', 19060046);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4104, 'human', 'null', 'Cynaroside', 'null', 'Three metabolites (acetylated luteoloside, luteolin and deoxygenated luteolin) were detected in the fractions isolated from the bacterial samples.', 25899973);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4106, 'human', 'null', 'D-(+)-Cellobiose', 'null', 'Acetate and succinate were the major end products of cellulose and cellobiose fermentation.', 21357460);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4107, 'human', 'null', 'D-(+)-Cellobiose', 'null', 'Acetate and succinate were the major end products of cellulose and cellobiose fermentation.', 21357460);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4112, 'human', 'null', 'null', 'null', 'Acetate and butyrate were produced by strain 3BBH22T as metabolic end-products.', 29745868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4113, 'human', 'null', 'null', 'null', 'Acetate and butyrate were produced by strain 3BBH22T as metabolic end-products.', 29745868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4114, 'human', 'null', 'Isoquercitrin', 'null', 'Fermentation of quercetin-3-glucoside by growing cultures of Eubacterium ramulus and Enterococcus casseliflavus.', 12747214);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4115, 'human', 'null', 'Isoquercitrin', 'null', 'Fermentation of quercetin-3-glucoside by growing cultures of Eubacterium ramulus and Enterococcus casseliflavus.', 12747214);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4116, 'human', 'null', 'null', 'null', 'We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.', 32392181);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4117, 'human', 'null', 'null', 'null', 'We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.', 32392181);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4118, 'human', 'null', 'null', 'null', 'We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.', 32392181);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4119, 'human', 'null', 'null', 'null', 'We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.', 32392181);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4120, 'human', 'null', 'null', 'null', 'We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.', 32392181);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4121, 'human', 'null', 'null', 'null', 'We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.', 32392181);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4122, 'human', 'null', 'Daidzein', 'null', 'In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.
', 32218321);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4123, 'human', 'null', 'Daidzein', 'null', 'In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.
', 32218321);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4124, 'human', 'null', 'Daidzein', 'null', 'In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.
', 32218321);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4125, 'human', 'null', 'Daidzein', 'null', 'In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.
', 32218321);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4126, 'human', 'null', 'Daidzein', 'null', 'In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.
', 32218321);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4127, 'human', 'null', 'Daidzein', 'null', 'In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.
', 32218321);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4131, 'human', 'null', 'Pyruvate', 'null', 'According to the FDR-adjusted p-values (q-values) in multivariable linear regression, p-cresyl sulfate showed a significant association with six major genera, with positive correlations observed with Alistipes, Oscillibacter, and Subdoligranulum (q < 0.001, q < 0.001, and q = 0.023, respectively) and negative correlations with Lachnospira, Veillonella, and Megamonas (q = 0.039, q = 0.014, and q = 0.034, respectively). Serum p-cresyl glucuronide and indoxyl sulfate levels were positively correlated with Alistipes (q = 0.010 and q = 0.035, respectively) and Oscillibacter (q = 0.001 and q = 0.037, respectively). Serum TMAO levels were associated only with Oscillibacter (q = 0.006). Unexpectedly, we found Oscillibacter to be associated with all four measured uremic toxins, although each of these metabolites is derived from different parent compounds.', 32560104);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4132, 'human', 'null', 'Pyruvate', 'null', 'According to the FDR-adjusted p-values (q-values) in multivariable linear regression, p-cresyl sulfate showed a significant association with six major genera, with positive correlations observed with Alistipes, Oscillibacter, and Subdoligranulum (q < 0.001, q < 0.001, and q = 0.023, respectively) and negative correlations with Lachnospira, Veillonella, and Megamonas (q = 0.039, q = 0.014, and q = 0.034, respectively). Serum p-cresyl glucuronide and indoxyl sulfate levels were positively correlated with Alistipes (q = 0.010 and q = 0.035, respectively) and Oscillibacter (q = 0.001 and q = 0.037, respectively). Serum TMAO levels were associated only with Oscillibacter (q = 0.006). Unexpectedly, we found Oscillibacter to be associated with all four measured uremic toxins, although each of these metabolites is derived from different parent compounds.', 32560104);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4133, 'human', 'null', 'Pyruvate', 'null', 'According to the FDR-adjusted p-values (q-values) in multivariable linear regression, p-cresyl sulfate showed a significant association with six major genera, with positive correlations observed with Alistipes, Oscillibacter, and Subdoligranulum (q < 0.001, q < 0.001, and q = 0.023, respectively) and negative correlations with Lachnospira, Veillonella, and Megamonas (q = 0.039, q = 0.014, and q = 0.034, respectively). Serum p-cresyl glucuronide and indoxyl sulfate levels were positively correlated with Alistipes (q = 0.010 and q = 0.035, respectively) and Oscillibacter (q = 0.001 and q = 0.037, respectively). Serum TMAO levels were associated only with Oscillibacter (q = 0.006). Unexpectedly, we found Oscillibacter to be associated with all four measured uremic toxins, although each of these metabolites is derived from different parent compounds.', 32560104);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4134, 'human', 'null', 'Pyruvate', 'null', 'According to the FDR-adjusted p-values (q-values) in multivariable linear regression, p-cresyl sulfate showed a significant association with six major genera, with positive correlations observed with Alistipes, Oscillibacter, and Subdoligranulum (q < 0.001, q < 0.001, and q = 0.023, respectively) and negative correlations with Lachnospira, Veillonella, and Megamonas (q = 0.039, q = 0.014, and q = 0.034, respectively). Serum p-cresyl glucuronide and indoxyl sulfate levels were positively correlated with Alistipes (q = 0.010 and q = 0.035, respectively) and Oscillibacter (q = 0.001 and q = 0.037, respectively). Serum TMAO levels were associated only with Oscillibacter (q = 0.006). Unexpectedly, we found Oscillibacter to be associated with all four measured uremic toxins, although each of these metabolites is derived from different parent compounds.', 32560104);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4135, 'human', 'increase', 'null', 'low-throughput', 'Here, we elucidate a micro biome-dependent pathway by which SG increases CA7S production. We show that a microbial metabolite,lithocholic acid (LCA), is increased in murine portal veins post-SG and by activating the vitamin D receptor, induces hepatic mSult2A1/hSULT2A expression to drive CA7S production. An SG-induced shift in the micro biome increases gut expression of the bile acid transporters Asbt and Osta, which in turn facilitate selective transport of LCA across the gut epithelium. Cecal microbiota transplant from SG animals is sufficient to recreate the pathway in germ-free (GF) animals. 
and GLP-1 secretion, causally connecting a microbial metabolite with the improvement of diabetic phe notypes.', 33434516);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4136, 'human', 'increase', 'null', 'low-throughput', 'Here, we elucidate a micro biome-dependent pathway by which SG increases CA7S production. We show that a microbial metabolite,lithocholic acid (LCA), is increased in murine portal veins post-SG and by activating the vitamin D receptor, induces hepatic mSult2A1/hSULT2A expression to drive CA7S production. An SG-induced shift in the micro biome increases gut expression of the bile acid transporters Asbt and Osta, which in turn facilitate selective transport of LCA across the gut epithelium. Cecal microbiota transplant from SG animals is sufficient to recreate the pathway in germ-free (GF) animals. 
and GLP-1 secretion, causally connecting a microbial metabolite with the improvement of diabetic phe notypes.', 33434516);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4137, 'human', 'increase', 'Trimethylamine', 'low-throughput', 'We found that TMAO treatment activated astrocytes, as indicated by a ~ 2-fold increase in LCN2 (Fig. 6a) and a ~ 3-fold increase in CD44 (Fig. 6b), a pro-inflammatory intercellular adhesion pro tein characteristic of reactive astrocytes [10].', 32862276);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4138, 'human', 'increase', 'Trimethylamine', 'low-throughput', 'We found that TMAO treatment activated astrocytes, as indicated by a ~ 2-fold increase in LCN2 (Fig. 6a) and a ~ 3-fold increase in CD44 (Fig. 6b), a pro-inflammatory intercellular adhesion pro tein characteristic of reactive astrocytes [10].', 32862276);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4139, 'human', 'decrease', 'Baicalin', 'low-throughput', 'As shown in Fig. 5a, treatment with 10�?0 μM of BE signifcantly reduced LPS-induced IL-8 secretion. Similar inhibitory efects were observed for BI at 15�?0 μM; however, BE showed more potent efects.', 31650468);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4140, 'human', 'decrease', 'null', 'low-throughput', 'As shown in Fig. 5a, treatment with 10�?0 μM of BE signifcantly reduced LPS-induced IL-8 secretion. Similar inhibitory efects were observed for BI at 15�?0 μM; however, BE showed more potent efects.', 31650468);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4141, 'human', 'increase', 'Trimethylamine', 'low-throughput', 'Plasma TMAO was positively associated with plasma levels of TNFα and IL-12 (Figures 3A,B), but not with IL-6 (p = 0.263, rho = 0.11) or IL-8.', 33042155);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4142, 'human', 'increase', 'Trimethylamine', 'low-throughput', 'Plasma TMAO was positively associated with plasma levels of TNFα and IL-12 (Figures 3A,B), but not with IL-6 (p = 0.263, rho = 0.11) or IL-8.', 33042155);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4143, 'human', 'increase', 'Trimethylamine', 'low-throughput', 'Herein, we found that plasma levels of LPS were significantly correlated with TMAOand the TMA precursors carnitine and y-BB in CVID patients (Figures 6A–C), indicating a relation between gut leakage and TMAO pathway metabolites.', 33042155);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4144, 'human', 'null', 'Secoisolariciresinol diglucoside', 'null', 'Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4145, 'human', 'null', 'Secoisolariciresinol diglucoside', 'null', 'Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4146, 'human', 'null', 'Secoisolariciresinol diglucoside', 'null', 'Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4147, 'human', 'null', 'Secoisolariciresinol diglucoside', 'null', 'Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4148, 'human', 'null', 'Secoisolariciresinol diglucoside', 'null', 'Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4149, 'human', 'null', 'Secoisolariciresinol diglucoside', 'null', 'Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4150, 'human', 'null', 'Lariciresinol', 'null', 'In addition,the previously isolated strain E. lenta SECO-Mt75m3 catalysed the reduction of pinoresinol to SECO via lariciresinol.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4151, 'human', 'null', 'Pinoresinol', 'null', 'In addition,the previously isolated strain E. lenta SECO-Mt75m3 catalysed the reduction of pinoresinol to SECO via lariciresinol.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4152, 'human', 'null', 'Matairesinol', 'null', 'However, methoxy or hydroxyl groups adjacent to the targeted methoxy groups may be required by P. productus SECO-Mt75m3 to catalyse demethylation, as evident from the inability of this organism to demethylate m-anisic acid. Incubation of P.productus SECO-Mt75m3 with ferulic acid led to complete conversion of the substrate without detection of any products by HPLC, suggesting that P. productus not only catalyses the demethylation of ferulic acid but also a number of additional reactions. More interestingly, P. productus SECO-Mt75m3 catalysed as well the demethyla tion of other lignans, namely lariciresinol (Fig. 2),
matairesinol and pinoresinol (data not shown). In addition, the previously isolated strain E. lenta SECO-Mt75m3 catalysed the reduction of pinoresinol to SECO via lariciresinol (Fig. 3).', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4153, 'human', 'null', 'Secoisolariciresinol', 'null', 'In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4154, 'human', 'null', 'Secoisolariciresinol', 'null', 'In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4155, 'human', 'null', 'Secoisolariciresinol', 'null', 'In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4156, 'human', 'null', 'Secoisolariciresinol', 'null', 'In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4157, 'human', 'null', 'Pinoresinol', 'null', 'In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.', 16765860);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4158, 'human', 'null', '5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one', 'null', 'A hypothetical pathway for the degradation of narin genin by E. ramulus has been proposed by Schneider and Blaut (2000) (Fig. 1a). The first step in this pathway is the isomerization of naringenin to naringenin chalcone, which is subsequently reduced to phloretin. Phloretin is further
hydrolyzed to phloroglucinol and 3(4-hydroxyphenyl)pro pionic acid.', 15127184);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4159, 'human', 'null', 'Naringenin chalcone', 'null', 'A hypothetical pathway for the degradation of narin genin by E. ramulus has been proposed by Schneider and Blaut (2000) (Fig. 1a). The first step in this pathway is the isomerization of naringenin to naringenin chalcone, which is subsequently reduced to phloretin. Phloretin is further
hydrolyzed to phloroglucinol and 3(4-hydroxyphenyl)pro pionic acid.', 15127184);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4160, 'human', 'null', 'Phloretin', 'null', 'A hypothetical pathway for the degradation of narin genin by E. ramulus has been proposed by Schneider and Blaut (2000) (Fig. 1a). The first step in this pathway is the isomerization of naringenin to naringenin chalcone, which is subsequently reduced to phloretin. Phloretin is further
hydrolyzed to phloroglucinol and 3(4-hydroxyphenyl)pro pionic acid.', 15127184);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4161, 'human', 'null', 'Phloretin', 'null', 'A hypothetical pathway for the degradation of narin genin by E. ramulus has been proposed by Schneider and Blaut (2000) (Fig. 1a). The first step in this pathway is the isomerization of naringenin to naringenin chalcone, which is subsequently reduced to phloretin. Phloretin is further
hydrolyzed to phloroglucinol and 3(4-hydroxyphenyl)pro pionic acid.', 15127184);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4162, 'human', 'null', 'Apigenin', 'null', 'The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which is followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.', 15466559);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4163, 'human', 'null', '5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one', 'null', 'The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which
is followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.', 15466559);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4164, 'human', 'null', 'Naringenin chalcone', 'null', 'The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which
is followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.', 15466559);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4165, 'human', 'null', 'Phloretin', 'null', 'The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which
is followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.', 15466559);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4166, 'human', 'null', 'Phloretin', 'null', 'The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which
is followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.', 15466559);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4167, 'human', 'null', 'Kaempferol-3-O-gal-rham-7-O-rham', 'null', 'Bacteroides distasonis hydrolyzed robinin to kaempferol, and Bacteroides uniformis and Bacteroides ovatus converted rutin to quercetin.', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4168, 'human', 'null', 'Rutin', 'null', 'Bacteroides distasonis hydrolyzed robinin to kaempferol, and Bacteroides uniformis and Bacteroides ovatus converted rutin to quercetin.', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4169, 'human', 'null', 'Rutin', 'null', 'Bacteroides distasonis hydrolyzed robinin to kaempferol, and Bacteroides uniformis and Bacteroides ovatus converted rutin to quercetin.', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4170, 'human', 'null', 'Glycitein', 'null', 'Glycitein is demethoxylated in vitro by Eubacterium limosum to 6,7,40-trihydroxyisoflavone .', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4171, 'human', 'null', 'Daidzein', 'null', 'Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4172, 'human', 'null', 'Daidzein', 'null', 'Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4173, 'human', 'null', 'Daidzein', 'null', 'Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4174, 'human', 'null', 'Daidzein', 'null', 'Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4175, 'human', 'null', 'Daidzein', 'null', 'Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4176, 'human', 'null', 'Daidzein', 'null', 'Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4177, 'human', 'null', 'Daidzein', 'null', 'Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4178, 'human', 'null', 'Dihydrodaidzein', 'null', 'The bacterial strain named Julong 732 is also able to metabolize dihydrodaidzein to S-equol (123). Thus, other bacteria have to be involved in the previous steps of the conversion of daidzein to equol .', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4179, 'human', 'null', 'Isoxanthohumol', 'null', 'Isoxanthohumol is a prenylated flavanone from hops and beer that has interesting biological activities. It can be metabo lized in vivo by colonic microbiota to render 8-prenylnaringenin (133, 134). This metabolic O-demethylation is carried out by Eubacterium species and particularly by Eubacterium limosum.', 19580283);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4180, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4181, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4182, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4183, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4184, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4185, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4186, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4187, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4188, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4189, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4190, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4191, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4192, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4193, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4194, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4195, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4196, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4197, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4198, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4199, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4200, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4201, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4202, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4203, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4204, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4205, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4206, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4207, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4208, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4209, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4210, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4211, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4212, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4213, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4214, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4215, 'mouse', 'null', 'Cholesterol', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4216, 'mouse', 'null', 'Cholesterol', 'null', 'Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4217, 'mouse', 'null', 'Cholesterol', 'null', 'Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4218, 'mouse', 'null', 'Cholesterol', 'null', 'Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4219, 'mouse', 'null', 'Cholesterol', 'null', 'Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4220, 'mouse', 'null', 'Cholesterol', 'null', 'Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4221, 'mouse', 'null', 'Cholesterol', 'null', 'Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4222, 'mouse', 'null', 'Cholesterol', 'null', 'Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4223, 'mouse', 'null', 'Cholesterol', 'null', 'Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4224, 'mouse', 'null', 'null', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4225, 'mouse', 'null', 'null', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4226, 'mouse', 'null', 'null', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4227, 'mouse', 'null', 'null', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4228, 'mouse', 'null', 'null', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4229, 'mouse', 'null', 'null', 'null', 'unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.', 31780788);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4233, 'mouse', 'null', 'Epigallocatechin', 'null', 'Biotransformation of both EGC and GC was observed with Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.', 25747993);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4234, 'mouse', 'null', 'Epigallocatechin', 'null', 'Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.', 25747993);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4235, 'mouse', 'null', 'Epigallocatechin', 'null', 'Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.', 25747993);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4236, 'mouse', 'null', '4''-dehydroxylated (�?-epigallocatechin', 'null', 'Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.', 25747993);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4237, 'mouse', 'null', 'null', 'null', 'Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.', 25747993);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4238, 'mouse', 'null', 'null', 'null', 'Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.', 25747993);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4239, 'mouse', 'null', '1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol', 'null', 'Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.', 25747993);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4240, 'mouse', 'null', '(-)-Gallocatechin', 'null', 'The�?bioconversion�?of�?GC�?was�?examined�?here.�? Strain MT4s-5 was�?observed to convert GC into 1 and 8  with�?slight production of 2. In the presence of hydrogen, strain MT4s-5�?accelerated�?the�?formation�?of�?2�?(data�?not�?shown).�?The�?results�?were�?very�?similar�?to�?those�?obtained�?in�?the�?bioconversion of EGC by strain  MT4s-5.', 25747993);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4241, 'mouse', 'null', '(-)-Gallocatechin', 'null', 'The�?bioconversion�?of�?GC�?was�?examined�?here.�? Strain MT4s-5 was�?observed to convert GC into 1 and 8  with�?slight production of 2. In the presence of hydrogen, strain MT4s-5�?accelerated�?the�?formation�?of�?2�?(data�?not�?shown).�?The�?results�?were�?very�?similar�?to�?those�?obtained�?in�?the�?bioconversion of EGC by strain  MT4s-5.', 25747993);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4242, 'mouse', 'null', '(-)-Gallocatechin', 'null', 'The�?bioconversion�?of�?GC�?was�?examined�?here.�? Strain MT4s-5 was�?observed to convert GC into 1 and 8  with�?slight production of 2. In the presence of hydrogen, strain MT4s-5�?accelerated�?the�?formation�?of�?2�?(data�?not�?shown).�?The�?results�?were�?very�?similar�?to�?those�?obtained�?in�?the�?bioconversion of EGC by strain  MT4s-5.', 25747993);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4243, 'mouse', 'decrease', 'null', 'low-throughput', 'Ratelimited enzymes in the process of TG synthesis of the DGAT1/2 mRNA level in the HFD group were signifificantly increased compared with the LFD group. The DGAT2 mRNA level in the HFD-CBM group decreased in the same range as the LFD group, whereas signifificant differences of the DGAT1 mRNA level were not found between the HFD group and the HFD-CBM group.', 24166662);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4249, 'mouse', 'decrease', 'null', 'low-throughput', 'TMAO inhibits SIRT1 expression, induces oxidative stress, and impairs NO production both in vivo and in vitro.', 29325896);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4250, 'mouse', 'decrease', 'null', 'low-throughput', 'By conducting a miRNA microarray screening, we revealed that expression of miRNA-10b in macrophages can be reduced by PCA. Functional analyses demonstrated that miRNA-10b directly represses ABCA1 and ABCG1 and negatively regulates cholesterol efflux from murine- and human-derived macrophages. Further in vitro and ex vivo analyses verified that PCA accelerates macrophage cholesterol efflux, correlating with the regulation of miRNA-10b-ABCA1/ABCG1 cascade, whereas Cy-3-G consumption promoted macrophage RCT and regressed atherosclerotic lesion in a gut microbiotaendependent manner.', 22821931);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4251, 'mouse', 'increase', 'null', 'low-throughput', 'By conducting a miRNA microarray screening, we revealed that expression of miRNA-10b in macrophages can be reduced by PCA. Functional analyses demonstrated that miRNA-10b directly represses ABCA1 and ABCG1 and negatively regulates cholesterol efflux from murine- and human-derived macrophages. Further in vitro and ex vivo analyses verified that PCA accelerates macrophage cholesterol efflux, correlating with the regulation of miRNA-10b-ABCA1/ABCG1 cascade, whereas Cy-3-G consumption promoted macrophage RCT and regressed atherosclerotic lesion in a gut microbiotaendependent manner.', 22821931);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4252, 'mouse', 'increase', 'null', 'low-throughput', 'By conducting a miRNA microarray screening, we revealed that expression of miRNA-10b in macrophages can be reduced by PCA. Functional analyses demonstrated that miRNA-10b directly represses ABCA1 and ABCG1 and negatively regulates cholesterol efflux from murine- and human-derived macrophages. Further in vitro and ex vivo analyses verified that PCA accelerates macrophage cholesterol efflux, correlating with the regulation of miRNA-10b-ABCA1/ABCG1 cascade, whereas Cy-3-G consumption promoted macrophage RCT and regressed atherosclerotic lesion in a gut microbiotaendependent manner.', 22821931);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4253, 'mouse', 'decrease', 'null', 'low-throughput', 'We extended our study to human samples to determine the effect of bile acids on CXCL16 mRNA expression of human liver sinusoidal endothelial cells. Similar to the mouse study, primary bile acids CDCA and TCA induced CXCL16 mRNA expression in SK-HEP1 cells (Fig. 5H). Next, the correlation between bile acids and CXCL16 expression was tested in nontumor liver tissues from HCC or cholangiocarcinoma patients of the Thailand Initiative for Genomics and Expression Research in Liver Cancer (TIGER-LC) cohort (34). Primary bile acid CDCA levels correlated with CXCL16 expression (Fig. 5I), whereas inverse correlation was seen for the secondary bile acid glycolithocholate (GLCA) (fig. S6I). The primary/secondary ratio was associated with CXCL16 increase (fig. S6J), indicating that the opposing effect of bile acids on CXCL16 expression also exists in humans.', 29798856);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4254, 'mouse', 'decrease', 'null', 'low-throughput', 'In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.', 29796620);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4255, 'mouse', 'decrease', 'null', 'low-throughput', 'In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.', 29796620);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4256, 'mouse', 'decrease', 'null', 'low-throughput', 'In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.', 29796620);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4257, 'mouse', 'decrease', 'null', 'low-throughput', 'In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.', 29796620);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4258, 'mouse', 'increase', 'null', 'low-throughput', 'In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.', 29796620);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4259, 'mouse', 'null', '4-Hydroxyphenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4260, 'mouse', 'null', '4-Hydroxyphenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4261, 'mouse', 'null', '4-Hydroxyphenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4262, 'mouse', 'null', '4-Hydroxyphenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4263, 'mouse', 'null', '4-Hydroxyphenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4264, 'mouse', 'null', '4-Hydroxyphenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4265, 'mouse', 'null', '4-Hydroxyphenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4266, 'mouse', 'null', '4-Hydroxyphenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4267, 'mouse', 'null', '4-Hydroxyphenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4268, 'mouse', 'null', 'D-Glucose', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4269, 'mouse', 'null', 'D-Glucose', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4270, 'mouse', 'null', 'D-Glucose', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4271, 'mouse', 'null', 'D-Glucose', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4272, 'mouse', 'null', 'D-Glucose', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4273, 'mouse', 'null', 'D-Glucose', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4274, 'mouse', 'null', 'D-Glucose', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4275, 'mouse', 'null', 'D-Glucose', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4276, 'mouse', 'null', 'D-Glucose', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4277, 'mouse', 'null', 'D-Glucose', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4278, 'mouse', 'null', 'D-Glucose', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4279, 'mouse', 'null', 'D-Glucose', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4280, 'mouse', 'null', 'Indole-3-acetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4281, 'mouse', 'null', 'Indole-3-acetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4282, 'mouse', 'null', 'Indole-3-acetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4283, 'mouse', 'null', 'Indole-3-acetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4284, 'mouse', 'null', 'Indole-3-acetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4285, 'mouse', 'null', 'Indole-3-acetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4286, 'mouse', 'null', 'Indole-3-acetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4287, 'mouse', 'null', 'Indole-3-acetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4288, 'mouse', 'null', 'Indole-3-acetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4289, 'mouse', 'null', 'Phenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4290, 'mouse', 'null', 'Phenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4291, 'mouse', 'null', 'Phenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4292, 'mouse', 'null', 'Phenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4293, 'mouse', 'null', 'Phenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4294, 'mouse', 'null', 'Phenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4295, 'mouse', 'null', 'Phenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4296, 'mouse', 'null', 'Phenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4297, 'mouse', 'null', 'Phenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4298, 'mouse', 'null', 'Phenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4299, 'mouse', 'null', 'Phenylacetic acid', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4300, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4301, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4302, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4303, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4304, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4305, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4306, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4307, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4308, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4309, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4310, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4311, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4312, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4313, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4314, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4315, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4316, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4317, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4318, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4319, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4320, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4321, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4322, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4323, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4324, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4325, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4326, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4327, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4328, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4329, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4330, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4331, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4332, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4333, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4334, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4335, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4336, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4337, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4338, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4339, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4340, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4341, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4342, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4343, 'mouse', 'null', 'Tryptophan', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4344, 'mouse', 'null', 'phenyalanine', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4345, 'mouse', 'null', 'phenyalanine', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4346, 'mouse', 'null', 'phenyalanine', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4347, 'mouse', 'null', 'phenyalanine', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4348, 'mouse', 'null', 'phenyalanine', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4349, 'mouse', 'null', 'phenyalanine', 'null', 'To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.', 23349065);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4350, 'mouse', 'increase', 'null', 'low-throughput', 'The main metabolites produced by B. thetaiotaomicron in the GIT were acetate (up to 6 mM) and propionate (up to 2 mM) (Figure 1D). We evaluated the effects of these bacterial metabolites on the amount of KLF4 protein produced by the mucus producing cell line HT29-MTX [31]. The amount of KLF4 protein was higher in the presence than in the absence of acetate (10 and 20 mM) (Figure 3A)whereas propionate (5 and 10 mM) and butyrate (1 and 5 mM) had no effect (Figure 3B, C).', 23692866);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4351, 'mouse', 'null', 'null', 'null', 'The main metabolites produced by B. thetaiotaomicron in the GIT were acetate (up to 6 mM) and propionate (up to 2 mM) (Figure 1D). We evaluated the effects of these bacterial metabolites on the amount of KLF4 protein produced by the mucus producing cell line HT29-MTX [31]. The amount of KLF4 protein was higher in the presence than in the absence of acetate (10 and 20 mM) (Figure 3A)whereas propionate (5 and 10 mM) and butyrate (1 and 5 mM) had no effect (Figure 3B, C).', 23692866);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4357, 'mouse', 'increase', 'Acetate', 'low-throughput', 'We first checked  in vitro  that  B. thetaiotaomicron produced acetate and propionate and that  F. prausnitzii consumed acetate and produced butyrate (Figure 4A). When both bacteria were grown together, butyrate was produced rather than acetate. The butyrate production by F. prausnitzii as higher in the presence than absence of B. thetaiotaomicron  (10.9 ± 0.3 mM versus 7.8 ± 0.5 mM) suggesting that  F. prausnitzii  synthesized butyrate from the acetate produced by  B. thetaiotaomicron .', 23692866);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4361, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4362, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4363, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4364, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4365, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4366, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4367, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4368, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4369, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4370, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4371, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4372, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4373, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4374, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4375, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4376, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4377, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4378, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4379, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4380, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4381, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4382, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4383, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4384, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4385, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4386, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4387, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4388, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4389, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4390, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4391, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4392, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4393, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4394, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4395, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4396, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4397, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4398, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4399, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4400, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4401, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4402, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4403, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4404, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4405, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4406, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4407, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4408, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4409, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4410, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4411, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4412, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4413, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4414, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4415, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4416, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4417, 'mouse', 'null', 'null', 'null', 'Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.', 31214133);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4418, 'mouse', 'increase', 'null', 'low-throughput', 'IVAS stimulated a dose-dependent increase in glucose transport through glucose transporter GLUT4- and PI3K-dependent mechanisms. IVAS also up regulated GLUT1, GLUT4, and PI3K p85a protein, and increased phosphorylation of Akt.', 30183376);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4419, 'mouse', 'increase', 'null', 'low-throughput', 'IVAS stimulated a dose-dependent increase in glucose transport through glucose transporter GLUT4- and PI3K-dependent mechanisms. IVAS also up regulated GLUT1, GLUT4, and PI3K p85a protein, and increased phosphorylation of Akt.', 30183376);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4420, 'mouse', 'increase', 'null', 'low-throughput', 'IVAS stimulated a dose-dependent increase in glucose transport through glucose transporter GLUT4- and PI3K-dependent mechanisms. IVAS also up regulated GLUT1, GLUT4, and PI3K p85a protein, and increased phosphorylation of Akt.', 30183376);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4425, 'mouse', 'null', 'D-Glucose', 'null', 'R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4426, 'mouse', 'null', 'D-Glucose', 'null', 'R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4427, 'mouse', 'null', 'D-Glucose', 'null', 'R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4428, 'mouse', 'null', 'D-Glucose', 'null', 'R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4429, 'mouse', 'null', 'D-Glucose', 'null', 'R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4430, 'mouse', 'null', 'D-Glucose', 'null', 'R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4431, 'mouse', 'null', 'D-Glucose', 'null', 'Propionate formation from fucose via propanediol has also been described in the human gut anaerobe Roseburia inulinivorans, which produces butyrate rather than propionate when grown on glucose.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4432, 'mouse', 'null', 'L-Fucose', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4433, 'mouse', 'null', 'L-Fucose', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4434, 'mouse', 'null', 'L-Fucose', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4435, 'mouse', 'null', 'Lactic acid', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4436, 'mouse', 'null', 'Lactic acid', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4437, 'mouse', 'null', 'Lactic acid', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4438, 'mouse', 'null', 'Lactic acid', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4439, 'mouse', 'null', 'Lactic acid', 'null', 'We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4440, 'mouse', 'null', 'Lactic acid', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4441, 'mouse', 'null', 'Lactic acid', 'null', 'We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4442, 'mouse', 'null', 'Lactic acid', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4443, 'mouse', 'null', 'Lactic acid', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4444, 'mouse', 'null', 'Lactic acid', 'null', 'We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4445, 'mouse', 'null', 'Lactic acid', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4446, 'mouse', 'null', 'Formate', 'null', 'We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4447, 'mouse', 'null', 'Formate', 'null', 'We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4448, 'mouse', 'null', 'Acrylate', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4449, 'mouse', 'null', 'Acrylate', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4450, 'mouse', 'null', 'Acrylate', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4451, 'mouse', 'null', 'Butyrate', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4452, 'mouse', 'null', 'Butyrate', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4453, 'mouse', 'null', 'Propanediol', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4454, 'mouse', 'null', 'Propanediol', 'null', 'The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4455, 'mouse', 'null', 'Succinate', 'null', 'The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4456, 'mouse', 'null', 'Succinate', 'null', 'The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4457, 'mouse', 'null', 'Succinate', 'null', 'The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4458, 'mouse', 'null', 'Succinate', 'null', 'The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4459, 'mouse', 'null', 'Succinate', 'null', 'The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4460, 'mouse', 'null', 'Succinate', 'null', 'The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4461, 'mouse', 'null', 'Succinate', 'null', 'The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.', 24553467);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4494, 'mouse', 'increase', 'null', 'low-throughput', 'succinate augmented the expression of TNF and IL-1b by iMoDCs when used in combination with the TLR3 ligand poly(I:C) or the TLR7 agonist imiquimod.', 18820681);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4495, 'mouse', 'increase', 'null', 'low-throughput', 'succinate augmented the expression of TNF and IL-1b by iMoDCs when used in combination with the TLR3 ligand poly(I:C) or the TLR7 agonist imiquimod.', 18820681);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4496, 'mouse', 'increase', 'null', 'low-throughput', 'Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.', 21531334);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4502, 'mouse', 'increase', 'null', 'low-throughput', 'Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.', 21531334);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4503, 'mouse', 'increase', 'null', 'low-throughput', 'Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.', 21531334);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4504, 'mouse', 'increase', 'null', 'low-throughput', 'Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.', 21531334);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4505, 'mouse', 'increase', 'null', 'low-throughput', 'Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.', 21531334);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4506, 'mouse', 'increase', 'null', 'low-throughput', 'Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.', 21531334);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4507, 'mouse', 'decrease', 'null', 'low-throughput', 'We also tested whether butyrate inhibits autophagy in GF colonocytes. As expected, it decreases p27 phosphorylation and LC3-II compared to untreated GF colonocytes.', 21531334);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4508, 'mouse', 'decrease', 'null', 'low-throughput', 'We also tested whether butyrate inhibits autophagy in GF colonocytes. As expected, it decreases p27 phosphorylation and LC3-II compared to untreated GF colonocytes.', 21531334);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4512, 'mouse', 'null', 'null', 'null', 'It has been reported that B. thetaiotaomicron and B. ovatus have the ability to ferment AAA  to produce phenylacetic acid.', 28628112);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4513, 'mouse', 'null', 'null', 'null', 'It has been reported that B. thetaiotaomicron and B. ovatus have the ability to ferment AAA  to produce phenylacetic acid.', 28628112);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4523, 'mouse', 'increase', 'Tyrosine', 'low-throughput', 'Here we show using untargeted metabolomics that levels of phenyl sulfate, a gut microbiota-derived metabolite, increase with the progression of diabetes in rats overexpressing human uremic toxin transporter SLCO4C1 in the kidney, and are decreased in rats with limited proteinuria.', 31015435);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4524, 'mouse', 'increase', 'Tyrosine', 'low-throughput', 'although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.', 31015435);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4525, 'mouse', 'increase', 'Tyrosine', 'low-throughput', 'although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.', 31015435);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4526, 'mouse', 'increase', 'Tyrosine', 'low-throughput', 'although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.', 31015435);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4527, 'mouse', 'increase', 'Tyrosine', 'low-throughput', 'although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.', 31015435);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4528, 'mouse', 'increase', 'Tyrosine', 'low-throughput', 'although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.', 31015435);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4530, 'mouse', 'null', 'Daidzein', 'null', 'Equol is a metabolite of daidzein that is produced by intestinal microbiota.The rod-shaped and Gram-positive anaerobic equol-producing intestinal bacterium Slackia TM-30 was isolated from healthy human faeces.', 26455424);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4534, 'mouse', 'increase', 'null', 'low-throughput', 'Live B. dentium-monoassociated mice exhibited increased colonic expression of goblet cell markers Krüppel-like factor 4 (Klf4), Trefoil factor 3 (Tff3), Relm-β, Muc2, and several glycosyltransferases compared to both heat-killed B. dentium and germfree counterparts.', 31213556);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4540, 'mouse', 'increase', 'null', 'low-throughput', 'We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.', 31213556);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4541, 'mouse', 'increase', 'null', 'low-throughput', 'We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.', 31213556);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4542, 'mouse', 'increase', 'null', 'low-throughput', 'We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.', 31213556);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4543, 'mouse', 'increase', 'null', 'low-throughput', 'We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.', 31213556);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4544, 'mouse', 'increase', 'null', 'low-throughput', 'We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.', 31213556);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4545, 'mouse', 'increase', 'null', 'low-throughput', 'B. dentium can secrete products, such as acetate, that are capable of increasing MUC2 synthesis.', 31213556);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4549, 'mouse', 'null', 'null', 'null', 'A specific human-associated gut microbe, Clostridium orbiscindens,produced DAT and rescued antibiotic-treated influenza-infected mice.Consistent with our hypothesis that DAT protects from influenza infection by enhancing type I IFN signaling before infection.', 28774928);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4550, 'mouse', 'increase', 'null', 'low-throughput', 'Oxidative stress and inflammation in the kidney have been demonstrated to play a pivotal role in the initiation and progression of obesity-related renal injury [20e22]. To further determine the possible molecular mechanisms by which elevated TMAO levels induce renal interstitial fibrosis and dysfunction, we measured expression of NOX-4 (a constitutive active enzyme responsible for the production of reactive oxygen species) and proinflammatory cytokines in the kidney. As presented in Fig. 4, expression of NOX-4(Fig. 4A and B) and pro-inflammatory cytokines TNF-a (Fig. 4A and C) and IL-1 b (Fig. 4A and D) was significantly increased in mice fed a HFD, compared with mice fed a LFD, but was reduced by DMB treatment. Expression of NOX-4, TNF-a and IL-1 b in mice fed a LFD was not affected by DMB treatment.', 28942145);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4551, 'mouse', 'increase', 'null', 'low-throughput', 'Oxidative stress and inflammation in the kidney have been demonstrated to play a pivotal role in the initiation and progression of obesity-related renal injury [20e22]. To further determine the possible molecular mechanisms by which elevated TMAO levels induce renal interstitial fibrosis and dysfunction, we measured expression of NOX-4 (a constitutive active enzyme responsible for the production of reactive oxygen species) and proinflammatory cytokines in the kidney. As presented in Fig. 4, expression of NOX-4(Fig. 4A and B) and pro-inflammatory cytokines TNF-a (Fig. 4A and C) and IL-1 b (Fig. 4A and D) was significantly increased in mice fed a HFD, compared with mice fed a LFD, but was reduced by DMB treatment. Expression of NOX-4, TNF-a and IL-1 b in mice fed a LFD was not affected by DMB treatment.', 28942145);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4552, 'mouse', 'increase', 'null', 'low-throughput', 'Oxidative stress and inflammation in the kidney have been demonstrated to play a pivotal role in the initiation and progression of obesity-related renal injury [20e22]. To further determine the possible molecular mechanisms by which elevated TMAO levels induce renal interstitial fibrosis and dysfunction, we measured expression of NOX-4 (a constitutive active enzyme responsible for the production of reactive oxygen species) and proinflammatory cytokines in the kidney. As presented in Fig. 4, expression of NOX-4(Fig. 4A and B) and pro-inflammatory cytokines TNF-a (Fig. 4A and C) and IL-1 b (Fig. 4A and D) was significantly increased in mice fed a HFD, compared with mice fed a LFD, but was reduced by DMB treatment. Expression of NOX-4, TNF-a and IL-1 b in mice fed a LFD was not affected by DMB treatment.', 28942145);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4553, 'mouse', 'increase', 'null', 'low-throughput', 'Butyrate or niacin induces IL-18 expression in colonic epithelium in a Gpr109a-dependent manner.', 24412617);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4554, 'mouse', 'increase', 'null', 'low-throughput', 'Butyrate or niacin induces IL-18 expression in colonic epithelium in a Gpr109a-dependent manner.', 24412617);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4555, 'mouse', 'decrease', 'null', 'low-throughput', 'Our data indicate that WAT miR-181 expression is controlled by the relative abundance of microbiota-derived metabolites acting as positive and negative regulators of miR-181 expression and that  indole and indole-derived metabolites  including indoxyl sulfate  are key negative regulators of miR-181 expression in WAT.', 31189717);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4556, 'mouse', 'decrease', 'null', 'low-throughput', 'Our data indicate that WAT miR-181 expression is controlled by the relative abundance of microbiota-derived metabolites acting as positive and negative regulators of miR-181 expression and that  indole and indole-derived metabolites  including indoxyl sulfate  are key negative regulators of miR-181 expression in WAT.', 31189717);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4560, 'mouse', 'increase', 'null', 'low-throughput', 'D8 up-regulates IL-22 expression ex vivo and in vivo.', 29459771);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4561, 'mouse', 'null', 'null', 'null', 'Thus, exposure to S. cerevisiae in the gut led to enhanced purine metabolism and promoted increased uric acid.', 28275154);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4562, 'mouse', 'null', 'null', 'null', 'Thus, exposure to S. cerevisiae in the gut led to enhanced purine metabolism and promoted increased uric acid.', 28275154);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4563, 'mouse', 'increase', 'null', 'low-throughput', 'Butyrate promotes the increased expression of IFN-γ and granzyme B in CTLs and Tc17 cells.', 30258117);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4564, 'mouse', 'increase', 'null', 'low-throughput', 'Butyrate promotes the increased expression of IFN-γ and granzyme B in CTLs and Tc17 cells.', 30258117);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4568, 'mouse', 'null', 'Sennoside', 'null', 'We demonstrated that intestinal peristalsis was promoted by rheinanthrone produced by hydrolysis of sennoside by strain LKM512 and LKM10070.', 22384059);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4569, 'mouse', 'null', 'Sennoside', 'null', 'We demonstrated that intestinal peristalsis was promoted by rheinanthrone produced by hydrolysis of sennoside by strain LKM512 and LKM10070.', 22384059);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4575, 'mouse', 'null', 'null', 'null', 'Fecal butyrate levels significantly decreased in the DSS mice as compared to the control mice, but increased significantly in the DSS plus TN-3 mice as compared to the DSS mice.', 27438072);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4576, 'mouse', 'decrease', 'null', 'low-throughput', 'Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.', 25456122);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4577, 'mouse', 'decrease', 'null', 'low-throughput', 'Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.', 25456122);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4578, 'mouse', 'decrease', 'null', 'low-throughput', 'Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.', 25456122);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4579, 'mouse', 'null', 'null', 'null', 'TL133 also increasing the caecal concentration of acetate, propionate and butyrate.', 17349084);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4580, 'mouse', 'null', 'null', 'null', 'TL134 also increasing the caecal concentration of acetate, propionate and butyrate.', 17349084);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4581, 'mouse', 'null', 'null', 'null', 'TL135 also increasing the caecal concentration of acetate, propionate and butyrate.', 17349084);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4582, 'mouse', 'null', 'L-Fucose', 'null', 'Accordingly, the mouse model provided L-fucose, enabling B. thetaiotaomicron to synthesize L-fucose containing polysaccharides.', 23022739);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4583, 'mouse', 'null', 'L-Fucose', 'null', 'Accordingly, the mouse model provided L-fucose, enabling B. thetaiotaomicron to synthesize L-fucose containing polysaccharides.', 23022739);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4584, 'mouse', 'null', 'L-Fucose', 'null', 'Accordingly, the mouse model provided L-fucose, enabling B. thetaiotaomicron to synthesize L-fucose containing polysaccharides.', 23022739);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4585, 'mouse', 'null', 'null', 'null', 'We were able to identify a few metabolites specific to the humanized Salmonella-infected group, including two metabolites in the glutathione pathway. In particular, we identified glutathione disulfide and glutathione cysteine disulfide in higher abundance in humanized Salmonella-infected mice. Salmonella infection triggers vast amounts of oxidative stress, and glutathione metabolism is important for protection against oxidative stress.', 30401779);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4586, 'mouse', 'null', 'null', 'null', 'We were able to identify a few metabolites specific to the humanized Salmonella-infected group, including two metabolites in the glutathione pathway. In particular, we identified glutathione disulfide and glutathione cysteine disulfide in higher abundance in humanized Salmonella-infected mice. Salmonella infection triggers vast amounts of oxidative stress, and glutathione metabolism is important for protection against oxidative stress.', 30401779);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4587, 'mouse', 'increase', 'null', 'low-throughput', 'SCFAs binding to GPR43 on colonic epithelial cells stimulates K þ efflux and hyperpolarization,which lead to NLRP3 inflammasome activation.', 25828455);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4588, 'mouse', 'increase', 'null', 'low-throughput', 'NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.', 28104981);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4589, 'mouse', 'increase', 'null', 'low-throughput', 'NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.', 28104981);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4590, 'mouse', 'increase', 'null', 'low-throughput', 'NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.', 28104981);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4591, 'mouse', 'increase', 'null', 'low-throughput', 'Taken together, these experiments suggest that RYGB restores intestinal NLRP6 inflammasome expression by increasing taurine and decreasing histamine in the gut.', 31602628);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4592, 'mouse', 'decrease', 'null', 'low-throughput', 'Taken together, these experiments suggest that RYGB restores intestinal NLRP6 inflammasome expression by increasing taurine and decreasing histamine in the gut.', 31602628);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4593, 'mouse', 'decrease', 'Lipopolysaccharide', 'low-throughput', 'Equol inhibited the lipopolysaccharide (LPS)-induced TLR4 activation, MAPK activation,NF-kB-mediated transcription of inflammatory mediators, production of nitric oxide (NO), release of prostaglandin E2 (PGE-2), secretion of tumor necrosis factor-α (TNF-α) and interleukin 6 (IL-6),in Lipopolysaccharide (LPS)-activated murine microglia cells.', 28264445);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4594, 'mouse', 'decrease', 'Lipopolysaccharide', 'low-throughput', 'Equol inhibited the lipopolysaccharide (LPS)-induced TLR4 activation, MAPK activation,NF-kB-mediated transcription of inflammatory mediators, production of nitric oxide (NO), release of prostaglandin E2 (PGE-2), secretion of tumor necrosis factor-α (TNF-α) and interleukin 6 (IL-6),in Lipopolysaccharide (LPS)-activated murine microglia cells.', 28264445);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4595, 'mouse', 'decrease', 'Lipopolysaccharide', 'low-throughput', 'Equol inhibited the lipopolysaccharide (LPS)-induced TLR4 activation, MAPK activation,NF-kB-mediated transcription of inflammatory mediators, production of nitric oxide (NO), release of prostaglandin E2 (PGE-2), secretion of tumor necrosis factor-α (TNF-α) and interleukin 6 (IL-6),in Lipopolysaccharide (LPS)-activated murine microglia cells.', 28264445);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4596, 'mouse', 'decrease', 'Lipopolysaccharide', 'low-throughput', 'Equol inhibited the lipopolysaccharide (LPS)-induced TLR4 activation, MAPK activation,NF-kB-mediated transcription of inflammatory mediators, production of nitric oxide (NO), release of prostaglandin E2 (PGE-2), secretion of tumor necrosis factor-α (TNF-α) and interleukin 6 (IL-6),in Lipopolysaccharide (LPS)-activated murine microglia cells.', 28264445);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4597, 'mouse', 'increase', 'null', 'low-throughput', 'The helminth Nippostrongylus brasiliensis and a tritrichomonad protist both secreted succinate as a metabolite. In vivo sensing of the tritrichomonad required SUCNR1, whereas N. brasiliensis was SUCNR1 independent. These findings define a paradigm wherein tuft cells monitor microbial metabolites to initiate type 2 immunity and suggest the existence of other sensing pathways triggering the response to helminths.', 30021144);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4598, 'mouse', 'increase', 'null', 'low-throughput', 'The helminth Nippostrongylus brasiliensis and a tritrichomonad protist both secreted succinate as a metabolite. In vivo sensing of the tritrichomonad required SUCNR1, whereas N. brasiliensis was SUCNR1 independent. These findings define a paradigm wherein tuft cells monitor microbial metabolites to initiate type 2 immunity and suggest the existence of other sensing pathways triggering the response to helminths.', 30021144);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4599, 'mouse', 'increase', 'omega-6 Fatty Acids', 'low-throughput', 'Here, we demonstrated that gut microbiota exerted protective effects against HFD-induced obesity and improved glucose homeostasis associated with several mechanisms involving gut microbial metabolites of dietary PUFAs: (1) gut microbiota converted LA to HYA, reducing the adipose inflammation otherwise induced by excessive dietary LA via the AA cascade; (2) HYA activated GPR40 and GPR120 and promoted GLP-1 secretion; and (3) HYA suppressed lipid absorption by promoting intestinal peristalsis via EP3.', 31488836);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4600, 'mouse', 'increase', 'null', 'low-throughput', 'Our study shows that MAM from F. prausnitzii can restore the intestinal barrier structure and function in DM conditions via the regulation of the tight junction pathway and ZO-1 expression.Collectively, our data reveal that MAM protein upregulates ZO-1 expression in intestinal epithelial cells.', 31503404);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4601, 'mouse', 'null', 'Melamine', 'null', 'We further demonstrated that melamine is converted to cyanuric acid in vitro by bacteria cultured from normal rat feces;Klebsiella was subsequently identified in fecal samples by 16S ribosomal DNA sequencing. In culture, Klebsiella terrigena was shown to convert melamine to cyanuric acid directly.', 23408055);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4602, 'mouse', 'null', 'null', 'null', 'We demonstrate that commensal Bacteroides species limit S. Typhimurium infection in the distalgut. We find that Bacteroides spp. production of the short-chain fatty acid (SCFA) metabolic byproduct propionate directly inhibits S. Typhimurium growth in vitro by disrupting intracellular pH homeostasis and limits distal intestinal expansion and fecal shedding in vivo.', 30057174);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4603, 'mouse', 'null', 'null', 'null', 'We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.', 25784704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4604, 'mouse', 'null', 'null', 'null', 'We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.', 25784704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4605, 'mouse', 'null', 'null', 'null', 'We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.', 25784704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4606, 'mouse', 'null', 'null', 'null', 'We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.', 25784704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4607, 'mouse', 'null', 'null', 'null', 'We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.', 25784704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4608, 'mouse', 'null', 'null', 'null', 'We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.', 25784704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4609, 'mouse', 'null', 'null', 'null', 'We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.', 25784704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4610, 'mouse', 'null', 'null', 'null', 'We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.', 25784704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4611, 'mouse', 'null', 'null', 'null', 'We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.', 25784704);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4612, 'mouse', 'decrease', 'null', 'low-throughput', 'Faecalibacterium prausnitzii produces butyrate to decrease c-Myc-related metabolism and Th17 differentiation by inhibiting histone deacetylase 3.', 30809639);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4613, 'mouse', 'null', 'null', 'null', 'However, PPD  synergistically enhanced APAP induced hepatic injury in vivo and in vitro. Finally, we  found Saccharomyces cerevisiae, which could reduce intestinal PPD levels, was able  to markedly alleviate APAP induced liver damage at ZT12.', 29524531);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4616, 'mouse', 'null', 'null', 'null', 'Accordingly, dietary succinate improved glucose and insulin tolerance in wild-type mice, but those effects were absent in mice deficient in IGN. Conventional mice colonized with the succinate producer Prevotella copri exhibited metabolic benefits, which could be related to succinate-activated IGN. Thus, microbiota-produced succinate is a previously unsuspected bacterial metabolite improving glycemic control through activation of IGN.', 27411015);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4617, 'mouse', 'increase', 'null', 'low-throughput', 'Butyrate can induce AMPK activation and GLUT4 expression in the adipose tissue, improving cardiovascular disease (CVD)-related metabolic disorder, resisting HFD-induced gut microbiome dysbiosis, and promoting resolvin E1 and lipoxin biosynthesis. Oral supplement of the natural fermentation product butyrate can be a potential strategy for preventing CVD.', 31607907);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4618, 'mouse', 'increase', 'null', 'low-throughput', 'Butyrate can induce AMPK activation and GLUT4 expression in the adipose tissue, improving cardiovascular disease (CVD)-related metabolic disorder, resisting HFD-induced gut microbiome dysbiosis, and promoting resolvin E1 and lipoxin biosynthesis. Oral supplement of the natural fermentation product butyrate can be a potential strategy for preventing CVD.', 31607907);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4619, 'mouse', 'decrease', 'null', 'low-throughput', 'Colonic cDNA concentrations of IL-1b and TNF-a were significantly down regulated in DSS-treated E. durans-fed mice but not in control or DSS-treated L.delbrueckii- fed mice.', 17349085);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4620, 'mouse', 'decrease', 'null', 'low-throughput', 'Colonic cDNA concentrations of IL-1b and TNF-a were significantly down regulated in DSS-treated E. durans-fed mice but not in control or DSS-treated L.delbrueckii- fed mice.', 17349085);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4621, 'mouse', 'increase', 'null', 'low-throughput', 'we demonstrate here that α-defensin 5 expression can also be regulated by microbial metabolites. Among these, lactate appeared to significantly suppress α-defensin 5 gene transcription.', 27760493);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4622, 'mouse', 'null', 'null', 'null', 'We report that the human gut bacterium Bacteroides thetaiotaomicron relies on the stringent response, an intracellular signaling pathway that allocates resources away from growth, to survive carbon starvation and persist in the gut. Genome-scale transcriptomics, 13C-labeling, and metabolomics analyses reveal that B. thetaiotaomicron uses the alarmone (p)ppGpp to repress multiple biosynthetic pathways and upregulate tricarboxylic acid (TCA) cycle genes in these conditions.', 30008292);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4623, 'mouse', 'increase', 'null', 'low-throughput', 'Lactobacilli Induce IL-22 via Iald.', 23973224);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4624, 'mouse', 'null', 'Ferulic acid', 'null', 'Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.', 31075850);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4625, 'mouse', 'null', 'Ferulic acid', 'null', 'Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.', 31075850);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4626, 'mouse', 'null', 'Ferulic acid', 'null', 'Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.', 31075850);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4627, 'mouse', 'null', 'Ferulic acid', 'null', 'Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.', 31075850);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4628, 'mouse', 'null', 'Ferulic acid', 'null', 'Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.', 31075850);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4629, 'mouse', 'null', 'Ferulic acid', 'null', 'Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.', 31075850);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4630, 'mouse', 'null', 'Ferulic acid', 'null', 'Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.', 31075850);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4635, 'mouse', 'increase', 'null', 'low-throughput', 'To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.', 33006494);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4636, 'mouse', 'increase', 'null', 'low-throughput', 'To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.', 33006494);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4637, 'mouse', 'increase', 'null', 'low-throughput', 'To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.', 33006494);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4638, 'mouse', 'increase', 'null', 'low-throughput', 'To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.', 33006494);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4639, 'mouse', 'increase', 'null', 'low-throughput', 'Indeed, DCA stimulation could induce potent increase of mRNA expression level of M2-mAchR in macrophages, and selective inhibition of M2-mAchR significantly reduced theproduction of TNF-α, NO and IL-6 triggered by DCA (Figure 4(d,e) and Supplementary Figure 4c).', 33006494);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4640, 'mouse', 'increase', 'null', 'low-throughput', 'Toll-like receptor 2 (TLR2) is up-regulated upon DCA stimulation and involved in DCA-induced macrophage polarization.', 33006494);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4641, 'mouse', 'increase', 'null', 'low-throughput', 'Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4642, 'mouse', 'increase', 'null', 'low-throughput', 'Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4643, 'mouse', 'increase', 'null', 'low-throughput', 'Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4644, 'mouse', 'increase', 'null', 'low-throughput', 'Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4645, 'mouse', 'increase', 'null', 'low-throughput', 'At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4646, 'mouse', 'increase', 'null', 'low-throughput', 'At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4647, 'mouse', 'increase', 'null', 'low-throughput', 'At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4648, 'mouse', 'increase', 'null', 'low-throughput', 'At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4649, 'mouse', 'increase', 'null', 'low-throughput', 'At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.', 32783770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4650, 'mouse', 'increase', 'null', 'low-throughput', 'Furthermore, TGP failed to blunt the increased of colonic IL-1β in colitis mice receiving ILA (Fig. 6E).', 33002829);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4651, 'mouse', 'null', 'Tryptophan', 'null', 'We have shown that breast milk tryptophan and Bifidobacterium infantis result in a metabolite (indole-3-lactic acid) response, which is anti-inflammatory via inhibition of the aryl hydrocarbon receptor transcription factor which stimulates an IL-8 response.', 32229728);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4656, 'mouse', 'increase', 'null', 'low-throughput', 'Duodenal murine enteroids were exposed to 1 and 5 mM SCFAs for 24 h. Expression of stem cell marker leucine-rich repeat-containing G-protein coupled receptor 5 (Lgr5) was increased more than 2-fold in murine enteroids treated with 5 mM ACET (P < 0.05). Conversely, enteroids treated with 5 mM BUT, or PROP had greater than 5-fold decreases in Lgr5 compared to CON (P < 0.05; Fig 2b). There was no difference in Lgr5 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4657, 'mouse', 'decrease', 'null', 'low-throughput', 'Duodenal murine enteroids were exposed to 1 and 5 mM SCFAs for 24 h. Expression of stem cell marker leucine-rich repeat-containing G-protein coupled receptor 5 (Lgr5) was increased more than 2-fold in murine enteroids treated with 5 mM ACET (P < 0.05). Conversely, enteroids treated with 5 mM BUT, or PROP had greater than 5-fold decreases in Lgr5 compared to CON (P < 0.05; Fig 2b). There was no difference in Lgr5 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4658, 'mouse', 'decrease', 'null', 'low-throughput', 'Duodenal murine enteroids were exposed to 1 and 5 mM SCFAs for 24 h. Expression of stem cell marker leucine-rich repeat-containing G-protein coupled receptor 5 (Lgr5) was increased more than 2-fold in murine enteroids treated with 5 mM ACET (P < 0.05). Conversely, enteroids treated with 5 mM BUT, or PROP had greater than 5-fold decreases in Lgr5 compared to CON (P < 0.05; Fig 2b). There was no difference in Lgr5 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4659, 'mouse', 'increase', 'null', 'low-throughput', 'Enterocyte biomarker alkaline phosphatase (Alpi) was significantly increased in response to all SCFA treatments but was highest (3.5-fold;P < 0.05) in enteroids treated with PROP. There was no difference in Alpi expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4660, 'mouse', 'increase', 'null', 'low-throughput', 'Enterocyte biomarker alkaline phosphatase (Alpi) was significantly increased in response to all SCFA treatments but was highest (3.5-fold;P < 0.05) in enteroids treated with PROP. There was no difference in Alpi expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4661, 'mouse', 'increase', 'null', 'low-throughput', 'Enterocyte biomarker alkaline phosphatase (Alpi) was significantly increased in response to all SCFA treatments but was highest (3.5-fold;P < 0.05) in enteroids treated with PROP. There was no difference in Alpi expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4662, 'mouse', 'decrease', 'null', 'low-throughput', 'Gene expression of goblet cell marker mucin 2 (Muc2) was not significantly affected by ACET treatment, however, Muc2 was significantly decreased due to both BUT and PROP treatment (3-fold and 10-fold respectively;P < 0.05). Muc2 expression was not altered in murine enteroids treated with 1 mM SCFAs(P > 0.05; Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4663, 'mouse', 'decrease', 'null', 'low-throughput', 'Gene expression of goblet cell marker mucin 2 (Muc2) was not significantly affected by ACET treatment, however, Muc2 was significantly decreased due to both BUT and PROP treatment (3-fold and 10-fold respectively;P < 0.05). Muc2 expression was not altered in murine enteroids treated with 1 mM SCFAs(P > 0.05; Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4664, 'mouse', 'increase', 'null', 'low-throughput', 'Paneth cell marker lysozyme (Lyz) had a tendency (P = 0.08) to increase due to BUT treatment but there was no effect observed when treated with 1 mM SCFAs (P > 0.05;Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4665, 'mouse', 'increase', 'null', 'low-throughput', 'Enteroendocrine cell marker chromogranin A (Chga) was only significantly increased in BUT treated enteroids at both 1 mM and 5 mM concentrations (P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4666, 'mouse', 'increase', 'null', 'low-throughput', 'Tuft cell marker doublecortin-like kinase 1 (Dclk1) was significantly elevated in response to 5 mM ACET (3-fold) and PROP (2-fold) treatments compared to CON (P < 0.05; Fig 2b). There was no difference in Dclk1 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4667, 'mouse', 'increase', 'null', 'low-throughput', 'Tuft cell marker doublecortin-like kinase 1 (Dclk1) was significantly elevated in response to 5 mM ACET (3-fold) and PROP (2-fold) treatments compared to CON (P < 0.05; Fig 2b). There was no difference in Dclk1 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4668, 'mouse', 'increase', 'null', 'low-throughput', 'Antimicrobial peptide gene expression of c-type lectins regenerating islet-derived protein 3β (Reg3β) and 3γ (Reg3γ; Fig 2b) were increased moderately due to ACET treatment (20-fold and 40-fold), but were much more significantly increased due to BUT (150-fold and 350-fold,respectively) and PROP (250-fold, and 480-fold, respectively, P < 0.05) compared to CON.Only 1 mM BUT increased Reg3β and Reg3γ expression (80-fold and 100-fold respectively;P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4669, 'mouse', 'increase', 'null', 'low-throughput', 'Antimicrobial peptide gene expression of c-type lectins regenerating islet-derived protein 3β (Reg3β) and 3γ (Reg3γ; Fig 2b) were increased moderately due to ACET treatment (20-fold and 40-fold), but were much more significantly increased due to BUT (150-fold and 350-fold,respectively) and PROP (250-fold, and 480-fold, respectively, P < 0.05) compared to CON.Only 1 mM BUT increased Reg3β and Reg3γ expression (80-fold and 100-fold respectively;P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4670, 'mouse', 'increase', 'null', 'low-throughput', 'Antimicrobial peptide, β-defensin 1 (Defb1), was increased 3.5-fold due to ACET and BUT treatment, and 6-fold due to PROP treatment (P < 0.05); Fig 2b). 1 mM PROP was sufficient to increase Defb1 expression 2-fold compared to CON (P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4671, 'mouse', 'increase', 'null', 'low-throughput', 'Antimicrobial peptide, β-defensin 1 (Defb1), was increased 3.5-fold due to ACET and BUT treatment, and 6-fold due to PROP treatment (P < 0.05); Fig 2b). 1 mM PROP was sufficient to increase Defb1 expression 2-fold compared to CON (P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4672, 'mouse', 'increase', 'null', 'low-throughput', 'Antimicrobial peptide, β-defensin 1 (Defb1), was increased 3.5-fold due to ACET and BUT treatment, and 6-fold due to PROP treatment (P < 0.05); Fig 2b). 1 mM PROP was sufficient to increase Defb1 expression 2-fold compared to CON (P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4673, 'mouse', 'increase', 'null', 'low-throughput', 'Tight junction markers occludin (Ocln), and claudin-3 (Cldn3) were increased nearly 2-fold in enteroids treated with 5 mM ACET (P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4674, 'mouse', 'increase', 'null', 'low-throughput', 'Tight junction markers occludin (Ocln), and claudin-3 (Cldn3) were increased nearly 2-fold in enteroids treated with 5 mM ACET (P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4675, 'mouse', 'decrease', 'null', 'low-throughput', 'BUT decreased Cldn3 expression at 5 mM but Ocln was not altered (P > 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4676, 'mouse', 'decrease', 'null', 'low-throughput', '5 mM PROP significantly decreased Cldn3 (P < 0.05).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4677, 'mouse', 'decrease', 'null', 'low-throughput', 'Tight junction protein 1 (Tjp1) was not altered by ACET treatment, but was decreased 1.7 and 2-fold respectively due to BUT and PROP treatment (P < 0.05; Fig 2b). There was no
difference in Tjp1 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4678, 'mouse', 'decrease', 'null', 'low-throughput', 'Tight junction protein 1 (Tjp1) was not altered by ACET treatment, but was decreased 1.7 and 2-fold respectively due to BUT and PROP treatment (P < 0.05; Fig 2b). There was no
difference in Tjp1 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).', 32240190);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4679, 'mouse', 'null', 'null', 'null', 'Ergothioneine, a metabolite of the gut bacterium Lactobacillus reuteri, protects against stress-induced sleep disturbances.', 32467627);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4680, 'mouse', 'decrease', 'null', 'low-throughput', 'Moreover, the expression of Cyp3a11 in GF mice also remained exten sively downregulated after butyrate supplementation rela tive to the corresponding conventional group (P < 0.01;P < 0.001, respectively).', 32337713);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4681, 'mouse', 'increase', 'null', 'low-throughput', 'In the presence of a complex microbiota, butyrate only had a significant effect on the hepatic expression of Cyp2b10 (2.85-fold higher relative to conventional mice; P < 0.05).', 32337713);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4682, 'mouse', 'decrease', 'null', 'low-throughput', 'Butyrate decreased the expression of Mdr1a in GF mice(P < 0.05) relative to the butyrate-treated conventional group, despite no evident changes in this isoform under GF conditions or by colonisation.', 32337713);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4683, 'mouse', 'increase', 'null', 'low-throughput', 'Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).', 33192557);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4684, 'mouse', 'decrease', 'null', 'low-throughput', 'Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).', 33192557);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4685, 'mouse', 'increase', 'null', 'low-throughput', 'Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).', 33192557);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4686, 'mouse', 'increase', 'null', 'low-throughput', 'Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).', 33192557);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4687, 'mouse', 'decrease', 'null', 'low-throughput', 'Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).', 33192557);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4688, 'mouse', 'decrease', 'null', 'low-throughput', 'In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).', 33192557);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4689, 'mouse', 'decrease', 'null', 'low-throughput', 'In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).', 33192557);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4690, 'mouse', 'decrease', 'null', 'low-throughput', 'In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).', 33192557);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4691, 'mouse', 'decrease', 'null', 'low-throughput', 'Relative to the SAP + IAH group, the levels of claudin-2, MMP9, and TNF-α were significantly lower in the two treatment groups (P < 0.05; Figures 5A,B).', 33192557);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4692, 'mouse', 'increase', 'null', 'low-throughput', 'LPS significantly decreased the level of TEER (P < 0.01), while treatment with PA moderated the reduced TEER (P < 0.05). Secondly, we utilized immunoblotting to analyze the protein expression levels of ZO-1, occludin, and claudin-1. As shown in Figures 2B–E, the protein expression levels of ZO-1, occludin, and claudin-1 were lower in the LPS group than in the control group (P < 0.01). This LPS-induced alteration of tight junction protein expression was inhibited by PA (P < 0.05). These results suggest that PA can enhance and protect intestinal barrier function against LPS in IEC-6 cells.', 33041816);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4693, 'mouse', 'increase', 'null', 'low-throughput', 'LPS significantly decreased the level of TEER (P < 0.01), while treatment with PA moderated the reduced TEER (P < 0.05). Secondly, we utilized immunoblotting to analyze the protein expression levels of ZO-1, occludin, and claudin-1. As shown in Figures 2B–E, the protein expression levels of ZO-1, occludin, and claudin-1 were lower in the LPS group than in the control group (P < 0.01). This LPS-induced alteration of tight junction protein expression was inhibited by PA (P < 0.05). These results suggest that PA can enhance and protect intestinal barrier function against LPS in IEC-6 cells.', 33041816);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4694, 'mouse', 'increase', 'null', 'low-throughput', 'LPS significantly decreased the level of TEER (P < 0.01), while treatment with PA moderated the reduced TEER (P < 0.05). Secondly, we utilized immunoblotting to analyze the protein expression levels of ZO-1, occludin, and claudin-1. As shown in Figures 2B–E, the protein expression levels of ZO-1, occludin, and claudin-1 were lower in the LPS group than in the control group (P < 0.01). This LPS-induced alteration of tight junction protein expression was inhibited by PA (P < 0.05). These results suggest that PA can enhance and protect intestinal barrier function against LPS in IEC-6 cells.', 33041816);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4695, 'mouse', 'decrease', 'null', 'low-throughput', 'These data indicate that PA promotes epithelial cell migration via inhibition of expression of b1-integrin induced by LPS.', 33041816);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4696, 'mouse', 'decrease', 'null', 'low-throughput', 'Treatment with PA rescued the obvious increases in protein expression in NLRP3, caspase-1, IL-1b, and IL-18 induced by LPS.', 33041816);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4697, 'mouse', 'decrease', 'null', 'low-throughput', 'Treatment with PA rescued the obvious increases in protein expression in NLRP3, caspase-1, IL-1b, and IL-18 induced by LPS.', 33041816);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4698, 'mouse', 'decrease', 'null', 'low-throughput', 'Treatment with PA rescued the obvious increases in protein expression in NLRP3, caspase-1, IL-1b, and IL-18 induced by LPS.', 33041816);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4699, 'mouse', 'decrease', 'null', 'low-throughput', 'Treatment with PA rescued the obvious increases in protein expression in NLRP3, caspase-1, IL-1b, and IL-18 induced by LPS.', 33041816);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4700, 'mouse', 'decrease', 'null', 'low-throughput', 'The research results show that LPS strongly stimulated TLR4, the phosphorylation of IkBa and p65 in whole cell extracts and IkBa degradation in the cytoplasm. It also caused the translocation of p65 into the nucleus. However, these effects were significantly inhibited by PA (Figures 5A–G).', 33041816);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4701, 'mouse', 'increase', 'null', 'low-throughput', 'SLC5A8 and GPR43 expressions in butyrate‑treated CRC cells were quantified (Fig. 5B and C for SLC5A8 and GPR43, respectively). In comparison with untreated cells, the NaB‑treated SW480 cells only presented higher GPR43 (P<0.05), but the NaB‑treated SW620 cells expressed the most SLC5A8 (P<0.01) and GPR43 (P<0.05).', 33101496);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4704, 'mouse', 'increase', 'Cholic acid', 'low-throughput', 'Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4705, 'mouse', 'increase', 'Cholic acid', 'low-throughput', 'FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4706, 'mouse', 'increase', 'Cholic acid', 'low-throughput', 'In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4707, 'mouse', 'decrease', 'Cholic acid', 'low-throughput', 'In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4708, 'mouse', 'decrease', 'Cholic acid', 'low-throughput', 'Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4709, 'mouse', 'decrease', 'Cholic acid', 'low-throughput', 'Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4710, 'mouse', 'increase', 'Cholic acid', 'low-throughput', 'Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4711, 'mouse', 'increase', 'Cholic acid', 'low-throughput', 'Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4712, 'mouse', 'increase', 'Cholic acid', 'low-throughput', 'FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4713, 'mouse', 'increase', 'Cholic acid', 'low-throughput', 'In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4714, 'mouse', 'decrease', 'Cholic acid', 'low-throughput', 'In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4715, 'mouse', 'decrease', 'Cholic acid', 'low-throughput', 'Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4716, 'mouse', 'decrease', 'Cholic acid', 'low-throughput', 'Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4717, 'mouse', 'increase', 'Cholic acid', 'low-throughput', 'Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4718, 'mouse', 'increase', 'null', 'low-throughput', 'Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4719, 'mouse', 'increase', 'null', 'low-throughput', 'FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4720, 'mouse', 'increase', 'null', 'low-throughput', 'In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4721, 'mouse', 'decrease', 'null', 'low-throughput', 'In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4722, 'mouse', 'decrease', 'null', 'low-throughput', 'Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4723, 'mouse', 'decrease', 'null', 'low-throughput', 'Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4724, 'mouse', 'increase', 'null', 'low-throughput', 'Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.', 27111464);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4725, 'mouse', 'null', 'Indole', 'null', 'Production of IPA was shown to be completely dependent on the presence of gut microflora and could be established by colonization with the bacterium Clostridium sporogenes.', 19234110);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4726, 'mouse', 'null', 'Rutin', 'null', 'LY1 also metabolized rutin to quercetin and 3,4-DPA, although the amount of 3,4-DPA formed was minute compared to that of quercetin.', 14629841);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4727, 'mouse', 'null', 'Rutin', 'null', 'LY1 also metabolized rutin to quercetin and 3,4-DPA, although the amount of 3,4-DPA formed was minute compared to that of quercetin.', 14629841);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4728, 'mouse', 'null', 'Quercitrin', 'null', 'A significant portion of flavonoids in EGb 761 was converted to the main aglycone forms (quercetin and kaempferol) (Fig. 4B) after 36 hours.', 14629841);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4729, 'mouse', 'null', 'Quercitrin', 'null', 'A significant portion of flavonoids in EGb 761 was converted to the main aglycone forms (quercetin and kaempferol) (Fig. 4B) after 36 hours.', 14629841);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4730, 'mouse', 'null', 'Quercitrin', 'null', 'When enrichment cultures derived from either rat fecal materials was used, glycosides in EGb 761 were completely consumed but quercetin, kaempferol, and the possible isorhamnetin were only transiently detected during the growth of the bacteria in enrichment cultures.', 14629841);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4731, 'mouse', 'null', 'Isoquercitrin', 'null', 'A significant portion of flavonoids in EGb 761 was converted to the main aglycone forms (quercetin and kaempferol) (Fig. 4B) after 36 hours.', 14629841);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4732, 'mouse', 'null', 'Isoquercitrin', 'null', 'A significant portion of flavonoids in EGb 761 was converted to the main aglycone forms (quercetin and kaempferol) (Fig. 4B) after 36 hours.', 14629841);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4733, 'mouse', 'null', 'Isoquercitrin', 'null', 'When enrichment cultures derived from either rat fecal materials was used, glycosides in EGb 761 were completely consumed but quercetin, kaempferol, and the possible isorhamnetin were only transiently detected during the growth of the bacteria in enrichment cultures.', 14629841);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4734, 'mouse', 'null', 'null', 'null', 'For example, quinic acid, a cyclic polyol obtained from plant-derived products, reached its highest levels when Odoribacter splanchnicus was present but Escherichia coli was not.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4735, 'mouse', 'null', 'null', 'null', 'One such example is our observation that concentrations of a lysophosphatidylethanolamine (LysoPE15) can be increased by either B. ovatus or B. vulgatus, but reaches maximal cecal concentrations when both strains were present in a community (Fig. 3B).', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4736, 'mouse', 'null', 'null', 'null', 'One such example is our observation that concentrations of a lysophosphatidylethanolamine (LysoPE15) can be increased by either B. ovatus or B. vulgatus, but reaches maximal cecal concentrations when both strains were present in a community (Fig. 3B).', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4737, 'mouse', 'null', 'null', 'null', 'While another metabolite (assignment, 4�?hydroxy-3�?5,6,7,8-pentamethoxyflavone) required both B. ovatus and O.splanchnicus to consistently reach high levels.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4738, 'mouse', 'null', 'null', 'null', 'While another metabolite (assignment, 4�?hydroxy-3�?5,6,7,8-pentamethoxyflavone) required both B. ovatus and O.splanchnicus to consistently reach high levels.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4747, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4748, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4749, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4750, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4751, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4752, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4753, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4754, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4755, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4756, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4757, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4758, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4759, 'mouse', 'null', 'null', 'null', 'Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.', 24452263);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4760, 'mouse', 'increase', 'Lactic acid', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4761, 'mouse', 'increase', 'Lactic acid', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4762, 'mouse', 'increase', 'Lactic acid', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4763, 'mouse', 'decrease', 'Lactic acid', 'low-throughput', 'Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4764, 'mouse', 'decrease', 'Lactic acid', 'low-throughput', 'Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4765, 'mouse', 'decrease', 'Lactic acid', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4766, 'mouse', 'decrease', 'Lactic acid', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4767, 'mouse', 'decrease', 'Lactic acid', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4768, 'mouse', 'increase', 'Pyruvate', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4769, 'mouse', 'increase', 'Pyruvate', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4770, 'mouse', 'increase', 'Pyruvate', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4771, 'mouse', 'increase', 'null', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4772, 'mouse', 'increase', 'null', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4773, 'mouse', 'increase', 'null', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4774, 'mouse', 'increase', 'null', 'low-throughput', 'GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4775, 'mouse', 'increase', 'null', 'low-throughput', 'Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4776, 'mouse', 'decrease', 'null', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4777, 'mouse', 'decrease', 'null', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4778, 'mouse', 'decrease', 'null', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4779, 'mouse', 'increase', 'null', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4780, 'mouse', 'increase', 'null', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4781, 'mouse', 'increase', 'null', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4782, 'mouse', 'increase', 'null', 'low-throughput', 'GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4783, 'mouse', 'increase', 'null', 'low-throughput', 'Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4784, 'mouse', 'decrease', 'null', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4785, 'mouse', 'decrease', 'null', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4786, 'mouse', 'decrease', 'null', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4787, 'mouse', 'increase', 'null', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4788, 'mouse', 'increase', 'null', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4789, 'mouse', 'increase', 'null', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4790, 'mouse', 'increase', 'null', 'low-throughput', 'GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4791, 'mouse', 'increase', 'null', 'low-throughput', 'Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4792, 'mouse', 'decrease', 'null', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4793, 'mouse', 'decrease', 'null', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4794, 'mouse', 'decrease', 'null', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4795, 'mouse', 'increase', 'Starch', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4796, 'mouse', 'increase', 'Starch', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4797, 'mouse', 'increase', 'Starch', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4798, 'mouse', 'increase', 'Starch', 'low-throughput', 'GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4799, 'mouse', 'increase', 'Starch', 'low-throughput', 'Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4800, 'mouse', 'decrease', 'Starch', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4801, 'mouse', 'decrease', 'Starch', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4802, 'mouse', 'decrease', 'Starch', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4803, 'mouse', 'increase', 'Succinate', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4804, 'mouse', 'increase', 'Succinate', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4805, 'mouse', 'increase', 'Succinate', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4806, 'mouse', 'decrease', 'Succinate', 'low-throughput', 'Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4807, 'mouse', 'decrease', 'Succinate', 'low-throughput', 'Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4808, 'mouse', 'decrease', 'Succinate', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4809, 'mouse', 'decrease', 'Succinate', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4810, 'mouse', 'decrease', 'Succinate', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 27448578);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4811, 'mouse', 'increase', 'Tryptophan', 'low-throughput', 'Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.', 29454749);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4812, 'mouse', 'increase', 'Tryptophan', 'low-throughput', 'Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.', 29454749);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4813, 'mouse', 'increase', 'Tryptophan', 'low-throughput', 'IAld has recently been identified as an aryl hydrocarbon receptor (AHR) ligand.', 29454749);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4814, 'mouse', 'increase', 'Tryptophan', 'low-throughput', 'Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.', 29454749);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4815, 'mouse', 'increase', 'Tryptophan', 'low-throughput', 'Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.', 29454749);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4816, 'mouse', 'increase', 'Tryptophan', 'low-throughput', 'IAld has recently been identified as an aryl hydrocarbon receptor (AHR) ligand.', 29454749);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4817, 'mouse', 'increase', 'null', 'low-throughput', 'In mice, nicotinic acid inhibits lipolysis via PUMA-G, a Gi/o-coupled seven-transmembrane receptor expressed in adipocytes and activated macrophages.', 15929991);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4818, 'mouse', 'increase', 'null', 'low-throughput', 'Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.', 15929991);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4819, 'mouse', 'increase', 'null', 'low-throughput', 'Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.', 15929991);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4820, 'mouse', 'increase', 'null', 'low-throughput', 'Both (D)- and (L)-OHB, but not acetoacetate or free acetone, elicited a Ca2 response in cells expressing HM74a but not HM74.', 15929991);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4821, 'mouse', 'increase', 'null', 'low-throughput', 'Racemic (DL)-OHB also showed some degree of activity on all receptors; however, it was more potent on HM74a and its ortholog PUMA-G  than on HM74.', 15929991);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4825, 'mouse', 'increase', 'Tryptophan', 'high-throughput', 'Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4826, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4827, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4828, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4829, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4830, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4831, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4832, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4833, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4834, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4835, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4836, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4837, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4838, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4839, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4840, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4841, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4842, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4843, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4844, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4845, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4846, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4847, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4848, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4849, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4850, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4851, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4852, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4853, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4854, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4855, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4856, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4857, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4858, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4859, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4860, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4861, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4862, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4863, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4864, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4865, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4866, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4867, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4868, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4869, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4870, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4871, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4872, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4873, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4874, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4875, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4876, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4877, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4878, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4879, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4880, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4881, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4882, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4883, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4884, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4885, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4886, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4887, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4888, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4889, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4890, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4891, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4892, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4893, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4894, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4895, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4896, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4897, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4898, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4899, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4900, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4901, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4902, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4903, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4904, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4905, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4906, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4907, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4908, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4909, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4910, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4911, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4912, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4913, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4914, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4915, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4916, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4917, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4918, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4919, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4920, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4921, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4922, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4923, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4924, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4925, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4926, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4927, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4928, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4929, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4930, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4931, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4932, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4933, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4934, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4935, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4936, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4937, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4938, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4939, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4940, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4941, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4942, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4943, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4944, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4945, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4946, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4947, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4948, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4949, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4950, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4951, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4952, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4953, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4954, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4955, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4956, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4957, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4958, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4959, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4960, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4961, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4962, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4963, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4964, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4965, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4966, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4967, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4968, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4969, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4970, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4971, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4972, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4973, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4974, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4975, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4976, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4977, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4978, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4979, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4980, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4981, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4982, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4983, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4984, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4985, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4986, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4987, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4988, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4989, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4990, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4991, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4992, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4993, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4994, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4995, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4996, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4997, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4998, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (4999, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5000, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5001, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5002, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5003, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5004, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5005, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5006, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5007, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5008, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5009, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5010, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5011, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5012, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5013, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5014, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5015, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5016, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5017, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5018, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5019, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5020, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5021, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5022, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5023, 'mouse', 'decrease', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5024, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5025, 'mouse', 'increase', 'null', 'high-throughput', 'Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.', 31123037);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5027, 'mouse', 'increase', 'D-Fructose', 'low-throughput', 'A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice', 31255176);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5028, 'mouse', 'increase', 'D-Fructose', 'low-throughput', 'A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice', 31255176);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5029, 'mouse', 'increase', 'D-Fructose', 'low-throughput', 'A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice', 31255176);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5030, 'mouse', 'decrease', 'D-Fructose', 'low-throughput', 'A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice', 31255176);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5031, 'mouse', 'decrease', '(-)-Epigallocatechin gallate', 'low-throughput', 'Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.', 26323573);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5032, 'mouse', 'decrease', '(-)-Epigallocatechin gallate', 'low-throughput', 'Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.', 26323573);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5033, 'mouse', 'decrease', '(-)-Epigallocatechin gallate', 'low-throughput', 'Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.', 26323573);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5034, 'mouse', 'decrease', '(-)-Epigallocatechin gallate', 'low-throughput', 'Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.', 26323573);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5035, 'mouse', 'decrease', '(-)-Epigallocatechin gallate', 'low-throughput', 'Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.', 26323573);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5036, 'mouse', 'decrease', '(-)-Epigallocatechin gallate', 'low-throughput', 'Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.', 26323573);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5037, 'mouse', 'null', 'null', 'null', 'Firstly, studies have shown that both Treponema.spp and glycerophospholipids are  associated with brain development and diseases. Iron metabolism might provide  substantial contribution to the cross talk of Treponema.spp and glycerophospholipids .  Secondly, in this report, the Facklamia genus was negatively correlated with 4 saturated  and 15 unsaturated fatty acids. Among them, linoleic acid , eicosapentaenoic acid, arachidonic acid  and docosahexaenoic acid  are  well-known dietary nutrients for neural development and brain functions especially in infants and the elderly. Facklamia.spp was associated with invasive diseases although  the pathogenic potential is still unclear. Thirdly, the close correlation of Ruminococcus  gnavus (a bile salt hydrolase (BSH) bacterium) and ursodeoxycholic acid (UDCA), deoxycholic acid (DCA) and chenodeoxycholic acid (CDCA) were in line with the  consensus that BSH bacteria participant directly in the connection between brain and gut and that modulation of BSH activity may be of benefit to human health, Additionally, the links of UDCA, DCA, CDCA and Ruminococcus genuswith  typical intestinal, brain and neuropathic diseases were demonstrated separately by numerous reports which were circumstantial evidence of these association pairs. Fourthly,  oral administration of glutamic acid (the dominant neurotransmitter in brain) , GABA  and glycinewere proposed as new therapeutic strategies for  the regulation of both neurobehaviors and gastrointestinal tract health, although further  studies are needed to define their possible roles in the gut-brain axis. We here observed that these neurotransmitters were positively correlated with Haliscomenobacter.spp,  Pedobacter.spp and Dyadobacter.spp, suggesting new mechanisms underlying healthy  diet and antibiotic interventions.', 29353471);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5038, 'mouse', 'null', 'null', 'null', 'Firstly, studies have shown that both Treponema.spp and glycerophospholipids are  associated with brain development and diseases. Iron metabolism might provide  substantial contribution to the cross talk of Treponema.spp and glycerophospholipids .  Secondly, in this report, the Facklamia genus was negatively correlated with 4 saturated  and 15 unsaturated fatty acids. Among them, linoleic acid , eicosapentaenoic acid, arachidonic acid  and docosahexaenoic acid  are  well-known dietary nutrients for neural development and brain functions especially in infants and the elderly. Facklamia.spp was associated with invasive diseases although  the pathogenic potential is still unclear. Thirdly, the close correlation of Ruminococcus  gnavus (a bile salt hydrolase (BSH) bacterium) and ursodeoxycholic acid (UDCA), deoxycholic acid (DCA) and chenodeoxycholic acid (CDCA) were in line with the  consensus that BSH bacteria participant directly in the connection between brain and gut and that modulation of BSH activity may be of benefit to human health, Additionally, the links of UDCA, DCA, CDCA and Ruminococcus genuswith  typical intestinal, brain and neuropathic diseases were demonstrated separately by numerous reports which were circumstantial evidence of these association pairs. Fourthly,  oral administration of glutamic acid (the dominant neurotransmitter in brain) , GABA  and glycinewere proposed as new therapeutic strategies for  the regulation of both neurobehaviors and gastrointestinal tract health, although further  studies are needed to define their possible roles in the gut-brain axis. We here observed that these neurotransmitters were positively correlated with Haliscomenobacter.spp,  Pedobacter.spp and Dyadobacter.spp, suggesting new mechanisms underlying healthy  diet and antibiotic interventions.', 29353471);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5039, 'mouse', 'null', 'null', 'null', 'Firstly, studies have shown that both Treponema.spp and glycerophospholipids are  associated with brain development and diseases. Iron metabolism might provide  substantial contribution to the cross talk of Treponema.spp and glycerophospholipids .  Secondly, in this report, the Facklamia genus was negatively correlated with 4 saturated  and 15 unsaturated fatty acids. Among them, linoleic acid , eicosapentaenoic acid, arachidonic acid  and docosahexaenoic acid  are  well-known dietary nutrients for neural development and brain functions especially in infants and the elderly. Facklamia.spp was associated with invasive diseases although  the pathogenic potential is still unclear. Thirdly, the close correlation of Ruminococcus  gnavus (a bile salt hydrolase (BSH) bacterium) and ursodeoxycholic acid (UDCA), deoxycholic acid (DCA) and chenodeoxycholic acid (CDCA) were in line with the  consensus that BSH bacteria participant directly in the connection between brain and gut and that modulation of BSH activity may be of benefit to human health, Additionally, the links of UDCA, DCA, CDCA and Ruminococcus genuswith  typical intestinal, brain and neuropathic diseases were demonstrated separately by numerous reports which were circumstantial evidence of these association pairs. Fourthly,  oral administration of glutamic acid (the dominant neurotransmitter in brain) , GABA  and glycinewere proposed as new therapeutic strategies for  the regulation of both neurobehaviors and gastrointestinal tract health, although further  studies are needed to define their possible roles in the gut-brain axis. We here observed that these neurotransmitters were positively correlated with Haliscomenobacter.spp,  Pedobacter.spp and Dyadobacter.spp, suggesting new mechanisms underlying healthy  diet and antibiotic interventions.', 29353471);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5047, 'mouse', 'null', 'Vorinostat', 'null', 'Metabolite profiles of P. mallochii CCH01 were obviously changed by SAHA treatment. Four metabolites (1�?), including two new natural sclerotioramine derivatives, isochromophilone XIV (1) and isochromophilone XV (2), and two known compounds, sclerotioramine (3) and (+)- sclerotiorin (4), were isolated and purified from P. mallochii CCH01 treated with SAHA.', 31632360);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5048, 'mouse', 'null', 'Vorinostat', 'null', 'Metabolite profiles of P. mallochii CCH01 were obviously changed by SAHA treatment. Four metabolites (1�?), including two new natural sclerotioramine derivatives, isochromophilone XIV (1) and isochromophilone XV (2), and two known compounds, sclerotioramine (3) and (+)- sclerotiorin (4), were isolated and purified from P. mallochii CCH01 treated with SAHA.', 31632360);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5053, 'mouse', 'null', 'Isoxanthohumol', 'null', 'Recently, it was shown that the exposure to the potent hop phytoestrogen 8-prenylnaringenin (8-PN) depends on intestinal bacterial activation of isoxanthohumol (IX), but this occurs in only one-third of tested individuals. As the butyrate-producing Eubacterium limosum can produce 8-PN from IX, a probiotic strategy was applied to investigate whether 8-PN production could be increased in low 8-PN producers, thus balancing phytoestrogen exposure.', 18567753);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5054, 'mouse', 'decrease', 'P-Cresol', 'low-throughput', 'In the present study, we investigated the immunological effects of pCS derived from intestinal bacteria in mice. For this purpose, we fed mice a tyrosine-rich diet that causes the accumulation of pCS in their blood. The mice were shown to exhibit decreased Th1-driven 2, 4-dinitrofluorobenzene-induced contact hypersensitivity response. The concentration of pCS in blood was negatively correlated with the degree of the contact hypersensitivity response. In contrast, the T cell-dependent antibody response was not influenced by the accumulated pCS. We also examined the in vitro cytokine responses by T cells in the presence of pCS. The production of IFN-γ was suppressed by pCS. Further, pCS decreased the percentage of IFN-γ-producing Th1 cells. Our results suggest that intestinal bacteria-derived pCS suppressesTh1-type cellular immune responses', 24161588);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5055, 'mouse', 'increase', 'P-Cresol', 'low-throughput', 'In the present study, we investigated the immunological effects of pCS derived from intestinal bacteria in mice. For this purpose, we fed mice a tyrosine-rich diet that causes the accumulation of pCS in their blood. The mice were shown to exhibit decreased Th1-driven 2, 4-dinitrofluorobenzene-induced contact hypersensitivity response. The concentration of pCS in blood was negatively correlated with the degree of the contact hypersensitivity response. In contrast, the T cell-dependent antibody response was not influenced by the accumulated pCS. We also examined the in vitro cytokine responses by T cells in the presence of pCS. The production of IFN-γ was suppressed by pCS. Further, pCS decreased the percentage of IFN-γ-producing Th1 cells. Our results suggest that intestinal bacteria-derived pCS suppressesTh1-type cellular immune responses', 24161588);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5056, 'mouse', 'null', 'Pectin', 'null', 'The purpose of this study was to elucidate the effect of pectin on the polyamine production by defined bacterial species in vivo. Germfree male Wistar rats (n = 18) were randomly assigned to one of three treatments: (i) monoassociation with Bacteroides thetaiotaomicron + fiber-free diet; (ii) diassociation with B. thetaiotaomicron + Fusobacterium varium + fiber-free diet or (iii) diassociation with B. thetaiotaomicron + F. varium + fiber-free diet + 10% pectin. The cecal contents of monoassociated rats fed fiber-free diet contained large amounts (1.51+/-0.21 micromol/dry total cecum content) of spermidine which was the major polyamine. The cecum of diassociated rats fed the fiber-free diet contained even higher concentrations of spermidine (2.53+/-0.21 micromol/dry total cecum content) and also putrescine, which was now the dominant polyamine (putrescine 0.32+/-0.28 vs. 3.01+/-0.28 micromol/dry total cecum content; monoassociation vs. diassociation).', 10801923);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5057, 'mouse', 'null', 'Pectin', 'null', 'The purpose of this study was to elucidate the effect of pectin on the polyamine production by defined bacterial species in vivo. Germfree male Wistar rats (n = 18) were randomly assigned to one of three treatments: (i) monoassociation with Bacteroides thetaiotaomicron + fiber-free diet; (ii) diassociation with B. thetaiotaomicron + Fusobacterium varium + fiber-free diet or (iii) diassociation with B. thetaiotaomicron + F. varium + fiber-free diet + 10% pectin. The cecal contents of monoassociated rats fed fiber-free diet contained large amounts (1.51+/-0.21 micromol/dry total cecum content) of spermidine which was the major polyamine. The cecum of diassociated rats fed the fiber-free diet contained even higher concentrations of spermidine (2.53+/-0.21 micromol/dry total cecum content) and also putrescine, which was now the dominant polyamine (putrescine 0.32+/-0.28 vs. 3.01+/-0.28 micromol/dry total cecum content; monoassociation vs. diassociation).', 10801923);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5058, 'mouse', 'null', 'Pectin', 'null', 'The purpose of this study was to elucidate the effect of pectin on the polyamine production by defined bacterial species in vivo. Germfree male Wistar rats (n = 18) were randomly assigned to one of three treatments: (i) monoassociation with Bacteroides thetaiotaomicron + fiber-free diet; (ii) diassociation with B. thetaiotaomicron + Fusobacterium varium + fiber-free diet or (iii) diassociation with B. thetaiotaomicron + F. varium + fiber-free diet + 10% pectin. The cecal contents of monoassociated rats fed fiber-free diet contained large amounts (1.51+/-0.21 micromol/dry total cecum content) of spermidine which was the major polyamine. The cecum of diassociated rats fed the fiber-free diet contained even higher concentrations of spermidine (2.53+/-0.21 micromol/dry total cecum content) and also putrescine, which was now the dominant polyamine (putrescine 0.32+/-0.28 vs. 3.01+/-0.28 micromol/dry total cecum content; monoassociation vs. diassociation).', 10801923);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5059, 'mouse', 'null', 'Pectin', 'null', 'The purpose of this study was to elucidate the effect of pectin on the polyamine production by defined bacterial species in vivo. Germfree male Wistar rats (n = 18) were randomly assigned to one of three treatments: (i) monoassociation with Bacteroides thetaiotaomicron + fiber-free diet; (ii) diassociation with B. thetaiotaomicron + Fusobacterium varium + fiber-free diet or (iii) diassociation with B. thetaiotaomicron + F. varium + fiber-free diet + 10% pectin. The cecal contents of monoassociated rats fed fiber-free diet contained large amounts (1.51+/-0.21 micromol/dry total cecum content) of spermidine which was the major polyamine. The cecum of diassociated rats fed the fiber-free diet contained even higher concentrations of spermidine (2.53+/-0.21 micromol/dry total cecum content) and also putrescine, which was now the dominant polyamine (putrescine 0.32+/-0.28 vs. 3.01+/-0.28 micromol/dry total cecum content; monoassociation vs. diassociation).', 10801923);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5088, 'mouse', 'increase', 'null', 'low-throughput', 'Transverse aortic constriction (TAC) was performed to induce cardiac hypertrophy in Sprague-Dawley (SD) rats. We observed that TMAO levels were significantly elevated in SD rats after 6 weeks of TAC, suggesting the potential role of TMAO in regulating cardiac hypertrophy. In cultured cardiomyocytes, TMAO treatment stimulated cardiac hypertrophy, as indicated by increased cell area of cardiomyocytes and expression of hypertrophic markers including atrial natriuretic peptide (ANP) and beta-myosin heavy chain (β-MHC). Additionally, TMAO treatment induced cardiac hypertrophy and cardiac fibrosis in SD rats. Reducing TMAO synthesis by antibiotics (Abs) attenuated TAC-induced cardiac hypertrophy and fibrosis. Furthermore, pharmacological inhibition of Smad3 by SIS3 significantly reduced the expression of ANP and β-MHC, and cardiomyocyte cell size in TMAO-treated group. These data for the first time demonstrate that gut microbe-derived metabolite TMAO induces cardiac hypertrophy and fibrosis involving Smad3 signaling, suggesting that inhibition of gut microbes or generation of TMAO may become a potential target for the prevention and treatment of cardiac hypertrophy.', 30068915);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5089, 'mouse', 'increase', 'null', 'low-throughput', 'Transverse aortic constriction (TAC) was performed to induce cardiac hypertrophy in Sprague-Dawley (SD) rats. We observed that TMAO levels were significantly elevated in SD rats after 6 weeks of TAC, suggesting the potential role of TMAO in regulating cardiac hypertrophy. In cultured cardiomyocytes, TMAO treatment stimulated cardiac hypertrophy, as indicated by increased cell area of cardiomyocytes and expression of hypertrophic markers including atrial natriuretic peptide (ANP) and beta-myosin heavy chain (β-MHC). Additionally, TMAO treatment induced cardiac hypertrophy and cardiac fibrosis in SD rats. Reducing TMAO synthesis by antibiotics (Abs) attenuated TAC-induced cardiac hypertrophy and fibrosis. Furthermore, pharmacological inhibition of Smad3 by SIS3 significantly reduced the expression of ANP and β-MHC, and cardiomyocyte cell size in TMAO-treated group. These data for the first time demonstrate that gut microbe-derived metabolite TMAO induces cardiac hypertrophy and fibrosis involving Smad3 signaling, suggesting that inhibition of gut microbes or generation of TMAO may become a potential target for the prevention and treatment of cardiac hypertrophy.', 30068915);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5090, 'mouse', 'increase', 'Linoelaidic acid', 'low-throughput', 'Therefore, in this study, we investigated the effects of KetoA [10-oxo-12(Z)-octadecenoic acid]—a linoleic acid metabolite produced by gut lactic acid bacteria—on whole-body energy metabolism and found that dietary intake of KetoA could enhance energy expenditure in mice, thereby protecting mice from diet-induced obesity.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5091, 'mouse', 'increase', 'Linoelaidic acid', 'low-throughput', 'Ucp1 expression levels were also observed to have increased after administration of KetoA.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5092, 'mouse', 'increase', 'Linoelaidic acid', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT
mice.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5093, 'mouse', 'increase', 'Linoelaidic acid', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT
mice.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5094, 'mouse', 'increase', 'Linoelaidic acid', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT
mice.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5095, 'mouse', 'increase', 'Linoelaidic acid', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT
mice.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5096, 'mouse', 'increase', 'Linoelaidic acid', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT
mice.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5097, 'mouse', 'increase', '(Z)-10-Hydroxy-12-octadecenoic acid', 'low-throughput', 'Therefore, in this study, we investigated the effects of KetoA [10-oxo-12(Z)-octadecenoic acid]—a linoleic acid metabolite produced by gut lactic acid bacteria—on whole-body energy metabolism and found that dietary intake of KetoA could enhance energy expenditure in mice, thereby protecting mice from diet-induced obesity.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5098, 'mouse', 'increase', '(Z)-10-Hydroxy-12-octadecenoic acid', 'low-throughput', 'Ucp1 expression levels were also observed to have increased after administration of KetoA.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5099, 'mouse', 'increase', '(Z)-10-Hydroxy-12-octadecenoic acid', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5100, 'mouse', 'increase', '(Z)-10-Hydroxy-12-octadecenoic acid', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5101, 'mouse', 'increase', '(Z)-10-Hydroxy-12-octadecenoic acid', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5102, 'mouse', 'increase', '(Z)-10-Hydroxy-12-octadecenoic acid', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5103, 'mouse', 'increase', '(Z)-10-Hydroxy-12-octadecenoic acid', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.', 28754711);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5104, 'mouse', 'increase', 'null', 'high-throughput', 'It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5105, 'mouse', 'decrease', 'null', 'high-throughput', 'Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5106, 'mouse', 'increase', 'null', 'high-throughput', 'It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5107, 'mouse', 'increase', 'null', 'high-throughput', 'It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5108, 'mouse', 'decrease', 'null', 'high-throughput', 'Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5109, 'mouse', 'increase', 'null', 'high-throughput', 'Acetate had no significant effect on the expression of Fiaf, Gpr43, Hdac5, or Ppar, while the expression of Hdac3 was significantly increased by acetate.', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5110, 'mouse', 'increase', 'null', 'high-throughput', 'It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5111, 'mouse', 'decrease', 'null', 'high-throughput', 'Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5112, 'mouse', 'increase', 'null', 'high-throughput', 'It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5113, 'mouse', 'increase', 'null', 'high-throughput', 'It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5114, 'mouse', 'decrease', 'null', 'high-throughput', 'Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5115, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5116, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5117, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5118, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5119, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5120, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5121, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5122, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5123, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5124, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5125, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5126, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5127, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5128, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5129, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5130, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5131, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5132, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5133, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5134, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5135, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5136, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5137, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5138, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5139, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5140, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5141, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5142, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5143, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5144, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5145, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5146, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5147, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5148, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5149, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5150, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5151, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5152, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5153, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5154, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5155, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5156, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5157, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5158, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5159, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5160, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5161, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5162, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5163, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5164, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5165, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5166, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5167, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5168, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5169, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5170, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5171, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5172, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5173, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5174, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5175, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5203, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5204, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5205, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5206, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5207, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5208, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5209, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5210, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5211, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5212, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5213, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5214, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5215, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5216, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5217, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5218, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5219, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5220, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5221, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5222, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5223, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5224, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5225, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5226, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5227, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5228, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5229, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5230, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5231, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5232, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5233, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5234, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5235, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5236, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5237, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5238, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5239, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5240, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5241, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5242, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5243, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5244, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5245, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5246, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5247, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5248, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5249, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5250, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5252, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5253, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5254, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5255, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5256, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5257, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5258, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5260, 'mouse', 'increase', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5262, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5264, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5265, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5266, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5267, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5268, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5269, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5270, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5272, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5273, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5274, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5275, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5276, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5277, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5278, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5280, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5281, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5282, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5283, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5284, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5285, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5286, 'mouse', 'decrease', 'null', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 25118238);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5304, 'mouse', 'null', 'null', 'null', 'The results clearly showed that the high dose of PIPA could improve the intestinal movement and alleviate constipation symptoms in AD induced constipation mice.', 30846982);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5305, 'mouse', 'null', 'null', 'null', 'The results showed that the AD groups had lower levels of ACh and 5-HT in colonic tissue and that the H-PIPA dose induced an increase in both colonic ACh and 5-HT but without significant differences from those in the AD group.', 30846982);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5306, 'mouse', 'null', 'null', 'null', 'The results showed that the AD groups had lower levels of ACh and 5-HT in colonic tissue and that the H-PIPA dose induced an increase in both colonic ACh and 5-HT but without significant differences from those in the AD group.', 30846982);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5307, 'mouse', 'increase', 'null', 'low-throughput', 'Butyrate is a short-chain fatty acid derived from the metabolism of indigestible carbohydrates by the gut microbiota. Butyrate contributes to gut homeostasis, but it may also control inflammatory responses and host physiology in other tissues. Butyrate inhibits histone deacetylases, thereby affecting gene transcription, and also signals through the metabolite-sensing G protein receptor (GPR)109a.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5308, 'mouse', 'increase', 'null', 'low-throughput', 'The protective effect of butyrate was further confirmed in situ by enhanced expression of WT-1 (Fig. 1D),podocin, and podoplanin (Supplemental Fig. S3). In ad_x0002_dition, analysis of glomeruli ultrastructure revealed a well-preserved podocytelayer surrounding the GBM with presence of slit diaphragm and reduced FP effacement in butyrate-treated mice at early and later time points (Fig.1E). Over 60 d, all butyrate-treated mice survived ADRinduced nephropathy, compared with only about 20% of nontreated mice (Fig. 1F).', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5309, 'mouse', 'decrease', 'null', 'low-throughput', 'We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5310, 'mouse', 'decrease', 'null', 'low-throughput', 'We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5311, 'mouse', 'decrease', 'null', 'low-throughput', 'We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5312, 'mouse', 'decrease', 'null', 'low-throughput', 'We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5313, 'mouse', 'decrease', 'null', 'low-throughput', 'We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5314, 'mouse', 'decrease', 'null', 'low-throughput', 'Renal inflammation is a common finding in ADR nephropathy. Injured resident kidney cells release chemokines that promote the recruitment ofmacrophages, which play a pivotal role as disease progresses. CD68 staining in renal cortex showed, at early stages of the disease, a slightly positive staining in nontreated mice, whereby we identified several foci of infiltrating CD68-positive cells at d 14 surrounding the glomeruli (Fig. 4A). Butyrate treatment significantly reduced the infiltration of CD68-positive cells, as well as gene expression of chemokines such as ccl2 and ccl3 and important proinflammatory cytokines such as Tnf-a and Il-1b in renal tissue (Fig. 4B). Protein levels for the proinflammatory cytokines in renal tissue were also significantly reduced by butyrate treatment (Fig. 4C).', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5315, 'mouse', 'decrease', 'null', 'low-throughput', 'Renal inflammation is a common finding in ADR nephropathy. Injured resident kidney cells release chemokines that promote the recruitment ofmacrophages, which play a pivotal role as disease progresses. CD68 staining in renal cortex showed, at early stages of the disease, a slightly positive staining in nontreated mice, whereby we identified several foci of infiltrating CD68-positive cells at d 14 surrounding the glomeruli (Fig. 4A). Butyrate treatment significantly reduced the infiltration of CD68-positive cells, as well as gene expression of chemokines such as ccl2 and ccl3 and important proinflammatory cytokines such as Tnf-a and Il-1b in renal tissue (Fig. 4B). Protein levels for the proinflammatory cytokines in renal tissue were also significantly reduced by butyrate treatment (Fig. 4C).', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5316, 'mouse', 'decrease', 'null', 'low-throughput', 'Renal inflammation is a common finding in ADR nephropathy. Injured resident kidney cells release chemokines that promote the recruitment ofmacrophages, which play a pivotal role as disease progresses. CD68 staining in renal cortex showed, at early stages of the disease, a slightly positive staining in nontreated mice, whereby we identified several foci of infiltrating CD68-positive cells at d 14 surrounding the glomeruli (Fig. 4A). Butyrate treatment significantly reduced the infiltration of CD68-positive cells, as well as gene expression of chemokines such as ccl2 and ccl3 and important proinflammatory cytokines such as Tnf-a and Il-1b in renal tissue (Fig. 4B). Protein levels for the proinflammatory cytokines in renal tissue were also significantly reduced by butyrate treatment (Fig. 4C).', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5317, 'mouse', 'decrease', 'null', 'low-throughput', 'Renal inflammation is a common finding in ADR nephropathy. Injured resident kidney cells release chemokines that promote the recruitment ofmacrophages, which play a pivotal role as disease progresses. CD68 staining in renal cortex showed, at early stages of the disease, a slightly positive staining in nontreated mice, whereby we identified several foci of infiltrating CD68-positive cells at d 14 surrounding the glomeruli (Fig. 4A). Butyrate treatment significantly reduced the infiltration of CD68-positive cells, as well as gene expression of chemokines such as ccl2 and ccl3 and important proinflammatory cytokines such as Tnf-a and Il-1b in renal tissue (Fig. 4B). Protein levels for the proinflammatory cytokines in renal tissue were also significantly reduced by butyrate treatment (Fig. 4C).', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5318, 'mouse', 'increase', 'null', 'low-throughput', 'In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5319, 'mouse', 'increase', 'null', 'low-throughput', 'In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5320, 'mouse', 'increase', 'null', 'low-throughput', 'In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5321, 'mouse', 'increase', 'null', 'low-throughput', 'In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5322, 'mouse', 'increase', 'null', 'low-throughput', 'In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5323, 'mouse', 'increase', 'null', 'low-throughput', 'To test this hypothesis, we evaluated the histone modifications H3K27Ac and H3K4me3, which are typically regarded as markers that lead to transcriptional activation, andH3K9me3,which correlates to transcriptional repression (Fig. 5F). ADR led to significant reductions of the activation marker H3K27Ac at the promoter region of both Synpo and Nphs1 genes, which was recovered in the presence of butyrate. In parallel, the increase of the repressive marker H3K9me3 that is due to ADR at the promoter region on both Synpo and Kirrel genes was significantly reduced in the presence of butyrate. Indeed, podocytes grown in vitro showed preserved expression of synaptopodin (Fig. 5G) after treatmentwith butyrate in the presence of ADR.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5324, 'mouse', 'increase', 'null', 'low-throughput', 'To test this hypothesis, we evaluated the histone modifications H3K27Ac and H3K4me3, which are typically regarded as markers that lead to transcriptional activation, andH3K9me3,which correlates to transcriptional repression (Fig. 5F). ADR led to significant reductions of the activation marker H3K27Ac at the promoter region of both Synpo and Nphs1 genes, which was recovered in the presence of butyrate. In parallel, the increase of the repressive marker H3K9me3 that is due to ADR at the promoter region on both Synpo and Kirrel genes was significantly reduced in the presence of butyrate. Indeed, podocytes grown in vitro showed preserved expression of synaptopodin (Fig. 5G) after treatmentwith butyrate in the presence of ADR.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5325, 'mouse', 'increase', 'null', 'low-throughput', 'To test this hypothesis, we evaluated the histone modifications H3K27Ac and H3K4me3, which are typically regarded as markers that lead to transcriptional activation, andH3K9me3,which correlates to transcriptional repression (Fig. 5F). ADR led to significant reductions of the activation marker H3K27Ac at the promoter region of both Synpo and Nphs1 genes, which was recovered in the presence of butyrate. In parallel, the increase of the repressive marker H3K9me3 that is due to ADR at the promoter region on both Synpo and Kirrel genes was significantly reduced in the presence of butyrate. Indeed, podocytes grown in vitro showed preserved expression of synaptopodin (Fig. 5G) after treatmentwith butyrate in the presence of ADR.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5326, 'mouse', 'decrease', 'null', 'low-throughput', 'Podocytes grown in vitro in the presence of ADR exhibited significantly increased TRPC5 and TRPC6 gene transcriptions, which were abolished when podocytes were incubated with buty rate.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5327, 'mouse', 'decrease', 'null', 'low-throughput', 'Podocytes grown in vitro in the presence of ADR exhibited significantly increased TRPC5 and TRPC6 gene transcriptions, which were abolished when podocytes were incubated with buty rate.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5328, 'mouse', 'increase', 'null', 'low-throughput', 'Here, the presence of ADR in podocytes grown in vitro decreased the expression levels of RHOA, RAC1, and CDC42, whereas treatmentwith butyrate significantly recovered the levels of both RAC1 and CDC42 but not RHOA.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5329, 'mouse', 'increase', 'null', 'low-throughput', 'Here, the presence of ADR in podocytes grown in vitro decreased the expression levels of RHOA, RAC1, and CDC42, whereas treatmentwith butyrate significantly recovered the levels of both RAC1 and CDC42 but not RHOA.', 31366236);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5330, 'mouse', 'decrease', 'Ellagic acid', 'low-throughput', 'Thus, the gut bacterial metabolite UA increases miR-10a-5p levels thereby downregulating Orai1/STIM1/STIM2 expression, store operated Ca2+ entry, and proliferation of murine CD4+ T cells.', 31417547);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5331, 'mouse', 'decrease', 'Ellagic acid', 'low-throughput', 'Thus, the gut bacterial metabolite UA increases miR-10a-5p levels thereby downregulating Orai1/STIM1/STIM2 expression, store operated Ca2+ entry, and proliferation of murine CD4+ T cells.', 31417547);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5332, 'mouse', 'decrease', 'Ellagic acid', 'low-throughput', 'Thus, the gut bacterial metabolite UA increases miR-10a-5p levels thereby downregulating Orai1/STIM1/STIM2 expression, store operated Ca2+ entry, and proliferation of murine CD4+ T cells.', 31417547);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5333, 'mouse', 'increase', 'Ellagic acid', 'low-throughput', 'Thus, the present observations uncover a novel action of UA, i.e., the upregulation of miR-10a-5p with subsequent downregulation of store operated Ca2+ influx in CD4+ T cells.', 31417547);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5337, 'mouse', 'decrease', 'Cholesterol', 'low-throughput', 'Altered substrate oxidation and fecal fat excretion in LHFD-fed GF mice is linked to decreased hepatic Cyp7a1 and Nr1h4 expression.', 27900259);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5338, 'mouse', 'decrease', 'Cholesterol', 'low-throughput', 'Altered substrate oxidation and fecal fat excretion in LHFD-fed GF mice is linked to decreased hepatic Cyp7a1 and Nr1h4 expression.', 27900259);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5339, 'mouse', 'decrease', 'Linoleic acid', 'low-throughput', '10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.', 25505251);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5340, 'mouse', 'decrease', 'Linoleic acid', 'low-throughput', '10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.', 25505251);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5341, 'mouse', 'increase', 'Linoleic acid', 'low-throughput', '10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.', 25505251);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5342, 'mouse', 'decrease', 'Linoleic acid', 'low-throughput', 'IFN-TNF-induced an increase in FD-4 permeability;however, the pretreatment of HYA and linoleic acid signififcantly suppressed this increase (Fig. 2C). In contrast, the pretreatment of HYB did not alter IFN-TNF-induced changes in TER and FD-4 permeability. As shown in Fig. 2D,IFN-TNF-induced a marked increase in the secretion of IL-8, which is a major pro-inflammatory cytokine. However,the pretreatment of HYA significantly suppressed IL-8 secretion.', 25505251);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5343, 'mouse', 'increase', 'Linoleic acid', 'low-throughput', 'Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.', 25505251);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5344, 'mouse', 'increase', 'Linoleic acid', 'low-throughput', 'Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.', 25505251);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5345, 'mouse', 'increase', 'Linoleic acid', 'low-throughput', 'Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.', 25505251);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5346, 'mouse', 'decrease', 'Linoleic acid', 'low-throughput', 'Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.', 25505251);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5347, 'mouse', 'decrease', 'Linoleic acid', 'low-throughput', 'The pretreatment of HYA suppressed IFN-induced up-regulation of TNFR2 (but not TNFR1), and it subsequently decreased the ratio of phospho-INF-B p65 expression induced by cytokines (Fig. 3, B and C). In contrast, pretreatment of HYB did not show the same inhibitory effects as HYA.', 25505251);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5348, 'mouse', 'increase', 'Linoleic acid', 'low-throughput', 'It was noteworthy that the pretreatment of HYA augmented the mRNA expression of GPR40 and GPR43 but not GPR120, PPAR, and GPR41 in Caco-2 cells (Fig. 4A). However, the pretreatment of HYB did not up-regulate GPR40 and GPR43 expression (Fig. 4A).', 25505251);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5349, 'mouse', 'decrease', 'null', 'low-throughput', 'The expression levels of IL�? (Fig. 5A and B) and its receptors CXCR1 (Fig. 5A and C) and CXCR2 (Fig. 5A and D) were significantly increased in the non‑infarct LV of MI + VEH rats compared with the LV of SHAM + VEH rats; however, the expression of these proteins was normalized by DMB treatment.', 31180562);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5350, 'mouse', 'decrease', 'null', 'low-throughput', 'The expression levels of IL�? (Fig. 5A and B) and its receptors CXCR1 (Fig. 5A and C) and CXCR2 (Fig. 5A and D) were significantly increased in the non‑infarct LV of MI + VEH rats compared with the LV of SHAM + VEH rats; however, the expression of these proteins was normalized by DMB treatment.', 31180562);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5351, 'mouse', 'decrease', 'null', 'low-throughput', 'The expression levels of IL�? (Fig. 5A and B) and its receptors CXCR1 (Fig. 5A and C) and CXCR2 (Fig. 5A and D) were significantly increased in the non‑infarct LV of MI + VEH rats compared with the LV of SHAM + VEH rats; however, the expression of these proteins was normalized by DMB treatment.', 31180562);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5352, 'mouse', 'increase', 'null', 'low-throughput', 'Accordingly, in livers from TLR2-defcient mice, the expression of SASP factors, such as IL1β,  Groα, and IL6, was significantly reduced in HSCs, suggesting that TLR2-mediated signaling induced the expression of SASP factors (Fig. 2F). Notably, however, the presence of p21 and 53BP1 foci was still observed in activated HSCs from TLR2-deficient mice (Fig. 2F). The majority of senescent 
signals are detected in the HSCs but not in HCC cells in the tumor region (Supplementary Fig. S2). Taken together, these results suggest that TLR2-mediated signaling is important for the upregulation of SASP factors in senescent HSCs but not for the induction of senescent cell-cycle arrest.', 28202625);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5353, 'mouse', 'increase', 'null', 'low-throughput', 'Accordingly, in livers from TLR2-defcient mice, the expression of SASP factors, such as IL1β,  Groα, and IL6, was significantly reduced in HSCs, suggesting that TLR2-mediated signaling induced the expression of SASP factors (Fig. 2F). Notably, however, the presence of p21 and 53BP1 foci was still observed in activated HSCs from TLR2-deficient mice (Fig. 2F). The majority of senescent 
signals are detected in the HSCs but not in HCC cells in the tumor region (Supplementary Fig. S2). Taken together, these results suggest that TLR2-mediated signaling is important for the upregulation of SASP factors in senescent HSCs but not for the induction of senescent cell-cycle arrest.', 28202625);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5354, 'mouse', 'increase', 'null', 'low-throughput', 'Accordingly, in livers from TLR2-defcient mice, the expression of SASP factors, such as IL1β,  Groα, and IL6, was significantly reduced in HSCs, suggesting that TLR2-mediated signaling induced the expression of SASP factors (Fig. 2F). Notably, however, the presence of p21 and 53BP1 foci was still observed in activated HSCs from TLR2-deficient mice (Fig. 2F). The majority of senescent 
signals are detected in the HSCs but not in HCC cells in the tumor region (Supplementary Fig. S2). Taken together, these results suggest that TLR2-mediated signaling is important for the upregulation of SASP factors in senescent HSCs but not for the induction of senescent cell-cycle arrest.', 28202625);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5355, 'mouse', 'increase', 'null', 'low-throughput', 'Consistent with our previous report, the expression of the senescence inducers p21 (encoded by Cdkn1a) and p16 (encoded by Cdkn2a) was upregulated by DCA treatment (Fig. 3B). Notably, the expression levels of TLR2 and SASP factors, such as IL1β, Groα, and IL6, were significantly enhanced by LTA in DCA-induced senescent HSCs (Fig. 3C).', 28202625);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5356, 'mouse', 'increase', 'null', 'low-throughput', 'Consistent with our previous report, the expression of the senescence inducers p21 (encoded by Cdkn1a) and p16 (encoded by Cdkn2a) was upregulated by DCA treatment (Fig. 3B). Notably, the expression levels of TLR2 and SASP factors, such as IL1β, Groα, and IL6, were significantly enhanced by LTA in DCA-induced senescent HSCs (Fig. 3C).', 28202625);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5357, 'mouse', 'increase', 'null', 'low-throughput', 'The expression of COX2, but not COX1, in DCA-treated primary HSCs was significantly induced by increasing amounts of LTA (Fig. 4C), and the COX2 expression was dependent on TLR2 (Supplementary 
Fig. S3). Indeed, we confirmed that TLR2 was expressed in HSCs and this expression was enhanced by LTA alone or DCA plus LTA treatment (Supplementary Fig. S8A).', 28202625);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5358, 'mouse', 'decrease', 'null', 'low-throughput', 'The production of IFNγ and TNFα, which are known to activate antitumor immune responses and inhibit cancer cell growth, was significantly and specifically attenuated by PGE2.', 28202625);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5359, 'mouse', 'decrease', 'null', 'low-throughput', 'The production of IFNγ and TNFα, which are known to activate antitumor immune responses and inhibit cancer cell growth, was significantly and specifically attenuated by PGE2.', 28202625);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5360, 'mouse', 'increase', 'null', 'low-throughput', 'In contrast, the production of IL10, a potent anti-inflammatory cytokine, and IL6, a known tumor-promoting cytokine, in immune cells was slightly but significantly enhanced (Supplementary Fig. S8D).', 28202625);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5361, 'mouse', 'increase', 'null', 'low-throughput', 'In contrast, the production of IL10, a potent anti-inflammatory cytokine, and IL6, a known tumor-promoting cytokine, in immune cells was slightly but significantly enhanced (Supplementary Fig. S8D).', 28202625);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5362, 'mouse', 'increase', 'null', 'low-throughput', 'PGE2 is known to be recognized by four types of receptors: PTGER1, PTGER2, PTGER3, and PTGER4 (37, 38). In our mouse model, the expression of PTGER4, but not other PGE2 receptors, was significantly upregulated in tumor tissues (Fig. 5C),', 28202625);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5363, 'mouse', 'increase', 'null', 'low-throughput', 'Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.', 25065623);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5364, 'mouse', 'decrease', 'null', 'low-throughput', 'Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.', 25065623);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5365, 'mouse', 'increase', 'null', 'low-throughput', 'Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.', 25065623);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5366, 'mouse', 'increase', 'null', 'low-throughput', 'For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of
L-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to
C. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa
exposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).', 25065623);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5367, 'mouse', 'increase', 'null', 'low-throughput', 'For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of
L-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to
C. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa
exposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).', 25065623);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5368, 'mouse', 'increase', 'null', 'low-throughput', 'For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of
L-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to
C. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa
exposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).', 25065623);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5369, 'mouse', 'decrease', 'N-Trimethyllysine', 'low-throughput', 'We found TMAVA to bind and inhibit BBOX, reducing synthesis of carnitine.We found the intestinal bacteria Enterococcus faecalis and Pseudomonas aeruginosa to metabolize trimethyllysine to TMAVA.', 32105727);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5370, 'mouse', 'decrease', 'N-Trimethyllysine', 'low-throughput', 'We found TMAVA to bind and inhibit BBOX, reducing synthesis of carnitine.We found the intestinal bacteria Enterococcus faecalis and Pseudomonas aeruginosa to metabolize trimethyllysine to TMAVA.', 32105727);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5371, 'mouse', 'null', 'null', 'null', 'In addition, Parabacteroides_sp_HGS0025 positively correlated with butyrate, IgM, IL-10, and TNF-α products in intestine and mice blood, respectively.', 32014483);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5375, 'mouse', 'increase', 'beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)', 'low-throughput', 'We found that UB treatment resulted in significant p62 accumulation.', 31996327);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5376, 'mouse', 'increase', 'beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)', 'low-throughput', 'The nuclear Nrf2 level was increased with UB treatment.', 31996327);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5377, 'mouse', 'increase', 'beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)', 'low-throughput', 'UB treatment significantly enhanced nuclear Nrf2 level and its downstream genes, HO-1, NQO1 and GSTP1 in the rat heart subjected to IR.', 31996327);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5378, 'mouse', 'increase', 'beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)', 'low-throughput', 'UB treatment significantly enhanced nuclear Nrf2 level and its downstream genes, HO-1, NQO1 and GSTP1 in the rat heart subjected to IR.', 31996327);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5379, 'mouse', 'increase', 'beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)', 'low-throughput', 'UB treatment significantly enhanced nuclear Nrf2 level and its downstream genes, HO-1, NQO1 and GSTP1 in the rat heart subjected to IR.', 31996327);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5380, 'mouse', 'null', 'null', 'null', 'The amino acid metabolite, N-delta-acetylornithine, was notably increased by B. longum rice bran fermentation when compared to non-fermented rice bran in food, colon, and blood.', 31965839);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5381, 'mouse', 'increase', 'Mogroside V', 'low-throughput', 'Only 11-oxo-mogrol could increase the mRNA expression of both Camk2α and Calbindin genes', 31900523);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5382, 'mouse', 'increase', 'Mogroside V', 'low-throughput', 'Only 11-oxo-mogrol could increase the mRNA expression of both Camk2α and Calbindin genes', 31900523);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5383, 'mouse', 'increase', 'null', 'low-throughput', 'Treatment of naive T cells under the Treg-cell-polarizing conditions with butyrate enhanced histone H3 acetylation in the promoter and conserved non-coding sequence regions of the Foxp3 locus, suggesting a possible mechanism for how microbial-derived butyrate regulates the differentiation of Treg cells.', 24226770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5384, 'mouse', 'increase', 'null', 'low-throughput', 'Treatment of naive T cells under the Treg-cell-polarizing conditions with butyrate enhanced histone H3 acetylation in the promoter and conserved non-coding sequence regions of the Foxp3 locus, suggesting a possible mechanism for how microbial-derived butyrate regulates the differentiation of Treg cells.', 24226770);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5385, 'mouse', 'null', 'Tryptophan', 'null', 'Bt and B. ovatus produced indole in both media conditions.', 27916477);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5386, 'mouse', 'null', 'Tryptophan', 'null', 'Bt and B. ovatus produced indole in both media conditions.', 27916477);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5387, 'mouse', 'increase', 'null', 'low-throughput', 'We found that butyrate enhanced phosphorylated-STAT3, mTOR, and ERK1/2 as evidenced by Western blots.', 30177845);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5388, 'mouse', 'increase', 'null', 'low-throughput', 'We found that butyrate enhanced phosphorylated-STAT3, mTOR, and ERK1/2 as evidenced by Western blots.', 30177845);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5389, 'mouse', 'increase', 'null', 'low-throughput', 'Mechanistically, SCFAs activate Th1 cell STAT3 and mTOR, and consequently upregulate transcription factor B lymphocyte-induced maturation protein 1 (Blimp-1), which mediates SCFA-induction of IL-10.', 30177845);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5390, 'mouse', 'increase', 'null', 'low-throughput', 'Mechanistically, SCFAs activate Th1 cell STAT3 and mTOR, and consequently upregulate transcription factor B lymphocyte-induced maturation protein 1 (Blimp-1), which mediates SCFA-induction of IL-10.', 30177845);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5391, 'mouse', 'increase', 'null', 'low-throughput', 'SCFA Propionate Induced Sympathetic Activation via GPR41-Gβγ in Sympathetic Neurons.', 21518883);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5392, 'mouse', 'increase', 'Linolenic acid', 'low-throughput', 'Treatments of BMDCs with the fatty acids significantly increased mRNA expression of Arg1 in a dose-dependent manner in comparison with that of IL-4 treatment alone.', 28904023);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5393, 'mouse', 'increase', 'Linolenic acid', 'low-throughput', 'Treatments of BMDCs with the fatty acids significantly increased mRNA expression of Arg1 in a dose-dependent manner in comparison with that of IL-4 treatment alone.', 28904023);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5399, 'mouse', 'increase', 'null', 'low-throughput', 'AM attenuates ALS in mice by increasing NAM levels', 31330533);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5400, 'mouse', 'increase', 'null', 'low-throughput', 'AM attenuates ALS in mice by increasing NAM levels', 31330533);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5406, 'mouse', 'decrease', 'null', 'low-throughput', 'Butyrate treatment significantly decreased expression of the CD11b.', 31835282);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5407, 'mouse', 'increase', 'Tryptophan', 'low-throughput', 'Here, we show that the biological effects of tryptamine are mediated through the 5-HT4 receptor (5-HT4R), a G-protein-coupled receptor (GPCR) uniquely expressed in the colonic epithelium.', 29902441);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5408, 'mouse', 'increase', 'Tryptophan', 'low-throughput', 'Here, we show that the biological effects of tryptamine are mediated through the 5-HT4 receptor (5-HT4R), a G-protein-coupled receptor (GPCR) uniquely expressed in the colonic epithelium.', 29902441);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5409, 'mouse', 'decrease', 'null', 'high-throughput', 'UroA or UAS03 treatment significantly reduced the LPS-induced increase in serum IL-6 and TNF-α levels (Fig. 1c). These results suggest that UAS03 is a potent structural analogue of UroA with increased anti-inflammatory activities.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5410, 'mouse', 'decrease', 'null', 'high-throughput', 'UroA or UAS03 treatment significantly reduced the LPS-induced increase in serum IL-6 and TNF-α levels (Fig. 1c). These results suggest that UAS03 is a potent structural analogue of UroA with increased anti-inflammatory activities.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5411, 'mouse', 'increase', 'null', 'high-throughput', 'UroA/UAS03 treatment attenuates TNBS-induced colitis in mice. Colitis was induced by intrarectal administration of TNBS (2.5 mg/mouse) in C57BL/6 (8 week age old, n = 5/group) mice. Mice were orally treated with vehicle or UroA (20 mg/kg) or UAS03 (20 mg/kg body weight) every 12 h post-TNBS instillation for 60 h and the experiment terminated at 72 h. Representative data from one of three independent experiments is shown. a Percent body weight loss (No TNBS- Solid black line; Veh + TNBS- Solid red line; UroA + TNBS- Solid blue line; UAS03 + TNBS- Solid purple line). b disease activity index, c intestinal permeability, d colon lengths were measured. e Gross morphological changes of colon, f ratio of colon weight/length, g colonic myeloperoxidase (MPO) levels, h serum IL-6, TNF-α, CXCL1, and IL-1β levels, i microphotographs of hematoxylin and eosin (H&E) stained sections of colons and inflammation scores are shown. Scale bar indicates 300 μm. j Cldn4 expression in the colons of these mice (n = 3) was measured by immunoblots and quantified. Statistical analysis was performed (unpaired t-test) using Graphpad Prism software. Error bars, ±SEM ***p < 0.001; **p < 0.01
*p < 0.05. Source Data are provided as a Source Data File.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5412, 'mouse', 'increase', 'null', 'high-throughput', 'UroA/UAS03 treatment attenuates TNBS-induced colitis in mice. Colitis was induced by intrarectal administration of TNBS (2.5 mg/mouse) in C57BL/6 (8 week age old, n = 5/group) mice. Mice were orally treated with vehicle or UroA (20 mg/kg) or UAS03 (20 mg/kg body weight) every 12 h post-TNBS instillation for 60 h and the experiment terminated at 72 h. Representative data from one of three independent experiments is shown. a Percent body weight loss (No TNBS- Solid black line; Veh + TNBS- Solid red line; UroA + TNBS- Solid blue line; UAS03 + TNBS- Solid purple line). b disease activity index, c intestinal permeability, d colon lengths were measured. e Gross morphological changes of colon, f ratio of colon weight/length, g colonic myeloperoxidase (MPO) levels, h serum IL-6, TNF-α, CXCL1, and IL-1β levels, i microphotographs of hematoxylin and eosin (H&E) stained sections of colons and inflammation scores are shown. Scale bar indicates 300 μm. j Cldn4 expression in the colons of these mice (n = 3) was measured by immunoblots and quantified. Statistical analysis was performed (unpaired t-test) using Graphpad Prism software. Error bars, ±SEM ***p < 0.001; **p < 0.01
*p < 0.05. Source Data are provided as a Source Data File.', 30626868);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5416, 'mouse', 'increase', 'null', 'low-throughput', 'Interestingly, metabolites produced by gut microbiota, including propionate, butyrate, and the secondary bile acid deoxycolate directly induced expression of tryptophan hydroxylase 1 (Tph1).', 28961041);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5417, 'mouse', 'increase', 'null', 'low-throughput', 'Interestingly, metabolites produced by gut microbiota, including propionate, butyrate, and the secondary bile acid deoxycolate directly induced expression of tryptophan hydroxylase 1 (Tph1).', 28961041);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5418, 'mouse', 'increase', 'null', 'low-throughput', 'Interestingly, metabolites produced by gut microbiota, including propionate, butyrate, and the secondary bile acid deoxycolate directly induced expression of tryptophan hydroxylase 1 (Tph1).', 28961041);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5426, 'mouse', 'null', 'null', 'null', 'CB0313.1 upregulates butyrate production.', 28765642);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5432, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5433, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5434, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5435, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5436, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5437, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5438, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5439, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5440, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5441, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5442, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5443, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5444, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5445, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5446, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5447, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5448, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5449, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5450, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5451, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5452, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5453, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5454, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5455, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5456, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5457, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5458, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5459, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5460, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5461, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5462, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5463, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5464, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5465, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5466, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5467, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5468, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5469, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5470, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5471, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5472, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5473, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5474, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5475, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5476, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5477, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5478, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5479, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5480, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5481, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5482, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5483, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5484, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5485, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5486, 'mouse', 'increase', 'Berberine', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5487, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5488, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5489, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5490, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5491, 'mouse', 'decrease', 'Berberine', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 31863867);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5492, 'mouse', 'decrease', 'null', 'low-throughput', 'In line with these clinical findings, VSL#3 and CLA significantly ameliorated gross pathology in colon, MLN and spleen, colonic histopathology such as leukocyte infiltration  and mucosal thickening , and colonic mRNA expression such as TNF-a and MCP-1 in comparison to untreated control mice with DSS colitis.', 22363592);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5493, 'mouse', 'decrease', 'null', 'low-throughput', 'In line with these clinical findings, VSL#3 and CLA significantly ameliorated gross pathology in colon, MLN and spleen, colonic histopathology such as leukocyte infiltration  and mucosal thickening , and colonic mRNA expression such as TNF-a and MCP-1 in comparison to untreated control mice with DSS colitis.', 22363592);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5498, 'mouse', 'increase', 'null', 'low-throughput', 'Here, we tested the hypothesis that the gut microbiota could induce maturation of the adult ENS through release of 5-HT and activation of 5-HT4 receptors.', 29866843);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5568, 'mouse', 'null', 'Daidzein', 'null', 'We isolated, for the first time, an anaerobic gram-positive rod-shaped strain capable of producing equol from daidzein.', 17046543);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5569, 'mouse', 'null', 'Genistein', 'null', 'We isolated a strictly anaerobic bacterium (Mt1B8) from the mouse intestine which converted daidzein via dihydrodaidzein to equol as well as genistein via dihydrogenistein to 5-hydroxy-equol.', 18539813);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5570, 'mouse', 'null', 'Daidzein', 'null', 'We isolated a strictly anaerobic bacterium (Mt1B8) from the mouse intestine which converted daidzein via dihydrodaidzein to equol as well as genistein via dihydrogenistein to 5-hydroxy-equol.', 18539813);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5571, 'mouse', 'null', 'Dihydrodaidzein', 'null', 'Strain Mt1B8 also transformed dihydrodaidzein and dihydrogenistein to equol and 5-hydroxy-equol, respectively.', 18539813);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5572, 'mouse', 'null', 'Dihydrogenistein', 'null', 'Strain Mt1B8 also transformed dihydrodaidzein and dihydrogenistein to equol and 5-hydroxy-equol, respectively.', 18539813);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5573, 'mouse', 'increase', '3-Dehydroshikimate', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 32728212);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5574, 'mouse', 'increase', '3-Dehydroshikimate', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 32728212);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5575, 'mouse', 'increase', '3-Dehydroshikimate', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 32728212);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5576, 'mouse', 'increase', '3-Dehydroshikimate', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 32728212);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5577, 'mouse', 'increase', '3-Dehydroshikimate', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 32728212);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5578, 'mouse', 'increase', '3-Dehydroshikimate', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 32728212);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5579, 'mouse', 'increase', '3-Dehydroshikimate', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 32728212);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5580, 'mouse', 'increase', '3-Dehydroshikimate', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 32728212);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5581, 'mouse', 'increase', '3-Dehydroshikimate', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 32728212);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5582, 'mouse', 'increase', '3-Dehydroshikimate', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 32728212);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5583, 'mouse', 'increase', '3-Dehydroshikimate', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 32728212);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5584, 'mouse', 'increase', '3-Dehydroshikimate', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 32728212);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5585, 'mouse', 'increase', 'Phytic acid', 'low-throughput', 'Taken together, these data demonstrate bacterial-intrinsic phytate metabolism and provoke the hypothesis that breakdown of phytate by microbiota promotes intestinal HDAC activity.', 32731255);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5586, 'mouse', 'decrease', 'null', 'low-throughput', 'Further, colonization of germ-free (GF) mice with butyrate producing bacteria,Faecalibacterium prausnitzii, inhibited IEC-intrinsic HDAC activity.', 32731255);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5587, 'mouse', 'increase', 'null', 'low-throughput', 'CNV mice exhibited significantly higher IP3 in both the intestinal lumen (Fig. 1e) and colonic IECs (Fig. 1f), supporting that microbiota promote increased IP3. To assess the effect of IP3 on epithelial HDAC3 activity, HDAC3 was purified by immunoprecipitation from IECs and incubated with IP3. IP3 increased endogenous HDAC3 activity (Fig. 1g), supporting that IP3 directly activates HDAC3 in IECs.', 32731255);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5588, 'mouse', 'increase', 'null', 'low-throughput', 'Although less potent than IP3, IP4 also increased HDAC activity in IECs (Extended Data Fig. 5f), suggesting that other inositol phosphates simultaneously generated by phytate metabolism may provide synergistic effects.', 32731255);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5589, 'mouse', 'increase', 'Glutamic acid', 'low-throughput', 'Glutamate, the agonist of NMDAR and a major excitatory neurotransmitter in the mammalian central nervous system (CNS) [6], is intracellularly located and extensively distributed in the brain.', 32290475);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5590, 'mouse', 'increase', 'Glutamic acid', 'low-throughput', 'Glutamate, the agonist of NMDAR and a major excitatory neurotransmitter in the mammalian central nervous system (CNS) [6], is intracellularly located and extensively distributed in the brain.', 32290475);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5591, 'mouse', 'increase', 'Glutamic acid', 'low-throughput', 'Non-pathogenic bacteria with glutamate racemase including Corynebacterium glutamicum, Brevibacterium lactofermentum, Brevibacterium avium [121], Mycobacterium smegmatis[122], and Bacillus subtilis[123] convert l-glutamate to d-glutamate.', 32290475);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5592, 'mouse', 'increase', 'Glutamic acid', 'low-throughput', 'Non-pathogenic bacteria with glutamate racemase including Corynebacterium glutamicum, Brevibacterium lactofermentum, Brevibacterium avium [121], Mycobacterium smegmatis[122], and Bacillus subtilis[124] convert l-glutamate to d-glutamate.', 32290475);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5593, 'mouse', 'increase', 'null', 'low-throughput', 'Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ', 32290475);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5594, 'mouse', 'increase', 'null', 'low-throughput', 'Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ', 32290475);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5595, 'mouse', 'increase', 'null', 'low-throughput', 'Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ', 32290475);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5596, 'mouse', 'increase', 'null', 'low-throughput', 'Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ', 32290475);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5597, 'mouse', 'increase', 'null', 'low-throughput', 'Furthermore, benzoate, an inhibitor of d-amino acid oxidases (DAAO), can increase d-serine,thereby enhancing NMDAR activation [125,126].', 32290475);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5598, 'mouse', 'increase', 'Bile acid', 'low-throughput', 'Reconstitution with a single bacterial species, Clostridium scindens, or its derived metabolite, the secondary bile acid deoxycholic acid, can restore pDC- and MyD88-dependent type I IFN responses to restrict systemic CHIKV infection and transmission back to vector mosquitoes. ', 32668198);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5599, 'mouse', 'increase', 'Bile acid', 'low-throughput', 'Reconstitution with a single bacterial species, Clostridium scindens, or its derived metabolite, the secondary bile acid deoxycholic acid, can restore pDC- and MyD88-dependent type I IFN responses to restrict systemic CHIKV infection and transmission back to vector mosquitoes. ', 32668198);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5600, 'mouse', 'null', 'Tryptophan', 'null', 'HSD significantly reduces intestinal Lactobacillus murinus in hypertensive FVB/N mice, while L murinus can metabolize trypto phan in the diet to produce indole, thereby inhibiting T lymphocyte differentiation into Th17 cells, alleviating inflammation and lowering BP.', 32078445);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5601, 'mouse', 'decrease', 'null', 'low-throughput', 'Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.', 32078445);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5602, 'mouse', 'decrease', 'null', 'low-throughput', 'Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.', 32078445);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5603, 'mouse', 'decrease', 'null', 'low-throughput', 'Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.', 32078445);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5604, 'mouse', 'increase', 'null', 'low-throughput', 'Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.', 32078445);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5605, 'mouse', 'increase', 'null', 'low-throughput', 'Here, we elucidate a micro biome-dependent pathway by which SG increases CA7S production. We show that a microbial metabolite,lithocholic acid (LCA), is increased in murine portal veins post-SG and by activating the vitamin D receptor, induces hepatic mSult2A1/hSULT2A expression to drive CA7S production. An SG-induced shift in the micro biome increases gut expression of the bile acid transporters Asbt and Osta, which in turn facilitate selective transport of LCA across the gut epithelium. Cecal microbiota transplant from SG animals is sufficient to recreate the pathway in germ-free (GF) animals. 
and GLP-1 secretion, causally connecting a microbial metabolite with the improvement of diabetic phe notypes.', 33434516);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5606, 'mouse', 'increase', 'null', 'low-throughput', 'Here, we elucidate a micro biome-dependent pathway by which SG increases CA7S production. We show that a microbial metabolite,lithocholic acid (LCA), is increased in murine portal veins post-SG and by activating the vitamin D receptor, induces hepatic mSult2A1/hSULT2A expression to drive CA7S production. An SG-induced shift in the micro biome increases gut expression of the bile acid transporters Asbt and Osta, which in turn facilitate selective transport of LCA across the gut epithelium. Cecal microbiota transplant from SG animals is sufficient to recreate the pathway in germ-free (GF) animals. 
and GLP-1 secretion, causally connecting a microbial metabolite with the improvement of diabetic phe notypes.', 33434516);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5607, 'mouse', 'increase', 'null', 'low-throughput', 'We determined that CA7S is a potent agonist of the G-protein-coupled recep tor TGR5 (GPBAR1) and induces GLP-1 secretion. When administered to diet-induced obese (DIO) mice, CA7S in creases insulin secretion and induces systemic glucose clearance.', 33434516);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5608, 'mouse', 'increase', 'null', 'low-throughput', 'We determined that CA7S is a potent agonist of the G-protein-coupled recep tor TGR5 (GPBAR1) and induces GLP-1 secretion. When administered to diet-induced obese (DIO) mice, CA7S in creases insulin secretion and induces systemic glucose clearance.', 33434516);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5609, 'mouse', 'decrease', 'null', 'low-throughput', 'Consistent with our in vitro results showing that LCA inhibits expression of Asbt and Osta, SG-CMT mice exhibited significantly higher levels of Asbt and Osta expression in the distal ileum compared with sham-CMT (Figure 7E).', 33434516);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5610, 'mouse', 'decrease', 'null', 'low-throughput', 'Consistent with our in vitro results showing that LCA inhibits expression of Asbt and Osta, SG-CMT mice exhibited significantly higher levels of Asbt and Osta expression in the distal ileum compared with sham-CMT (Figure 7E).', 33434516);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5611, 'mouse', 'increase', 'Trimethylamine', 'low-throughput', 'Concentrations of pro-inflammatory cytokines com monly associated with neuroinflammation (IL1-β, IL 6, and TNF-α) and the anti-inflammatory cytokine IL-10 were measured in whole-brain lysates from young and old mice and are presented in Fig. 3a. Whole-brain concentrations of pro-inflammatory cytokines were all ~ 50�?00% higher in old vs. young mice (all p < 0.05).Old mice also tended to have (presumably compensato ry) higher brain IL-10 compared to young mice (p =0.07).', 32862276);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5612, 'mouse', 'increase', 'Trimethylamine', 'low-throughput', 'Concentrations of pro-inflammatory cytokines com monly associated with neuroinflammation (IL1-β, IL 6, and TNF-α) and the anti-inflammatory cytokine IL-10 were measured in whole-brain lysates from young and old mice and are presented in Fig. 3a. Whole-brain concentrations of pro-inflammatory cytokines were all ~ 50�?00% higher in old vs. young mice (all p < 0.05).Old mice also tended to have (presumably compensato ry) higher brain IL-10 compared to young mice (p =0.07).', 32862276);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5613, 'mouse', 'increase', 'Trimethylamine', 'low-throughput', 'Concentrations of pro-inflammatory cytokines com monly associated with neuroinflammation (IL1-β, IL 6, and TNF-α) and the anti-inflammatory cytokine IL-10 were measured in whole-brain lysates from young and old mice and are presented in Fig. 3a. Whole-brain concentrations of pro-inflammatory cytokines were all ~ 50�?00% higher in old vs. young mice (all p < 0.05).Old mice also tended to have (presumably compensato ry) higher brain IL-10 compared to young mice (p =0.07).', 32862276);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5614, 'mouse', 'increase', 'Trimethylamine', 'low-throughput', 'Concentrations of pro-inflammatory cytokines com monly associated with neuroinflammation (IL1-β, IL 6, and TNF-α) and the anti-inflammatory cytokine IL-10 were measured in whole-brain lysates from young and old mice and are presented in Fig. 3a. Whole-brain concentrations of pro-inflammatory cytokines were all ~ 50�?00% higher in old vs. young mice (all p < 0.05).Old mice also tended to have (presumably compensato ry) higher brain IL-10 compared to young mice (p =0.07).', 32862276);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5615, 'mouse', 'increase', 'Trimethylamine', 'low-throughput', 'Next, we measured the pro-inflammatory tran scription factor nuclear factor kappa B (NF-κB), which is a key regulator of pro-inflammatory cytokine signal ing. Although abundance of phosphorylated (i.e., acti vated) NF-κB was similar between brains from young and old mice (p = 0.49), old mice had ~ 50% higher abundance of total NF-κB (p < 0.001; Fig. 3b), suggest ing a greater capacity to upregulate pro-inflammatory cytokine production.', 32862276);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5616, 'mouse', 'increase', 'Trimethylamine', 'low-throughput', 'Finally, LCN2, an established marker of reactive astrocytes involved in the innate immune response to bacteria [5, 63], was higher in brain lysates from old compared to young mice (p < 0.05;Fig. 3c).', 32862276);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5617, 'mouse', 'increase', 'null', 'low-throughput', 'In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.', 33256731);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5618, 'mouse', 'increase', 'null', 'low-throughput', 'In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.', 33256731);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5619, 'mouse', 'increase', 'null', 'low-throughput', 'In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.', 33256731);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5620, 'mouse', 'increase', 'null', 'low-throughput', 'mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).', 32296105);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5621, 'mouse', 'increase', 'null', 'low-throughput', 'mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).', 32296105);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5622, 'mouse', 'increase', 'null', 'low-throughput', 'mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).', 32296105);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5623, 'mouse', 'increase', 'null', 'low-throughput', 'mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).', 32296105);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5624, 'mouse', 'increase', 'null', 'low-throughput', 'mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).', 32296105);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5625, 'mouse', 'increase', 'null', 'low-throughput', 'As a result, stercobilin signifcantly induced NF-κB reporter gene activities in a dose-dependent manner (Fig. 6A). Additionally, ster cobilin signifcantly induced expression of several infammation-related genes, including TNF-α, IL-1β, IL-6, iNOS, and COX-2 (Fig. 6B).', 32296105);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5626, 'mouse', 'decrease', 'Baicalin', 'low-throughput', 'Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ', 31650468);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5627, 'mouse', 'decrease', 'Baicalin', 'low-throughput', 'Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ', 31650468);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5628, 'mouse', 'decrease', 'Baicalin', 'low-throughput', 'Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ', 31650468);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5629, 'mouse', 'decrease', 'Baicalin', 'low-throughput', 'Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ', 31650468);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5630, 'mouse', 'decrease', 'Baicalin', 'low-throughput', 'Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ', 31650468);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5631, 'mouse', 'decrease', 'Baicalin', 'low-throughput', 'Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ', 31650468);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5632, 'mouse', 'increase', 'Tryptophan', 'low-throughput', 'Interestingly, in the same mice, indole induced a dramatic increase of the expression of the adrenomedullary Pnmt gene, which is involved in catecholamine biosynthesis.', 32569990);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5633, 'mouse', 'decrease', 'null', 'low-throughput', 'To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5634, 'mouse', 'decrease', 'null', 'low-throughput', 'To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5635, 'mouse', 'decrease', 'null', 'low-throughput', 'To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5636, 'mouse', 'decrease', 'null', 'low-throughput', 'To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5637, 'mouse', 'decrease', 'null', 'low-throughput', 'To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5638, 'mouse', 'decrease', 'null', 'low-throughput', 'To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5639, 'mouse', 'decrease', 'null', 'low-throughput', 'To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5640, 'mouse', 'decrease', 'null', 'low-throughput', 'To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5641, 'mouse', 'decrease', 'null', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5642, 'mouse', 'decrease', 'null', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5643, 'mouse', 'decrease', 'null', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5644, 'mouse', 'decrease', 'null', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5645, 'mouse', 'decrease', 'null', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5646, 'mouse', 'decrease', 'null', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5647, 'mouse', 'decrease', 'null', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5648, 'mouse', 'decrease', 'null', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 33024047);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5649, 'mouse', 'null', 'Stachyose', 'null', 'In the low-dose and high dose groups, elevated concentrations of acetic acid and propionic acid were produced, indicating that stachyose intake promoted the proliferation of acetic acid- and propionic acid-producing microorganisms.', 33233042);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5650, 'mouse', 'null', 'Stachyose', 'null', 'In the low-dose and high dose groups, elevated concentrations of acetic acid and propionic acid were produced, indicating that stachyose intake promoted the proliferation of acetic acid- and propionic acid-producing microorganisms.', 33233042);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5651, 'mouse', 'null', 'Stachyose', 'null', 'In the low-dose and high dose groups, elevated concentrations of acetic acid and propionic acid were produced, indicating that stachyose intake promoted the proliferation of acetic acid- and propionic acid-producing microorganisms.', 33233042);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5652, 'mouse', 'null', 'Stachyose', 'null', 'In the low-dose and high dose groups, elevated concentrations of acetic acid and propionic acid were produced, indicating that stachyose intake promoted the proliferation of acetic acid- and propionic acid-producing microorganisms.', 33233042);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5653, 'mouse', 'decrease', 'null', 'low-throughput', 'The expression of IL-1, IL-17 and TNF-α were higher in the intestinal tract of the germ free mice that had not received a human flora fecal transplant than in those that had received a fecal transplant, and the expression of IL-1, IL 17 and TNF-α decreased significantly after the fecal transplant (Fig. 4c, p < 0.01).', 33233042);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5654, 'mouse', 'decrease', 'null', 'low-throughput', 'The expression of IL-1, IL-17 and TNF-α were higher in the intestinal tract of the germ free mice that had not received a human flora fecal transplant than in those that had received a fecal transplant, and the expression of IL-1, IL 17 and TNF-α decreased significantly after the fecal transplant (Fig. 4c, p < 0.01).', 33233042);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5655, 'mouse', 'decrease', 'null', 'low-throughput', 'The expression of IL-1, IL-17 and TNF-α were higher in the intestinal tract of the germ free mice that had not received a human flora fecal transplant than in those that had received a fecal transplant, and the expression of IL-1, IL 17 and TNF-α decreased significantly after the fecal transplant (Fig. 4c, p < 0.01).', 33233042);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5656, 'mouse', 'decrease', 'null', 'low-throughput', 'Compared with the germ free group, the expression of NF-κB decreased significantly in the colon after transplantation of intestinal flora (p < 0.05), and the intake of stachyose downregulated the expression of NF-κB (p < 0.05).', 33233042);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5657, 'mouse', 'increase', 'null', 'low-throughput', 'As shown in Fig. 4h and i, the expression of occludin and ZO-1 in the colon tissue of germ-free mice significantly increased after human fecal transplantation (p < 0.05).', 33233042);
INSERT INTO gutmgene.metabolite_gene_change_results (`index`, host_type, Alteration, substrate_name, throughput, description, Evidence) VALUES (5658, 'mouse', 'increase', 'null', 'low-throughput', 'As shown in Fig. 4h and i, the expression of occludin and ZO-1 in the colon tissue of germ-free mice significantly increased after human fecal transplantation (p < 0.05).', 33233042);