create table gut_microbiota_gene_change_results
(
    `index`               int  null,
    host_type             text null,
    Alteration            text null,
    throughput            text null,
    description           text null,
    experimental_method   text null,
    sequencing_technology text null,
    Evidence              int  null
)collate = utf8mb4_bin;

INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3078, 'human', 'decrease', 'low-throughput', 'The expression of several chemokine genes, some of which have reported antimicrobial properties37 (CXCL6, CCL20), were negatively correlated with the relative abundance of Eubacterium rectale (OTU 120) in the ileum, and Streptococcus (OTU 37) and Eikenella (OTU 39) in the rectum, suggesting that these species are the most susceptible to the activity of these chemokines.', 'cell culture', 'HPLC analysis,qPCR,LC/MS analysis', 31142855);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3079, 'human', 'decrease', 'low-throughput', 'The expression of several chemokine genes, some of which have reported antimicrobial properties37 (CXCL6, CCL20), were negatively correlated with the relative abundance of Eubacterium rectale (OTU 120) in the ileum, and Streptococcus (OTU 37) and Eikenella (OTU 39) in the rectum, suggesting that these species are the most susceptible to the activity of these chemokines.', 'cell culture', 'HPLC analysis,qPCR,LC/MS analysis', 31142855);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3080, 'human', 'increase', 'low-throughput', 'Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.', 'cell culture', 'HPLC analysis,qPCR,LC/MS analysis', 31142855);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3081, 'human', 'increase', 'low-throughput', 'Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.', 'cell culture', 'HPLC analysis,qPCR,LC/MS analysis', 31142855);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3082, 'human', 'increase', 'low-throughput', 'Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.', 'cell culture', 'HPLC analysis,qPCR,LC/MS analysis', 31142855);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3084, 'human', 'decrease', 'low-throughput', 'The expression of several chemokine genes, some of which have reported antimicrobial properties37 (CXCL6, CCL20), were negatively correlated with the relative abundance of Eubacterium rectale (OTU 120) in the ileum, and Streptococcus (OTU 37) and Eikenella (OTU 39) in the rectum, suggesting that these species are the most susceptible to the activity of these chemokines.', 'cell culture', 'HPLC analysis,qPCR,LC/MS analysis', 31142855);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3085, 'human', 'decrease', 'low-throughput', 'The expression of several chemokine genes, some of which have reported antimicrobial properties37 (CXCL6, CCL20), were negatively correlated with the relative abundance of Eubacterium rectale (OTU 120) in the ileum, and Streptococcus (OTU 37) and Eikenella (OTU 39) in the rectum, suggesting that these species are the most susceptible to the activity of these chemokines.', 'cell culture', 'HPLC analysis,qPCR,LC/MS analysis', 31142855);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3086, 'human', 'decrease', 'low-throughput', 'The genes involved included known IBD-associated host–microbial interaction factors, including DUOX2 and its maturation factor DUOXA231,36, both of which were negatively associated with the abundance of Ruminococcaceae UCG 005 (OTU 89) in the ileum.', 'cell culture', 'HPLC analysis,qPCR,LC/MS analysis', 31142855);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3087, 'human', 'decrease', 'low-throughput', 'The expression of several chemokine genes, some of which have reported antimicrobial properties37 (CXCL6, CCL20), were negatively correlated with the relative abundance of Eubacterium rectale (OTU 120) in the ileum, and Streptococcus (OTU 37) and Eikenella (OTU 39) in the rectum, suggesting that these species are the most susceptible to the activity of these chemokines.', 'cell culture', 'HPLC analysis,qPCR,LC/MS analysis', 31142855);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3109, 'human', 'decrease', 'low-throughput', 'Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).', 'in vitro experiments', 'GC/MS analysis', 26241311);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3110, 'human', 'decrease', 'low-throughput', 'Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).', 'in vitro experiments', 'GC/MS analysis', 26241311);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3111, 'human', 'decrease', 'low-throughput', 'Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).', 'in vitro experiments', 'GC/MS analysis', 26241311);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3141, 'human', 'increase', 'low-throughput', 'In the evaluation of our study, a lower count of A.muciniphila and F. prausnitzii was observed in the micro biota of allergic children. F. prausnitzii and A.muciniphila may have induced anti-inflammatory cytokine IL-10 and may prevent the secretion of pro-inflammatory cytokines like IL-12. These findings suggest that A. muciniphila and F.prausnitzii may suppress inflammation through its secreted metabolites.29 We suggest that all of the aforementioned mechanisms that result from the decrease of F.prausnitzii and A. muciniphila may contribute to allergic asthma in our patient group.', 'cell culture', 'qPCR', 30765132);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3142, 'human', 'decrease', 'low-throughput', 'In the evaluation of our study, a lower count of A.muciniphila and F. prausnitzii was observed in the micro biota of allergic children. F. prausnitzii and A.muciniphila may have induced anti-inflammatory cytokine IL-10 and may prevent the secretion of pro-inflammatory cytokines like IL-12. These findings suggest that A. muciniphila and F.prausnitzii may suppress inflammation through its secreted metabolites.29 We suggest that all of the aforementioned mechanisms that result from the decrease of F.prausnitzii and A. muciniphila may contribute to allergic asthma in our patient group.', 'cell culture', 'qPCR', 30765132);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3143, 'human', 'decrease', 'low-throughput', 'In the evaluation of our study, a lower count of A.muciniphila and F. prausnitzii was observed in the micro biota of allergic children. F. prausnitzii and A.muciniphila may have induced anti-inflammatory cytokine IL-10 and may prevent the secretion of pro-inflammatory cytokines like IL-12. These findings suggest that A. muciniphila and F.prausnitzii may suppress inflammation through its secreted metabolites.29 We suggest that all of the aforementioned mechanisms that result from the decrease of F.prausnitzii and A. muciniphila may contribute to allergic asthma in our patient group.', 'cell culture', 'qPCR', 30765132);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3144, 'human', 'increase', 'low-throughput', 'In the evaluation of our study, a lower count of A.muciniphila and F. prausnitzii was observed in the micro biota of allergic children. F. prausnitzii and A.muciniphila may have induced anti-inflammatory cytokine IL-10 and may prevent the secretion of pro-inflammatory cytokines like IL-12. These findings suggest that A. muciniphila and F.prausnitzii may suppress inflammation through its secreted metabolites.29 We suggest that all of the aforementioned mechanisms that result from the decrease of F.prausnitzii and A. muciniphila may contribute to allergic asthma in our patient group.', 'cell culture', 'qPCR', 30765132);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3145, 'human', 'decrease', 'low-throughput', 'In the evaluation of our study, a lower count of A.muciniphila and F. prausnitzii was observed in the micro biota of allergic children. F. prausnitzii and A.muciniphila may have induced anti-inflammatory cytokine IL-10 and may prevent the secretion of pro-inflammatory cytokines like IL-12. These findings suggest that A. muciniphila and F.prausnitzii may suppress inflammation through its secreted metabolites.29 We suggest that all of the aforementioned mechanisms that result from the decrease of F.prausnitzii and A. muciniphila may contribute to allergic asthma in our patient group.', 'cell culture', 'qPCR', 30765132);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3146, 'human', 'decrease', 'low-throughput', 'In the evaluation of our study, a lower count of A.muciniphila and F. prausnitzii was observed in the micro biota of allergic children. F. prausnitzii and A.muciniphila may have induced anti-inflammatory cytokine IL-10 and may prevent the secretion of pro-inflammatory cytokines like IL-12. These findings suggest that A. muciniphila and F.prausnitzii may suppress inflammation through its secreted metabolites.29 We suggest that all of the aforementioned mechanisms that result from the decrease of F.prausnitzii and A. muciniphila may contribute to allergic asthma in our patient group.', 'cell culture', 'qPCR', 30765132);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3149, 'human', 'increase', 'low-throughput', 'Based on these results and on the frequent expression of CCR6, CXCR3, and CXCR6 by colon-derived DP8a cells, we then further focused on CCR6, CXCR3, and CXCR6 as potential markers for F prausnitzii–specific DP8a cells: in comparison to LPLs, CCR6þ, CXCR6þ, and CXCR3þ cells represented only 55%, 32%, and 37%, respectively, of DP8a cells derived from PBMCs, and almost 18% of the latter cells expressed the 3 molecules.', 'cell culture', 'ELISA', 29981781);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3150, 'human', 'increase', 'low-throughput', 'Based on these results and on the frequent expression of CCR6, CXCR3, and CXCR6 by colon-derived DP8a cells, we then further focused on CCR6, CXCR3, and CXCR6 as potential markers for F prausnitzii–specific DP8a cells: in comparison to LPLs, CCR6þ, CXCR6þ, and CXCR3þ cells represented only 55%, 32%, and 37%, respectively, of DP8a cells derived from PBMCs, and almost 18% of the latter cells expressed the 3 molecules.', 'cell culture', 'ELISA', 29981781);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3151, 'human', 'increase', 'low-throughput', 'Based on these results and on the frequent expression of CCR6, CXCR3, and CXCR6 by colon-derived DP8a cells, we then further focused on CCR6, CXCR3, and CXCR6 as potential markers for F prausnitzii–specific DP8a cells: in comparison to LPLs, CCR6þ, CXCR6þ, and CXCR3þ cells represented only 55%, 32%, and 37%, respectively, of DP8a cells derived from PBMCs, and almost 18% of the latter cells expressed the 3 molecules.', 'cell culture', 'ELISA', 29981781);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3152, 'human', 'increase', 'low-throughput', 'For the TJ protein ZO-2 we found a slight upregulation of mRNA after incubation with EcN.', 'cell culture', 'RT-qPCR,immunofluorescence staining,Western blot assay', 17087734);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3153, 'human', 'increase', 'low-throughput', 'For the TJ protein ZO-2 we found a slight upregulation of mRNA after incubation with EcN.', 'cell culture', 'RT-qPCR,immunofluorescence staining,Western blot assay', 17087734);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3154, 'human', 'decrease', 'low-throughput', 'The activity of PKCz is reduced during incubation with EcN as well as during co-incubation with EcN and EPEC.', 'cell culture', 'RT-qPCR,immunofluorescence staining,Western blot assay', 17087734);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3155, 'human', 'decrease', 'low-throughput', 'The activity of PKCz is reduced during incubation with EcN as well as during co-incubation with EcN and EPEC.', 'cell culture', 'RT-qPCR,immunofluorescence staining,Western blot assay', 17087734);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3162, 'human', 'increase', 'low-throughput', 'butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.', 'cell culture', 'HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay', 23300800);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3163, 'human', 'increase', 'low-throughput', 'butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.', 'cell culture', 'HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay', 23300800);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3164, 'human', 'increase', 'low-throughput', 'butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.', 'cell culture', 'HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay', 23300800);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3165, 'human', 'increase', 'low-throughput', 'The activation of AP-1 was strongly correlated with the presence of butyrate in the CM.', 'cell culture', 'HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay', 23300800);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3166, 'human', 'increase', 'low-throughput', 'butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.', 'cell culture', 'HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay', 23300800);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3167, 'human', 'increase', 'low-throughput', 'butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.', 'cell culture', 'HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay', 23300800);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3168, 'human', 'increase', 'low-throughput', 'butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.', 'cell culture', 'HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay', 23300800);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3169, 'human', 'increase', 'low-throughput', 'The activation of AP-1 was strongly correlated with the presence of butyrate in the CM.', 'cell culture', 'HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay', 23300800);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3180, 'human', 'decrease', 'low-throughput', 'Both JIM8772 and CIP102503T supernatants and their corresponding <3-kDa fractions provoked a drastic decrease of IL-8 secretion induced by TNF-α.', 'cell culture', 'HPLC analysis', 21602373);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3181, 'human', 'decrease', 'low-throughput', 'We showed that the supernatant of S. salivarius also inhibited NF-κB activity induced in HT-29 cells with another proinflammatory cytokine, IL-1β, and the TLR5 ligand, flagellin.', 'cell culture', 'HPLC analysis', 21602373);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3182, 'human', 'decrease', 'low-throughput', 'We showed that the supernatant of S. salivarius also inhibited NF-κB activity induced in HT-29 cells with another proinflammatory cytokine, IL-1β, and the TLR5 ligand, flagellin.', 'cell culture', 'HPLC analysis', 21602373);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3183, 'human', 'decrease', 'low-throughput', 'Both JIM8772 and CIP102503T supernatants and their corresponding <3-kDa fractions provoked a drastic decrease of IL-8 secretion induced by TNF-α.', 'cell culture', 'HPLC analysis', 21602373);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3184, 'human', 'decrease', 'low-throughput', 'We showed that the supernatant of S. salivarius also inhibited NF-κB activity induced in HT-29 cells with another proinflammatory cytokine, IL-1β, and the TLR5 ligand, flagellin.', 'cell culture', 'HPLC analysis', 21602373);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3185, 'human', 'decrease', 'low-throughput', 'We showed that the supernatant of S. salivarius also inhibited NF-κB activity induced in HT-29 cells with another proinflammatory cytokine, IL-1β, and the TLR5 ligand, flagellin.', 'cell culture', 'HPLC analysis', 21602373);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3192, 'human', 'increase', 'low-throughput', 'A. muciniphila activates the NF-κB pathway through TLR4 and TLR2 receptors.', 'cell culture', 'LC/MS/MS analysis', 28249045);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3193, 'human', 'increase', 'low-throughput', 'A. muciniphila activates the NF-κB pathway through TLR4 and TLR2 receptors.', 'cell culture', 'LC/MS/MS analysis', 28249045);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3237, 'human', 'increase', 'low-throughput', 'The expression of barrier-forming genes, viz. ZO-1, Claudin-1, Occludin and Cingulin were elevated significantly (p < 0.05) by 2.33 ± 0.4, 2.47 ± 0.3, 2.70 ± 0.5 and 3.13 ± 0.6 folds, respectively, after exposure of Caco-2 cells to L. rhamnosus for 6 h as compared to untreated cells (Fig. 2A-D).', 'control experiment,in vitro bacterial culture,cell culture', 'real-time qPCR,real-time PCR,qPCR', 31732863);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3238, 'human', 'increase', 'low-throughput', 'The expression of barrier-forming genes, viz. ZO-1, Claudin-1, Occludin and Cingulin were elevated significantly (p < 0.05) by 2.33 ± 0.4, 2.47 ± 0.3, 2.70 ± 0.5 and 3.13 ± 0.6 folds, respectively, after exposure of Caco-2 cells to L. rhamnosus for 6 h as compared to untreated cells (Fig. 2A-D).', 'control experiment,in vitro bacterial culture,cell culture', 'real-time qPCR,real-time PCR,qPCR', 31732863);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3239, 'human', 'increase', 'low-throughput', 'The expression of barrier-forming genes, viz. ZO-1, Claudin-1, Occludin and Cingulin were elevated significantly (p < 0.05) by 2.33 ± 0.4, 2.47 ± 0.3, 2.70 ± 0.5 and 3.13 ± 0.6 folds, respectively, after exposure of Caco-2 cells to L. rhamnosus for 6 h as compared to untreated cells (Fig. 2A-D).', 'control experiment,in vitro bacterial culture,cell culture', 'real-time qPCR,real-time PCR,qPCR', 31732863);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3240, 'human', 'increase', 'low-throughput', 'The expression of barrier-forming genes, viz. ZO-1, Claudin-1, Occludin and Cingulin were elevated significantly (p < 0.05) by 2.33 ± 0.4, 2.47 ± 0.3, 2.70 ± 0.5 and 3.13 ± 0.6 folds, respectively, after exposure of Caco-2 cells to L. rhamnosus for 6 h as compared to untreated cells (Fig. 2A-D).', 'control experiment,in vitro bacterial culture,cell culture', 'real-time qPCR,real-time PCR,qPCR', 31732863);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3241, 'human', 'decrease', 'low-throughput', 'In contrast to the remarkable elevated levels of hBD-2 mRNA after stimulation with E. coli individually, exclusion, competition and displacement of E. coli by L. rhamnosus considerably reduced its expression which was indicative of the induction of healthy conditions in presence of L. rhamnosus under the pathogenic milieu set in by E. coli alone.', 'control experiment,in vitro bacterial culture,cell culture', 'real-time qPCR,real-time PCR,qPCR', 31732863);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3345, 'human', 'increase', 'low-throughput', 'For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3346, 'human', 'increase', 'low-throughput', 'For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3347, 'human', 'increase', 'low-throughput', 'For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3348, 'human', 'increase', 'low-throughput', 'For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3349, 'human', 'decrease', 'low-throughput', 'As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3350, 'human', 'decrease', 'low-throughput', 'As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3351, 'human', 'decrease', 'low-throughput', 'As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3352, 'human', 'decrease', 'low-throughput', 'As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3353, 'human', 'decrease', 'low-throughput', 'As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3354, 'human', 'decrease', 'low-throughput', 'As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3355, 'human', 'decrease', 'low-throughput', 'As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3356, 'human', 'decrease', 'low-throughput', 'As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3357, 'human', 'decrease', 'low-throughput', 'As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).', 'in vitro bacterial culture', '16S rRNA sequence,ELISA', 28713353);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3367, 'human', 'increase', 'low-throughput', 'In the present study, we focused on the impact of SCFAs and the role of PPAR-γin intestinal ANGPTL4 expression. Here, we show that ANGPTL4 expression can be regulated by SCFAs both in vitro and in vivo, with butyrate being the most efficient activator.', 'control experiment', 'real-time qPCR，Western blot assay，ELISA', 23518684);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3417, 'human', 'decrease', 'low-throughput', 'In the present study, LP mitigated the UCB-induced redistribution of occludin, ZO-1, claudin-1, claudin-4, JAM-1 and F-actin.', 'control experiment,in vitro bacterial culture,cell culture', 'Western blot assay, real-time qPCR', 20412608);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3418, 'human', 'decrease', 'low-throughput', 'In the present study, LP mitigated the UCB-induced redistribution of occludin, ZO-1, claudin-1, claudin-4, JAM-1 and F-actin.', 'control experiment,in vitro bacterial culture,cell culture', 'Western blot assay, real-time qPCR', 20412608);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3419, 'human', 'decrease', 'low-throughput', 'In the present study, LP mitigated the UCB-induced redistribution of occludin, ZO-1, claudin-1, claudin-4, JAM-1 and F-actin.', 'control experiment,in vitro bacterial culture,cell culture', 'Western blot assay, real-time qPCR', 20412608);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3420, 'human', 'decrease', 'low-throughput', 'In the present study, LP mitigated the UCB-induced redistribution of occludin, ZO-1, claudin-1, claudin-4, JAM-1 and F-actin.', 'control experiment,in vitro bacterial culture,cell culture', 'Western blot assay, real-time qPCR', 20412608);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3421, 'human', 'decrease', 'low-throughput', 'In the present study, LP mitigated the UCB-induced redistribution of occludin, ZO-1, claudin-1, claudin-4, JAM-1 and F-actin.', 'control experiment,in vitro bacterial culture,cell culture', 'Western blot assay, real-time qPCR', 20412608);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3422, 'human', 'increase', 'low-throughput', 'F. prausnitzii, B. longum and F. prausnitzii supernatant clearly facilitated the induction of IL-10 and TGF-β1, while induced relatively mild production of IL-12p70 in both cellular and animal models. The F. prausnitzii, B. longum and supernatant differed in their capacity to induce T-bet, GATA-3 and ROR-γt mRNA expression in human PBMCs (both bacterial strains inhibited the expression of ROR-γt while supernatant inhibited the T-bet and GATA-3). However, all of them induced the Foxp3 and Treg production and ameliorated the TNBS-induced colitis. In addition, F. prausnitzii supernatant exhibited the supreme anti-inflammatory capacity.', 'control experiment,in vitro bacterial culture', 'ELISA,real-time qPCR', 23643066);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3423, 'human', 'increase', 'low-throughput', 'F. prausnitzii, B. longum and F. prausnitzii supernatant clearly facilitated the induction of IL-10 and TGF-β1, while induced relatively mild production of IL-12p70 in both cellular and animal models. The F. prausnitzii, B. longum and supernatant differed in their capacity to induce T-bet, GATA-3 and ROR-γt mRNA expression in human PBMCs (both bacterial strains inhibited the expression of ROR-γt while supernatant inhibited the T-bet and GATA-3). However, all of them induced the Foxp3 and Treg production and ameliorated the TNBS-induced colitis. In addition, F. prausnitzii supernatant exhibited the supreme anti-inflammatory capacity.', 'control experiment,in vitro bacterial culture', 'ELISA,real-time qPCR', 23643066);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3424, 'human', 'increase', 'low-throughput', 'F. prausnitzii, B. longum and F. prausnitzii supernatant clearly facilitated the induction of IL-10 and TGF-β1, while induced relatively mild production of IL-12p70 in both cellular and animal models. The F. prausnitzii, B. longum and supernatant differed in their capacity to induce T-bet, GATA-3 and ROR-γt mRNA expression in human PBMCs (both bacterial strains inhibited the expression of ROR-γt while supernatant inhibited the T-bet and GATA-3). However, all of them induced the Foxp3 and Treg production and ameliorated the TNBS-induced colitis. In addition, F. prausnitzii supernatant exhibited the supreme anti-inflammatory capacity.', 'control experiment,in vitro bacterial culture', 'ELISA,real-time qPCR', 23643066);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3451, 'human', 'increase', 'low-throughput', 'These data show that B dentium–secreted products, including acetate, can modulate 5-HT production by host enterochromaffin cells.', 'in vitro bacterial culture', 'immunostaining, qPCR，LC-MS/MS analysis', 32795610);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3504, 'human', 'decrease', 'low-throughput', 'A Gram-negative anaerobic bacterium, strain 4, capable of metabolizing diosmetin-7-O-glucoside was newly isolated.Its 16S rRNA sequence displayed 99% similarity with that of Escherichia. Then strain 4 was identified as a species of thegenus Escherichia and was named Escherichia sp. 4. Additionally,  an ultraperformance liquid chromatography/quadrupole-time-of-flight mass spectrometry  technique combined with Metabolynx software method was established to screenthe metabolites of diosmetin-7-O-glucoside.', 'in vitro bacterial culture,control experiment', 'HPLC analysis,16S rRNA sequence', 25382172);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3511, 'human', 'decrease', 'low-throughput', 'Inactivated probiotic strains can affect TLR4 expression in a down-regulatory direction as with lipopolysaccharides  after 12 hours. LGG significantly down-regulated expression of p38 while IκB expression was significantly reduced  in L.del-treated DCs. Moreover, we found these Lactobacillus strains could even modify the immune response at  post-transcriptional level by modifying miRNAs expression. Based on our results LGG induced a significant down_regulatory effect on miR-146a expression which is known as a novel fine negative regulator of immune response  targeting NFκB. On the other hand, miR-155 was up-regulated by LGG which is consistent with down-regulation  of p38 and in LGG-treated DCs.', 'cell culture', 'fluorescence microscopy,RT-PCR', 22476320);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3512, 'human', 'decrease', 'low-throughput', 'Inactivated probiotic strains can affect TLR4 expression in a down-regulatory direction as with lipopolysaccharides  after 12 hours. LGG significantly down-regulated expression of p38 while IκB expression was significantly reduced  in L.del-treated DCs. Moreover, we found these Lactobacillus strains could even modify the immune response at  post-transcriptional level by modifying miRNAs expression. Based on our results LGG induced a significant down_regulatory effect on miR-146a expression which is known as a novel fine negative regulator of immune response  targeting NFκB. On the other hand, miR-155 was up-regulated by LGG which is consistent with down-regulation  of p38 and in LGG-treated DCs.', 'cell culture', 'fluorescence microscopy,RT-PCR', 22476320);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3513, 'human', 'decrease', 'low-throughput', 'Inactivated probiotic strains can affect TLR4 expression in a down-regulatory direction as with lipopolysaccharides  after 12 hours. LGG significantly down-regulated expression of p38 while IκB expression was significantly reduced  in L.del-treated DCs. Moreover, we found these Lactobacillus strains could even modify the immune response at  post-transcriptional level by modifying miRNAs expression. Based on our results LGG induced a significant down_regulatory effect on miR-146a expression which is known as a novel fine negative regulator of immune response  targeting NFκB. On the other hand, miR-155 was up-regulated by LGG which is consistent with down-regulation  of p38 and in LGG-treated DCs.', 'cell culture', 'fluorescence microscopy,RT-PCR', 22476320);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3514, 'human', 'decrease', 'low-throughput', 'Inactivated probiotic strains can affect TLR4 expression in a down-regulatory direction as with lipopolysaccharides  after 12 hours. LGG significantly down-regulated expression of p38 while IκB expression was significantly reduced  in L.del-treated DCs. Moreover, we found these Lactobacillus strains could even modify the immune response at  post-transcriptional level by modifying miRNAs expression. Based on our results LGG induced a significant down_regulatory effect on miR-146a expression which is known as a novel fine negative regulator of immune response  targeting NFκB. On the other hand, miR-155 was up-regulated by LGG which is consistent with down-regulation  of p38 and in LGG-treated DCs.', 'cell culture', 'fluorescence microscopy,RT-PCR', 22476320);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3515, 'human', 'increase', 'low-throughput', 'Inactivated probiotic strains can affect TLR4 expression in a down-regulatory direction as with lipopolysaccharides  after 12 hours. LGG significantly down-regulated expression of p38 while IκB expression was significantly reduced  in L.del-treated DCs. Moreover, we found these Lactobacillus strains could even modify the immune response at  post-transcriptional level by modifying miRNAs expression. Based on our results LGG induced a significant down_regulatory effect on miR-146a expression which is known as a novel fine negative regulator of immune response  targeting NFκB. On the other hand, miR-155 was up-regulated by LGG which is consistent with down-regulation  of p38 and in LGG-treated DCs.', 'cell culture', 'fluorescence microscopy,RT-PCR', 22476320);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3529, 'human', 'increase', 'low-throughput', 'DIF-3 induces proteolysis of cyclin D1 and c-Myc in HCT-116 cells via activation of GSK-3b.', 'cell culture', 'Western blot assay,RT-PCR', 25913757);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3530, 'human', 'decrease', 'low-throughput', 'The amount of TCF7L2 increased gradually over time in control cells, but it was markedly reduced by DIF-3 in a time and dose-dependent manner.', 'cell culture', 'Western blot assay,RT-PCR', 25913757);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3531, 'human', 'decrease', 'low-throughput', 'DIF-3 decreased the expression of cyclin D1 and cMyc to a level comparable to the reduction induced by DIF-1.', 'cell culture', 'Western blot assay,RT-PCR', 25913757);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3532, 'human', 'decrease', 'low-throughput', 'DIF-3 decreased the expression of cyclin D1 and cMyc to a level comparable to the reduction induced by DIF-1.', 'cell culture', 'Western blot assay,RT-PCR', 25913757);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3533, 'human', 'decrease', 'low-throughput', 'Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC3) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.', 'in vitro bacterial culture,cell culture', 'LC/MS analysis,qPCR', 30683619);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3534, 'human', 'decrease', 'low-throughput', 'Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC4) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.', 'in vitro bacterial culture,cell culture', 'LC/MS analysis,qPCR', 30683619);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3535, 'human', 'decrease', 'low-throughput', 'Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC5) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.', 'in vitro bacterial culture,cell culture', 'LC/MS analysis,qPCR', 30683619);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3540, 'human', 'increase', 'low-throughput', 'In summary, our results clearly demonstrate that different E. faecalis strains differentially induce colitis in monoassociated IL-10-/- mice and that severe E. faecalis-induced colitis resulted in decreased Alix/AIP1 protein expression as well as ERK1/2 phosphorylation. Functional studies revealed no effect of Alix/AIP1 on bacteria- and EGF-mediated effects on the epithelium. The 13 differentially regulated proteins might be ofinterest aspotentialmarkers for severeintestinalinflammation.', 'in vitro bacterial culture,cell culture', 'qPCR,ELISA,Western blot assay', 19166300);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3541, 'human', 'decrease', 'low-throughput', 'In summary, our results clearly demonstrate that different E. faecalis strains differentially induce colitis in monoassociated IL-10-/- mice and that severe E. faecalis-induced colitis resulted in decreased Alix/AIP1 protein expression as well as ERK1/2 phosphorylation. Functional studies revealed no effect of Alix/AIP1 on bacteria- and EGF-mediated effects on the epithelium. The 13 differentially regulated proteins might be ofinterest aspotentialmarkers for severeintestinalinflammation.', 'in vitro bacterial culture,cell culture', 'qPCR,ELISA,Western blot assay', 19166300);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3566, 'human', 'increase', 'low-throughput', 'Regulatory T cells (Tregs) that express the transcription factor Foxp3 are critical for regulating intestinal inflammation. Candidate microbe approaches have identified bacterial species and strain-specific molecules that can affect intestinal immune responses, including species that modulate Treg responses. Because neither all humans nor mice harbor the same bacterial strains, we posited that more prevalent factors exist that regulate the number and function of colonic Tregs. We determined that short chain fatty acids (SCFA), gut microbiota-derived bacterial fermentation products, regulate the size and function of the colonic Treg pool and protect against colitis in a Ffar2(GPR43)-dependent manner in mice. Our study reveals that a class of abundant microbial metabolites underlies adaptive immune microbiota co-adaptation and promotes colonic homeostasis and health.', 'cell culture', 'null', 23828891);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3567, 'human', 'increase', 'low-throughput', 'Regulatory T cells (Tregs) that express the transcription factor Foxp3 are critical for regulating intestinal inflammation. Candidate microbe approaches have identified bacterial species and strain-specific molecules that can affect intestinal immune responses, including species that modulate Treg responses. Because neither all humans nor mice harbor the same bacterial strains, we posited that more prevalent factors exist that regulate the number and function of colonic Tregs. We determined that short chain fatty acids (SCFA), gut microbiota-derived bacterial fermentation products, regulate the size and function of the colonic Treg pool and protect against colitis in a Ffar2(GPR43)-dependent manner in mice. Our study reveals that a class of abundant microbial metabolites underlies adaptive immune microbiota co-adaptation and promotes colonic homeostasis and health.', 'cell culture', 'null', 23828891);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3568, 'human', 'increase', 'low-throughput', 'Regulatory T cells (Tregs) that express the transcription factor Foxp3 are critical for regulating intestinal inflammation. Candidate microbe approaches have identified bacterial species and strain-specific molecules that can affect intestinal immune responses, including species that modulate Treg responses. Because neither all humans nor mice harbor the same bacterial strains, we posited that more prevalent factors exist that regulate the number and function of colonic Tregs. We determined that short chain fatty acids (SCFA), gut microbiota-derived bacterial fermentation products, regulate the size and function of the colonic Treg pool and protect against colitis in a Ffar2(GPR43)-dependent manner in mice. Our study reveals that a class of abundant microbial metabolites underlies adaptive immune microbiota co-adaptation and promotes colonic homeostasis and health.', 'cell culture', 'null', 23828891);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3569, 'human', 'decrease', 'low-throughput', 'F. prausnitzii exhibits anti-inflammatory effects, partly due to secreted metabolites able to block nuclear factor-κB activation and the secretion of proinflammatory mediators (41). Oral administration of F. prausnitzii or of supernatant from F. prausnitzii cultures increased the production of IL-10 by blood mononuclear cells and reduced the production of the proinflammatory mediator-like IL-12 in the colon.', 'in vitro bacterial culture', 'RT-qPCR', 20876719);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3570, 'human', 'decrease', 'low-throughput', 'F. prausnitzii exhibits anti-inflammatory effects, partly due to secreted metabolites able to block nuclear factor-κB activation and the secretion of proinflammatory mediators (41). Oral administration of F. prausnitzii or of supernatant from F. prausnitzii cultures increased the production of IL-10 by blood mononuclear cells and reduced the production of the proinflammatory mediator-like IL-12 in the colon.', 'in vitro bacterial culture', 'RT-qPCR', 20876719);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3571, 'human', 'decrease', 'low-throughput', 'F. prausnitzii exhibits anti-inflammatory effects, partly due to secreted metabolites able to block nuclear factor-κB activation and the secretion of proinflammatory mediators (41). Oral administration of F. prausnitzii or of supernatant from F. prausnitzii cultures increased the production of IL-10 by blood mononuclear cells and reduced the production of the proinflammatory mediator-like IL-12 in the colon.', 'in vitro bacterial culture', 'RT-qPCR', 20876719);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3572, 'human', 'increase', 'low-throughput', 'F. prausnitzii exhibits anti-inflammatory effects, partly due to secreted metabolites able to block nuclear factor-κB activation and the secretion of proinflammatory mediators (41). Oral administration of F. prausnitzii or of supernatant from F. prausnitzii cultures increased the production of IL-10 by blood mononuclear cells and reduced the production of the proinflammatory mediator-like IL-12 in the colon.', 'in vitro bacterial culture', 'RT-qPCR', 20876719);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3586, 'human', 'decrease', 'low-throughput', 'F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.', 'in vivo experiment,in vitro bacterial culture', 'null', 25900655);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3587, 'human', 'decrease', 'low-throughput', 'F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.', 'in vivo experiment,in vitro bacterial culture', 'null', 25900655);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3588, 'human', 'decrease', 'low-throughput', 'F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.', 'in vivo experiment,in vitro bacterial culture', 'null', 25900655);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3589, 'human', 'decrease', 'low-throughput', 'Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.', 'in vitro bacterial culture', '16S rRNA sequence,Western blot assay,RT-PCR', 30566883);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3590, 'human', 'increase', 'low-throughput', 'Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.', 'in vitro bacterial culture', '16S rRNA sequence,Western blot assay,RT-PCR', 30566883);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3591, 'human', 'increase', 'low-throughput', 'Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.', 'in vitro bacterial culture', '16S rRNA sequence,Western blot assay,RT-PCR', 30566883);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3592, 'human', 'increase', 'low-throughput', 'Our data demonstrate that a dysbiotic gut microbiota in RA patients, characterized by a decrease in Faecalibacterium and expansion of C. aerofaciens and E. lenta, could trigger inflammatory conditions in the gut that depend on the production of chemokines and IL-17A and compromise the gut epithelium integrity. It is possible that the inflammatory conditions can be modulated by prebiotics or probiotics. The therapeutic potential of the only probiotics, lactobacilli, used as treatment for RA is inconclusive, with some studies in favor of their use while others did not show significant improvement with lactobacilli using the American College of Rheumatology (ACR) response criteria for RA .', 'cell culture', '16S rRNA sequence,RT-qPCR', 27102666);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3593, 'human', 'increase', 'low-throughput', 'Our data demonstrate that a dysbiotic gut microbiota in RA patients, characterized by a decrease in Faecalibacterium and expansion of C. aerofaciens and E. lenta, could trigger inflammatory conditions in the gut that depend on the production of chemokines and IL-17A and compromise the gut epithelium integrity. It is possible that the inflammatory conditions can be modulated by prebiotics or probiotics. The therapeutic potential of the only probiotics, lactobacilli, used as treatment for RA is inconclusive, with some studies in favor of their use while others did not show significant improvement with lactobacilli using the American College of Rheumatology (ACR) response criteria for RA .', 'cell culture', '16S rRNA sequence,RT-qPCR', 27102666);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3594, 'human', 'increase', 'low-throughput', 'Our data demonstrate that a dysbiotic gut microbiota in RA patients, characterized by a decrease in Faecalibacterium and expansion of C. aerofaciens and E. lenta, could trigger inflammatory conditions in the gut that depend on the production of chemokines and IL-17A and compromise the gut epithelium integrity. It is possible that the inflammatory conditions can be modulated by prebiotics or probiotics. The therapeutic potential of the only probiotics, lactobacilli, used as treatment for RA is inconclusive, with some studies in favor of their use while others did not show significant improvement with lactobacilli using the American College of Rheumatology (ACR) response criteria for RA .', 'cell culture', '16S rRNA sequence,RT-qPCR', 27102666);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3595, 'human', 'increase', 'low-throughput', 'Our data demonstrate that a dysbiotic gut microbiota in RA patients, characterized by a decrease in Faecalibacterium and expansion of C. aerofaciens and E. lenta, could trigger inflammatory conditions in the gut that depend on the production of chemokines and IL-17A and compromise the gut epithelium integrity. It is possible that the inflammatory conditions can be modulated by prebiotics or probiotics. The therapeutic potential of the only probiotics, lactobacilli, used as treatment for RA is inconclusive, with some studies in favor of their use while others did not show significant improvement with lactobacilli using the American College of Rheumatology (ACR) response criteria for RA .', 'cell culture', '16S rRNA sequence,RT-qPCR', 27102666);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3598, 'human', 'increase', 'low-throughput', 'Here, we show that in colonic cell lines and in primary colonic cells, Enterococcus faecalis isolated from newborn babies possess the ability to regulate peroxisome proliferator-activated receptor-γ1 (PPARγ1) activity through phosphorylation. This results in elevated DNA binding and transcriptional activation of downstream target genes, including IL-10, a cytokine known to modulate innate immune function. Furthermore, phosphorylation appears tightly regulated as phospho-PPARγ1 becomes an immediate substrate for degradation possibly to curtail any extended transactivation. The involvement of PPARγ1 in a myriad of physiological processes further confirms that microflora-driven regulation might be important for a number of homeostatic strategies in the gut.', 'cell culture', 'Western blot assay,RT-PCR', 18234854);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3599, 'human', 'increase', 'low-throughput', 'Here, we show that in colonic cell lines and in primary colonic cells, Enterococcus faecalis isolated from newborn babies possess the ability to regulate peroxisome proliferator-activated receptor-γ1 (PPARγ1) activity through phosphorylation. This results in elevated DNA binding and transcriptional activation of downstream target genes, including IL-10, a cytokine known to modulate innate immune function. Furthermore, phosphorylation appears tightly regulated as phospho-PPARγ1 becomes an immediate substrate for degradation possibly to curtail any extended transactivation. The involvement of PPARγ1 in a myriad of physiological processes further confirms that microflora-driven regulation might be important for a number of homeostatic strategies in the gut.', 'cell culture', 'Western blot assay,RT-PCR', 18234854);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3600, 'human', 'increase', 'low-throughput', 'Here, we show that in colonic cell lines and in primary colonic cells, Enterococcus faecalis isolated from newborn babies possess the ability to regulate peroxisome proliferator-activated receptor-γ1 (PPARγ1) activity through phosphorylation. This results in elevated DNA binding and transcriptional activation of downstream target genes, including IL-10, a cytokine known to modulate innate immune function. Furthermore, phosphorylation appears tightly regulated as phospho-PPARγ1 becomes an immediate substrate for degradation possibly to curtail any extended transactivation. The involvement of PPARγ1 in a myriad of physiological processes further confirms that microflora-driven regulation might be important for a number of homeostatic strategies in the gut.', 'cell culture', 'Western blot assay,RT-PCR', 18234854);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3601, 'human', 'increase', 'low-throughput', 'Here, we show that in colonic cell lines and in primary colonic cells, Enterococcus faecalis isolated from newborn babies possess the ability to regulate peroxisome proliferator-activated receptor-γ1 (PPARγ1) activity through phosphorylation. This results in elevated DNA binding and transcriptional activation of downstream target genes, including IL-10, a cytokine known to modulate innate immune function. Furthermore, phosphorylation appears tightly regulated as phospho-PPARγ1 becomes an immediate substrate for degradation possibly to curtail any extended transactivation. The involvement of PPARγ1 in a myriad of physiological processes further confirms that microflora-driven regulation might be important for a number of homeostatic strategies in the gut.', 'cell culture', 'Western blot assay,RT-PCR', 18234854);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3602, 'human', 'decrease', 'low-throughput', 'Human and murine pregnancy were associated with reduced intestinal FXR signaling, with lower FGF19/15 and resultant increased hepatic bile acid synthesis. Terminal ileal ASBT protein was reduced in murine pregnancy. Cecal bile acid conjugation was reduced in pregnancy because of elevated bile salt hydrolase‐producing Bacteroidetes. CA supplementation induced intestinal FXR signaling, which was not abrogated by pregnancy, with strikingly similar changes to the microbiota and metabonome as identified in pregnancy. Conclusion: The altered intestinal microbiota of pregnancy enhance bile acid deconjugation, reducing ileal bile acid uptake and lowering FXR induction in enterocytes. This exacerbates the effects mediated by reduced bile acid uptake transporters in pregnancy. Thus, in pregnant women and mice, there is reduced FGF19/15‐mediated hepatic repression of hepatic bile acid synthesis, resulting in hypercholanemia.', 'in vivo experiment', 'RT-PCR,16S rRNA sequence', 30983011);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3603, 'human', 'increase', 'low-throughput', 'Human and murine pregnancy were associated with reduced intestinal FXR signaling, with lower FGF19/15 and resultant increased hepatic bile acid synthesis. Terminal ileal ASBT protein was reduced in murine pregnancy. Cecal bile acid conjugation was reduced in pregnancy because of elevated bile salt hydrolase‐producing Bacteroidetes. CA supplementation induced intestinal FXR signaling, which was not abrogated by pregnancy, with strikingly similar changes to the microbiota and metabonome as identified in pregnancy. Conclusion: The altered intestinal microbiota of pregnancy enhance bile acid deconjugation, reducing ileal bile acid uptake and lowering FXR induction in enterocytes. This exacerbates the effects mediated by reduced bile acid uptake transporters in pregnancy. Thus, in pregnant women and mice, there is reduced FGF19/15‐mediated hepatic repression of hepatic bile acid synthesis, resulting in hypercholanemia.', 'in vivo experiment', 'RT-PCR,16S rRNA sequence', 30983011);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3604, 'human', 'increase', 'low-throughput', 'The effect of Lactobacillus plantarum MB452 on tight junction integrity was determined by measuring trans-epithelial electrical resistance (TEER) across Caco-2 cell layers. L. plantarum MB452 caused a dose-dependent TEER increase across Caco-2 cell monolayers compared to control medium. Gene expression was compared in Caco-2 cells untreated or treated with L. plantarum MB452 for 10 hours. Caco-2 cell RNA was hybridised to human oligonucleotide arrays. Data was analysed using linear models and differently expressed genes were examined using pathway analysis tools. Nineteen tight junction-related genes had altered expression levels in response to L. plantarum MB452 (modified-P < 0.05, fold-change > 1.2), including those encoding occludin and its associated plaque proteins that anchor it to the cytoskeleton. L. plantarum MB452 also caused changes in tubulin and proteasome gene expression levels which may be linked to intestinal barrier function.', 'cell culture', 'null', 21143932);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3605, 'human', 'increase', 'low-throughput', 'The effect of Lactobacillus plantarum MB452 on tight junction integrity was determined by measuring trans-epithelial electrical resistance (TEER) across Caco-2 cell layers. L. plantarum MB452 caused a dose-dependent TEER increase across Caco-2 cell monolayers compared to control medium. Gene expression was compared in Caco-2 cells untreated or treated with L. plantarum MB452 for 10 hours. Caco-2 cell RNA was hybridised to human oligonucleotide arrays. Data was analysed using linear models and differently expressed genes were examined using pathway analysis tools. Nineteen tight junction-related genes had altered expression levels in response to L. plantarum MB452 (modified-P < 0.05, fold-change > 1.2), including those encoding occludin and its associated plaque proteins that anchor it to the cytoskeleton. L. plantarum MB452 also caused changes in tubulin and proteasome gene expression levels which may be linked to intestinal barrier function.', 'cell culture', 'null', 21143932);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3606, 'human', 'increase', 'low-throughput', 'The effect of Lactobacillus plantarum MB452 on tight junction integrity was determined by measuring trans-epithelial electrical resistance (TEER) across Caco-2 cell layers. L. plantarum MB452 caused a dose-dependent TEER increase across Caco-2 cell monolayers compared to control medium. Gene expression was compared in Caco-2 cells untreated or treated with L. plantarum MB452 for 10 hours. Caco-2 cell RNA was hybridised to human oligonucleotide arrays. Data was analysed using linear models and differently expressed genes were examined using pathway analysis tools. Nineteen tight junction-related genes had altered expression levels in response to L. plantarum MB452 (modified-P < 0.05, fold-change > 1.2), including those encoding occludin and its associated plaque proteins that anchor it to the cytoskeleton. L. plantarum MB452 also caused changes in tubulin and proteasome gene expression levels which may be linked to intestinal barrier function.', 'cell culture', 'null', 21143932);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3607, 'human', 'increase', 'low-throughput', 'The effect of Lactobacillus plantarum MB452 on tight junction integrity was determined by measuring trans-epithelial electrical resistance (TEER) across Caco-2 cell layers. L. plantarum MB452 caused a dose-dependent TEER increase across Caco-2 cell monolayers compared to control medium. Gene expression was compared in Caco-2 cells untreated or treated with L. plantarum MB452 for 10 hours. Caco-2 cell RNA was hybridised to human oligonucleotide arrays. Data was analysed using linear models and differently expressed genes were examined using pathway analysis tools. Nineteen tight junction-related genes had altered expression levels in response to L. plantarum MB452 (modified-P < 0.05, fold-change > 1.2), including those encoding occludin and its associated plaque proteins that anchor it to the cytoskeleton. L. plantarum MB452 also caused changes in tubulin and proteasome gene expression levels which may be linked to intestinal barrier function.', 'cell culture', 'null', 21143932);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3608, 'human', 'decrease', 'low-throughput', 'The effect of Lactobacillus plantarum MB452 on tight junction integrity was determined by measuring trans-epithelial electrical resistance (TEER) across Caco-2 cell layers. L. plantarum MB452 caused a dose-dependent TEER increase across Caco-2 cell monolayers compared to control medium. Gene expression was compared in Caco-2 cells untreated or treated with L. plantarum MB452 for 10 hours. Caco-2 cell RNA was hybridised to human oligonucleotide arrays. Data was analysed using linear models and differently expressed genes were examined using pathway analysis tools. Nineteen tight junction-related genes had altered expression levels in response to L. plantarum MB452 (modified-P < 0.05, fold-change > 1.2), including those encoding occludin and its associated plaque proteins that anchor it to the cytoskeleton. L. plantarum MB452 also caused changes in tubulin and proteasome gene expression levels which may be linked to intestinal barrier function.', 'cell culture', 'null', 21143932);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3609, 'human', 'increase', 'low-throughput', 'In this study, we investigated the effects of LA and LA-derived fatty acids on the activation of peroxisome proliferator-activated receptors (PPARs) which regulate whole-body energy metabolism. None of the fatty acids activated PPARd, whereas almost all activated PPARa in luciferase assays. Two fatty acids potently activated PPARg, a master regulator of adipocyte differentiation, with 10-oxo-12(Z)-octadecenoic acid (KetoA) having the most potency. In 3T3- L1 cells, KetoA induced adipocyte differentiation via the activation of PPARg, and increased adiponectin production and insulin-stimulated glucose uptake. These findings suggest that fatty acids, including KetoA, generated in gut by lactic acid bacteria may be involved in the regulation of host energy metabolism.', 'cell culture', 'RT-PCR', 25749343);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3611, 'human', 'decrease', 'low-throughput', 'The addition of the EPM had no significant effect on transcription of IL-1β, TNF-α or IL-10 in hDCs stimulated with L. plantarum (not shown) but significantly decreased the transcript levels of IL-12 at 20h by about 2 fold.', 'in vitro bacterial culture', 'RT-qPCR', 25910186);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3612, 'human', 'decrease', 'low-throughput', 'The addition of the EPM had no significant effect on transcription of IL-1β, TNF-α or IL-10 in hDCs stimulated with L. plantarum (not shown) but significantly decreased the transcript levels of IL-12 at 20h by about 2 fold.', 'in vitro bacterial culture', 'RT-qPCR', 25910186);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3613, 'human', 'decrease', 'low-throughput', 'The EPM of F. prausnitzii HTF-F decreases transcription and production of pro-inflammatory IL-12p70 in L. plantarum-activated hDCs', 'in vitro bacterial culture', 'RT-qPCR', 25910186);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3623, 'human', 'decrease', 'low-throughput', 'F. prausnitzii was found to be underrepresented in the gut microbiotas of patients with Crohn''s disease, active ulcerative colitis, alternating-type irritable bowel syndrome (IBS-A), and diabetes type II. One of its main fermentation products, the short-chain fatty acid butyrate, serves as the main energy source for colonocytes and has anti-inflammatory properties . F. prausnitzii has been proposed as a potential probiotic for treatment of gut inflammation, as the microbe stimulates the expression of the anti-inflammatory cytokine interleukin 10 (IL-10) in peripheral blood mononuclear cells in vitro. Furthermore, cell supernatants of F. prausnitzii culture reduced the secretion of proinflammatory IL-8 and strongly inhibited NF-κB activation in cancer cells . Butyrate alone did not provoke the observed inhibitory effect , demonstrating that F. prausnitzii likely secretes an unknown anti-inflammatory metabolite apart from butyrate.', 'in vitro bacterial culture', 'LC/MS analysis', 25002542);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3624, 'human', 'decrease', 'low-throughput', 'F. prausnitzii was found to be underrepresented in the gut microbiotas of patients with Crohn''s disease, active ulcerative colitis, alternating-type irritable bowel syndrome (IBS-A), and diabetes type II. One of its main fermentation products, the short-chain fatty acid butyrate, serves as the main energy source for colonocytes and has anti-inflammatory properties . F. prausnitzii has been proposed as a potential probiotic for treatment of gut inflammation, as the microbe stimulates the expression of the anti-inflammatory cytokine interleukin 10 (IL-10) in peripheral blood mononuclear cells in vitro. Furthermore, cell supernatants of F. prausnitzii culture reduced the secretion of proinflammatory IL-8 and strongly inhibited NF-κB activation in cancer cells . Butyrate alone did not provoke the observed inhibitory effect , demonstrating that F. prausnitzii likely secretes an unknown anti-inflammatory metabolite apart from butyrate.', 'in vitro bacterial culture', 'LC/MS analysis', 25002542);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3625, 'human', 'increase', 'low-throughput', 'F. prausnitzii was found to be underrepresented in the gut microbiotas of patients with Crohn''s disease, active ulcerative colitis, alternating-type irritable bowel syndrome (IBS-A), and diabetes type II. One of its main fermentation products, the short-chain fatty acid butyrate, serves as the main energy source for colonocytes and has anti-inflammatory properties . F. prausnitzii has been proposed as a potential probiotic for treatment of gut inflammation, as the microbe stimulates the expression of the anti-inflammatory cytokine interleukin 10 (IL-10) in peripheral blood mononuclear cells in vitro. Furthermore, cell supernatants of F. prausnitzii culture reduced the secretion of proinflammatory IL-8 and strongly inhibited NF-κB activation in cancer cells . Butyrate alone did not provoke the observed inhibitory effect , demonstrating that F. prausnitzii likely secretes an unknown anti-inflammatory metabolite apart from butyrate.', 'in vitro bacterial culture', 'LC/MS analysis', 25002542);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3629, 'human', 'increase', 'low-throughput', 'Here we reveal a relationship between gut microbiota and caspase recruitment domain family member 9 (CARD9), a susceptibility gene for inflammatory bowel disease (IBD) that functions in the immune response against microorganisms. CARD9 promotes recovery from colitis by promoting interleukin (IL)-22 production, and Card9�?�?mice are more susceptible to colitis. The microbiota is altered in Card9�?�?mice, and transfer of the microbiota from Card9�?�?to wild-type, germ-free recipients increases their susceptibility to colitis. The microbiota from Card9�?�?mice fails to metabolize tryptophan into metabolites that act as aryl hydrocarbon receptor (AHR) ligands. Intestinal inflammation is attenuated after inoculation of mice with three Lactobacillus strains capable of metabolizing tryptophan or by treatment with an AHR agonist. Reduced production of AHR ligands is also observed in the microbiota from individuals with IBD, particularly in those with CARD9 risk alleles associated with IBD. Our findings reveal that host genes affect the composition and function of the gut microbiota, altering the production of microbial metabolites and intestinal inflammation', 'in vitro bacterial culture', 'qPCR,16S rDNA sequence,HPLC analysis', 27158904);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3630, 'human', 'increase', 'low-throughput', 'Here we reveal a relationship between gut microbiota and caspase recruitment domain family member 9 (CARD9), a susceptibility gene for inflammatory bowel disease (IBD) that functions in the immune response against microorganisms. CARD9 promotes recovery from colitis by promoting interleukin (IL)-22 production, and Card9�?�?mice are more susceptible to colitis. The microbiota is altered in Card9�?�?mice, and transfer of the microbiota from Card9�?�?to wild-type, germ-free recipients increases their susceptibility to colitis. The microbiota from Card9�?�?mice fails to metabolize tryptophan into metabolites that act as aryl hydrocarbon receptor (AHR) ligands. Intestinal inflammation is attenuated after inoculation of mice with three Lactobacillus strains capable of metabolizing tryptophan or by treatment with an AHR agonist. Reduced production of AHR ligands is also observed in the microbiota from individuals with IBD, particularly in those with CARD9 risk alleles associated with IBD. Our findings reveal that host genes affect the composition and function of the gut microbiota, altering the production of microbial metabolites and intestinal inflammation', 'in vitro bacterial culture', 'qPCR,16S rDNA sequence,HPLC analysis', 27158904);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3631, 'human', 'decrease', 'low-throughput', 'Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host''s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p38 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.', 'in vivo experiment', 'RT-PCR', 27505423);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3645, 'human', 'decrease', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 'cell culture、in vitro bacterial culture、control experiment', 'Western blot assay、RT-PCR、HPLC analysis', 21889493);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3646, 'human', 'increase', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 'cell culture、in vitro bacterial culture、control experiment', 'Western blot assay、RT-PCR、HPLC analysis', 21889493);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3647, 'human', 'increase', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 'cell culture、in vitro bacterial culture、control experiment', 'Western blot assay、RT-PCR、HPLC analysis', 21889493);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3648, 'human', 'decrease', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 'cell culture、in vitro bacterial culture、control experiment', 'Western blot assay、RT-PCR、HPLC analysis', 21889493);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3649, 'human', 'increase', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 'cell culture、in vitro bacterial culture、control experiment', 'Western blot assay、RT-PCR、HPLC analysis', 21889493);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3650, 'human', 'increase', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 'cell culture、in vitro bacterial culture、control experiment', 'Western blot assay、RT-PCR、HPLC analysis', 21889493);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3651, 'human', 'decrease', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 'cell culture、in vitro bacterial culture、control experiment', 'Western blot assay、RT-PCR、HPLC analysis', 21889493);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3652, 'human', 'increase', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 'cell culture、in vitro bacterial culture、control experiment', 'Western blot assay、RT-PCR、HPLC analysis', 21889493);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3653, 'human', 'increase', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 'cell culture、in vitro bacterial culture、control experiment', 'Western blot assay、RT-PCR、HPLC analysis', 21889493);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3654, 'human', 'decrease', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 'cell culture、in vitro bacterial culture、control experiment', 'Western blot assay、RT-PCR、HPLC analysis', 21889493);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3655, 'human', 'increase', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 'cell culture、in vitro bacterial culture、control experiment', 'Western blot assay、RT-PCR、HPLC analysis', 21889493);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3656, 'human', 'increase', 'low-throughput', 'In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.', 'cell culture、in vitro bacterial culture、control experiment', 'Western blot assay、RT-PCR、HPLC analysis', 21889493);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3657, 'human', 'increase', 'low-throughput', 'S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'null', 18077037);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3658, 'human', 'increase', 'low-throughput', 'S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'null', 18077037);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3659, 'human', 'increase', 'low-throughput', 'S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'null', 18077037);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3660, 'human', 'increase', 'low-throughput', 'S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'null', 18077037);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3661, 'human', 'increase', 'low-throughput', 'S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'null', 18077037);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3662, 'human', 'increase', 'low-throughput', 'S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'null', 18077037);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3663, 'human', 'increase', 'low-throughput', 'S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'null', 18077037);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3664, 'human', 'increase', 'low-throughput', 'S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'null', 18077037);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3665, 'human', 'increase', 'low-throughput', 'S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'null', 18077037);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3666, 'human', 'decrease', 'low-throughput', 'B. infantis and MO decreased hepatic CYP7A1 and induced Sult2a1, Sult2a2, and Sult2a3 suggesting reduced BA synthesis and increased detoxification. Furthermore, B. infantis and MO increased ileal BA membrane receptor TGR5-regulated signaling. Together, via BA-regulated signaling, synbiotics B. infantis and MO have their unique and combined effects in reversing NASH.', 'in vitro bacterial culture', 'qPCR,LC/MS analysis', 29800811);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3676, 'human', 'increase', 'low-throughput', 'B. infantis and MO decreased hepatic CYP7A1 and induced Sult2a1, Sult2a2, and Sult2a3 suggesting reduced BA synthesis and increased detoxification. Furthermore, B. infantis and MO increased ileal BA membrane receptor TGR5-regulated signaling. Together, via BA-regulated signaling, synbiotics B. infantis and MO have their unique and combined effects in reversing NASH.', 'in vitro bacterial culture', 'qPCR,LC/MS analysis', 11040423);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3677, 'human', 'increase', 'low-throughput', 'There was a significant difference in TLR2 signalling induced by live and UV‐killed bacteria. However, the activation by live bacteria was only 1.1‐fold higher than UV‐killed bacteria; therefore, the difference was less than that seen in apical anaerobic conditions and may not be biologically relevant.', 'in vitro bacterial culture,cell culture,control experiment', 'null', 29112296);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3678, 'human', 'increase', 'low-throughput', 'For the activation of TLR2/6, the interaction between the environment and live/dead status was significant as shown in Figure 5b(p < .001). In apical anaerobic conditions, TLR2/6 activation by live F. prausnitzii was 1.6‐fold higher than by UV‐killed bacteria.', 'in vitro bacterial culture,cell culture,control experiment', 'null', 29112296);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3679, 'human', 'decrease', 'low-throughput', 'There was no TLR4 activation by either live or UV‐killed F. prausnitzii in both conventional and apical anaerobic conditions. The raw luminescence values for the activation of TLR2, TLR2/6,TLR4, and the activation of the control cell line (TLR‐null) are shown in Figure S3.', 'in vitro bacterial culture,cell culture,control experiment', 'null', 29112296);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3712, 'human', 'decrease', 'low-throughput', 'The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.', 'in vitro bacterial culture,cell culture', 'RT-PCR,fluorescence microscopy', 16534848);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3713, 'human', 'decrease', 'low-throughput', 'The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.', 'in vitro bacterial culture,cell culture', 'RT-PCR,fluorescence microscopy', 16534848);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3714, 'human', 'decrease', 'low-throughput', 'The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.', 'in vitro bacterial culture,cell culture', 'RT-PCR,fluorescence microscopy', 16534848);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3715, 'human', 'decrease', 'low-throughput', 'The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.', 'in vitro bacterial culture,cell culture', 'RT-PCR,fluorescence microscopy', 16534848);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3727, 'human', 'decrease', 'low-throughput', 'The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 17485456);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3728, 'human', 'decrease', 'low-throughput', 'The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 17485456);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3729, 'human', 'decrease', 'low-throughput', 'The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 17485456);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3730, 'human', 'decrease', 'low-throughput', 'The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 17485456);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3731, 'human', 'decrease', 'low-throughput', 'The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 17485456);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3732, 'human', 'decrease', 'low-throughput', 'The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 17485456);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3733, 'human', 'decrease', 'low-throughput', 'The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 17485456);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3734, 'human', 'decrease', 'low-throughput', 'The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 17485456);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3735, 'human', 'decrease', 'low-throughput', 'The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 17485456);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3736, 'human', 'decrease', 'low-throughput', 'L. plantarum prevents EIEC-induced expression of Claudin-1, Occludin, JAM-1 and ZO-1 proteins.', 'in vitro bacterial culture', 'fluorescence microscopy', 19331693);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3737, 'human', 'decrease', 'low-throughput', 'L. plantarum prevents EIEC-induced expression of Claudin-1, Occludin, JAM-1 and ZO-1 proteins.', 'in vitro bacterial culture', 'fluorescence microscopy', 19331693);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3738, 'human', 'decrease', 'low-throughput', 'L. plantarum prevents EIEC-induced expression of Claudin-1, Occludin, JAM-1 and ZO-1 proteins.', 'in vitro bacterial culture', 'fluorescence microscopy', 19331693);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3739, 'human', 'decrease', 'low-throughput', 'L. plantarum prevents EIEC-induced expression of Claudin-1, Occludin, JAM-1 and ZO-1 proteins.', 'in vitro bacterial culture', 'fluorescence microscopy', 19331693);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3748, 'human', 'increase', 'low-throughput', 'Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a ''diet-gut microbiota-physiology axis'' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind ''metabolite-sensing'' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.', 'in vitro bacterial culture', 'RT-PCR', 28658542);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3749, 'human', 'increase', 'low-throughput', 'In this study, we investigated the effects of novel fatty acid metabolite derivatives of linoleic acid generated by the gut lactic acid bacteria Lactobacillus plantarum on the Nrf2-ARE pathway. 10-Oxo-trans-11-octadecenoic acid (KetoC) protected HepG2 cells from cytotoxicity induced by hydrogen peroxide. KetoC also significantly increased cellular Nrf2 protein levels, ARE-dependent transcription, and the gene expression of antioxidative enzymes such as heme oxygenase-1 (HO-1), glutamate-cysteine ligase modifier subunit (GCLM), and Nad(p)h: quinone oxidoreductase 1 (NQO1) in HepG2 cells.', 'cell culture', 'null', 26879219);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3750, 'human', 'increase', 'low-throughput', 'In this study, we investigated the effects of novel fatty acid metabolite derivatives of linoleic acid generated by the gut lactic acid bacteria Lactobacillus plantarum on the Nrf2-ARE pathway. 10-Oxo-trans-11-octadecenoic acid (KetoC) protected HepG2 cells from cytotoxicity induced by hydrogen peroxide. KetoC also significantly increased cellular Nrf2 protein levels, ARE-dependent transcription, and the gene expression of antioxidative enzymes such as heme oxygenase-1 (HO-1), glutamate-cysteine ligase modifier subunit (GCLM), and Nad(p)h: quinone oxidoreductase 1 (NQO1) in HepG2 cells.', 'cell culture', 'null', 26879219);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3753, 'human', 'decrease', 'low-throughput', 'A feature of human ulcerative colitis and other colitic diseases is a change in ''healthy'' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.', 'in vitro bacterial culture', 'null', 19865172);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3754, 'human', 'decrease', 'low-throughput', 'A feature of human ulcerative colitis and other colitic diseases is a change in ''healthy'' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.', 'in vitro bacterial culture', 'null', 19865172);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3755, 'human', 'decrease', 'low-throughput', 'A feature of human ulcerative colitis and other colitic diseases is a change in ''healthy'' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.', 'in vitro bacterial culture', 'null', 19865172);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3756, 'human', 'decrease', 'low-throughput', 'A feature of human ulcerative colitis and other colitic diseases is a change in ''healthy'' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.', 'in vitro bacterial culture', 'null', 19865172);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3757, 'human', 'decrease', 'low-throughput', 'The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.', 'in vitro bacterial culture,cell culture', 'HPLC analysis,RT-PCR', 30767231);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3758, 'human', 'decrease', 'low-throughput', 'The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.', 'in vitro bacterial culture,cell culture', 'HPLC analysis,RT-PCR', 30767231);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3759, 'human', 'decrease', 'low-throughput', 'The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.', 'in vitro bacterial culture,cell culture', 'HPLC analysis,RT-PCR', 30767231);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3760, 'human', 'decrease', 'low-throughput', 'The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.', 'in vitro bacterial culture,cell culture', 'HPLC analysis,RT-PCR', 30767231);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3761, 'human', 'decrease', 'low-throughput', 'The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.', 'in vitro bacterial culture,cell culture', 'HPLC analysis,RT-PCR', 30767231);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3762, 'human', 'decrease', 'low-throughput', 'The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.', 'in vitro bacterial culture,cell culture', 'HPLC analysis,RT-PCR', 30767231);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3765, 'human', 'increase', 'low-throughput', 'A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host''s health through transcriptional regulation.', 'in vitro bacterial culture', 'HPLC analysis,GC/MS analysis', 28266623);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3766, 'human', 'increase', 'low-throughput', 'A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host''s health through transcriptional regulation.', 'in vitro bacterial culture', 'HPLC analysis,GC/MS analysis', 28266623);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3767, 'human', 'increase', 'low-throughput', 'A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host''s health through transcriptional regulation.', 'in vitro bacterial culture', 'HPLC analysis,GC/MS analysis', 28266623);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3768, 'human', 'increase', 'low-throughput', 'A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host''s health through transcriptional regulation.', 'in vitro bacterial culture', 'HPLC analysis,GC/MS analysis', 28266623);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3779, 'human', 'increase', 'low-throughput', 'A. muciniphila increased the expression of tlr2 and tlr4. However, EVs at all of the concentrations showed a decrease in tlr4 expression. EVs at the concentrations of 0.1 and 0.5 µg/ml decreased the expression of tlr2. A. muciniphila significantly increased the expression of ocldn and cldn4. Both this bacterium and EVs increased the expression of zo2 in the cell line. Furthermore, this data show that A. muciniphila derived EVs have a dose-independent effect on Caco-2 cells.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 31191842);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3780, 'human', 'increase', 'low-throughput', 'A. muciniphila increased the expression of tlr2 and tlr4. However, EVs at all of the concentrations showed a decrease in tlr4 expression. EVs at the concentrations of 0.1 and 0.5 µg/ml decreased the expression of tlr2. A. muciniphila significantly increased the expression of ocldn and cldn4. Both this bacterium and EVs increased the expression of zo2 in the cell line. Furthermore, this data show that A. muciniphila derived EVs have a dose-independent effect on Caco-2 cells.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 31191842);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3781, 'human', 'increase', 'low-throughput', 'A. muciniphila increased the expression of tlr2 and tlr4. However, EVs at all of the concentrations showed a decrease in tlr4 expression. EVs at the concentrations of 0.1 and 0.5 µg/ml decreased the expression of tlr2. A. muciniphila significantly increased the expression of ocldn and cldn4. Both this bacterium and EVs increased the expression of zo2 in the cell line. Furthermore, this data show that A. muciniphila derived EVs have a dose-independent effect on Caco-2 cells.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 31191842);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3782, 'human', 'increase', 'low-throughput', 'A. muciniphila increased the expression of tlr2 and tlr4. However, EVs at all of the concentrations showed a decrease in tlr4 expression. EVs at the concentrations of 0.1 and 0.5 µg/ml decreased the expression of tlr2. A. muciniphila significantly increased the expression of ocldn and cldn4. Both this bacterium and EVs increased the expression of zo2 in the cell line. Furthermore, this data show that A. muciniphila derived EVs have a dose-independent effect on Caco-2 cells.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 31191842);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3783, 'human', 'increase', 'low-throughput', 'A. muciniphila increased the expression of tlr2 and tlr4. However, EVs at all of the concentrations showed a decrease in tlr4 expression. EVs at the concentrations of 0.1 and 0.5 µg/ml decreased the expression of tlr2. A. muciniphila significantly increased the expression of ocldn and cldn4. Both this bacterium and EVs increased the expression of zo2 in the cell line. Furthermore, this data show that A. muciniphila derived EVs have a dose-independent effect on Caco-2 cells.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 31191842);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3784, 'human', 'increase', 'low-throughput', 'A. muciniphila increased the expression of tlr2 and tlr4. However, EVs at all of the concentrations showed a decrease in tlr4 expression. EVs at the concentrations of 0.1 and 0.5 µg/ml decreased the expression of tlr2. A. muciniphila significantly increased the expression of ocldn and cldn4. Both this bacterium and EVs increased the expression of zo2 in the cell line. Furthermore, this data show that A. muciniphila derived EVs have a dose-independent effect on Caco-2 cells.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 31191842);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3785, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3786, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3787, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3788, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3789, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3790, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3791, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3792, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3793, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3794, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3795, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3796, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3797, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3798, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3799, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3800, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3801, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3802, 'human', 'increase', 'low-throughput', 'On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.', 'in vitro bacterial culture,cell culture', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3811, 'human', 'increase', 'low-throughput', 'Here we analyse the ability of OMVs and soluble secreted factors to protect epithelial barrier function in polarized T-84 and Caco-2 cells infected with enteropathogenic Escherichia coli (EPEC). Transepithelial electrical resistance, paracellular permeability, mRNA levels and subcellular distribution of tight junction proteins were monitored in the absence or presence of EcN and ECOR63 extracellular fractions. EPEC downregulated expression of ZO-1 ZO-2, occludin and claudin-14 and altered the subcellular localization of ZO-1, occludin and F-actin cytoskeleton. OMVs and soluble factors secreted by EcN and ECOR63 counteracted EPEC-altered transepithelial resistance and paracellular permeability, preserved occludin and claudin-14 mRNA levels, retained ZO-1 and occludin at tight junctions in the cell boundaries and ameliorated F-actin disorganization. Redistribution of ZO-1 was not accompanied by changes at mRNA level.', 'in vitro bacterial culture,cell culture', 'null', 31315566);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3812, 'human', 'increase', 'low-throughput', 'Here we analyse the ability of OMVs and soluble secreted factors to protect epithelial barrier function in polarized T-84 and Caco-2 cells infected with enteropathogenic Escherichia coli (EPEC). Transepithelial electrical resistance, paracellular permeability, mRNA levels and subcellular distribution of tight junction proteins were monitored in the absence or presence of EcN and ECOR63 extracellular fractions. EPEC downregulated expression of ZO-1 ZO-2, occludin and claudin-14 and altered the subcellular localization of ZO-1, occludin and F-actin cytoskeleton. OMVs and soluble factors secreted by EcN and ECOR63 counteracted EPEC-altered transepithelial resistance and paracellular permeability, preserved occludin and claudin-14 mRNA levels, retained ZO-1 and occludin at tight junctions in the cell boundaries and ameliorated F-actin disorganization. Redistribution of ZO-1 was not accompanied by changes at mRNA level.', 'in vitro bacterial culture,cell culture', 'null', 31315566);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3813, 'human', 'increase', 'low-throughput', 'Here we analyse the ability of OMVs and soluble secreted factors to protect epithelial barrier function in polarized T-84 and Caco-2 cells infected with enteropathogenic Escherichia coli (EPEC). Transepithelial electrical resistance, paracellular permeability, mRNA levels and subcellular distribution of tight junction proteins were monitored in the absence or presence of EcN and ECOR63 extracellular fractions. EPEC downregulated expression of ZO-1 ZO-2, occludin and claudin-14 and altered the subcellular localization of ZO-1, occludin and F-actin cytoskeleton. OMVs and soluble factors secreted by EcN and ECOR63 counteracted EPEC-altered transepithelial resistance and paracellular permeability, preserved occludin and claudin-14 mRNA levels, retained ZO-1 and occludin at tight junctions in the cell boundaries and ameliorated F-actin disorganization. Redistribution of ZO-1 was not accompanied by changes at mRNA level.', 'in vitro bacterial culture,cell culture', 'null', 31315566);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3814, 'human', 'increase', 'low-throughput', 'Here we analyse the ability of OMVs and soluble secreted factors to protect epithelial barrier function in polarized T-84 and Caco-2 cells infected with enteropathogenic Escherichia coli (EPEC). Transepithelial electrical resistance, paracellular permeability, mRNA levels and subcellular distribution of tight junction proteins were monitored in the absence or presence of EcN and ECOR63 extracellular fractions. EPEC downregulated expression of ZO-1 ZO-2, occludin and claudin-14 and altered the subcellular localization of ZO-1, occludin and F-actin cytoskeleton. OMVs and soluble factors secreted by EcN and ECOR63 counteracted EPEC-altered transepithelial resistance and paracellular permeability, preserved occludin and claudin-14 mRNA levels, retained ZO-1 and occludin at tight junctions in the cell boundaries and ameliorated F-actin disorganization. Redistribution of ZO-1 was not accompanied by changes at mRNA level.', 'in vitro bacterial culture,cell culture', 'null', 31315566);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3815, 'human', 'increase', 'low-throughput', 'Here we analyse the ability of OMVs and soluble secreted factors to protect epithelial barrier function in polarized T-84 and Caco-2 cells infected with enteropathogenic Escherichia coli (EPEC). Transepithelial electrical resistance, paracellular permeability, mRNA levels and subcellular distribution of tight junction proteins were monitored in the absence or presence of EcN and ECOR63 extracellular fractions. EPEC downregulated expression of ZO-1 ZO-2, occludin and claudin-14 and altered the subcellular localization of ZO-1, occludin and F-actin cytoskeleton. OMVs and soluble factors secreted by EcN and ECOR63 counteracted EPEC-altered transepithelial resistance and paracellular permeability, preserved occludin and claudin-14 mRNA levels, retained ZO-1 and occludin at tight junctions in the cell boundaries and ameliorated F-actin disorganization. Redistribution of ZO-1 was not accompanied by changes at mRNA level.', 'in vitro bacterial culture,cell culture', 'null', 31315566);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3816, 'human', 'increase', 'low-throughput', 'Here we analyse the ability of OMVs and soluble secreted factors to protect epithelial barrier function in polarized T-84 and Caco-2 cells infected with enteropathogenic Escherichia coli (EPEC). Transepithelial electrical resistance, paracellular permeability, mRNA levels and subcellular distribution of tight junction proteins were monitored in the absence or presence of EcN and ECOR63 extracellular fractions. EPEC downregulated expression of ZO-1 ZO-2, occludin and claudin-14 and altered the subcellular localization of ZO-1, occludin and F-actin cytoskeleton. OMVs and soluble factors secreted by EcN and ECOR63 counteracted EPEC-altered transepithelial resistance and paracellular permeability, preserved occludin and claudin-14 mRNA levels, retained ZO-1 and occludin at tight junctions in the cell boundaries and ameliorated F-actin disorganization. Redistribution of ZO-1 was not accompanied by changes at mRNA level.', 'in vitro bacterial culture,cell culture', 'null', 31315566);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3824, 'human', 'decrease', 'low-throughput', 'It has been shown to down-regulate nuclear transcription factor (NF-кB) in human intestinal cells, a central regulator of the host mucosal immune system response to the microbiota.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 25946041);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3825, 'human', 'decrease', 'low-throughput', 'In order to evaluate its impact on a further important transcription factor shown to link metabolism and inflammation in the intestine, namely PPARγ (peroxisome proliferator-activated receptor), we used human intestinal epithelial cell-lines engineered to monitor PPARγ transcriptional activity in response to a wide range of S. salivarius strains.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 25946041);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3826, 'human', 'decrease', 'low-throughput', 'To assess the impact of S. salivarius-dependent inhibition on PPARγ target genes, we quantified by RT-qPCR the expression of Angptl4 and I-FABP (intestinal fatty acid binding protein).The expression of these two well-characterized PPARγ-regulated genes were strongly reduced by S. salivarius supernatant in the presence of the PPARγ-specific ligand, rosiglitazone.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 25946041);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3827, 'human', 'decrease', 'low-throughput', 'To assess the impact of S. salivarius-dependent inhibition on PPARγ target genes, we quantified by RT-qPCR the expression of Angptl4 and I-FABP (intestinal fatty acid binding protein).The expression of these two well-characterized PPARγ-regulated genes were strongly reduced by S. salivarius supernatant in the presence of the PPARγ-specific ligand, rosiglitazone.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 25946041);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3828, 'human', 'decrease', 'low-throughput', 'It has been shown to down-regulate nuclear transcription factor (NF-кB) in human intestinal cells, a central regulator of the host mucosal immune system response to the microbiota.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 25946041);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3829, 'human', 'decrease', 'low-throughput', 'In order to evaluate its impact on a further important transcription factor shown to link metabolism and inflammation in the intestine, namely PPARγ (peroxisome proliferator-activated receptor), we used human intestinal epithelial cell-lines engineered to monitor PPARγ transcriptional activity in response to a wide range of S. salivarius strains.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 25946041);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3830, 'human', 'decrease', 'low-throughput', 'To assess the impact of S. salivarius-dependent inhibition on PPARγ target genes, we quantified by RT-qPCR the expression of Angptl4 and I-FABP (intestinal fatty acid binding protein).The expression of these two well-characterized PPARγ-regulated genes were strongly reduced by S. salivarius supernatant in the presence of the PPARγ-specific ligand, rosiglitazone.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 25946041);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3831, 'human', 'decrease', 'low-throughput', 'To assess the impact of S. salivarius-dependent inhibition on PPARγ target genes, we quantified by RT-qPCR the expression of Angptl4 and I-FABP (intestinal fatty acid binding protein).The expression of these two well-characterized PPARγ-regulated genes were strongly reduced by S. salivarius supernatant in the presence of the PPARγ-specific ligand, rosiglitazone.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 25946041);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3832, 'human', 'decrease', 'low-throughput', 'In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment,Supplementation of metabolite', 'RT-PCR', 21178864);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3833, 'human', 'decrease', 'low-throughput', 'In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment,Supplementation of metabolite', 'RT-PCR', 21178864);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3834, 'human', 'decrease', 'low-throughput', 'In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment,Supplementation of metabolite', 'RT-PCR', 21178864);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3835, 'human', 'increase', 'low-throughput', 'In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment,Supplementation of metabolite', 'RT-PCR', 21178864);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3836, 'human', 'decrease', 'low-throughput', 'In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment,Supplementation of metabolite', 'RT-PCR', 21178864);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3837, 'human', 'decrease', 'low-throughput', 'Incubation with LGG significantly decreased the concentration of IL-8 compared with the control, but no other significant effects were observed for LGG or its genomic DNA isolate.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment,Supplementation of metabolite', 'RT-PCR', 21178864);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3839, 'human', 'increase', 'low-throughput', 'To examine how C. difficile induces indole production, the expression of the tryptophanase gene (tnaA) in the E. coli strains was measured by quantitative real-time PCR. E. coli strains exposed to C. difficile stationary phase supernatant expressed significantly (P   0.0025) higher tnaA levels ( 4-fold) than when cultured alone (Fig. 4A). The increased expression of tnaA in the presence of the stationary-phase culture supernatant suggested that C. difficile likely induces indole at the level of transcription.', 'in vitro bacterial culture、in vitro bacterial culture', 'RT-PCR', 30944877);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3840, 'human', 'increase', 'low-throughput', 'To examine how C. difficile induces indole production, the expression of the tryptophanase gene (tnaA) in the E. coli strains was measured by quantitative real-time PCR. E. coli strains exposed to C. difficile stationary phase supernatant expressed significantly (P   0.0025) higher tnaA levels ( 4-fold) than when cultured alone (Fig. 4A). The increased expression of tnaA in the presence of the stationary-phase culture supernatant suggested that C. difficile likely induces indole at the level of transcription.', 'in vitro bacterial culture、in vitro bacterial culture', 'RT-PCR', 30944877);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3845, 'human', 'decrease', 'low-throughput', 'We examined the effects of UV-killed F. prausnitzii, bacterial fractions,supernatants, or sterile medium on IL-8 secretion in Caco-2 cells,with or without IL-1 stimulation (SI Materials and Methods and Fig. 3A). Either UV-killed F. prausnitzii, membrane extracts, or cytoplasmic extracts, but not DNA or sterile medium, induce low levels of IL-8 secretion. UV-killed F. prausnitzii, bacterial fractions,and sterile medium did not inhibit IL-8 secretion induced by IL-1,compared with negative control. In contrast, supernatant from F.prausnitzii cultures significantly reduces IL-8 secretion induced by IL-1.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 18936492);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3846, 'human', 'decrease', 'low-throughput', 'We examined the effects of UV-killed F. prausnitzii, bacterial fractions,supernatants, or sterile medium on IL-8 secretion in Caco-2 cells,with or without IL-1 stimulation (SI Materials and Methods and Fig. 3A). Either UV-killed F. prausnitzii, membrane extracts, or cytoplasmic extracts, but not DNA or sterile medium, induce low levels of IL-8 secretion. UV-killed F. prausnitzii, bacterial fractions,and sterile medium did not inhibit IL-8 secretion induced by IL-1,compared with negative control. In contrast, supernatant from F.prausnitzii cultures significantly reduces IL-8 secretion induced by IL-1.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 18936492);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3847, 'human', 'decrease', 'low-throughput', 'In vitro peripheral blood mononuclear cell stimulation by F. prausnitzii led to significantly lower IL-12 and IFN-γ production levels and higher secretion of IL-10.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 18936492);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3848, 'human', 'decrease', 'low-throughput', 'In vitro peripheral blood mononuclear cell stimulation by F. prausnitzii led to significantly lower IL-12 and IFN-γ production levels and higher secretion of IL-10.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 18936492);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3849, 'human', 'increase', 'low-throughput', 'In vitro peripheral blood mononuclear cell stimulation by F. prausnitzii led to significantly lower IL-12 and IFN-γ production levels and higher secretion of IL-10.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 18936492);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3870, 'human', 'increase', 'low-throughput', 'Compared with the control, the stimulation of HCT116 cells with C. butyricum supernatant and sodium butyrate induced 4.5-fold and 4.2-fold upregulation of the GPR43 mRNA levels, 2.6-fold and 7.0-fold increases in the GPR109A mRNA levels.', 'cell culture,in vitro bacterial culture', 'null', 31734354);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3871, 'human', 'increase', 'low-throughput', 'Compared with the control, the stimulation of HCT116 cells with C. butyricum supernatant and sodium butyrate induced 4.5-fold and 4.2-fold upregulation of the GPR43 mRNA levels, 2.6-fold and 7.0-fold increases in the GPR109A mRNA levels.', 'cell culture,in vitro bacterial culture', 'null', 31734354);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3872, 'human', 'increase', 'low-throughput', 'The mRNA level of P21WAF1 was increased in HCT116 cells treated with C. butyricum.', 'cell culture,in vitro bacterial culture', 'null', 31734354);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3882, 'human', 'increase', 'low-throughput', 'Exposure to 1 mM indole led to an increase in the expression of two Toll-like receptor (TLR) genes—TLR-3 and TLR-9.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3883, 'human', 'increase', 'low-throughput', 'Exposure to 1 mM indole led to an increase in the expression of two Toll-like receptor (TLR) genes—TLR-3 and TLR-9.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3884, 'human', 'increase', 'low-throughput', 'Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3885, 'human', 'increase', 'low-throughput', 'Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3886, 'human', 'increase', 'low-throughput', 'Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3887, 'human', 'increase', 'low-throughput', 'Similarly, genes involved in the phosphoinositide-3-kinase (PI3K) pathway, which modulates TLR signaling and leads to antiinflammatory cytokine IL-10 induction, were also induced by indole.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3888, 'human', 'increase', 'low-throughput', 'Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3889, 'human', 'increase', 'low-throughput', 'Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3890, 'human', 'increase', 'low-throughput', 'Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3891, 'human', 'increase', 'low-throughput', 'Downstream target genes of AKT1, including the cAMP response element-binding protein and IL-10, were also induced upon addition of indole.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3892, 'human', 'increase', 'low-throughput', 'In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3893, 'human', 'increase', 'low-throughput', 'In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3894, 'human', 'increase', 'low-throughput', 'In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3895, 'human', 'increase', 'low-throughput', 'In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3896, 'human', 'increase', 'low-throughput', 'In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3897, 'human', 'increase', 'low-throughput', 'In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3898, 'human', 'increase', 'low-throughput', 'In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3899, 'human', 'increase', 'low-throughput', 'In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3900, 'human', 'increase', 'low-throughput', 'In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3901, 'human', 'increase', 'low-throughput', 'In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3902, 'human', 'increase', 'low-throughput', 'In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3903, 'human', 'increase', 'low-throughput', 'Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3904, 'human', 'increase', 'low-throughput', 'Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3905, 'human', 'increase', 'low-throughput', 'Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3906, 'human', 'increase', 'low-throughput', 'Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3907, 'human', 'increase', 'low-throughput', 'Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.', 'cell culture', 'RT-qPCR', 19966295);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3936, 'human', 'decrease', 'low-throughput', 'We also noted that IRS1 levels were reduced by 8 hr treatment of imidazole propionate in the human embryonic kidney cell line HEK293.', 'cell culture', 'UPLC-MS/MS', 30401435);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3937, 'human', 'increase', 'low-throughput', 'Imidazole propionate activates mTORC1.', 'cell culture', 'UPLC-MS/MS', 30401435);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3938, 'human', 'decrease', 'low-throughput', 'We also noted that IRS1 levels were reduced by 8 hr treatment of imidazole propionate in the human embryonic kidney cell line HEK293.', 'cell culture', 'UPLC-MS/MS', 30401435);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3939, 'human', 'increase', 'low-throughput', 'Imidazole propionate activates mTORC1.', 'cell culture', 'UPLC-MS/MS', 30401435);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3944, 'human', 'increase', 'low-throughput', 'EcN positively modulates the intestinal epithelial barrier through upregulation and redistribution of the tight junction proteins ZO-1, ZO-2 and claudin-14.', 'cell culture', 'RT-qPCR', 28018313);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3945, 'human', 'increase', 'low-throughput', 'EcN positively modulates the intestinal epithelial barrier through upregulation and redistribution of the tight junction proteins ZO-1, ZO-2 and claudin-14.', 'cell culture', 'RT-qPCR', 28018313);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3946, 'human', 'increase', 'low-throughput', 'EcN positively modulates the intestinal epithelial barrier through upregulation and redistribution of the tight junction proteins ZO-1, ZO-2 and claudin-14.', 'cell culture', 'RT-qPCR', 28018313);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3947, 'human', 'increase', 'low-throughput', 'There was a strong correlation between the amount of Firmicutes and TLR2 gene expression levels.', 'in vivo experiment', 'qPCR', 31677982);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (3948, 'human', 'decrease', 'low-throughput', 'Preincubation with BCM significantly reduced the occludin expression compared with the control and LCM group.', 'cell culture', 'RT-qPCR', 28230606);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4017, 'human', 'increase', 'low-throughput', 'B. lactis augmented TNF-α compared to controlas time and dose dependent.', 'cell culture', 'qPCR', 31780030);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4018, 'human', 'decrease', 'low-throughput', 'B.fragilis significantly decreased TLR2 gene expression.TLR4 gene expression was slightly increased by this bacterium.', 'cell culture', 'RT-qPCR', 30507089);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4019, 'human', 'increase', 'low-throughput', 'B.fragilis significantly decreased TLR2 gene expression.TLR4 gene expression was slightly increased by this bacterium.', 'cell culture', 'RT-qPCR', 30507089);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4020, 'human', 'increase', 'low-throughput', 'B. fragilis signifcantly elevated IFNγ concentration.', 'cell culture', 'RT-qPCR', 30507089);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4021, 'human', 'increase', 'low-throughput', 'B. fragilis was able to increase IL-4 and IL-10 concentrations.', 'cell culture', 'RT-qPCR', 30507089);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4022, 'human', 'increase', 'low-throughput', 'B. fragilis was able to increase IL-4 and IL-10 concentrations.', 'cell culture', 'RT-qPCR', 30507089);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4074, 'human', 'increase', 'low-throughput', 'CS of LP increases SMCT1 mRNA and protein levels in IEC-6 cells.', 'cell culture', 'RT-PCR', 20671196);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4078, 'human', 'increase', 'low-throughput', 'The results showed a significant increase in the expression of MUC2 and MUC1 genes in HT29-MTX cells with RSN.', 'cell culture', 'RT-PCR', 26868655);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4079, 'human', 'increase', 'low-throughput', 'The results showed a significant increase in the expression of MUC2 and MUC1 genes in HT29-MTX cells with RSN.', 'cell culture', 'RT-PCR', 26868655);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4080, 'human', 'increase', 'low-throughput', 'The results showed an increase in MGAT3, B4GALT1 and ST6GAL1 expression in RSN-BHI compared to BHI-treated HT29-MTX cells.', 'cell culture', 'RT-PCR', 26868655);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4081, 'human', 'increase', 'low-throughput', 'The results showed an increase in MGAT3, B4GALT1 and ST6GAL1 expression in RSN-BHI compared to BHI-treated HT29-MTX cells.', 'cell culture', 'RT-PCR', 26868655);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4082, 'human', 'increase', 'low-throughput', 'The results showed an increase in MGAT3, B4GALT1 and ST6GAL1 expression in RSN-BHI compared to BHI-treated HT29-MTX cells.', 'cell culture', 'RT-PCR', 26868655);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4086, 'human', 'decrease', 'low-throughput', 'Secretion of TNF-α-triggered IL-8 was significantly reduced in cells pre-incubated with L. plantarum L9.', 'cell culture', 'ELISA', 28441885);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4087, 'human', 'decrease', 'low-throughput', 'Secretion of TNF-α-triggered IL-8 was significantly reduced in cells pre-incubated with L. plantarum L9.', 'cell culture', 'ELISA', 28441885);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4088, 'human', 'increase', 'low-throughput', 'LA CS induces Erk1/2 MAPK activation and c-Fos or c-Jun protein expression in Caco-2 cells.', 'cell culture', 'Western blot assay', 26867563);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4089, 'human', 'increase', 'low-throughput', 'LA CS induces Erk1/2 MAPK activation and c-Fos or c-Jun protein expression in Caco-2 cells.', 'cell culture', 'Western blot assay', 26867563);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4090, 'human', 'increase', 'low-throughput', 'These results further suggest that c-Fos and c-Jun play an important role in stimulation of Pgp gene expression in Caco-2 cells in response to LA CS.', 'cell culture', 'Western blot assay', 26867563);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4097, 'human', 'decrease', 'low-throughput', 'The results showed the mRNA levels of both hTHTR-1 and hTHTR-2 were significantly (P < 0.01 for both) lower in EPEC-treated cells compared with controls.', 'cell culture', 'RT-PCR', 19628653);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4098, 'human', 'decrease', 'low-throughput', 'The results showed the mRNA levels of both hTHTR-1 and hTHTR-2 were significantly (P < 0.01 for both) lower in EPEC-treated cells compared with controls.', 'cell culture', 'RT-PCR', 19628653);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4105, 'human', 'increase', 'low-throughput', 'Treatment of Caco2 cells with LA or its conditioned culture supernatant (CS) for 8�?4 h resulted in a significant increase in Na+/H+ exchange activity, mRNA, and protein levels of NHE3.', 'in vitro bacterial culture', 'RT-PCR', 23086913);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4108, 'human', 'decrease', 'low-throughput', 'L. acidophilus blocks PAF-induced NF-kB activation and IL-8 production in human intestinal epithelial cells.', 'in vitro bacterial culture', 'ELISA,RT-PCR', 24130731);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4109, 'human', 'decrease', 'low-throughput', 'L. acidophilus blocks PAF-induced NF-kB activation and IL-8 production in human intestinal epithelial cells.', 'in vitro bacterial culture', 'ELISA,RT-PCR', 24130731);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4110, 'human', 'decrease', 'low-throughput', 'LA-CS also attenuated PAF-induced increase in Bcl10 mRNA and protein levels and Bcl10 promoter activity.', 'in vitro bacterial culture', 'ELISA,RT-PCR', 24130731);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4111, 'human', 'increase', 'low-throughput', 'LA-CS Increases DRA mRNA Levels and Blocks IFN-γ-Induced Decrease in DRA', 'in vitro bacterial culture', 'ELISA,RT-PCR', 25059823);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4128, 'human', 'decrease', 'low-throughput', 'Using a publically available database (GSE75214) 25, we found that the expression of host glycosyltransferases, such as MGTA4A and MGAT4B, was significantly impaired in UC patients compared to normal controls (Fig. 4a). The expression of IL22RA2 (also known as IL-22 binding protein (IL-22BP), a soluble IL-22 receptor that blocks IL-22 signaling, was markedly up-regulated in patients with UC, while IL-22 expression was not affected (Fig. 4a and Extended Data Fig. 7a).', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,qPCR', 32066975);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4129, 'human', 'decrease', 'low-throughput', 'Using a publically available database (GSE75214) 25, we found that the expression of host glycosyltransferases, such as MGTA4A and MGAT4B, was significantly impaired in UC patients compared to normal controls (Fig. 4a). The expression of IL22RA2 (also known as IL-22 binding protein (IL-22BP), a soluble IL-22 receptor that blocks IL-22 signaling, was markedly up-regulated in patients with UC, while IL-22 expression was not affected (Fig. 4a and Extended Data Fig. 7a).', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,qPCR', 32066975);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4130, 'human', 'increase', 'low-throughput', 'Using a publically available database (GSE75214) 25, we found that the expression of host glycosyltransferases, such as MGTA4A and MGAT4B, was significantly impaired in UC patients compared to normal controls (Fig. 4a). The expression of IL22RA2 (also known as IL-22 binding protein (IL-22BP), a soluble IL-22 receptor that blocks IL-22 signaling, was markedly up-regulated in patients with UC, while IL-22 expression was not affected (Fig. 4a and Extended Data Fig. 7a).', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,qPCR', 32066975);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4230, 'mouse', 'decrease', 'low-throughput', 'Using human intestinal epithelial cells (IECs), we discovered that SCFAs, particularly butyrate, enhanced IEC barrier formation, induced IL10RA mRNA, IL-10RA protein, and transactivation through activated Stat3 and HDAC inhibition. Loss and gain of IL10RA expression directly correlates with IEC barrier formation and butyrate represses permeability-promoting claudin-2 (Cldn2) tight-junction protein expression through an IL-10RA-dependent mechanism. Our findings provide a novel mechanism by which microbial derived butyrate promotes barrier through IL-10RA-dependent repression of Cldn2.', 'cell culture', 'real-time PCR,qPCR,Western blot assay', 28893958);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4231, 'mouse', 'increase', 'low-throughput', 'Using human intestinal epithelial cells (IECs), we discovered that SCFAs, particularly butyrate, enhanced IEC barrier formation, induced IL10RA mRNA, IL-10RA protein, and transactivation through activated Stat3 and HDAC inhibition. Loss and gain of IL10RA expression directly correlates with IEC barrier formation and butyrate represses permeability-promoting claudin-2 (Cldn2) tight-junction protein expression through an IL-10RA-dependent mechanism. Our findings provide a novel mechanism by which microbial derived butyrate promotes barrier through IL-10RA-dependent repression of Cldn2.', 'cell culture', 'real-time PCR,qPCR,Western blot assay', 28893958);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4232, 'mouse', 'decrease', 'low-throughput', 'Using human intestinal epithelial cells (IECs), we discovered that SCFAs, particularly butyrate, enhanced IEC barrier formation, induced IL10RA mRNA, IL-10RA protein, and transactivation through activated Stat3 and HDAC inhibition. Loss and gain of IL10RA expression directly correlates with IEC barrier formation and butyrate represses permeability-promoting claudin-2 (Cldn2) tight-junction protein expression through an IL-10RA-dependent mechanism. Our findings provide a novel mechanism by which microbial derived butyrate promotes barrier through IL-10RA-dependent repression of Cldn2.', 'cell culture', 'real-time PCR,qPCR,Western blot assay', 28893958);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4243, 'mouse', 'decrease', 'low-throughput', 'Ratelimited enzymes in the process of TG synthesis of the DGAT1/2 mRNA level in the HFD group were signifificantly increased compared with the LFD group. The DGAT2 mRNA level in the HFD-CBM group decreased in the same range as the LFD group, whereas signifificant differences of the DGAT1 mRNA level were not found between the HFD group and the HFD-CBM group.', 'cell culture', 'Western blot assay,GC/MS Analysis, real-time PCR', 24166662);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4244, 'mouse', 'decrease', 'low-throughput', 'several candidate strains were screened to find some probiotics as therapeutics using dextran sodium sulfate (DSS)-induced colitis in mice, and China Center for Type Culture Collection (CCTCC) M206119 strain led to an exacerbated phenotype of DSS-colitis mice. Expression of protective factors zonula occludens-1 and β-defensin 2 was downregulated after CCTCC M206119 treatment. There was an increase in the nuclear translocation of nuclear factor-κB in epithelial cells. Then, intestinal proinflammatory and anti-inflammatory cytokine responses were evaluated. Proinflammatory colonic cytokines [interleukin (IL)-1β, IL-6 and tumor necrosis factor-α] levels were clearly increased in CCTCC-M206119-treated animals, whereas anti-inflammatory colonic cytokine (IL-10) level was lowered compared with saline- or 5-ASA-treated DSS-colitis mice. Next, CCTCC M206119 strain was identified as Lactobacillus crispatus, so it was concluded that not all lactobacilli strains have beneficial effects on intestinal inflammation and that Lactobacillus crispatus (L. crispatus) CCTCC M206119 is involved in exacerbation of intestinal inflammation in DSS-colitis mice.', 'cell culture', 'RT-PCR,HPLC analysis', 22654425);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4245, 'mouse', 'decrease', 'low-throughput', 'several candidate strains were screened to find some probiotics as therapeutics using dextran sodium sulfate (DSS)-induced colitis in mice, and China Center for Type Culture Collection (CCTCC) M206119 strain led to an exacerbated phenotype of DSS-colitis mice. Expression of protective factors zonula occludens-1 and β-defensin 2 was downregulated after CCTCC M206119 treatment. There was an increase in the nuclear translocation of nuclear factor-κB in epithelial cells. Then, intestinal proinflammatory and anti-inflammatory cytokine responses were evaluated. Proinflammatory colonic cytokines [interleukin (IL)-1β, IL-6 and tumor necrosis factor-α] levels were clearly increased in CCTCC-M206119-treated animals, whereas anti-inflammatory colonic cytokine (IL-10) level was lowered compared with saline- or 5-ASA-treated DSS-colitis mice. Next, CCTCC M206119 strain was identified as Lactobacillus crispatus, so it was concluded that not all lactobacilli strains have beneficial effects on intestinal inflammation and that Lactobacillus crispatus (L. crispatus) CCTCC M206119 is involved in exacerbation of intestinal inflammation in DSS-colitis mice.', 'cell culture', 'RT-PCR,HPLC analysis', 22654425);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4246, 'mouse', 'increase', 'low-throughput', 'several candidate strains were screened to find some probiotics as therapeutics using dextran sodium sulfate (DSS)-induced colitis in mice, and China Center for Type Culture Collection (CCTCC) M206119 strain led to an exacerbated phenotype of DSS-colitis mice. Expression of protective factors zonula occludens-1 and β-defensin 2 was downregulated after CCTCC M206119 treatment. There was an increase in the nuclear translocation of nuclear factor-κB in epithelial cells. Then, intestinal proinflammatory and anti-inflammatory cytokine responses were evaluated. Proinflammatory colonic cytokines [interleukin (IL)-1β, IL-6 and tumor necrosis factor-α] levels were clearly increased in CCTCC-M206119-treated animals, whereas anti-inflammatory colonic cytokine (IL-10) level was lowered compared with saline- or 5-ASA-treated DSS-colitis mice. Next, CCTCC M206119 strain was identified as Lactobacillus crispatus, so it was concluded that not all lactobacilli strains have beneficial effects on intestinal inflammation and that Lactobacillus crispatus (L. crispatus) CCTCC M206119 is involved in exacerbation of intestinal inflammation in DSS-colitis mice.', 'cell culture', 'RT-PCR,HPLC analysis', 22654425);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4247, 'mouse', 'increase', 'low-throughput', 'several candidate strains were screened to find some probiotics as therapeutics using dextran sodium sulfate (DSS)-induced colitis in mice, and China Center for Type Culture Collection (CCTCC) M206119 strain led to an exacerbated phenotype of DSS-colitis mice. Expression of protective factors zonula occludens-1 and β-defensin 2 was downregulated after CCTCC M206119 treatment. There was an increase in the nuclear translocation of nuclear factor-κB in epithelial cells. Then, intestinal proinflammatory and anti-inflammatory cytokine responses were evaluated. Proinflammatory colonic cytokines [interleukin (IL)-1β, IL-6 and tumor necrosis factor-α] levels were clearly increased in CCTCC-M206119-treated animals, whereas anti-inflammatory colonic cytokine (IL-10) level was lowered compared with saline- or 5-ASA-treated DSS-colitis mice. Next, CCTCC M206119 strain was identified as Lactobacillus crispatus, so it was concluded that not all lactobacilli strains have beneficial effects on intestinal inflammation and that Lactobacillus crispatus (L. crispatus) CCTCC M206119 is involved in exacerbation of intestinal inflammation in DSS-colitis mice.', 'cell culture', 'RT-PCR,HPLC analysis', 22654425);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4248, 'mouse', 'increase', 'low-throughput', 'several candidate strains were screened to find some probiotics as therapeutics using dextran sodium sulfate (DSS)-induced colitis in mice, and China Center for Type Culture Collection (CCTCC) M206119 strain led to an exacerbated phenotype of DSS-colitis mice. Expression of protective factors zonula occludens-1 and β-defensin 2 was downregulated after CCTCC M206119 treatment. There was an increase in the nuclear translocation of nuclear factor-κB in epithelial cells. Then, intestinal proinflammatory and anti-inflammatory cytokine responses were evaluated. Proinflammatory colonic cytokines [interleukin (IL)-1β, IL-6 and tumor necrosis factor-α] levels were clearly increased in CCTCC-M206119-treated animals, whereas anti-inflammatory colonic cytokine (IL-10) level was lowered compared with saline- or 5-ASA-treated DSS-colitis mice. Next, CCTCC M206119 strain was identified as Lactobacillus crispatus, so it was concluded that not all lactobacilli strains have beneficial effects on intestinal inflammation and that Lactobacillus crispatus (L. crispatus) CCTCC M206119 is involved in exacerbation of intestinal inflammation in DSS-colitis mice.', 'cell culture', 'RT-PCR,HPLC analysis', 22654425);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4253, 'mouse', 'decrease', 'low-throughput', 'We extended our study to human samples to determine the effect of bile acids on CXCL16 mRNA expression of human liver sinusoidal endothelial cells. Similar to the mouse study, primary bile acids CDCA and TCA induced CXCL16 mRNA expression in SK-HEP1 cells (Fig. 5H). Next, the correlation between bile acids and CXCL16 expression was tested in nontumor liver tissues from HCC or cholangiocarcinoma patients of the Thailand Initiative for Genomics and Expression Research in Liver Cancer (TIGER-LC) cohort (34). Primary bile acid CDCA levels correlated with CXCL16 expression (Fig. 5I), whereas inverse correlation was seen for the secondary bile acid glycolithocholate (GLCA) (fig. S6I). The primary/secondary ratio was associated with CXCL16 increase (fig. S6J), indicating that the opposing effect of bile acids on CXCL16 expression also exists in humans.', 'cell culture', 'real-time PCR', 29798856);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4254, 'mouse', 'decrease', 'low-throughput', 'In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.', 'cell culture', 'GC/MS analysis', 29796620);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4255, 'mouse', 'decrease', 'low-throughput', 'In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.', 'cell culture', 'GC/MS analysis', 29796620);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4256, 'mouse', 'decrease', 'low-throughput', 'In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.', 'cell culture', 'GC/MS analysis', 29796620);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4257, 'mouse', 'decrease', 'low-throughput', 'In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.', 'cell culture', 'GC/MS analysis', 29796620);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4258, 'mouse', 'increase', 'low-throughput', 'In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.', 'cell culture', 'GC/MS analysis', 29796620);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4350, 'mouse', 'increase', 'low-throughput', 'The main metabolites produced by B. thetaiotaomicron in the GIT were acetate (up to 6 mM) and propionate (up to 2 mM) (Figure 1D). We evaluated the effects of these bacterial metabolites on the amount of KLF4 protein produced by the mucus producing cell line HT29-MTX [31]. The amount of KLF4 protein was higher in the presence than in the absence of acetate (10 and 20 mM) (Figure 3A)whereas propionate (5 and 10 mM) and butyrate (1 and 5 mM) had no effect (Figure 3B, C).', 'cell culture', 'Western blot assay,real-time qPC,immunohistochemisty assay', 23692866);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4352, 'mouse', 'increase', 'low-throughput', 'The amount of KLF4 protein, a transcription factor involved in goblet cell terminal differentiation [30], was higher in Bt-2d and Bt-30d than GF rats (Figure 2D). Also, muc2, muc4, c1galt1 and b4galt4 mRNAs, encoding proteins involved in mucin synthesis and glycosylation, were significantly more abundant in Bt-2d and Bt-30d rats than in GF rats(Table 1). The klf4 mRNA level was higher in Bt-2d, but not in Bt-30d, than in controls (Table 1).', 'cell culture', 'Western blot assay,real-time qPC,immunohistochemisty assay', 23692866);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4353, 'mouse', 'increase', 'low-throughput', 'The amount of KLF4 protein, a transcription factor involved in goblet cell terminal differentiation [30], was higher in Bt-2d and Bt-30d than GF rats (Figure 2D). Also, muc2, muc4, c1galt1 and b4galt4 mRNAs, encoding proteins involved in mucin synthesis and glycosylation, were significantly more abundant in Bt-2d and Bt-30d rats than in GF rats(Table 1). The klf4 mRNA level was higher in Bt-2d, but not in Bt-30d, than in controls (Table 1).', 'cell culture', 'Western blot assay,real-time qPC,immunohistochemisty assay', 23692866);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4354, 'mouse', 'increase', 'low-throughput', 'The amount of KLF4 protein, a transcription factor involved in goblet cell terminal differentiation [30], was higher in Bt-2d and Bt-30d than GF rats (Figure 2D). Also, muc2, muc4, c1galt1 and b4galt4 mRNAs, encoding proteins involved in mucin synthesis and glycosylation, were significantly more abundant in Bt-2d and Bt-30d rats than in GF rats(Table 1). The klf4 mRNA level was higher in Bt-2d, but not in Bt-30d, than in controls (Table 1).', 'cell culture', 'Western blot assay,real-time qPC,immunohistochemisty assay', 23692866);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4355, 'mouse', 'increase', 'low-throughput', 'The amount of KLF4 protein, a transcription factor involved in goblet cell terminal differentiation [30], was higher in Bt-2d and Bt-30d than GF rats (Figure 2D). Also, muc2, muc4, c1galt1 and b4galt4 mRNAs, encoding proteins involved in mucin synthesis and glycosylation, were significantly more abundant in Bt-2d and Bt-30d rats than in GF rats(Table 1). The klf4 mRNA level was higher in Bt-2d, but not in Bt-30d, than in controls (Table 1).', 'cell culture', 'Western blot assay,real-time qPC,immunohistochemisty assay', 23692866);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4356, 'mouse', 'increase', 'low-throughput', 'We observed an increase in the expression of the sialyltransferase st3gal4.', 'cell culture', 'Western blot assay,real-time qPC,immunohistochemisty assay', 23692866);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4357, 'mouse', 'increase', 'low-throughput', 'We first checked  in vitro  that  B. thetaiotaomicron produced acetate and propionate and that  F. prausnitzii consumed acetate and produced butyrate (Figure 4A). When both bacteria were grown together, butyrate was produced rather than acetate. The butyrate production by F. prausnitzii as higher in the presence than absence of B. thetaiotaomicron  (10.9 ± 0.3 mM versus 7.8 ± 0.5 mM) suggesting that  F. prausnitzii  synthesized butyrate from the acetate produced by  B. thetaiotaomicron .', 'cell culture', 'Western blot assay,real-time qPC,immunohistochemisty assay', 23692866);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4358, 'mouse', 'increase', 'low-throughput', 'In fact,miR-21 was reportedly induced by pathogenic microbes such as Helicobacter pylori, Salmonella Typhimurium, and Mycobacterium species, leading to excessive immune responses (24, 25). Neither LPS nor P3CSK4 acted in a dose-dependent manner, suggesting that cellular mechanisms suppress the excessive induction of miR-21-5p.', 'cell culture', 'RT-qPCR,Western blot assay', 28760826);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4359, 'mouse', 'increase', 'low-throughput', 'In fact,miR-21 was reportedly induced by pathogenic microbes such as Helicobacter pylori, Salmonella Typhimurium, and Mycobacterium species, leading to excessive immune responses (24, 25). Neither LPS nor P3CSK4 acted in a dose-dependent manner, suggesting that cellular mechanisms suppress the excessive induction of miR-21-5p.', 'cell culture', 'RT-qPCR,Western blot assay', 28760826);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4360, 'mouse', 'increase', 'low-throughput', 'In fact,miR-21 was reportedly induced by pathogenic microbes such as Helicobacter pylori, Salmonella Typhimurium, and Mycobacterium species, leading to excessive immune responses (24, 25). Neither LPS nor P3CSK4 acted in a dose-dependent manner, suggesting that cellular mechanisms suppress the excessive induction of miR-21-5p.', 'cell culture', 'RT-qPCR,Western blot assay', 28760826);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4421, 'mouse', 'decrease', 'low-throughput', 'C. butyricum suppresses the expression of MMP-9 in asthmatic mice.', 'cell culture', 'ELISA', 28122397);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4422, 'mouse', 'decrease', 'low-throughput', 'A recent study suggests that for an OVA-sensitized/ challenged mouse, oral L. rhamnosus GG (LGG) can not only reduce lung resistance and pulmonary inflam mation, but also suppress the expression of T helper 2 (Th2) cytokines, MMP-9 and OVA-specific IgE.16 Sim ilarly, oral administration of C. butyricum to asthmatic mice is effective in reducing allergic airway inflamma tion and remodelling in our study.', 'cell culture', 'ELISA', 28122397);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4423, 'mouse', 'decrease', 'low-throughput', 'Both INF-γ(Fig. S2E, Supplementary Information) and IL-10 (Fig. S2F,I, Supplementary Information) in the PC group were decreased compared with the counterparts of the NC group. They increased in the groups that received C. butyricum orally except for BALF IL-10 in the Tre group.', 'cell culture', 'ELISA', 28122397);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4424, 'mouse', 'decrease', 'low-throughput', 'Compared with the NC group, IgE (Fig. 4A) and IgG1(Fig. 4B) were signifificantly elevated in the PC group (P < 0.001). However, they decreased dramatically in the Pre and Tre groups, especially the level of IgG1 in the Tre group.', 'cell culture', 'ELISA', 28122397);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4462, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4463, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4464, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4465, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4466, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4467, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4468, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4469, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4470, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4471, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4472, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4473, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4474, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4475, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4476, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4477, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4478, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4479, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4480, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4481, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4482, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4483, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4484, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4485, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4486, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4487, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4488, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4489, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4490, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4491, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4492, 'mouse', 'increase', 'low-throughput', 'The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.', 'cell culture', 'null', 26725514);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4493, 'mouse', 'increase', 'low-throughput', 'The metalloprotease GelE, produced by commensal strains of E faecalis, contributes to development of chronic intestinal inflflammation in mice that are susceptible to intestinal inflflammation by impairing epithelial barrier in tegrity.', 'cell culture', 'LC-MS/MS analysis', 21699778);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4496, 'mouse', 'increase', 'low-throughput', 'Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.', 'cell culture', 'RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis', 21531334);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4497, 'mouse', 'increase', 'low-throughput', 'we performed transcriptome profiling and quantitative mass spectrometry on freshly isolated colonocytes from GF and CONV-R mice. For the transcriptome profiling experiments, we identified 624 genes that are upregulated and 813 genes are downregulated in GF compared to CONV-R. All of these gene expression changes are statistically significant with false discovery rates below 0.01. The entire data set and statistical values are included as Table S1. Our transcriptome data includes genes previously shown to be regulated by microbiota in various tissues of zebrafish, mice, and piglets. For example, Angpt14/Fiaf is upregulated in GF compared to CONV while Gpx2, ApoB, Aqp4, and Pla2g4c are downregulated.', 'cell culture', 'RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis', 21531334);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4498, 'mouse', 'decrease', 'low-throughput', 'we performed transcriptome profiling and quantitative mass spectrometry on freshly isolated colonocytes from GF and CONV-R mice. For the transcriptome profiling experiments, we identified 624 genes that are upregulated and 813 genes are downregulated in GF compared to CONV-R. All of these gene expression changes are statistically significant with false discovery rates below 0.01. The entire data set and statistical values are included as Table S1. Our transcriptome data includes genes previously shown to be regulated by microbiota in various tissues of zebrafish, mice, and piglets. For example, Angpt14/Fiaf is upregulated in GF compared to CONV while Gpx2, ApoB, Aqp4, and Pla2g4c are downregulated.', 'cell culture', 'RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis', 21531334);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4499, 'mouse', 'decrease', 'low-throughput', 'we performed transcriptome profiling and quantitative mass spectrometry on freshly isolated colonocytes from GF and CONV-R mice. For the transcriptome profiling experiments, we identified 624 genes that are upregulated and 813 genes are downregulated in GF compared to CONV-R. All of these gene expression changes are statistically significant with false discovery rates below 0.01. The entire data set and statistical values are included as Table S1. Our transcriptome data includes genes previously shown to be regulated by microbiota in various tissues of zebrafish, mice, and piglets. For example, Angpt14/Fiaf is upregulated in GF compared to CONV while Gpx2, ApoB, Aqp4, and Pla2g4c are downregulated.', 'cell culture', 'RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis', 21531334);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4500, 'mouse', 'decrease', 'low-throughput', 'we performed transcriptome profiling and quantitative mass spectrometry on freshly isolated colonocytes from GF and CONV-R mice. For the transcriptome profiling experiments, we identified 624 genes that are upregulated and 813 genes are downregulated in GF compared to CONV-R. All of these gene expression changes are statistically significant with false discovery rates below 0.01. The entire data set and statistical values are included as Table S1. Our transcriptome data includes genes previously shown to be regulated by microbiota in various tissues of zebrafish, mice, and piglets. For example, Angpt14/Fiaf is upregulated in GF compared to CONV while Gpx2, ApoB, Aqp4, and Pla2g4c are downregulated.', 'cell culture', 'RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis', 21531334);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4501, 'mouse', 'decrease', 'low-throughput', 'we performed transcriptome profiling and quantitative mass spectrometry on freshly isolated colonocytes from GF and CONV-R mice. For the transcriptome profiling experiments, we identified 624 genes that are upregulated and 813 genes are downregulated in GF compared to CONV-R. All of these gene expression changes are statistically significant with false discovery rates below 0.01. The entire data set and statistical values are included as Table S1. Our transcriptome data includes genes previously shown to be regulated by microbiota in various tissues of zebrafish, mice, and piglets. For example, Angpt14/Fiaf is upregulated in GF compared to CONV while Gpx2, ApoB, Aqp4, and Pla2g4c are downregulated.', 'cell culture', 'RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis', 21531334);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4502, 'mouse', 'increase', 'low-throughput', 'Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.', 'cell culture', 'RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis', 21531334);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4503, 'mouse', 'increase', 'low-throughput', 'Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.', 'cell culture', 'RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis', 21531334);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4504, 'mouse', 'increase', 'low-throughput', 'Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.', 'cell culture', 'RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis', 21531334);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4505, 'mouse', 'increase', 'low-throughput', 'Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.', 'cell culture', 'RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis', 21531334);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4506, 'mouse', 'increase', 'low-throughput', 'Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.', 'cell culture', 'RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis', 21531334);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4509, 'mouse', 'decrease', 'low-throughput', 'mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.', 'cell culture', 'RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis', 28628112);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4510, 'mouse', 'decrease', 'low-throughput', 'mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.', 'cell culture', 'real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis', 28628112);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4511, 'mouse', 'decrease', 'low-throughput', 'mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.', 'cell culture', 'real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis', 28628112);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4514, 'mouse', 'increase', 'low-throughput', 'mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.', 'cell culture', 'real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis', 28628112);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4515, 'mouse', 'increase', 'low-throughput', 'mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.', 'cell culture', 'real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis', 28628112);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4516, 'mouse', 'increase', 'low-throughput', 'mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.', 'cell culture', 'real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis', 28628112);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4517, 'mouse', 'increase', 'low-throughput', 'mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.', 'cell culture', 'real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis', 28628112);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4518, 'mouse', 'increase', 'low-throughput', 'mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.', 'cell culture', 'real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis', 28628112);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4519, 'mouse', 'decrease', 'low-throughput', 'mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.', 'cell culture', 'real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis', 28628112);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4520, 'mouse', 'decrease', 'low-throughput', 'mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.', 'cell culture', 'real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis', 28628112);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4521, 'mouse', 'decrease', 'low-throughput', 'mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.', 'cell culture', 'real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis', 28628112);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4522, 'mouse', 'decrease', 'low-throughput', 'mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.', 'cell culture', 'real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis', 28628112);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4523, 'mouse', 'increase', 'low-throughput', 'Here we show using untargeted metabolomics that levels of phenyl sulfate, a gut microbiota-derived metabolite, increase with the progression of diabetes in rats overexpressing human uremic toxin transporter SLCO4C1 in the kidney, and are decreased in rats with limited proteinuria.', 'cell culture', 'UHPLC analysis,UHPLC-QTOF/MS analysis,LC/MS analysis', 31015435);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4524, 'mouse', 'increase', 'low-throughput', 'although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.', 'cell culture', 'UHPLC analysis,UHPLC-QTOF/MS analysis,LC/MS analysis', 31015435);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4525, 'mouse', 'increase', 'low-throughput', 'although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.', 'cell culture', 'UHPLC analysis,UHPLC-QTOF/MS analysis,LC/MS analysis', 31015435);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4526, 'mouse', 'increase', 'low-throughput', 'although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.', 'cell culture', 'UHPLC analysis,UHPLC-QTOF/MS analysis,LC/MS analysis', 31015435);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4527, 'mouse', 'increase', 'low-throughput', 'although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.', 'cell culture', 'UHPLC analysis,UHPLC-QTOF/MS analysis,LC/MS analysis', 31015435);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4528, 'mouse', 'increase', 'low-throughput', 'although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.', 'cell culture', 'UHPLC analysis,UHPLC-QTOF/MS analysis,LC/MS analysis', 31015435);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4529, 'mouse', 'increase', 'low-throughput', 'The expression of tight junction protein ZO-1, Occludin and Claudin-1 was measured by western blot. As shown in Fig. 2A, the expression of Claudin-1 was significantly decreased in irinotecan treated mice compared with control group. However, ZO-1 and Oc cludin expression was not modified by irinotecan. Interestingly, EcN significantly restored the decreased Claudin-1 expression.', 'cell culture', 'Western blot assay', 31173781);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4531, 'mouse', 'increase', 'low-throughput', 'Live B. dentium-monoassociated mice exhibited increased colonic expression of goblet cell markers Krüppel-like factor 4 (Klf4), Trefoil factor 3 (Tff3), Relm-β, Muc2, and several glycosyltransferases compared to both heat-killed B. dentium and germfree counterparts.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4532, 'mouse', 'increase', 'low-throughput', 'Live B. dentium-monoassociated mice exhibited increased colonic expression of goblet cell markers Krüppel-like factor 4 (Klf4), Trefoil factor 3 (Tff3), Relm-β, Muc2, and several glycosyltransferases compared to both heat-killed B. dentium and germfree counterparts.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4533, 'mouse', 'increase', 'low-throughput', 'Live B. dentium-monoassociated mice exhibited increased colonic expression of goblet cell markers Krüppel-like factor 4 (Klf4), Trefoil factor 3 (Tff3), Relm-β, Muc2, and several glycosyltransferases compared to both heat-killed B. dentium and germfree counterparts.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4534, 'mouse', 'increase', 'low-throughput', 'Live B. dentium-monoassociated mice exhibited increased colonic expression of goblet cell markers Krüppel-like factor 4 (Klf4), Trefoil factor 3 (Tff3), Relm-β, Muc2, and several glycosyltransferases compared to both heat-killed B. dentium and germfree counterparts.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4535, 'mouse', 'increase', 'low-throughput', 'Increases in B. dentium-monoassociated colons compared with germfree controls were observed in the following glycosyltransferases: B3gnt6 (β-1,3-N-acetylglucosaminyltransferase 6), C1galt1 (β-1,3-galactosyltransferase 1), C2gnt [glucosaminyl (N-acetyl)transferase 1], Fut2 (fucosyltransferase 2), C2GnT1 [glucosaminyl (N-acetyl)transferase 1], and St6gal1 (ST6 N-acetylgalactosaminide α-2,6-sialyltransferase).', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4536, 'mouse', 'increase', 'low-throughput', 'Increases in B. dentium-monoassociated colons compared with germfree controls were observed in the following glycosyltransferases: B3gnt6 (β-1,3-N-acetylglucosaminyltransferase 6), C1galt1 (β-1,3-galactosyltransferase 1), C2gnt [glucosaminyl (N-acetyl)transferase 1], Fut2 (fucosyltransferase 2), C2GnT1 [glucosaminyl (N-acetyl)transferase 1], and St6gal1 (ST6 N-acetylgalactosaminide α-2,6-sialyltransferase).', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4537, 'mouse', 'increase', 'low-throughput', 'Increases in B. dentium-monoassociated colons compared with germfree controls were observed in the following glycosyltransferases: B3gnt6 (β-1,3-N-acetylglucosaminyltransferase 6), C1galt1 (β-1,3-galactosyltransferase 1), C2gnt [glucosaminyl (N-acetyl)transferase 1], Fut2 (fucosyltransferase 2), C2GnT1 [glucosaminyl (N-acetyl)transferase 1], and St6gal1 (ST6 N-acetylgalactosaminide α-2,6-sialyltransferase).', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4538, 'mouse', 'increase', 'low-throughput', 'Increases in B. dentium-monoassociated colons compared with germfree controls were observed in the following glycosyltransferases: B3gnt6 (β-1,3-N-acetylglucosaminyltransferase 6), C1galt1 (β-1,3-galactosyltransferase 1), C2gnt [glucosaminyl (N-acetyl)transferase 1], Fut2 (fucosyltransferase 2), C2GnT1 [glucosaminyl (N-acetyl)transferase 1], and St6gal1 (ST6 N-acetylgalactosaminide α-2,6-sialyltransferase).', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4539, 'mouse', 'increase', 'low-throughput', 'Increases in B. dentium-monoassociated colons compared with germfree controls were observed in the following glycosyltransferases: B3gnt6 (β-1,3-N-acetylglucosaminyltransferase 6), C1galt1 (β-1,3-galactosyltransferase 1), C2gnt [glucosaminyl (N-acetyl)transferase 1], Fut2 (fucosyltransferase 2), C2GnT1 [glucosaminyl (N-acetyl)transferase 1], and St6gal1 (ST6 N-acetylgalactosaminide α-2,6-sialyltransferase).', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4540, 'mouse', 'increase', 'low-throughput', 'We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4541, 'mouse', 'increase', 'low-throughput', 'We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4542, 'mouse', 'increase', 'low-throughput', 'We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4543, 'mouse', 'increase', 'low-throughput', 'We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4544, 'mouse', 'increase', 'low-throughput', 'We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4545, 'mouse', 'increase', 'low-throughput', 'B. dentium can secrete products, such as acetate, that are capable of increasing MUC2 synthesis.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4546, 'mouse', 'increase', 'low-throughput', 'Of note, a trend toward increased expression of Relm-β and Muc2 was observed in heat-killed B. dentium-treated mice. Although not significant, this may point to a complementary role of B. dentium surface proteins in modulating goblet cells. Several cytokines are associated with increased mucus production, most notably interleukin-22 (IL-22), IL-33, and IL-13.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4547, 'mouse', 'increase', 'low-throughput', 'Of note, a trend toward increased expression of Relm-β and Muc2 was observed in heat-killed B. dentium-treated mice. Although not significant, this may point to a complementary role of B. dentium surface proteins in modulating goblet cells. Several cytokines are associated with increased mucus production, most notably interleukin-22 (IL-22), IL-33, and IL-13.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4548, 'mouse', 'increase', 'low-throughput', 'Of note, a trend toward increased expression of Relm-β and Muc2 was observed in heat-killed B. dentium-treated mice. Although not significant, this may point to a complementary role of B. dentium surface proteins in modulating goblet cells. Several cytokines are associated with increased mucus production, most notably interleukin-22 (IL-22), IL-33, and IL-13.', 'cell culture', 'real-time qPCR', 31213556);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4557, 'mouse', 'decrease', 'low-throughput', 'the secretion of NF-kB-dependent cytokines (TNF-a, IL-1b and IL-6) was dramatically inhibited by treatment with live EC-M17.', 'control experiment,in vitro bacterial culture', 'Western blot analysis', 18279557);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4558, 'mouse', 'decrease', 'low-throughput', 'the secretion of NF-kB-dependent cytokines (TNF-a, IL-1b and IL-6) was dramatically inhibited by treatment with live EC-M17.', 'control experiment,in vitro bacterial culture', 'Western blot analysis', 18279557);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4559, 'mouse', 'decrease', 'low-throughput', 'the secretion of NF-kB-dependent cytokines (TNF-a, IL-1b and IL-6) was dramatically inhibited by treatment with live EC-M17.', 'control experiment,in vitro bacterial culture', 'Western blot analysis', 18279557);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4560, 'mouse', 'increase', 'low-throughput', 'D8 up-regulates IL-22 expression ex vivo and in vivo.', 'control experiment,in vitro bacterial culture', 'RT-qPCR,real-time PCR,real-time qPCR,16s rRNA sequence', 29459771);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4565, 'mouse', 'decrease', 'low-throughput', 'The OLL2838 strain significantly suppressed TNF-α-induced IL-8 production.', 'control experiment,in vitro bacterial culture', 'ELISA,Western blot assay', 19447972);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4566, 'mouse', 'increase', 'low-throughput', 'The OLL2838 strain prevented the loss of ZO-1 and MLCK and suppressed the increase in mucosal permeability.', 'control experiment,in vitro bacterial culture', 'ELISA,Western blot assay', 19447972);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4567, 'mouse', 'increase', 'low-throughput', 'The OLL2838 strain prevented the loss of ZO-1 and MLCK and suppressed the increase in mucosal permeability.', 'control experiment,in vitro bacterial culture', 'ELISA,Western blot assay', 19447972);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4570, 'mouse', 'decrease', 'low-throughput', 'The mRNA expression of IL-1β, IL-6, IL-17A and IFN-γ decreased significantly in the DSS plus TN-3 mice as compared to the DSS mice.', 'control experiment', 'real-time PCR,HPLC analysis,T-RFLP', 27438072);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4571, 'mouse', 'decrease', 'low-throughput', 'The mRNA expression of IL-1β, IL-6, IL-17A and IFN-γ decreased significantly in the DSS plus TN-3 mice as compared to the DSS mice.', 'control experiment', 'real-time PCR,HPLC analysis,T-RFLP', 27438072);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4572, 'mouse', 'decrease', 'low-throughput', 'The mRNA expression of IL-1β, IL-6, IL-17A and IFN-γ decreased significantly in the DSS plus TN-3 mice as compared to the DSS mice.', 'control experiment', 'real-time PCR,HPLC analysis,T-RFLP', 27438072);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4573, 'mouse', 'decrease', 'low-throughput', 'The mRNA expression of IL-1β, IL-6, IL-17A and IFN-γ decreased significantly in the DSS plus TN-3 mice as compared to the DSS mice.', 'control experiment', 'real-time PCR,HPLC analysis,T-RFLP', 27438072);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4574, 'mouse', 'increase', 'low-throughput', 'The expression of IL-10 increased in the colon tissues of the DSS plus TN-3 mice as compared to the DSS mice using immunohistochemistry.', 'control experiment', 'real-time PCR,HPLC analysis,T-RFLP', 27438072);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4576, 'mouse', 'decrease', 'low-throughput', 'Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.', 'cell culture', 'null', 25456122);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4577, 'mouse', 'decrease', 'low-throughput', 'Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.', 'cell culture', 'null', 25456122);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4578, 'mouse', 'decrease', 'low-throughput', 'Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.', 'cell culture', 'null', 25456122);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4588, 'mouse', 'increase', 'low-throughput', 'NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.', 'control experiment', 'ELISA�?real-time PCR', 28104981);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4589, 'mouse', 'increase', 'low-throughput', 'NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.', 'control experiment', 'ELISA�?real-time PCR', 28104981);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4590, 'mouse', 'increase', 'low-throughput', 'NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.', 'control experiment', 'ELISA�?real-time PCR', 28104981);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4597, 'mouse', 'increase', 'low-throughput', 'The helminth Nippostrongylus brasiliensis and a tritrichomonad protist both secreted succinate as a metabolite. In vivo sensing of the tritrichomonad required SUCNR1, whereas N. brasiliensis was SUCNR1 independent. These findings define a paradigm wherein tuft cells monitor microbial metabolites to initiate type 2 immunity and suggest the existence of other sensing pathways triggering the response to helminths.', 'Cell culture,in vitro bacterial culture', 'RT-qPCR,RNA-seq,', 30021144);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4598, 'mouse', 'increase', 'low-throughput', 'The helminth Nippostrongylus brasiliensis and a tritrichomonad protist both secreted succinate as a metabolite. In vivo sensing of the tritrichomonad required SUCNR1, whereas N. brasiliensis was SUCNR1 independent. These findings define a paradigm wherein tuft cells monitor microbial metabolites to initiate type 2 immunity and suggest the existence of other sensing pathways triggering the response to helminths.', 'Cell culture,in vitro bacterial culture', 'RT-qPCR,RNA-seq,', 30021144);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4600, 'mouse', 'increase', 'low-throughput', 'Our study shows that MAM from F. prausnitzii can restore the intestinal barrier structure and function in DM conditions via the regulation of the tight junction pathway and ZO-1 expression.Collectively, our data reveal that MAM protein upregulates ZO-1 expression in intestinal epithelial cells.', 'control experiment,cell culture', '16S high-throughput sequencing,qRT-PCR,Western blot assay', 31503404);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4612, 'mouse', 'decrease', 'low-throughput', 'Faecalibacterium prausnitzii produces butyrate to decrease c-Myc-related metabolism and Th17 differentiation by inhibiting histone deacetylase 3.', 'control experiment,in vitro bacterial culture', 'qPCR,real-time PCR,Western blot assay,GC-MS analysis', 30809639);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4614, 'mouse', 'increase', 'low-throughput', 'F. prausnitzii, B. longum and F. prausnitzii supernatant clearly facilitated the induction of IL-10 and TGF-β1, while induced relatively mild production of IL-12p70 in both cellular and animal models. The F. prausnitzii, B. longum and supernatant differed in their capacity to induce T-bet, GATA-3 and ROR-γt mRNA expression in human PBMCs (both bacterial strains inhibited the expression of ROR-γt while supernatant inhibited the T-bet and GATA-3). However, all of them induced the Foxp3 and Treg production and ameliorated the TNBS-induced colitis. In addition, F. prausnitzii supernatant exhibited the supreme anti-inflammatory capacity.', 'control experiment,in vitro bacterial culture', 'ELISA,real-time qPCR', 23643066);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4615, 'mouse', 'increase', 'low-throughput', 'F. prausnitzii, B. longum and F. prausnitzii supernatant clearly facilitated the induction of IL-10 and TGF-β1, while induced relatively mild production of IL-12p70 in both cellular and animal models. The F. prausnitzii, B. longum and supernatant differed in their capacity to induce T-bet, GATA-3 and ROR-γt mRNA expression in human PBMCs (both bacterial strains inhibited the expression of ROR-γt while supernatant inhibited the T-bet and GATA-3). However, all of them induced the Foxp3 and Treg production and ameliorated the TNBS-induced colitis. In addition, F. prausnitzii supernatant exhibited the supreme anti-inflammatory capacity.', 'control experiment,in vitro bacterial culture', 'ELISA,real-time qPCR', 23643066);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4619, 'mouse', 'decrease', 'low-throughput', 'Colonic cDNA concentrations of IL-1b and TNF-a were significantly down regulated in DSS-treated E. durans-fed mice but not in control or DSS-treated L.delbrueckii- fed mice.', 'in vitro bacterial culture,control experiment', 'qPCR,real-time PCR', 17349085);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4620, 'mouse', 'decrease', 'low-throughput', 'Colonic cDNA concentrations of IL-1b and TNF-a were significantly down regulated in DSS-treated E. durans-fed mice but not in control or DSS-treated L.delbrueckii- fed mice.', 'in vitro bacterial culture,control experiment', 'qPCR,real-time PCR', 17349085);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4623, 'mouse', 'increase', 'low-throughput', 'Lactobacilli Induce IL-22 via Iald.', 'control experiment,in vitro bacterial culture', '16S rRNA sequence,qPCR ,HPLC-HRMS Analysis', 23973224);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4631, 'mouse', 'decrease', 'low-throughput', 'The mRNA levels of  -SMA and TIMP-1 were revealed to be markedly elevated in the livers of the MCD diet-fed mice, and these high expressions were normalized by LcS intervention (Fig. 3B). These data suggest LcS intervention to have suppressed HSC activation and liver fibrosis.', 'control experiment,in vitro bacterial culture', 'real-time RT-qPCR', 24113768);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4632, 'mouse', 'decrease', 'low-throughput', 'The mRNA levels of  -SMA and TIMP-1 were revealed to be markedly elevated in the livers of the MCD diet-fed mice, and these high expressions were normalized by LcS intervention (Fig. 3B). These data suggest LcS intervention to have suppressed HSC activation and liver fibrosis.', 'control experiment,in vitro bacterial culture', 'real-time RT-qPCR', 24113768);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4633, 'mouse', 'decrease', 'low-throughput', 'To investigate the molecular mechanisms underlying the resistance to NASH development conferred by LcS intervention, the mRNA expression levels of lipogenic enzyme genes such as SREBP-1c and FAS, oxidation enzyme genes, CPT-1, and CD36 were examined by quantitative real-time PCR. While MCD diet feeding signifi- cantly elevated the expression of SREBP-1c but not that of FAS, LcS intervention suppressed these mRNA levels compared with the MCD diet alone (Fig. 4A).', 'control experiment,in vitro bacterial culture', 'real-time RT-qPCR', 24113768);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4634, 'mouse', 'decrease', 'low-throughput', 'To investigate the molecular mechanisms underlying the resistance to NASH development conferred by LcS intervention, the mRNA expression levels of lipogenic enzyme genes such as SREBP-1c and FAS, oxidation enzyme genes, CPT-1, and CD36 were examined by quantitative real-time PCR. While MCD diet feeding signifi- cantly elevated the expression of SREBP-1c but not that of FAS, LcS intervention suppressed these mRNA levels compared with the MCD diet alone (Fig. 4A).', 'control experiment,in vitro bacterial culture', 'real-time RT-qPCR', 24113768);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4635, 'mouse', 'increase', 'low-throughput', 'To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.', 'control experiment', '16S rDNA sequence,LC-MS/MS analysis,RT-PCR,Western blot assay', 33006494);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4636, 'mouse', 'increase', 'low-throughput', 'To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.', 'control experiment', '16S rDNA sequence,LC-MS/MS analysis,RT-PCR,Western blot assay', 33006494);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4637, 'mouse', 'increase', 'low-throughput', 'To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.', 'control experiment', '16S rDNA sequence,LC-MS/MS analysis,RT-PCR,Western blot assay', 33006494);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4638, 'mouse', 'increase', 'low-throughput', 'To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.', 'control experiment', '16S rDNA sequence,LC-MS/MS analysis,RT-PCR,Western blot assay', 33006494);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4639, 'mouse', 'increase', 'low-throughput', 'Indeed, DCA stimulation could induce potent increase of mRNA expression level of M2-mAchR in macrophages, and selective inhibition of M2-mAchR significantly reduced theproduction of TNF-α, NO and IL-6 triggered by DCA (Figure 4(d,e) and Supplementary Figure 4c).', 'control experiment', '16S rDNA sequence,LC-MS/MS analysis,RT-PCR,Western blot assay', 33006494);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4640, 'mouse', 'increase', 'low-throughput', 'Toll-like receptor 2 (TLR2) is up-regulated upon DCA stimulation and involved in DCA-induced macrophage polarization.', 'control experiment', 'RT-PCR,Western blot assay,ChIP', 33006494);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4652, 'mouse', 'increase', 'low-throughput', 'When the complex carbohydrates of breast milk interact with B. fragilis, it produces short-chain fatty acids (SCFAs). We know from animal studies that SCFAs interacting with G-protein-coupled receptors can stimulate regulatory T cells via the GPR 43 receptor to activate the original cells and to produce IL-10, which inhibits inflammation [44].', 'null', 'null', 32229728);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4653, 'mouse', 'increase', 'low-throughput', 'To determine the functional impact of B dentium colonization on enterochromaffin-produced serotonin, we assessed 5-HT concentrations by immunostaining, qPCR,and LC-MS/MS. Increased numbers of 5-HT–positive cells were observed in both the ileum and colon of B dentium mono-associated mice relative to germ-free and heat-killed B dentium–treated mice (Figure 3A and B). Increases in 5-HT were observed in enteroendocrine cells directly above enteric neurons (Figure 3C). Tph-1 is the rate limiting enzyme for 5-HT production in enterochromaffin cells. The results of the immunostaining were corroborated by the significant increase in Tph-1 expression in both the ileum and colon of B dentium mono-associated mice relative to germ-free and heat-killed B dentium–treated mice(Figure 3D). Increased Tph-1 gene expression also was accompanied by significantly increased 5-HT concentrations in the intestinal lumen and feces of mice, as measured by LC-MS/MS (Figure 3E).', 'in vitro bacterial culture', 'immunostaining, qPCR，LC-MS/MS analysis', 32795610);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4654, 'mouse', 'increase', 'low-throughput', 'We found that expression of the colonic 5-HT receptors 2a and 4 (HTR2A and HTR4) were increased significantly in B dentium mono-associated mice (Figure 7A).', 'in vitro bacterial culture', 'qPCR，immunostaining', 32795610);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4655, 'mouse', 'increase', 'low-throughput', 'We found that expression of the colonic 5-HT receptors 2a and 4 (HTR2A and HTR4) were increased significantly in B dentium mono-associated mice (Figure 7A).', 'in vitro bacterial culture', 'qPCR，immunostaining', 32795610);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4683, 'mouse', 'increase', 'low-throughput', 'Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).', 'in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 33192557);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4684, 'mouse', 'decrease', 'low-throughput', 'Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).', 'in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 33192557);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4685, 'mouse', 'increase', 'low-throughput', 'Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).', 'in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 33192557);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4686, 'mouse', 'increase', 'low-throughput', 'Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).', 'in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 33192557);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4687, 'mouse', 'decrease', 'low-throughput', 'Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).', 'in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 33192557);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4688, 'mouse', 'decrease', 'low-throughput', 'In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).', 'in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 33192557);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4689, 'mouse', 'decrease', 'low-throughput', 'In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).', 'in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 33192557);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4690, 'mouse', 'decrease', 'low-throughput', 'In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).', 'in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 33192557);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4691, 'mouse', 'decrease', 'low-throughput', 'Relative to the SAP + IAH group, the levels of claudin-2, MMP9, and TNF-α were significantly lower in the two treatment groups (P < 0.05; Figures 5A,B).', 'in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay', 33192557);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4701, 'mouse', 'increase', 'low-throughput', 'SLC5A8 and GPR43 expressions in butyrate‑treated CRC cells were quantified (Fig. 5B and C for SLC5A8 and GPR43, respectively). In comparison with untreated cells, the NaB‑treated SW480 cells only presented higher GPR43 (P<0.05), but the NaB‑treated SW620 cells expressed the most SLC5A8 (P<0.01) and GPR43 (P<0.05).', 'cell culture,control experiment', 'GC/MS analysis', 33101496);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4702, 'mouse', 'increase', 'low-throughput', 'SLC5A8 and GPR43 expressions in butyrate‑treated CRC cells were quantified (Fig. 5B and C for SLC5A8 and GPR43, respectively). In comparison with untreated cells, the NaB‑treated SW480 cells only presented higher GPR43 (P<0.05), but the NaB‑treated SW620 cells expressed the most SLC5A8 (P<0.01) and GPR43 (P<0.05).', 'cell culture,control experiment', 'GC/MS analysis', 33101496);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4703, 'mouse', 'increase', 'low-throughput', 'SLC5A8 and GPR43 expressions in butyrate‑treated CRC cells were quantified (Fig. 5B and C for SLC5A8 and GPR43, respectively). In comparison with untreated cells, the NaB‑treated SW480 cells only presented higher GPR43 (P<0.05), but the NaB‑treated SW620 cells expressed the most SLC5A8 (P<0.01) and GPR43 (P<0.05).', 'cell culture,control experiment', 'GC/MS analysis', 33101496);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4704, 'mouse', 'increase', 'low-throughput', 'Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4705, 'mouse', 'increase', 'low-throughput', 'FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4706, 'mouse', 'increase', 'low-throughput', 'In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4707, 'mouse', 'decrease', 'low-throughput', 'In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4708, 'mouse', 'decrease', 'low-throughput', 'Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4709, 'mouse', 'decrease', 'low-throughput', 'Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4710, 'mouse', 'increase', 'low-throughput', 'Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4711, 'mouse', 'increase', 'low-throughput', 'Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4712, 'mouse', 'increase', 'low-throughput', 'FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4713, 'mouse', 'increase', 'low-throughput', 'In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4714, 'mouse', 'decrease', 'low-throughput', 'In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4715, 'mouse', 'decrease', 'low-throughput', 'Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4716, 'mouse', 'decrease', 'low-throughput', 'Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4717, 'mouse', 'increase', 'low-throughput', 'Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4718, 'mouse', 'increase', 'low-throughput', 'Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4719, 'mouse', 'increase', 'low-throughput', 'FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4720, 'mouse', 'increase', 'low-throughput', 'In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4721, 'mouse', 'decrease', 'low-throughput', 'In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4722, 'mouse', 'decrease', 'low-throughput', 'Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4723, 'mouse', 'decrease', 'low-throughput', 'Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4724, 'mouse', 'increase', 'low-throughput', 'Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.', 'control experiment', 'RT-PCR,16S rRNA', 27111464);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4739, 'mouse', 'increase', 'low-throughput', 'B. intestinalis produced the highest levels of colonic lamina propria Tregs [36.6% ±4.39 (mean ± SEM) FoxP3+ among CD4+ T cells], and reached high densities of colonization. Collinsella aerofaciens, which colonized to low density, did not induce an increase in Tregs in this initial screen.', 'in vitro bacterial culture,control experiment,in vivo infection experiment', 'GC/MS analysis,LC/MS analysis', 24452263);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4740, 'mouse', 'increase', 'low-throughput', 'B. intestinalis produced the highest levels of colonic lamina propria Tregs [36.6% ±4.39 (mean ± SEM) FoxP3+ among CD4+ T cells], and reached high densities of colonization. Collinsella aerofaciens, which colonized to low density, did not induce an increase in Tregs in this initial screen.', 'in vitro bacterial culture,control experiment,in vivo infection experiment', 'GC/MS analysis,LC/MS analysis', 24452263);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4741, 'mouse', 'increase', 'low-throughput', 'Each Bacteroides strain resulted in significant increases in the proportions of FoxP3+ Treg cells among CD4+ T cells in the colon lamina propria (Fig. 4A and fig. S5A; see table S9 for P-values).', 'in vitro bacterial culture,control experiment,in vivo infection experiment', 'GC/MS analysis,LC/MS analysis', 24452263);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4742, 'mouse', 'increase', 'low-throughput', 'Each Bacteroides strain resulted in significant increases in the proportions of FoxP3+ Treg cells among CD4+ T cells in the colon lamina propria (Fig. 4A and fig. S5A; see table S9 for P-values).', 'in vitro bacterial culture,control experiment,in vivo infection experiment', 'GC/MS analysis,LC/MS analysis', 24452263);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4743, 'mouse', 'increase', 'low-throughput', 'Each Bacteroides strain resulted in significant increases in the proportions of FoxP3+ Treg cells among CD4+ T cells in the colon lamina propria (Fig. 4A and fig. S5A; see table S9 for P-values).', 'in vitro bacterial culture,control experiment,in vivo infection experiment', 'GC/MS analysis,LC/MS analysis', 24452263);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4744, 'mouse', 'increase', 'low-throughput', 'Each Bacteroides strain resulted in significant increases in the proportions of FoxP3+ Treg cells among CD4+ T cells in the colon lamina propria (Fig. 4A and fig. S5A; see table S9 for P-values).', 'in vitro bacterial culture,control experiment,in vivo infection experiment', 'GC/MS analysis,LC/MS analysis', 24452263);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4745, 'mouse', 'increase', 'low-throughput', 'Each Bacteroides strain resulted in significant increases in the proportions of FoxP3+ Treg cells among CD4+ T cells in the colon lamina propria (Fig. 4A and fig. S5A; see table S9 for P-values).', 'in vitro bacterial culture,control experiment,in vivo infection experiment', 'GC/MS analysis,LC/MS analysis', 24452263);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4746, 'mouse', 'increase', 'low-throughput', 'Each Bacteroides strain resulted in significant increases in the proportions of FoxP3+ Treg cells among CD4+ T cells in the colon lamina propria (Fig. 4A and fig. S5A; see table S9 for P-values).', 'in vitro bacterial culture,control experiment,in vivo infection experiment', 'GC/MS analysis,LC/MS analysis', 24452263);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4760, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4761, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4762, 'mouse', 'increase', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4763, 'mouse', 'decrease', 'low-throughput', 'Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4764, 'mouse', 'decrease', 'low-throughput', 'Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4765, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4766, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4767, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4768, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4769, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4770, 'mouse', 'increase', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4771, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4772, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4773, 'mouse', 'increase', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4774, 'mouse', 'increase', 'low-throughput', 'GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4775, 'mouse', 'increase', 'low-throughput', 'Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4776, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4777, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4778, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4779, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4780, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4781, 'mouse', 'increase', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4782, 'mouse', 'increase', 'low-throughput', 'GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4783, 'mouse', 'increase', 'low-throughput', 'Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4784, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4785, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4786, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4787, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4788, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4789, 'mouse', 'increase', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4790, 'mouse', 'increase', 'low-throughput', 'GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4791, 'mouse', 'increase', 'low-throughput', 'Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4792, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4793, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4794, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4795, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4796, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4797, 'mouse', 'increase', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4798, 'mouse', 'increase', 'low-throughput', 'GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4799, 'mouse', 'increase', 'low-throughput', 'Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4800, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4801, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4802, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4803, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4804, 'mouse', 'increase', 'low-throughput', 'Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4805, 'mouse', 'increase', 'low-throughput', 'SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4806, 'mouse', 'decrease', 'low-throughput', 'Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4807, 'mouse', 'decrease', 'low-throughput', 'Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4808, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4809, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4810, 'mouse', 'decrease', 'low-throughput', 'Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.', 'null', '16S rRNA sequence', 27448578);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4811, 'mouse', 'increase', 'low-throughput', 'Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay,HPLC analysis', 29454749);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4812, 'mouse', 'increase', 'low-throughput', 'Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay,HPLC analysis', 29454749);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4813, 'mouse', 'increase', 'low-throughput', 'IAld has recently been identified as an aryl hydrocarbon receptor (AHR) ligand.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay,HPLC analysis', 29454749);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4814, 'mouse', 'increase', 'low-throughput', 'Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay,HPLC analysis', 29454749);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4815, 'mouse', 'increase', 'low-throughput', 'Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay,HPLC analysis', 29454749);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4816, 'mouse', 'increase', 'low-throughput', 'IAld has recently been identified as an aryl hydrocarbon receptor (AHR) ligand.', 'cell culture,in vitro bacterial culture,control experiment', 'RT-PCR,Western blot assay,HPLC analysis', 29454749);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4822, 'mouse', 'decrease', 'low-throughput', 'In summary, our results clearly demonstrate that different E. faecalis strains differentially induce colitis in monoassociated IL-10-/- mice and that severe E. faecalis-induced colitis resulted in decreased Alix/AIP1 protein expression as well as ERK1/2 phosphorylation. Functional studies revealed no effect of Alix/AIP1 on bacteria- and EGF-mediated effects on the epithelium. The 13 differentially regulated proteins might be ofinterest aspotentialmarkers for severeintestinalinflammation.', 'in vitro bacterial culture,cell culture', 'qPCR,ELISA,Western blot assay', 19166300);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4823, 'mouse', 'decrease', 'low-throughput', 'In summary, our results clearly demonstrate that different E. faecalis strains differentially induce colitis in monoassociated IL-10-/- mice and that severe E. faecalis-induced colitis resulted in decreased Alix/AIP1 protein expression as well as ERK1/2 phosphorylation. Functional studies revealed no effect of Alix/AIP1 on bacteria- and EGF-mediated effects on the epithelium. The 13 differentially regulated proteins might be ofinterest aspotentialmarkers for severeintestinalinflammation.', 'in vitro bacterial culture,cell culture', 'qPCR,ELISA,Western blot assay', 19166300);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (4824, 'mouse', 'decrease', 'low-throughput', 'In summary, our results clearly demonstrate that different E. faecalis strains differentially induce colitis in monoassociated IL-10-/- mice and that severe E. faecalis-induced colitis resulted in decreased Alix/AIP1 protein expression as well as ERK1/2 phosphorylation. Functional studies revealed no effect of Alix/AIP1 on bacteria- and EGF-mediated effects on the epithelium. The 13 differentially regulated proteins might be ofinterest aspotentialmarkers for severeintestinalinflammation.', 'in vitro bacterial culture,cell culture', 'qPCR,ELISA,Western blot assay', 19166300);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5026, 'mouse', 'increase', 'low-throughput', 'Human and murine pregnancy were associated with reduced intestinal FXR signaling, with lower FGF19/15 and resultant increased hepatic bile acid synthesis. Terminal ileal ASBT protein was reduced in murine pregnancy. Cecal bile acid conjugation was reduced in pregnancy because of elevated bile salt hydrolase‐producing Bacteroidetes. CA supplementation induced intestinal FXR signaling, which was not abrogated by pregnancy, with strikingly similar changes to the microbiota and metabonome as identified in pregnancy. Conclusion: The altered intestinal microbiota of pregnancy enhance bile acid deconjugation, reducing ileal bile acid uptake and lowering FXR induction in enterocytes. This exacerbates the effects mediated by reduced bile acid uptake transporters in pregnancy. Thus, in pregnant women and mice, there is reduced FGF19/15‐mediated hepatic repression of hepatic bile acid synthesis, resulting in hypercholanemia.', 'in vivo experiment', 'RT-PCR,16S rRNA sequence', 30983011);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5031, 'mouse', 'decrease', 'low-throughput', 'Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.', 'in vitro bacterial culture,in vivo infection experiment', 'HPLC analysis', 26323573);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5032, 'mouse', 'decrease', 'low-throughput', 'Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.', 'in vitro bacterial culture,in vivo infection experiment', 'HPLC analysis', 26323573);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5033, 'mouse', 'decrease', 'low-throughput', 'Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.', 'in vitro bacterial culture,in vivo infection experiment', 'HPLC analysis', 26323573);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5034, 'mouse', 'decrease', 'low-throughput', 'Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.', 'in vitro bacterial culture,in vivo infection experiment', 'HPLC analysis', 26323573);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5035, 'mouse', 'decrease', 'low-throughput', 'Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.', 'in vitro bacterial culture,in vivo infection experiment', 'HPLC analysis', 26323573);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5036, 'mouse', 'decrease', 'low-throughput', 'Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.', 'in vitro bacterial culture,in vivo infection experiment', 'HPLC analysis', 26323573);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5040, 'mouse', 'decrease', 'low-throughput', 'Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.', 'in vivo experiment,cell culture', 'qPCR', 23665276);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5041, 'mouse', 'decrease', 'low-throughput', 'Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.', 'in vivo experiment,cell culture', 'qPCR', 23665276);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5042, 'mouse', 'decrease', 'low-throughput', 'Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.', 'in vivo experiment,cell culture', 'qPCR', 23665276);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5043, 'mouse', 'decrease', 'low-throughput', 'Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.', 'in vivo experiment,cell culture', 'qPCR', 23665276);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5044, 'mouse', 'decrease', 'low-throughput', 'Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.', 'in vivo experiment,cell culture', 'qPCR', 23665276);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5045, 'mouse', 'decrease', 'low-throughput', 'Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.', 'in vivo experiment,cell culture', 'qPCR', 23665276);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5046, 'mouse', 'decrease', 'low-throughput', 'Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.', 'in vivo experiment,cell culture', 'qPCR', 23665276);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5049, 'mouse', 'increase', 'low-throughput', 'B. infantis and MO decreased hepatic CYP7A1 and induced Sult2a1, Sult2a2, and Sult2a3 suggesting reduced BA synthesis and increased detoxification. Furthermore, B. infantis and MO increased ileal BA membrane receptor TGR5-regulated signaling. Together, via BA-regulated signaling, synbiotics B. infantis and MO have their unique and combined effects in reversing NASH.', 'in vitro bacterial culture', 'qPCR,LC/MS analysis', 15640190);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5050, 'mouse', 'increase', 'low-throughput', 'B. infantis and MO decreased hepatic CYP7A1 and induced Sult2a1, Sult2a2, and Sult2a3 suggesting reduced BA synthesis and increased detoxification. Furthermore, B. infantis and MO increased ileal BA membrane receptor TGR5-regulated signaling. Together, via BA-regulated signaling, synbiotics B. infantis and MO have their unique and combined effects in reversing NASH.', 'in vitro bacterial culture', 'qPCR,LC/MS analysis', 31295342);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5051, 'mouse', 'increase', 'low-throughput', 'B. infantis and MO decreased hepatic CYP7A1 and induced Sult2a1, Sult2a2, and Sult2a3 suggesting reduced BA synthesis and increased detoxification. Furthermore, B. infantis and MO increased ileal BA membrane receptor TGR5-regulated signaling. Together, via BA-regulated signaling, synbiotics B. infantis and MO have their unique and combined effects in reversing NASH.', 'in vitro bacterial culture', 'qPCR,LC/MS analysis', 11040423);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5052, 'mouse', 'decrease', 'low-throughput', 'A 15kDa protein with anti-inflammatory properties is produced by F. prausnitzii, a commensal bacterium involved in CD pathogenesis. This protein is able to inhibit the NF-κB pathway in intestinal epithelial cells and to prevent colitis in an animal model.', 'in vitro bacterial culture,in vitro bacterial culture,cell culture', 'fluorescence microscopy', 26045134);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5060, 'mouse', 'decrease', 'low-throughput', 'L. rhamnosus strain CNCM I-3690 reduces NF-kB activation by approximately 40% confirming the previous results, whereas the L. paracasei CNCM I-3689 reduces this activation only by 20%', 'null', 'RT-PCR', 25517879);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5061, 'mouse', 'decrease', 'low-throughput', 'L. rhamnosus strain CNCM I-3690 reduces NF-kB activation by approximately 40% confirming the previous results, whereas the L. paracasei CNCM I-3689 reduces this activation only by 20%.', 'null', 'RT-PCR', 25517879);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5062, 'mouse', 'decrease', 'low-throughput', 'IL-4 and IFN-g levels were significantly reduced in mice treated with either L. rhamnosus CNCM I-3690 or F. prausnitzii but not with the L. paracasei CNCM I-3689 strain (P < 0.05). A significant reduction in IL-6 levels was observed after treatment with all bacteri (P < 0.05).', 'null', 'RT-PCR', 25517879);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5063, 'mouse', 'decrease', 'low-throughput', 'IL-4 and IFN-g levels were significantly reduced in mice treated with either L. rhamnosus CNCM I-3690 or F. prausnitzii but not with the L. paracasei CNCM I-3689 strain (P < 0.05). A significant reduction in IL-6 levels was observed after treatment with all bacteri (P < 0.05).', 'null', 'RT-PCR', 25517879);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5064, 'mouse', 'decrease', 'low-throughput', 'IL-4 and IFN-g levels were significantly reduced in mice treated with either L. rhamnosus CNCM I-3690 or F. prausnitzii but not with the L. paracasei CNCM I-3689 strain (P < 0.05). A significant reduction in IL-6 levels was observed after treatment with all bacteri (P < 0.05).', 'null', 'RT-PCR', 25517879);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5065, 'mouse', 'decrease', 'low-throughput', 'IL-4 and IFN-g levels were significantly reduced in mice treated with either L. rhamnosus CNCM I-3690 or F. prausnitzii but not with the L. paracasei CNCM I-3689 strain (P < 0.05). A significant reduction in IL-6 levels was observed after treatment with all bacteri (P < 0.05).', 'null', 'RT-PCR', 25517879);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5066, 'mouse', 'decrease', 'low-throughput', 'IL-4 and IFN-g levels were significantly reduced in mice treated with either L. rhamnosus CNCM I-3690 or F. prausnitzii but not with the L. paracasei CNCM I-3689 strain (P < 0.05). A significant reduction in IL-6 levels was observed after treatment with all bacteri (P < 0.05).', 'null', 'RT-PCR', 25517879);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5067, 'mouse', 'decrease', 'low-throughput', 'IL-4 and IFN-g levels were significantly reduced in mice treated with either L. rhamnosus CNCM I-3690 or F. prausnitzii but not with the L. paracasei CNCM I-3689 strain (P < 0.05). A significant reduction in IL-6 levels was observed after treatment with all bacteri (P < 0.05).', 'null', 'RT-PCR', 25517879);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5068, 'mouse', 'decrease', 'low-throughput', 'The analysis  of cytokines�?and chemokines�?expression showed a decrease  in mRNA amounts of the proinflammatory molecules il6, il1b, il12p70, and tnf-a after 7 days, but no differences in expression after 21 days of administration.', 'in vivo bacterial administration', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5069, 'mouse', 'decrease', 'low-throughput', 'The analysis  of cytokines�?and chemokines�?expression showed a decrease  in mRNA amounts of the proinflammatory molecules il6, il1b, il12p70, and tnf-a after 7 days, but no differences in expression after 21 days of administration.', 'in vivo bacterial administration', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5070, 'mouse', 'decrease', 'low-throughput', 'The analysis  of cytokines�?and chemokines�?expression showed a decrease  in mRNA amounts of the proinflammatory molecules il6, il1b, il12p70, and tnf-a after 7 days, but no differences in expression after 21 days of administration.', 'in vivo bacterial administration', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5071, 'mouse', 'decrease', 'low-throughput', 'The analysis  of cytokines�?and chemokines�?expression showed a decrease  in mRNA amounts of the proinflammatory molecules il6, il1b, il12p70, and tnf-a after 7 days, but no differences in expression after 21 days of administration.', 'in vivo bacterial administration', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5072, 'mouse', 'decrease', 'low-throughput', 'EP1 consumption decreased il6, il1b, and cxcl-1 expression after 7 days, and this downregulation was persistent for the first two genes after 21 days of probiotic treatment.', 'in vivo bacterial administration', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5073, 'mouse', 'decrease', 'low-throughput', 'Treatment with EP1 during 21 days decreased the expression of the proinflammatory molecules il1b, il6, ifng, and cxcl-1 only in PP while il 17 was reduced in both MLN and PP.', 'in vivo bacterial administration', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5074, 'mouse', 'decrease', 'low-throughput', 'Treatment with EP1 during 21 days decreased the expression of the proinflammatory molecules il1b, il6, ifng, and cxcl-1 only in PP while il 17 was reduced in both MLN and PP.', 'in vivo bacterial administration', 'RT-PCR', 28239378);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5075, 'mouse', 'increase', 'low-throughput', 'PPAR-α and PPAR-γ mRNAs in the epididymal fat tissue were significantly downregulated in the HFD group compared with the control group and significantly upregulated after CB intervention.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5076, 'mouse', 'increase', 'low-throughput', 'PPAR-α and PPAR-γ mRNAs in the epididymal fat tissue were significantly downregulated in the HFD group compared with the control group and significantly upregulated after CB intervention.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5077, 'mouse', 'decrease', 'low-throughput', 'The mRNA levels of TNF-α, MCP-1, IL-1β, IL-2, IL-6 and IL-10 were upregulated in the HFD group compared with the control group, all of which were reduced by CB  intervention.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5078, 'mouse', 'decrease', 'low-throughput', 'The mRNA levels of TNF-α, MCP-1, IL-1β, IL-2, IL-6 and IL-10 were upregulated in the HFD group compared with the control group, all of which were reduced by CB  intervention.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5079, 'mouse', 'decrease', 'low-throughput', 'The mRNA levels of TNF-α, MCP-1, IL-1β, IL-2, IL-6 and IL-10 were upregulated in the HFD group compared with the control group, all of which were reduced by CB  intervention.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5080, 'mouse', 'decrease', 'low-throughput', 'The mRNA levels of TNF-α, MCP-1, IL-1β, IL-2, IL-6 and IL-10 were upregulated in the HFD group compared with the control group, all of which were reduced by CB  intervention.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5081, 'mouse', 'decrease', 'low-throughput', 'The mRNA levels of TNF-α, MCP-1, IL-1β, IL-2, IL-6 and IL-10 were upregulated in the HFD group compared with the control group, all of which were reduced by CB  intervention.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5082, 'mouse', 'decrease', 'low-throughput', 'The mRNA levels of TNF-α, MCP-1, IL-1β, IL-2, IL-6 and IL-10 were upregulated in the HFD group compared with the control group, all of which were reduced by CB  intervention.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5083, 'mouse', 'increase', 'low-throughput', 'CB intervention effectively reversed these imbalanced immune factors, which increased the levels of Foxp3, IL-4 and IL-22 and decreased IFN-γ and IL-17 in the liver and ileum.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5084, 'mouse', 'increase', 'low-throughput', 'CB intervention effectively reversed these imbalanced immune factors, which increased the levels of Foxp3, IL-4 and IL-22 and decreased IFN-γ and IL-17 in the liver and ileum.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5085, 'mouse', 'increase', 'low-throughput', 'CB intervention effectively reversed these imbalanced immune factors, which increased the levels of Foxp3, IL-4 and IL-22 and decreased IFN-γ and IL-17 in the liver and ileum.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5086, 'mouse', 'decrease', 'low-throughput', 'CB intervention effectively reversed these imbalanced immune factors, which increased the levels of Foxp3, IL-4 and IL-22 and decreased IFN-γ and IL-17 in the liver and ileum.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5087, 'mouse', 'decrease', 'low-throughput', 'CB intervention effectively reversed these imbalanced immune factors, which increased the levels of Foxp3, IL-4 and IL-22 and decreased IFN-γ and IL-17 in the liver and ileum.', 'in vivo infection experiment', 'RT-PCR', 28109017);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5090, 'mouse', 'increase', 'low-throughput', 'Therefore, in this study, we investigated the effects of KetoA [10-oxo-12(Z)-octadecenoic acid]—a linoleic acid metabolite produced by gut lactic acid bacteria—on whole-body energy metabolism and found that dietary intake of KetoA could enhance energy expenditure in mice, thereby protecting mice from diet-induced obesity.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5091, 'mouse', 'increase', 'low-throughput', 'Ucp1 expression levels were also observed to have increased after administration of KetoA.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5092, 'mouse', 'increase', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT
mice.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5093, 'mouse', 'increase', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT
mice.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5094, 'mouse', 'increase', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT
mice.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5095, 'mouse', 'increase', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT
mice.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5096, 'mouse', 'increase', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT
mice.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5097, 'mouse', 'increase', 'low-throughput', 'Therefore, in this study, we investigated the effects of KetoA [10-oxo-12(Z)-octadecenoic acid]—a linoleic acid metabolite produced by gut lactic acid bacteria—on whole-body energy metabolism and found that dietary intake of KetoA could enhance energy expenditure in mice, thereby protecting mice from diet-induced obesity.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5098, 'mouse', 'increase', 'low-throughput', 'Ucp1 expression levels were also observed to have increased after administration of KetoA.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5099, 'mouse', 'increase', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5100, 'mouse', 'increase', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5101, 'mouse', 'increase', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5102, 'mouse', 'increase', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5103, 'mouse', 'increase', 'low-throughput', 'We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.', 'cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment', 'RT-PCR,Western blot assay', 28754711);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5104, 'mouse', 'increase', 'high-throughput', 'It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5105, 'mouse', 'decrease', 'high-throughput', 'Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5106, 'mouse', 'increase', 'high-throughput', 'It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5107, 'mouse', 'increase', 'high-throughput', 'It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5108, 'mouse', 'decrease', 'high-throughput', 'Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5109, 'mouse', 'increase', 'high-throughput', 'Acetate had no significant effect on the expression of Fiaf, Gpr43, Hdac5, or Ppar, while the expression of Hdac3 was significantly increased by acetate.', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5110, 'mouse', 'increase', 'high-throughput', 'It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5111, 'mouse', 'decrease', 'high-throughput', 'Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5112, 'mouse', 'increase', 'high-throughput', 'It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5113, 'mouse', 'increase', 'high-throughput', 'It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5114, 'mouse', 'decrease', 'high-throughput', 'Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5115, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5116, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5117, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5118, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5119, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5120, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5121, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5122, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5123, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5124, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5125, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5126, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5127, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5128, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5129, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5130, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5131, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5132, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5133, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5134, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5135, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5136, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5137, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5138, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5139, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5140, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5141, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5142, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5143, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5144, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5145, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5146, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5147, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5148, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5149, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5150, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5151, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5152, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5153, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5154, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5155, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5156, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5157, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5158, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5159, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5160, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5161, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5162, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5163, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5164, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5165, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5166, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5167, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5168, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5169, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5170, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5171, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5172, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5173, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5174, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5175, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5176, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5177, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5178, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5179, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5180, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5181, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5182, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5183, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5184, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5185, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5186, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5187, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5188, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5189, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5190, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5191, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5192, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5193, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5194, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5195, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5196, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5197, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5198, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5199, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5200, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5201, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5202, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5203, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5204, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5205, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5206, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5207, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5208, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5209, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5210, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5211, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5212, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5213, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5214, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5215, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5216, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5217, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5218, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5219, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5220, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5221, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5222, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5223, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5224, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5225, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5226, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5227, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5228, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5229, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5230, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5231, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5232, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5233, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5234, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5235, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5236, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5237, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5238, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5239, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5240, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5241, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5242, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5243, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5244, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5245, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5246, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5247, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5248, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5249, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5250, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5251, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5252, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5253, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5254, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5255, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5256, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5257, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5258, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5259, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5260, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5261, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5262, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5263, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5264, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5265, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5266, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5267, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5268, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5269, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5270, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5271, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5272, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5273, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5274, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5275, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5276, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5277, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5278, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5279, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5280, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5281, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5282, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5283, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5284, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5285, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5286, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5287, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5288, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5289, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5290, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5291, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5292, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5293, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5294, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5295, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5296, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5297, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5298, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5299, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5300, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5301, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5302, 'mouse', 'decrease', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5303, 'mouse', 'increase', 'high-throughput', 'To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).', 'cell culture,in vitro bacterial culture,control experiment', 'null', 25118238);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5334, 'mouse', 'increase', 'low-throughput', 'Forty-eight hours after TNBS instillation,colonic proinflammatory TNF and IL-12 cytokines and anti-inflammatory IL-10 were quantified by ELISA (SI Materials and Methods and Fig. 5). Compared with the no-colitis control group,TNF-α and IL-12 levels were increased, whereas IL-10 levels were barely modified. In the two TNBS groups treated with either F.prausnitzii or its supernatant, the secretion of TNF and IL-12 was significantly lower than in the colitis control group. Interestingly, IL-10 secretion was induced in the colon of mice treated with F.prausnitzii or its supernatant.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 18936492);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5335, 'mouse', 'decrease', 'low-throughput', 'Forty-eight hours after TNBS instillation,colonic proinflammatory TNF and IL-12 cytokines and anti-inflammatory IL-10 were quantified by ELISA (SI Materials and Methods and Fig. 5). Compared with the no-colitis control group,TNF-α and IL-12 levels were increased, whereas IL-10 levels were barely modified. In the two TNBS groups treated with either F.prausnitzii or its supernatant, the secretion of TNF and IL-12 was significantly lower than in the colitis control group. Interestingly, IL-10 secretion was induced in the colon of mice treated with F.prausnitzii or its supernatant.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 18936492);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5336, 'mouse', 'decrease', 'low-throughput', 'Forty-eight hours after TNBS instillation,colonic proinflammatory TNF and IL-12 cytokines and anti-inflammatory IL-10 were quantified by ELISA (SI Materials and Methods and Fig. 5). Compared with the no-colitis control group,TNF-α and IL-12 levels were increased, whereas IL-10 levels were barely modified. In the two TNBS groups treated with either F.prausnitzii or its supernatant, the secretion of TNF and IL-12 was significantly lower than in the colitis control group. Interestingly, IL-10 secretion was induced in the colon of mice treated with F.prausnitzii or its supernatant.', 'cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 18936492);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5337, 'mouse', 'decrease', 'low-throughput', 'Altered substrate oxidation and fecal fat excretion in LHFD-fed GF mice is linked to decreased hepatic Cyp7a1 and Nr1h4 expression.', 'control experiment', 'RT-PCR�?6S rRNA sequence', 27900259);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5338, 'mouse', 'decrease', 'low-throughput', 'Altered substrate oxidation and fecal fat excretion in LHFD-fed GF mice is linked to decreased hepatic Cyp7a1 and Nr1h4 expression.', 'control experiment', 'RT-PCR�?6S rRNA sequence', 27900259);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5339, 'mouse', 'decrease', 'low-throughput', '10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.', 'cell culture、control experiment', 'Western blot assay、RT-PCR', 25505251);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5340, 'mouse', 'decrease', 'low-throughput', '10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.', 'cell culture、control experiment', 'Western blot assay、RT-PCR', 25505251);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5341, 'mouse', 'increase', 'low-throughput', '10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.', 'cell culture、control experiment', 'Western blot assay、RT-PCR', 25505251);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5342, 'mouse', 'decrease', 'low-throughput', 'IFN-TNF-induced an increase in FD-4 permeability;however, the pretreatment of HYA and linoleic acid signififcantly suppressed this increase (Fig. 2C). In contrast, the pretreatment of HYB did not alter IFN-TNF-induced changes in TER and FD-4 permeability. As shown in Fig. 2D,IFN-TNF-induced a marked increase in the secretion of IL-8, which is a major pro-inflammatory cytokine. However,the pretreatment of HYA significantly suppressed IL-8 secretion.', 'cell culture、control experiment', 'Western blot assay、RT-PCR', 25505251);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5343, 'mouse', 'increase', 'low-throughput', 'Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.', 'cell culture、control experiment', 'Western blot assay、RT-PCR', 25505251);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5344, 'mouse', 'increase', 'low-throughput', 'Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.', 'cell culture、control experiment', 'Western blot assay、RT-PCR', 25505251);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5345, 'mouse', 'increase', 'low-throughput', 'Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.', 'cell culture、control experiment', 'Western blot assay、RT-PCR', 25505251);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5346, 'mouse', 'decrease', 'low-throughput', 'Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.', 'cell culture、control experiment', 'Western blot assay、RT-PCR', 25505251);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5347, 'mouse', 'decrease', 'low-throughput', 'The pretreatment of HYA suppressed IFN-induced up-regulation of TNFR2 (but not TNFR1), and it subsequently decreased the ratio of phospho-INF-B p65 expression induced by cytokines (Fig. 3, B and C). In contrast, pretreatment of HYB did not show the same inhibitory effects as HYA.', 'cell culture、control experiment', 'Western blot assay、RT-PCR', 25505251);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5348, 'mouse', 'increase', 'low-throughput', 'It was noteworthy that the pretreatment of HYA augmented the mRNA expression of GPR40 and GPR43 but not GPR120, PPAR, and GPR41 in Caco-2 cells (Fig. 4A). However, the pretreatment of HYB did not up-regulate GPR40 and GPR43 expression (Fig. 4A).', 'cell culture、control experiment', 'Western blot assay、RT-PCR', 25505251);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5363, 'mouse', 'increase', 'low-throughput', 'Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.', 'in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 25065623);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5364, 'mouse', 'decrease', 'low-throughput', 'Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.', 'in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 25065623);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5365, 'mouse', 'increase', 'low-throughput', 'Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.', 'in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 25065623);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5366, 'mouse', 'increase', 'low-throughput', 'For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of
L-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to
C. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa
exposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).', 'in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 25065623);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5367, 'mouse', 'increase', 'low-throughput', 'For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of
L-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to
C. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa
exposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).', 'in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 25065623);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5368, 'mouse', 'increase', 'low-throughput', 'For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of
L-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to
C. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa
exposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).', 'in vitro bacterial culture、in vivo infection experiment、control experiment', 'qPCR', 25065623);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5369, 'mouse', 'decrease', 'low-throughput', 'We found TMAVA to bind and inhibit BBOX, reducing synthesis of carnitine.We found the intestinal bacteria Enterococcus faecalis and Pseudomonas aeruginosa to metabolize trimethyllysine to TMAVA.', 'in vivo experiment', 'qPCR', 32105727);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5370, 'mouse', 'decrease', 'low-throughput', 'We found TMAVA to bind and inhibit BBOX, reducing synthesis of carnitine.We found the intestinal bacteria Enterococcus faecalis and Pseudomonas aeruginosa to metabolize trimethyllysine to TMAVA.', 'in vivo experiment', 'qPCR', 32105727);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5372, 'mouse', 'increase', 'low-throughput', 'In addition, Parabacteroides_sp_HGS0025 positively correlated with butyrate, IgM, IL-10, and TNF-α products in intestine and mice blood, respectively.', 'in vivo experiment', 'ELISA', 32014483);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5373, 'mouse', 'increase', 'low-throughput', 'In addition, Parabacteroides_sp_HGS0025 positively correlated with butyrate, IgM, IL-10, and TNF-α products in intestine and mice blood, respectively.', 'in vivo experiment', 'ELISA', 32014483);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5374, 'mouse', 'increase', 'low-throughput', 'In addition, Parabacteroides_sp_HGS0025 positively correlated with butyrate, IgM, IL-10, and TNF-α products in intestine and mice blood, respectively.', 'in vivo experiment', 'ELISA', 32014483);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5392, 'mouse', 'increase', 'low-throughput', 'Treatments of BMDCs with the fatty acids significantly increased mRNA expression of Arg1 in a dose-dependent manner in comparison with that of IL-4 treatment alone.', 'in vivo experiment', 'RT-qPCR', 28904023);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5393, 'mouse', 'increase', 'low-throughput', 'Treatments of BMDCs with the fatty acids significantly increased mRNA expression of Arg1 in a dose-dependent manner in comparison with that of IL-4 treatment alone.', 'in vivo experiment', 'RT-qPCR', 28904023);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5394, 'mouse', 'increase', 'low-throughput', 'The ratio of p-Akt/Akt in the C. butyricum-treated mice was significantly increased compared with the D-I/R group, suggesting activating Akt by C. butyricum treatment.', 'in vivo experiment', 'Western blot assay', 27037183);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5395, 'mouse', 'decrease', 'low-throughput', 'C. butyricum treatment resulted in decreases of caspase-3 protein.', 'in vivo experiment', 'Western blot assay', 27037183);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5396, 'mouse', 'increase', 'low-throughput', 'BDNF in the CUMS group was markedly decreased compared to that in the Con group, and Cb treatment significantly reversed this decrease.', 'in vivo experiment', 'ELISA', 30040410);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5397, 'mouse', 'increase', 'low-throughput', 'Cb treatment resulted in a significant increase in GLP-1.', 'in vivo experiment', 'ELISA', 30040410);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5398, 'mouse', 'decrease', 'low-throughput', 'Microbial suppression of Gpr41 expression.', 'in vivo experiment', 'RT-qPCR', 18931303);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5399, 'mouse', 'increase', 'low-throughput', 'AM attenuates ALS in mice by increasing NAM levels', 'in vivo experiment', 'RNA-seq', 31330533);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5400, 'mouse', 'increase', 'low-throughput', 'AM attenuates ALS in mice by increasing NAM levels', 'in vivo experiment', 'RNA-seq', 31330533);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5401, 'mouse', 'increase', 'low-throughput', 'Additionally, prophylactic treatment with C. butyricum increased the activity of both superoxide dismutase (SOD) and catalase (CAT).', 'in vivo experiment', 'Western blot assay', 28933492);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5402, 'mouse', 'increase', 'low-throughput', 'Additionally, prophylactic treatment with C. butyricum increased the activity of both superoxide dismutase (SOD) and catalase (CAT).', 'in vivo experiment', 'Western blot assay', 28933492);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5403, 'mouse', 'increase', 'low-throughput', 'Furthermore, C. butyricum up-regulated the nuclear factor (erythroid-derived 2)-like 2 (NRF2) content.', 'in vivo experiment', 'Western blot assay', 28933492);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5404, 'mouse', 'decrease', 'low-throughput', 'The levels of IL-1β and TNF-α in the brains of APP/PS1 mice were remarkably increased, while the levels in the CB-treated mice were significantly reduced.', 'in vivo experiment,cell culture', 'Western blot assay,immunofluorescence', 31835282);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5405, 'mouse', 'decrease', 'low-throughput', 'The levels of IL-1β and TNF-α in the brains of APP/PS1 mice were remarkably increased, while the levels in the CB-treated mice were significantly reduced.', 'in vivo experiment,cell culture', 'Western blot assay,immunofluorescence', 31835282);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5406, 'mouse', 'decrease', 'low-throughput', 'Butyrate treatment significantly decreased expression of the CD11b.', 'in vivo experiment,cell culture', 'Western blot assay,immunofluorescence', 31835282);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5407, 'mouse', 'increase', 'low-throughput', 'Here, we show that the biological effects of tryptamine are mediated through the 5-HT4 receptor (5-HT4R), a G-protein-coupled receptor (GPCR) uniquely expressed in the colonic epithelium.', 'in vivo experiment', 'LC/MS analysis,RT-PCR', 29902441);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5408, 'mouse', 'increase', 'low-throughput', 'Here, we show that the biological effects of tryptamine are mediated through the 5-HT4 receptor (5-HT4R), a G-protein-coupled receptor (GPCR) uniquely expressed in the colonic epithelium.', 'in vivo experiment', 'LC/MS analysis,RT-PCR', 29902441);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5413, 'mouse', 'decrease', 'low-throughput', 'A significant decrease in IL17, IFNγ and IL5 was found in supernatant of lymphocytes from LL-pILMAM treated mice compared to LL-pILEMPTY and control mice.', 'in vivo experiment', 'ELISA', 28203226);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5414, 'mouse', 'decrease', 'low-throughput', 'A significant decrease in IL17, IFNγ and IL5 was found in supernatant of lymphocytes from LL-pILMAM treated mice compared to LL-pILEMPTY and control mice.', 'in vivo experiment', 'ELISA', 28203226);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5415, 'mouse', 'decrease', 'low-throughput', 'A significant decrease in IL17, IFNγ and IL5 was found in supernatant of lymphocytes from LL-pILMAM treated mice compared to LL-pILEMPTY and control mice.', 'in vivo experiment', 'ELISA', 28203226);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5419, 'mouse', 'increase', 'low-throughput', 'The expression of colonic IDO was increased by Bifidobacterium.', 'in vivo experiment', 'Western blot assay', 29226383);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5420, 'mouse', 'decrease', 'low-throughput', 'Furthermore, CB0313.1-treated mice exhibited signifcantly lower serum fructosamine and glucose-6-phosphatase (G6Pase) levels.', 'in vivo experiment', 'ELISA,Western blot analysis,RT-PCR,GC-MS', 28765642);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5421, 'mouse', 'increase', 'low-throughput', 'In CB0313.1-treated mice, we observed an increase of mitochondrial function marker PPARγ (1.50-fold, p = 0.0339), carnitine palmitoyltransferase-1α (CPT1α) (1.85-fold, p = 0.0057), nuclear factor-like 2 (NRF2) (1.39-fold, p = 0.0012) in epididymal adipose tissue.', 'in vivo experiment', 'ELISA,Western blot analysis,RT-PCR,GC-MS', 28765642);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5422, 'mouse', 'increase', 'low-throughput', 'In CB0313.1-treated mice, we observed an increase of mitochondrial function marker PPARγ (1.50-fold, p = 0.0339), carnitine palmitoyltransferase-1α (CPT1α) (1.85-fold, p = 0.0057), nuclear factor-like 2 (NRF2) (1.39-fold, p = 0.0012) in epididymal adipose tissue.', 'in vivo experiment', 'ELISA,Western blot analysis,RT-PCR,GC-MS', 28765642);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5423, 'mouse', 'increase', 'low-throughput', 'In CB0313.1-treated mice, we observed an increase of mitochondrial function marker PPARγ (1.50-fold, p = 0.0339), carnitine palmitoyltransferase-1α (CPT1α) (1.85-fold, p = 0.0057), nuclear factor-like 2 (NRF2) (1.39-fold, p = 0.0012) in epididymal adipose tissue.', 'in vivo experiment', 'ELISA,Western blot analysis,RT-PCR,GC-MS', 28765642);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5424, 'mouse', 'increase', 'low-throughput', 'Notably, glucose transporter 4 (Glut4) mRNA levels in CB0313.1 and NaB-fed mice were increased', 'in vivo experiment', 'ELISA,Western blot analysis,RT-PCR,GC-MS', 28765642);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5425, 'mouse', 'increase', 'low-throughput', 'Ucp-1 is a key gene capable of retarding oxidative phosphorylation and thereby hindering the production of ATP17. Its mRNA levels in CB0313.1 and NaB-fed mice increased.', 'in vivo experiment', 'ELISA,Western blot analysis,RT-PCR,GC-MS', 28765642);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5427, 'mouse', 'increase', 'low-throughput', 'Similarly, F. prausnitziiexposed DC systematically secreted IL-27 and IL-6.', 'cell culture', 'ELISA', 30787928);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5428, 'mouse', 'increase', 'low-throughput', 'Similarly, F. prausnitziiexposed DC systematically secreted IL-27 and IL-6.', 'cell culture', 'ELISA', 30787928);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5429, 'mouse', 'increase', 'low-throughput', 'F. prausnitzii-exposed DC upregulated PDL-1,CD39,and IDO1.', 'cell culture', 'ELISA', 30787928);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5430, 'mouse', 'increase', 'low-throughput', 'F. prausnitzii-exposed DC upregulated PDL-1,CD39,and IDO1.', 'cell culture', 'ELISA', 30787928);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5431, 'mouse', 'increase', 'low-throughput', 'F. prausnitzii-exposed DC upregulated PDL-1,CD39,and IDO1.', 'cell culture', 'ELISA', 30787928);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5432, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5433, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5434, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5435, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5436, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5437, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5438, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5439, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5440, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5441, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5442, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5443, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5444, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5445, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5446, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5447, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5448, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5449, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5450, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5451, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5452, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5453, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5454, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5455, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5456, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5457, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5458, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5459, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5460, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5461, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5462, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5463, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5464, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5465, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5466, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5467, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5468, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5469, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5470, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5471, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5472, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5473, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5474, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5475, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5476, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5477, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5478, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5479, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5480, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5481, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5482, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5483, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5484, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5485, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5486, 'mouse', 'increase', 'low-throughput', 'By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5487, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5488, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5489, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5490, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5491, 'mouse', 'decrease', 'low-throughput', 'However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.', 'in vivo experiment', 'HPLC-ESI-MS analysis,RT-qPCR,Western blot assay', 31863867);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5494, 'mouse', 'decrease', 'low-throughput', 'In contrast, in livers of fructose + Lcs fed mice expression of TLR-4 and MyD88 was almost at the level of controls.', 'in vivo experiment', 'RNA-seq,Western blot assay', 22749137);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5495, 'mouse', 'decrease', 'low-throughput', 'In contrast, in livers of fructose + Lcs fed mice expression of TLR-4 and MyD88 was almost at the level of controls.', 'in vivo experiment', 'RNA-seq,Western blot assay', 22749137);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5496, 'mouse', 'increase', 'low-throughput', 'Lactobacillus casei Shirota treatment up-regulates peroxisome proliferator-activated receptor γ activity in the liver.', 'in vivo experiment', 'RNA-seq,Western blot assay', 22749137);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5497, 'mouse', 'decrease', 'low-throughput', 'Membrane expression of MCT1 is decreased by EPEC infection.', 'in vivo experiment', 'Western blot assay', 16150873);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5499, 'mouse', 'increase', 'low-throughput', 'Colonization with A. onderdonkii, B. fragilis, or B. thetaiotamicron led to variable IL-6 production by IELs that was substantially higher than that produced by IELs isolated from E. coli or PBS-gavaged germ-free mice.', 'in vivo experiment', 'qPCR', 28812548);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5500, 'mouse', 'increase', 'low-throughput', 'Colonization with A. onderdonkii, B. fragilis, or B. thetaiotamicron led to variable IL-6 production by IELs that was substantially higher than that produced by IELs isolated from E. coli or PBS-gavaged germ-free mice.', 'in vivo experiment', 'qPCR', 28812548);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5501, 'mouse', 'increase', 'low-throughput', 'Colonization with A. onderdonkii, B. fragilis, or B. thetaiotamicron led to variable IL-6 production by IELs that was substantially higher than that produced by IELs isolated from E. coli or PBS-gavaged germ-free mice.', 'in vivo experiment', 'qPCR', 28812548);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5502, 'mouse', 'increase', 'low-throughput', 'Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.', 'in vivo experiment', 'Western blot assay', 30141701);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5503, 'mouse', 'increase', 'low-throughput', 'Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.', 'in vivo experiment', 'Western blot assay', 30141701);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5504, 'mouse', 'increase', 'low-throughput', 'Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.', 'in vivo experiment', 'Western blot assay', 30141701);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5505, 'mouse', 'increase', 'low-throughput', 'Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.', 'in vivo experiment', 'Western blot assay', 30141701);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5506, 'mouse', 'decrease', 'low-throughput', 'Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.', 'in vivo experiment', 'Western blot assay', 30141701);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5507, 'mouse', 'decrease', 'low-throughput', 'Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.', 'in vivo experiment', 'Western blot assay', 30141701);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5508, 'mouse', 'decrease', 'low-throughput', 'Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.', 'in vivo experiment', 'Western blot assay', 30141701);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5509, 'mouse', 'decrease', 'low-throughput', 'Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.', 'in vivo experiment', 'Western blot assay', 30141701);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5510, 'mouse', 'decrease', 'low-throughput', 'Table 3 revealed that γ-GT, AST and ALT were signifcantly elevated in the M group, while these alterations were obviously reversed by treatment with L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5511, 'mouse', 'decrease', 'low-throughput', 'Table 3 revealed that γ-GT, AST and ALT were signifcantly elevated in the M group, while these alterations were obviously reversed by treatment with L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5512, 'mouse', 'decrease', 'low-throughput', 'Table 3 revealed that γ-GT, AST and ALT were signifcantly elevated in the M group, while these alterations were obviously reversed by treatment with L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5513, 'mouse', 'decrease', 'low-throughput', 'On the contrary, the increasing trend of these pro-infammatory factors in HFFD-fed mice were apparently down-regulated by the administration with L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5514, 'mouse', 'decrease', 'low-throughput', 'On the contrary, the increasing trend of these pro-infammatory factors in HFFD-fed mice were apparently down-regulated by the administration with L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5515, 'mouse', 'decrease', 'low-throughput', 'On the contrary, the increasing trend of these pro-infammatory factors in HFFD-fed mice were apparently down-regulated by the administration with L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5516, 'mouse', 'decrease', 'low-throughput', 'Results indicated that the mRNA expressions of hepatic lipogenic genes (Pparγ and Srebp-1c) were higher in the HFFD-fed mice than that in the C group mice, while the expression of these genes were signifcantly reduced via supplementation of L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5517, 'mouse', 'decrease', 'low-throughput', 'Results indicated that the mRNA expressions of hepatic lipogenic genes (Pparγ and Srebp-1c) were higher in the HFFD-fed mice than that in the C group mice, while the expression of these genes were signifcantly reduced via supplementation of L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5518, 'mouse', 'decrease', 'low-throughput', 'Moreover, the expressions of fatty acid metabolic genes, including CD36 and Fabp2, were also dramatically increased in the M group compared to the C group and showed a down-regulated expression in L. rhamnosus LS-8 and L. crustorum MN047 treatment group.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5519, 'mouse', 'decrease', 'low-throughput', 'Moreover, the expressions of fatty acid metabolic genes, including CD36 and Fabp2, were also dramatically increased in the M group compared to the C group and showed a down-regulated expression in L. rhamnosus LS-8 and L. crustorum MN047 treatment group.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5520, 'mouse', 'decrease', 'low-throughput', 'Table 3 revealed that γ-GT, AST and ALT were signifcantly elevated in the M group, while these alterations were obviously reversed by treatment with L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5521, 'mouse', 'decrease', 'low-throughput', 'Table 3 revealed that γ-GT, AST and ALT were signifcantly elevated in the M group, while these alterations were obviously reversed by treatment with L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5522, 'mouse', 'decrease', 'low-throughput', 'Table 3 revealed that γ-GT, AST and ALT were signifcantly elevated in the M group, while these alterations were obviously reversed by treatment with L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5523, 'mouse', 'decrease', 'low-throughput', 'On the contrary, the increasing trend of these pro-infammatory factors in HFFD-fed mice were apparently down-regulated by the administration with L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5524, 'mouse', 'decrease', 'low-throughput', 'On the contrary, the increasing trend of these pro-infammatory factors in HFFD-fed mice were apparently down-regulated by the administration with L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5525, 'mouse', 'decrease', 'low-throughput', 'On the contrary, the increasing trend of these pro-infammatory factors in HFFD-fed mice were apparently down-regulated by the administration with L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5526, 'mouse', 'decrease', 'low-throughput', 'Results indicated that the mRNA expressions of hepatic lipogenic genes (Pparγ and Srebp-1c) were higher in the HFFD-fed mice than that in the C group mice, while the expression of these genes were signifcantly reduced via supplementation of L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5527, 'mouse', 'decrease', 'low-throughput', 'Results indicated that the mRNA expressions of hepatic lipogenic genes (Pparγ and Srebp-1c) were higher in the HFFD-fed mice than that in the C group mice, while the expression of these genes were signifcantly reduced via supplementation of L. rhamnosus LS-8 and L. crustorum MN047.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5528, 'mouse', 'decrease', 'low-throughput', 'Moreover, the expressions of fatty acid metabolic genes, including CD36 and Fabp2, were also dramatically increased in the M group compared to the C group and showed a down-regulated expression in L. rhamnosus LS-8 and L. crustorum MN047 treatment group.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5529, 'mouse', 'decrease', 'low-throughput', 'Moreover, the expressions of fatty acid metabolic genes, including CD36 and Fabp2, were also dramatically increased in the M group compared to the C group and showed a down-regulated expression in L. rhamnosus LS-8 and L. crustorum MN047 treatment group.', 'in vivo experiment', 'qPCR', 31659451);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5530, 'mouse', 'increase', 'low-throughput', 'The culture supernatant of F. prausnitzii increased plasma anti-Th17 cytokines (IL-10 and IL-12)and suppressed IL-17 levels in both plasma and colonic mucosa, with ameliorated colonic colitis lesions.', 'cell culture,in vitro bacterial culture', 'ELISA', 25275569);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5531, 'mouse', 'increase', 'low-throughput', 'The culture supernatant of F. prausnitzii increased plasma anti-Th17 cytokines (IL-10 and IL-12)and suppressed IL-17 levels in both plasma and colonic mucosa, with ameliorated colonic colitis lesions.', 'cell culture,in vitro bacterial culture', 'ELISA', 25275569);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5532, 'mouse', 'increase', 'low-throughput', 'The culture supernatant of F. prausnitzii increased plasma anti-Th17 cytokines (IL-10 and IL-12)and suppressed IL-17 levels in both plasma and colonic mucosa, with ameliorated colonic colitis lesions.', 'cell culture,in vitro bacterial culture', 'ELISA', 25275569);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5533, 'mouse', 'decrease', 'low-throughput', 'The culture supernatant of F. prausnitzii increased plasma anti-Th17 cytokines (IL-10 and IL-12)and suppressed IL-17 levels in both plasma and colonic mucosa, with ameliorated colonic colitis lesions.', 'cell culture,in vitro bacterial culture', 'ELISA', 25275569);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5534, 'mouse', 'decrease', 'low-throughput', 'A. muciniphila treatment significantly reduced the Con A-induced elevation of serum ALT and AST activities.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5535, 'mouse', 'decrease', 'low-throughput', 'A. muciniphila treatment significantly reduced the Con A-induced elevation of serum ALT and AST activities.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5536, 'mouse', 'decrease', 'low-throughput', 'Serum levels of pro-inflammatory cytokines and chemokines (IL-2, IFN-γ, IL-12p40, MCP-1, MIP-1a, MIP-1b) were substantially attenuated.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5537, 'mouse', 'decrease', 'low-throughput', 'Serum levels of pro-inflammatory cytokines and chemokines (IL-2, IFN-γ, IL-12p40, MCP-1, MIP-1a, MIP-1b) were substantially attenuated.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5538, 'mouse', 'decrease', 'low-throughput', 'Serum levels of pro-inflammatory cytokines and chemokines (IL-2, IFN-γ, IL-12p40, MCP-1, MIP-1a, MIP-1b) were substantially attenuated.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5539, 'mouse', 'decrease', 'low-throughput', 'Serum levels of pro-inflammatory cytokines and chemokines (IL-2, IFN-γ, IL-12p40, MCP-1, MIP-1a, MIP-1b) were substantially attenuated.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5540, 'mouse', 'decrease', 'low-throughput', 'Serum levels of pro-inflammatory cytokines and chemokines (IL-2, IFN-γ, IL-12p40, MCP-1, MIP-1a, MIP-1b) were substantially attenuated.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5541, 'mouse', 'decrease', 'low-throughput', 'A. muciniphila significantly reduced the increase in IFN-g, IL-2, IL-1b, and IL-12p40.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5542, 'mouse', 'decrease', 'low-throughput', 'A. muciniphila significantly reduced the increase in IFN-g, IL-2, IL-1b, and IL-12p40.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5543, 'mouse', 'increase', 'low-throughput', 'A. muciniphila significantly decreased hepatocellular apoptosis; Bcl-2 expression increased, but Fas and DR5 decreased.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5544, 'mouse', 'decrease', 'low-throughput', 'A. muciniphila significantly decreased hepatocellular apoptosis; Bcl-2 expression increased, but Fas and DR5 decreased.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5545, 'mouse', 'decrease', 'low-throughput', 'A. muciniphila significantly decreased hepatocellular apoptosis; Bcl-2 expression increased, but Fas and DR5 decreased.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5546, 'mouse', 'increase', 'low-throughput', 'Further investigation showed that A. muciniphila enhanced expression of Occludin and Tjp-1 and inhibited CB1 receptor.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5547, 'mouse', 'increase', 'low-throughput', 'Further investigation showed that A. muciniphila enhanced expression of Occludin and Tjp-1 and inhibited CB1 receptor.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5548, 'mouse', 'decrease', 'low-throughput', 'Further investigation showed that A. muciniphila enhanced expression of Occludin and Tjp-1 and inhibited CB1 receptor.', 'in vivo experiment', 'qPCR', 29033903);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5549, 'mouse', 'increase', 'low-throughput', 'Besides this, not only Th1 cytokines but also regulatory T (Treg) cytokines including IL-12, IFN-g and TGF-b were significantly increased in the LGG pre- and posttreatment groups.', 'in vivo experiment', 'ELISA', 25440975);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5550, 'mouse', 'increase', 'low-throughput', 'Besides this, not only Th1 cytokines but also regulatory T (Treg) cytokines including IL-12, IFN-g and TGF-b were significantly increased in the LGG pre- and posttreatment groups.', 'in vivo experiment', 'ELISA', 25440975);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5551, 'mouse', 'increase', 'low-throughput', 'Besides this, not only Th1 cytokines but also regulatory T (Treg) cytokines including IL-12, IFN-g and TGF-b were significantly increased in the LGG pre- and posttreatment groups.', 'in vivo experiment', 'ELISA', 25440975);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5552, 'mouse', 'increase', 'low-throughput', 'Besides this, not only Th1 cytokines but also regulatory T (Treg) cytokines including IL-12, IFN-g and TGF-b were significantly increased in the LGG pre- and posttreatment groups.', 'in vivo experiment', 'ELISA', 25440975);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5553, 'mouse', 'increase', 'low-throughput', 'Besides this, not only Th1 cytokines but also regulatory T (Treg) cytokines including IL-12, IFN-g and TGF-b were significantly increased in the LGG pre- and posttreatment groups.', 'in vivo experiment', 'ELISA', 25440975);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5554, 'mouse', 'decrease', 'low-throughput', 'After the LGG treatment, in both of the groups treated with LGG in the allergic asthma models, there was significantly less MMP9 expression.', 'in vivo experiment', 'ELISA', 25440975);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5555, 'mouse', 'increase', 'low-throughput', 'LA counteracts CR-induced decrease in NHE3 and DRA expression.', 'in vivo experiment', 'RT-PCR', 27634011);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5556, 'mouse', 'increase', 'low-throughput', 'LA counteracts CR-induced decrease in NHE3 and DRA expression.', 'in vivo experiment', 'RT-PCR', 27634011);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5557, 'mouse', 'increase', 'low-throughput', 'LA alone had no effect but significantly attenuated CR-induced decrease in CAI and CAIV mRNA.', 'in vivo experiment', 'RT-PCR', 27634011);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5558, 'mouse', 'increase', 'low-throughput', 'LA alone had no effect but significantly attenuated CR-induced decrease in CAI and CAIV mRNA.', 'in vivo experiment', 'RT-PCR', 27634011);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5559, 'mouse', 'increase', 'low-throughput', 'LA alone had no effect on NHERF 1 and 2 but significantly increased NHERF3 mRNA level.', 'in vivo experiment', 'RT-PCR', 27634011);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5560, 'mouse', 'decrease', 'low-throughput', 'LA alone showed significantly blocked the CR-induced increase in their mRNA levels in the CR+LA group.', 'in vivo experiment', 'RT-PCR', 27634011);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5561, 'mouse', 'decrease', 'low-throughput', 'LA alone showed significantly blocked the CR-induced increase in their mRNA levels in the CR+LA group.', 'in vivo experiment', 'RT-PCR', 27634011);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5562, 'mouse', 'decrease', 'low-throughput', 'LA alone showed significantly blocked the CR-induced increase in their mRNA levels in the CR+LA group.', 'in vivo experiment', 'RT-PCR', 27634011);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5563, 'mouse', 'increase', 'low-throughput', 'Similar to the in vitro studies, in vivo studies using mice gavaged with LA also showed significantly increased DRA mRNA ( approximately 4-fold) and protein expression in the colonic regions as assessed by Western blot analysis and immunofluorescence.', 'in vivo experiment', 'Western blot assay', 20044511);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5564, 'mouse', 'decrease', 'low-throughput', 'Intrarectal administration of C. tyrobutyricum prevented appearance of clinical symptoms of DSS-colitis, restored normal MUC-2 production, unaltered expression of TJ protein ZO-1 and decreased levels of TNF-α and IL-18 in the descending colon of SCID and BALB/c mice, respectively.', 'in vivo experiment', 'ELISA,RT-PCR', 22236013);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5565, 'mouse', 'decrease', 'low-throughput', 'Intrarectal administration of C. tyrobutyricum prevented appearance of clinical symptoms of DSS-colitis, restored normal MUC-2 production, unaltered expression of TJ protein ZO-1 and decreased levels of TNF-α and IL-18 in the descending colon of SCID and BALB/c mice, respectively.', 'in vivo experiment', 'ELISA,RT-PCR', 22236013);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5566, 'mouse', 'increase', 'low-throughput', 'Intrarectal administration of C. tyrobutyricum prevented appearance of clinical symptoms of DSS-colitis, restored normal MUC-2 production, unaltered expression of TJ protein ZO-1 and decreased levels of TNF-α and IL-18 in the descending colon of SCID and BALB/c mice, respectively.', 'in vivo experiment', 'ELISA,RT-PCR', 22236013);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5567, 'mouse', 'increase', 'low-throughput', 'Intrarectal administration of C. tyrobutyricum prevented appearance of clinical symptoms of DSS-colitis, restored normal MUC-2 production, unaltered expression of TJ protein ZO-1 and decreased levels of TNF-α and IL-18 in the descending colon of SCID and BALB/c mice, respectively.', 'in vivo experiment', 'ELISA,RT-PCR', 22236013);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5573, 'mouse', 'increase', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 'control experiment,Supplementation of metabolite', '16S rRNA sequence,Western blot assay,RT–qPCR', 32728212);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5574, 'mouse', 'increase', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 'control experiment,Supplementation of metabolite', '16S rRNA sequence,Western blot assay,RT–qPCR', 32728212);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5575, 'mouse', 'increase', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 'control experiment,Supplementation of metabolite', '16S rRNA sequence,Western blot assay,RT–qPCR', 32728212);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5576, 'mouse', 'increase', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 'control experiment,Supplementation of metabolite', '16S rRNA sequence,Western blot assay,RT–qPCR', 32728212);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5577, 'mouse', 'increase', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 'control experiment,Supplementation of metabolite', '16S rRNA sequence,Western blot assay,RT–qPCR', 32728212);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5578, 'mouse', 'increase', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 'control experiment,Supplementation of metabolite', '16S rRNA sequence,Western blot assay,RT–qPCR', 32728212);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5579, 'mouse', 'increase', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 'control experiment,Supplementation of metabolite', '16S rRNA sequence,Western blot assay,RT–qPCR', 32728212);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5580, 'mouse', 'increase', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 'control experiment,Supplementation of metabolite', '16S rRNA sequence,Western blot assay,RT–qPCR', 32728212);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5581, 'mouse', 'increase', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 'control experiment,Supplementation of metabolite', '16S rRNA sequence,Western blot assay,RT–qPCR', 32728212);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5582, 'mouse', 'increase', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 'control experiment,Supplementation of metabolite', '16S rRNA sequence,Western blot assay,RT–qPCR', 32728212);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5583, 'mouse', 'increase', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 'control experiment,Supplementation of metabolite', '16S rRNA sequence,Western blot assay,RT–qPCR', 32728212);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5584, 'mouse', 'increase', 'low-throughput', 'Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.', 'control experiment,Supplementation of metabolite', '16S rRNA sequence,Western blot assay,RT–qPCR', 32728212);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5585, 'mouse', 'increase', 'low-throughput', 'Taken together, these data demonstrate bacterial-intrinsic phytate metabolism and provoke the hypothesis that breakdown of phytate by microbiota promotes intestinal HDAC activity.', 'in vitro bacterial culture,control experiment', 'qPCR', 32731255);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5586, 'mouse', 'decrease', 'low-throughput', 'Further, colonization of germ-free (GF) mice with butyrate producing bacteria,Faecalibacterium prausnitzii, inhibited IEC-intrinsic HDAC activity.', 'in vitro bacterial culture,control experiment', 'qPCR', 32731255);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5587, 'mouse', 'increase', 'low-throughput', 'CNV mice exhibited significantly higher IP3 in both the intestinal lumen (Fig. 1e) and colonic IECs (Fig. 1f), supporting that microbiota promote increased IP3. To assess the effect of IP3 on epithelial HDAC3 activity, HDAC3 was purified by immunoprecipitation from IECs and incubated with IP3. IP3 increased endogenous HDAC3 activity (Fig. 1g), supporting that IP3 directly activates HDAC3 in IECs.', 'in vitro bacterial culture,control experiment', 'qPCR', 32731255);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5588, 'mouse', 'increase', 'low-throughput', 'Although less potent than IP3, IP4 also increased HDAC activity in IECs (Extended Data Fig. 5f), suggesting that other inositol phosphates simultaneously generated by phytate metabolism may provide synergistic effects.', 'in vitro bacterial culture,control experiment', 'qPCR', 32731255);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5589, 'mouse', 'increase', 'low-throughput', 'Glutamate, the agonist of NMDAR and a major excitatory neurotransmitter in the mammalian central nervous system (CNS) [6], is intracellularly located and extensively distributed in the brain.', 'null', 'null', 32290475);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5590, 'mouse', 'increase', 'low-throughput', 'Glutamate, the agonist of NMDAR and a major excitatory neurotransmitter in the mammalian central nervous system (CNS) [6], is intracellularly located and extensively distributed in the brain.', 'null', 'null', 32290475);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5591, 'mouse', 'increase', 'low-throughput', 'Non-pathogenic bacteria with glutamate racemase including Corynebacterium glutamicum, Brevibacterium lactofermentum, Brevibacterium avium [121], Mycobacterium smegmatis[122], and Bacillus subtilis[123] convert l-glutamate to d-glutamate.', 'null', 'null', 32290475);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5592, 'mouse', 'increase', 'low-throughput', 'Non-pathogenic bacteria with glutamate racemase including Corynebacterium glutamicum, Brevibacterium lactofermentum, Brevibacterium avium [121], Mycobacterium smegmatis[122], and Bacillus subtilis[124] convert l-glutamate to d-glutamate.', 'null', 'null', 32290475);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5593, 'mouse', 'increase', 'low-throughput', 'Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ', 'null', 'null', 32290475);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5594, 'mouse', 'increase', 'low-throughput', 'Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ', 'null', 'null', 32290475);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5595, 'mouse', 'increase', 'low-throughput', 'Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ', 'null', 'null', 32290475);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5596, 'mouse', 'increase', 'low-throughput', 'Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ', 'null', 'null', 32290475);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5598, 'mouse', 'increase', 'low-throughput', 'Reconstitution with a single bacterial species, Clostridium scindens, or its derived metabolite, the secondary bile acid deoxycholic acid, can restore pDC- and MyD88-dependent type I IFN responses to restrict systemic CHIKV infection and transmission back to vector mosquitoes. ', 'cell culture,control experiment,in vitro bacterial culture', '16S rDNA sequence,qPCR', 32668198);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5599, 'mouse', 'increase', 'low-throughput', 'Reconstitution with a single bacterial species, Clostridium scindens, or its derived metabolite, the secondary bile acid deoxycholic acid, can restore pDC- and MyD88-dependent type I IFN responses to restrict systemic CHIKV infection and transmission back to vector mosquitoes. ', 'cell culture,control experiment,in vitro bacterial culture', '16S rDNA sequence,qPCR', 32668198);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5601, 'mouse', 'decrease', 'low-throughput', 'Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.', 'in vitro bacterial culture,control experiment', '16S rRNA sequence,HPLC analysis', 32078445);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5602, 'mouse', 'decrease', 'low-throughput', 'Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.', 'in vitro bacterial culture,control experiment', '16S rRNA sequence,HPLC analysis', 32078445);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5603, 'mouse', 'decrease', 'low-throughput', 'Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.', 'in vitro bacterial culture,control experiment', '16S rRNA sequence,HPLC analysis', 32078445);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5604, 'mouse', 'increase', 'low-throughput', 'Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.', 'in vitro bacterial culture,control experiment', '16S rRNA sequence,HPLC analysis', 32078445);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5617, 'mouse', 'increase', 'low-throughput', 'In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.', 'control experiment', '16S rRNA sequence,LC/MS analysis', 33256731);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5618, 'mouse', 'increase', 'low-throughput', 'In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.', 'control experiment', '16S rRNA sequence,LC/MS analysis', 33256731);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5619, 'mouse', 'increase', 'low-throughput', 'In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.', 'control experiment', '16S rRNA sequence,LC/MS analysis', 33256731);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5632, 'mouse', 'increase', 'low-throughput', 'Interestingly, in the same mice, indole induced a dramatic increase of the expression of the adrenomedullary Pnmt gene, which is involved in catecholamine biosynthesis.', 'control experiment', 'HPLC analysis,ELISA,RT-PCR', 32569990);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5641, 'mouse', 'decrease', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 'cell culture,control experiment,Supplementation of metabolite', '16S rRNA sequence,RT-PCR', 33024047);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5642, 'mouse', 'decrease', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 'cell culture,control experiment,Supplementation of metabolite', '16S rRNA sequence,RT-PCR', 33024047);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5643, 'mouse', 'decrease', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 'cell culture,control experiment,Supplementation of metabolite', '16S rRNA sequence,RT-PCR', 33024047);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5644, 'mouse', 'decrease', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 'cell culture,control experiment,Supplementation of metabolite', '16S rRNA sequence,RT-PCR', 33024047);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5645, 'mouse', 'decrease', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 'cell culture,control experiment,Supplementation of metabolite', '16S rRNA sequence,RT-PCR', 33024047);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5646, 'mouse', 'decrease', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 'cell culture,control experiment,Supplementation of metabolite', '16S rRNA sequence,RT-PCR', 33024047);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5647, 'mouse', 'decrease', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 'cell culture,control experiment,Supplementation of metabolite', '16S rRNA sequence,RT-PCR', 33024047);
INSERT INTO gutmgene.gut_microbiota_gene_change_results (`index`, host_type, Alteration, throughput, description, experimental_method, sequencing_technology, Evidence) VALUES (5648, 'mouse', 'decrease', 'low-throughput', 'Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).', 'cell culture,control experiment,Supplementation of metabolite', '16S rRNA sequence,RT-PCR', 33024047);